---

title: Methods and compositions involving microRNA
abstract: The present invention concerns methods and compositions for isolating, enriching, and/or labeling miRNA molecules and for preparing and using arrays or other detection techniques for miRNA analysis. Moreover, the present invention concerns methods and compositions for generating miRNA profiles and employing such profiles for therapeutic, diagnostic, and prognostic applications.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08568971&OS=08568971&RS=08568971
owner: Asuragen, Inc.
number: 08568971
owner_city: Austin
owner_country: US
publication_date: 20100924
---
This application is a divisional of U.S. patent application Ser. No. 11 141 707 filed May 31 2005 now U.S. Pat. No. 7 888 010 which claims the benefit of U.S. Provisional Patent Application Ser. No. 60 575 743 filed on May 28 2004 and also of U.S. Provisional Patent Application Ser. No. 60 649 584 filed on Feb. 3 2005. The entire text of each of the above referenced disclosures is specifically incorporated herein by reference without disclaimer.

Pursuant to 37 C.F.R. 1.53 e the Sequence Listing required under 37 C.F.R. 1.821 c for SEQ ID NOs 1 899 as set forth below is submitted on one compact disc Copy 1 together with a duplicate thereof Copy 2 . Each of Copy 1 and Copy 2 were created on May 31 2005 in PC format using MS Windows operating system. Copy 1 and Copy 2 each contain one 196 kb file entitled AMBI097US.APP.txt. The material contained on the compact disc is specifically incorporated herein by reference.

The present invention relates generally to the field of molecular biology. More particularly it concerns methods and compositions involving microRNA miRNAs molecules. Methods and compositions for isolating labeling preparing miRNAs for analysis or as a tool for analysis are described such as miRNA arrays. In addition there are applications for miRNAs in diagnostics therapeutics and prognostics.

In 2001 several groups used a novel cloning method to isolate and identify a large variety of micro RNAs miRNAs from and humans Lagos Quintana et al. 2001 Lau et al. 2001 Lee and Ambros 2001 . Several hundreds of miRNAs have been identified in plants and animals including humans which do not appear to have endogenous siRNAs. Thus while similar to siRNAs miRNAs are nonetheless distinct.

miRNAs thus far observed have been approximately 21 22 nucleotides in length and they arise from longer precursors which are transcribed from non protein encoding genes. See review of Carrington et al. 2003 . The precursors form structures that fold back on each other in self complementary regions they are then processed by the nuclease Dicer in animals or DCL1 in plants. miRNA molecules interrupt translation through imprecise base pairing with their targets.

The function of most miRNAs is not known. A number of miRNAs however seem to be involved in gene regulation. Some of these miRNAs including lin 4 and let 7 inhibit protein synthesis by binding to partially complementary 3 untranslated regions 3 UTRs of target mRNAs. Others including the Scarecrow miRNA found in plants function like siRNA and bind to perfectly complementary mRNA sequences to destroy the target transcript Grishok et al. 2001 .

Some miRNAs such as lin 4 let 7 mir 14 mir 23 and bantam have been shown to play critical roles in cell differentiation and tissue development Ambros 2003 Xu et al. 2003 . Others are believed to have similarly important roles because of their differential spatial and temporal expression patterns.

Research on microRNAs miRNAs is increasing as scientists are beginning to appreciate the broad role that these molecules play in the regulation of eukaryotic gene expression. The two best understood miRNAs lin 4 and let 7 regulate developmental timing in by regulating the translation of a family of key mRNAs reviewed in Pasquinelli 2002 . Several hundred miRNAs have been identified in mouse and humans. As would be expected for molecules that regulate gene expression miRNA levels have been shown to vary between tissues and developmental states. Characterization of a number of miRNAs indicates that they influence a variety of processes including early development Reinhart 2000 cell proliferation and cell death Brennecke 2003 and apoptosis and fat metabolism Xu 2003 . In addition one study shows a strong correlation between reduced expression of two miRNAs and chronic lymphocytic leukemia providing a possible link between miRNAs and cancer Calin 2002 . Although the field is still young there is speculation that miRNAs could be as important as transcription factors in regulating gene expression in higher eukaryotes.

Several publications describe labeling miRNAs for analysis. These publications describe appending a radioactive phosphate at the 5 end of the miRNA population using a polynucleotide kinase Krichevsky 2003 or a radiolabeled single nucleotide at the 3 end with RNA ligase Dostie 2003 . For the purpose of using arrays to estimate the relative abundances of miRNAs in samples these methods have two significant drawbacks 1 only a single label is appended per miRNA limiting the sensitivity that can be achieved and 2 the methods are compatible with radiolabeling only which has disadvantages as compared to non isotopic platforms for arrays. Furthermore while RNA oligonucleotides have been labeled with non isotopic labels Martin et al. 1998 there is no evidence that small RNA molecules from a cell lysate can be effectively labeled in a similar manner after they have been enriched or isolated from the lysate.

Because microarrays are typically used to analyze messenger RNAs that are hundreds or thousands of nucleotides in length we found that the 60 500 mer probes typically used in microarrays were not compatible with miRNA analysis.

Therefore there is a need for information about the function and activity of miRNAs as well as for methods and compositions that can be used for their characterization and analysis.

The present invention is based on the inventors studies regarding the manipulation of miRNA and the use of miRNAs to characterize their role and function in cells. It concerns methods and compositions for isolating miRNA labeling it preparing arrays directed to miRNAs miRNA array or microarray analyzing miRNAs using an array and characterizing miRNAs for diagnostic therapeutic and prognostic applications.

The term miRNA is used according to its ordinary and plain meaning and refers to a microRNA molecule found in eukaryotes that is involved in RNA based gene regulation. See e.g. Carrington et al. 2003 which is hereby incorporated by reference. The term will be used to refer to the RNA molecule processed from a precursor. Individual miRNAs have been identified and sequenced in different organisms and they have been given names. Names of miRNAs and their sequences are provided herein. Additionally other miRNAs are known to those of skill in the art and can be readily implemented in embodiments of the invention. The methods and compositions should not be limited to miRNAs identified in the application as they are provided as examples not necessarily as limitations of the invention.

Some embodiments of the invention concern methods for labeling miRNA. While it is contemplated that miRNA may be labeled with a single label in many embodiments of the invention multiple labels may be the same or different labels are attached to the miRNA. Consequently methods for multi labeling are specifically contemplated as part of the invention. In some embodiments the miRNA to be labeled is contacted with an enzyme that catalyzes the addition of di or tri phosphate nucleotides to the 3 end of the miRNA effectively adding a nucleotide tail to the miRNA. The miRNA and enzyme will be incubated under conditions to allow catalysis by the enzyme to occur such that a reaction mixture for tailing is formed. Moreover the reaction mixture will contain nucleotides that are labeled or unlabeled which are added to the 3 end of the miRNA by the enzyme to form a tailed miRNA molecule. The nucleotides may be DNA and or RNA though in some embodiments the nucleotide is a ribonucleotide. The term nucleotide refers to both ribonucleotides and deoxyribonucleotides though it is specifically contemplated that any embodiment discussed in the context of nucleotides may be implemented specifically with ribonucleotides or deoxyribonucleotides unless otherwise specified or limited. The nucleotide used in the reaction may be uridine adenosine guanosine and or cytosine including any combination thereof. In specific embodiments the nucleotide is uridine which may or may not be modified as discussed below. The tail added to the miRNA is at least or at most 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 441 450 460 470 480 490 500 nucleotides or more in length or any range derivable therein.

The nucleotides added to the miRNA may already have a label attached or the label may be attached after the miRNA is tailed. If unlabeled nucleotides are added to the miRNA the method further comprises attaching a label to the tailed miRNA molecules. Thereafter the multi labeled miRNA molecules may be isolated which means it may be separated from any or all of the following unincorporated label or nucleotides enzyme nontailed miRNA and other RNA. In the process of isolating the multi labeled miRNA molecules the molecules may be dried down for subsequent use.

miRNA used in the reaction may be obtained by a variety of methods and from a variety of sources. The miRNA may be obtained from a biological sample such as a cell tissue or organ. It may be isolated from a biological sample that contains other RNA molecules as well such as mRNA tRNA and or rRNA. In certain instances total RNA is first isolated from the sample and then the miRNA is separated from the other RNA thereby enriching for miRNA. In some embodiments the miRNA has been isolated away from other RNA to enrich for the miRNA such that the miRNA substantially pure meaning it is at least about 80 85 90 95 pure or more but less than 100 pure with respect to other RNA molecules. Alternatively enrichment of miRNA may be expressed in terms of fold enrichment. In certain embodiments miRNA is enriched by about at least about or at most about 5 10 20 30 40 50 60 70 80 90 100 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 450 460 470 480 490 500 600 700 800 900 1000 1100 1200 1300 1400 1500 1600 1700 1800 1900 2000 3000 4000 5000 6000 7000 8000 9000 10 000 or more or any range derivable therein with respect to the concentration of miRNA in an RNA isolate or the total RNA in the sample.

miRNA can be separated from other RNA molecules using methods known to those of ordinary skill in the art. In some embodiments miRNA are separated from other RNA molecules using chromatography. Gel chromatography can be implemented to isolate miRNA molecules. In certain embodiments gel chromatography can be performed using a polyacrylamide gel and tube electrophoresis.

The biological sample may be from any organism that has endogenous miRNA. Organisms include but should not be limited to arthopods nematodes and vertebrates plants and all of which have miRNA that has been sequenced. See the miRNA Registry on the world wide web at sanger.ac.uk cgi bin Rfam mirna browse.pl. Alternatively miRNA may be recombinant such that it is obtained from a cell free system or reaction mixture or from a recombinant host cell which may or may not have endogenous miRNA. Furthermore miRNA may be evaluated in samples that were previously fixed. In some embodiments of the invention the sample was fixed in formaldehyde or paraformaldehyde prior to taking steps to evaluate its miRNA. In additional embodiments samples that can be used according to the invention include those in which RNA in the sample has been degraded. Such samples include those in which about or at least about 50 60 70 80 90 95 or more or any range derivable therein of the mRNA and or rRNA in the sample is degraded. In particular embodiments samples in which there has been substantial degradation that is at least about 80 degradation of mRNA and or rRNA in the sample are analyzed according to methods and compositions of the invention.

In some embodiments enzymes used in methods and compositions of the invention are selected from the group consisting of poly A polymerase terminal transferase and polynucleotide phosphorylase. In particular embodiments the enzyme is poly A polymerase. In some cases poly A polymerase is from or yeast. The enzyme may be purified recombinant and or purchased commercially.

Labeling methods of the invention involve in many embodiments one or more modified nucleotides. The term modified nucleotide refers to a nucleotide the basic structural unit of nucleic acids RNA and DNA that has been chemically modified but still functions as a nucleotide. Modified nucleotides include nucleotides with covalently bound detectable moieties such as a fluorescent or chemi luminescent label. Modified nucleotides also include nucleotides with reactive groups that can react with a detectable moiety before during or after incorporation of the modified nucleotide to create a labeled nucleic acid. In specific embodiments the modified nucleotide is an amine modified nucleotide which means the nucleotide has been modified to have a reactive amine group. Modified nucleotides may be uridine adenosine guanosine and or cytosine. In specific embodiments the amine modified nucleotide is selected from the group consisting of 5 3 aminoallyl UTP 8 4 amino butyl amino ATP and 8 6 amino butyl amino ATP N 4 amino butyl ATP N 6 amino butyl ATP N 2 2 oxy bis ethylamine CTP N 6 Amino hexyl ATP 8 6 Amino hexyl amino ATP 5 propargylamino CTP 5 propargylamino UTP. However it is contemplated that other nucleotides may be similarly modified for example 5 3 aminoallyl GTP instead of 5 3 aminoallyl UTP.

In certain embodiments a reaction mixture includes modified nucleotides that have a labeled moiety attached labeled nucleotides . In this case a one step process a single enzymatic reaction is used to generate multi labeled miRNA which refers to an miRNA molecule having at least two labels attached to it. Labeled nucleotides are readily available they can be acquired commercially or they can be synthesized by reactions known to those of skill in the art. When the reaction includes labeled nucleotides it is contemplated that in other embodiments both labeled and unlabeled nucleotides are included. The labeled and unlabeled nucleotides may be the same nucleotides but labeled and unlabeled or they may be different nucleotides such as a labeled UTP but an unlabeled CTP GTP and or ATP.

In certain other embodiments the tailing reaction contains unlabeled but modified nucleotides unlabeled modified nucleotide that are labeled in a subsequent reaction two step process . It is contemplated that a reaction may contain modified and unmodified nucleotides to improve tailing efficiency and avoid dye quenching induced by having dye molecules to close together. The unmodified and modified nucleotides may be the same nucleotide but one modified and the other not or they may be different nucleotides such as a modified UTP but an unmodified CTP GTP and or ATP.

Thus in both the one step and two step processes of the invention a reaction mixture for tailing comprises modified and unmodified nucleotides. In some embodiments the ratio of modified to unmodified nucleotides in the reaction mixture is from about 1 1 to about 10 1 and specifically about 5 1. The ratio may be about 1 5 1 4 1 3 1 2 1 1 2 1 3 1 4 1 5 1 6 1 7 1 8 1 9 1 10 1 11 1 12 1 13 1 14 1 15 1 or more or any range derivable therein. It is understood that the ratio is based on concentrations.

If the reaction mixture contains initially unlabeled modified nucleotides the nucleotides are added to the miRNA which becomes tailed and one or more labels become subsequently attached to one or more of these nucleotides. The way in which the label becomes attached depends on the label being used and a person of ordinary skill in the art would know how to attach the label in a subsequent reaction.

Labels that can be attached to miRNA include those that are covalently attached to a nucleic acid. It is contemplated that the label on the labeled nucleotides or the label that becomes attached to the nucleotides in the tailed miRNA is biotin radioactivity or a dye. Alternatively the label may be qualified as positron emitting colorimetric enzymatic luminescent fluorescent or a ligand.

In specific embodiments of the invention the dye is selected from the group consisting of Alexa Fluor BODIPY Cascade Blue Cascade Yellow cyanine dye eosins and erythrosins fluorescein HEX Marina Blue Oregon Green Rhodamine Quantum Dye SYPRO tetramethylrhodamine Texas Red TOTAB and derivatives thereof.

It is contemplated that miRNA from a single sample can be labeled with the same label or with different labels.

Tailing reactions may be incubated for at least or at most 2 3 4 5 6 7 8 9 10 15 20 25 30 35 40 45 50 55 minutes 1 2 3 4 5 6 7 8 9 10 11 12 hours or more. Typically reactions are about 1 to 3 hours such as 2 hours. Incubations may be at temperatures of 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 or 40 C. or more though it is specifically contemplated that temperatures may be in the range of 30 C. to 40 C. particularly at about 37 C.

In some embodiments of the invention the reaction mixture in which nucleotides are added to miRNA further comprises a volume exclusion reagent. Examples of such reagents include but are not limited to a reagent selected from the group consisting of polyethylene glycol PEG and dextran. In some embodiments of the invention the concentration of the volume exclusion reagent in the reaction is between about 1 and about 20 or about 5 to 10 . The concentration of the volume exclusion reagent in the reaction is about at least about or at most about 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 or 20 or any range derivable therein in embodiments of the reaction. In certain cases the concentration of the volume exclusion reagent in the reaction is up to about 10 .

The reaction mixture for tailing miRNA molecules has a pH of about or at most about 5.5 5.6 5.7 5.8 5.9 6.0 6.1 6.2 6.3 6.4 6.5 6.6 6.7 6.8 6.9 7.0 7.1 7.2 7.3 7.4 7.5 7.6 7.7 or 7.8 or any range derivable therein. In particular embodiments the reaction mixture has a pH between about 6.0 and about 7.8 and in certain cases the pH is about 6.5.

The present invention also concerns arrays for evaluating miRNA molecules. Clearly contemplated is an array that is a microarray. The arrays have one or more probes directed to one or more miRNA molecules miRNA array . In some embodiments an miRNA array includes one or more miRNA probes immobilized on a solid support. An miRNA probe refers to a nucleic acid having a sequence that is complementary or identical to all or part of a miRNA precursor or gene such that it is capable of specifically hybridizing to an miRNA gene the cognate miRNA precursor or the processed miRNA. Typically the probe will contain at least ten contiguous nucleotides complementary to all or part of the miRNA precursor or at least ten contiguous nucleotides complementary or identical to all or part of an miRNA gene. It will be understood that DNA probes with sequences relative to an miRNA gene will be identical in sequence to all or part of the coding sequence of the gene and complementary in sequence to all or part of the noncoding sequence of the gene. In specific embodiments an miRNA probe contains the sequence encoding an miRNA miRNA coding sequence which refers to sequence encoding processed miRNA . Because the precise length and consequently sequence of a particular processed miRNA has been found to vary occasionally the predominant species will be understood as the sequence and length of the processed miRNA. The predominant species is usually the one observed at least 90 of the time.

In embodiments of the invention there is an amine attached to the 5 or 3 end of the probe. The amine was a reactive group on the probe that allowed for attachment of the probe to the array.

The number of different probes on the array is variable. It is contemplated that there may be be at least or be at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 441 450 460 470 480 490 500 510 520 530 540 550 560 570 580 590 600 610 620 630 640 650 660 670 680 690 700 710 720 730 740 750 760 770 780 790 800 810 820 830 840 850 860 870 880 890 900 910 920 930 940 950 960 970 980 990 1000 1100 1200 1300 1400 1500 1600 1700 1800 1900 2000 2100 2200 2300 2400 2500 2600 2700 2800 2900 3000 31 3200 3300 3400 3500 3600 3700 3800 3900 4000 4100 4200 4300 4400 4500 4600 4700 4800 4900 5000 6000 7000 8000 9000 10000 or more or any range derivable therein different miRNA probes on an array. In specific embodiments arrays have between 5 and 1000 different miRNA probes between 20 and 500 different miRNA probes between 50 and 250 different miRNA probes or between 100 and 225 different miRNA probes. Different probes refers to probes with different sequences. Therefore it is contemplated that different probes can be used to target the same miRNA. Moreover multiple and different probes to the same miRNA can be included on an array. For example one probe may target specifically a precursor miRNA or the miRNA gene depending on what sample is used to hybridize to the array i.e whether the sample contains DNA or RNA while another probe may be capable of hybridizing to the processed miRNA its precursor or the gene. Moreover miRNA probes targeting the same miRNA may be overlapping such that they share contiguous sequences. It is also contemplated that a single probe may target multiple miRNAs particularly miRNAs from the same gene family or related miRNAs distinguished by a letter . It is understood by those of skill in the art that a gene family refers to a group of genes having the same miRNA coding sequence. Typically members of a gene family are identified by a number following the initial designation. For example miR 16 1 and miR 16 2 are members of the miR 16 gene family. Also a probe may have a sequence that allows it to target more than 1 miRNA. It is understood that a 2 base pair mismatch between the probe and an miRNA is sufficient to hybridize with at least 90 of the mismatched miRNA under the conditions described in the Examples. Consequently it will be understood that unless otherwise indicated a probe for a particular miRNA will also pick up a related miRNA such as those designated with the same number but with an added letter designation. For example an miRNA probe that is fully complementary to miR 15a would also hybridize to miR 15b unless otherwise noted. Thus an miRNA probe can target 1 2 3 4 5 6 or more different miRNAs.

miRNA probes are contemplated to be made of DNA though in some embodiments they may be RNA nucleotide analogs PNAs or any combination of DNA RNA nucleotide analogs and PNAs.

As suggested above it is contemplated that arrays of the invention may target one or more miRNAs. In embodiments of the invention an array has probes that target 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 441 450 460 470 480 490 500 510 520 530 540 550 560 570 580 590 600 610 620 630 640 650 660 670 680 690 700 710 720 730 740 750 760 770 780 790 800 810 820 830 840 850 860 870 880 890 900 910 920 930 940 950 960 970 980 990 1000 1100 1200 1300 1400 1500 1600 1700 1800 1900 2000 2100 2200 2300 2400 2500 2600 2700 2800 2900 3000 31 3200 3300 3400 3500 3600 3700 3800 3900 4000 4100 4200 4300 4400 4500 4600 4700 4800 4900 5000 6000 7000 8000 9000 10000 or more or any range derivable therein different miRNAs. Accordingly it is understood that the array contains one or more probes for these different miRNAs. In specific embodiments human miRNAs are targeted though the targeting of multiple organisms or species using a single array or other method or technique is contemplated. In certain other embodiments miRNAs of a mammal are targeted. Such mammals include but are not limited to monkeys gorilla chimpanzees rats mice cats dogs rabbits hamsters ferrets cows sheep pigs humans rodents and goats. Thus arrays or methods including miRNA probes having human miRNA coding sequences or other mammalian miRNA coding sequences are part of the invention.

miRNA probes of the invention have an miRNA coding sequence that is between 19 34 nucleotides in length. Of course this is understood to mean that the probes have 19 34 contiguous nucleotides that are identical or nearly identical to the miRNA gene and complementary to the processed miRNA or its precursor. As discussed above a probe can be used to target an miRNA with which it has a 2 base pair mismatch in hybridization. Thus it is contemplated that miRNA probes of the invention may be almost fully complementary 2 base pair mismatches or fewer or fully complementary to any miRNA sequence or set of sequences such as related miRNAs or miRNAs from the same gene family that is targeted. The term nearly identical means that any difference in sequence is 2 bases or fewer. When an miRNA has a perfectly complementary stem loop in its precursor the miRNA coding sequence should be identical to a sequence in the precursor as well. In some embodiments of the invention a probe has an miRNA coding sequence that includes the entire processed miRNA sequence. It is contemplated that the probe has or has at least 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 or more contiguous nucleotides or any range derivable therein from an miRNA coding sequence. In specific embodiments an miRNA probe has a sequence identical or complementary or at least 90 or greater identity or complementarity across the lengths discussed in the previous sentence with respect to any of SEQ ID NOs 1 593 inclusive as well as any other sequence discussed herein including in the appendix. The appendix provides a list of 1 miRNAs that were screened any one of which can be screened for using any array or method of the present invention 2 the names of the probe used to screen for that miRNA and 3 the sequence of the named probe.

As discussed above miRNA are processed from a precursor molecule. In certain embodiments probes have an miRNA coding sequence that also includes at least 2 to 5 nucleotides of coding sequence upstream and or downstream of the processed miRNA sequence. Probes may have or have up to 1 2 3 4 5 6 7 or more contiguous nucleotides or any range derivable therein that flank the sequence encoding the predominant processed miRNA on one or both sides 5 and or 3 end . In particular embodiments probes have an miRNA coding sequence that includes 4 nucleotides of coding sequence upstream 5 and or downstream 3 of the processed miRNA sequence. On other embodiments miRNA probes also have one or more linkers flanking the miRNA coding sequence. In particular embodiments there is a linker at the 3 end of the miRNA coding sequence. In some embodiments a linker has a sequence that is between 3 to 25 nucleotides in length.

In some embodiments of the invention miRNA probes are attached to the array through an amine attached at the 3 end.

The invention is not limited to arrays constructed with particular materials. Typically arrays are made with materials that do not interfere with the hybridization between the probe and a sample. In some embodiments the array is a solid support that is made with glass plastic or metal.

The present invention concerns methods for identifying a correlation between miRNA expression and a disease or condition. In certain embodiments methods involve identifying miRNA differentially expressed in a sample representative of the disease or condition non normal sample compared to a normal sample. A sample representative of the disease or condition will be one that has the disease or condition is affected by the disease or condition and or causes the disease or condition. In certain embodiments identifying differentially expressed miRNA involves a labeling miRNA in the sample and b hybridizing the labelled miRNA to an miRNA array. In further embodiments the miRNA in the sample is isolated before or after labeling.

In certain embodiments isolating miRNA in a sample involves methods concerning electrophoresis. The present invention provides methods for the isolation of nucleic acids using electrophoresis. These methods may be used to isolate any nucleic acid molecules including but not limited to small nucleic acid molecules such as miRNA. In some embodiments the nucleic acid molecules are less than 200 nucleotides in length. In other embodiments the nucleic acid molecules are less than 100 nucleotides in length. In certain aspects the nucleic acid molecules are between 15 200 nucleotides in length 15 100 nucleotides in length 15 40 nucleotides in length or are nucleotides in length.

In one embodiment the method for isolating nucleic acid molecules from a sample comprises providing an electrophoresis apparatus comprising a lower buffer collection chamber electrically coupled to a positive electrode an upper buffer chamber electrically coupled to a negative electrode and a gel matrix disposed between the lower buffer collection chamber and the upper buffer chamber adding a running buffer to the lower buffer collection chamber and the upper buffer chamber applying the sample to a surface of the gel matrix moving the nucleic acid molecules in the sample through the gel matrix by electrophoresis and collecting the running buffer from the lower collection buffer chamber wherein the running buffer in the lower collection buffer chamber comprises nucleic acid molecules that passed through the gel matrix. The electrophoresis apparatus used in the practice of the present invention may be a micro electrophoresis apparatus such as the flashPAGE Fractionator Ambion Inc. . In some embodiments the method further comprises purifying the nucleic acid molecules present in the collected running buffer. In certain embodiments between about 1 g to about 100 g of nucleic acid are applied to the gel matrix.

In certain aspects of the invention the electrophoresis is performed at about 50 to about 100 Volts V and about 2 to about 5 milliAmps mA . In other aspects of the invention the electrophoresis is performed at about 60 80 V and about 3 mA. In some embodiments the electrophoresis is performed for about 10 11 12 13 14 15 16 17. 18 19 or about 20 minutes.

The nucleic acid molecules may be isolated in one fraction or they may be isolated in more than one fraction. Nucleic acid molecules of a desired size or size range may be isolated by collecting specific fractions as they migrate off of the gel. After the running buffer has been removed from the lower buffer collection chamber to collect a first fraction of nucleic acid molecules that has migrated off the gel it is necessary to add additional running buffer to the lower buffer collection chamber in order to continue electrophoresis and collect additional fractions of nucleic acid molecules. Where the nucleic acid molecules are isolated in multiple fractions it may be desirable that they are isolated at regular intervals such as 30 second 1 minute 2 minute or 3 minute intervals. By performing a time course and evaluating the size of nucleic acids collected at each time point in an initial run any desired size class of nucleic acid molecules can be isolated in subsequent runs because the nucleic acid size fractions will exit the gel at the same timepoint during each run as long as the conditions are kept consistent between runs.

A dye marker may also be applied to the surface of the gel matrix at the same time as the sample comprising nucleic acids is loaded onto the gel. A dye marker that is known to migrate through the gel matrix with a particular size of nucleic acid is useful in identifying an approximate time for elution of nucleic acids of the size or sizes of interest. For example a dye marker that migrates with 40 nucleotide nucleic acids would be useful in collecting nucleic acid fractions that contain nucleic acids that are smaller or larger than 40 nucleotides. To collect nucleic acid molecules that are less than 40 nucleotides in length electrophoresis should be stopped just as the dye begins to migrate off of the gel and the running buffer collected from the lower buffer collection chamber. For nucleic acids larger than 40 nucleotides collection of the lower running buffer would begin once the dye begins to migrate off of the gel.

Electrophoretic purification of nucleic acids can isolate nucleic acids of a desired size or size range away from nucleic acid molecules of other sizes such that the nucleic acids of the desired size or size range are substantially pure meaning they are at least about 80 85 90 95 pure or more but less than 100 pure with respect to other nucleic acid molecules in the sample. Alternatively enrichment of nucleic acids of the desired size or size range may be expressed in terms of fold enrichment. In certain embodiments the nucleic acids of the desired size or size range are enriched by about at least about or at most about 5 10 20 30 40 50 60 70 80 90 100 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 450 460 470 480 490 500 600 700 800 900 1000 1100 1200 1300 1400 1500 1600 1700 1800 1900 2000 3000 4000 5000 6000 7000 8000 9000 10 000 or more or any range derivable therein with respect to the concentration of the nucleic acids of the desired size or size range in the nucleic acid sample.

In further embodiments methods involve identifying an appropriate sample to analyze or evaluate. It is particularly contemplated that in some embodiments an appropriate sample is one that can provide information about a particular disease or condition or about some other phenotype.

Other methods of the invention concern analyzing miRNA in a sample comprising generating an miRNA profile for the sample and evaluating the miRNA profile to determine whether miRNA in the sample are differentially expressed compared to a normal sample. In specific embodiments methods of the invention include a method for evaluating miRNA in a biological sample. In certain instances the biological sample is from a patient. This method is implemented by analyzing one or more miRNAs in a sample using the array compositions and methods of the invention. In specific embodiments miRNA are evaluated by one or more of the following steps a isolating miRNA away from other RNA in the sample b labeling the miRNA c hybridizing the miRNA to an miRNA array and d determining miRNA hybridization to the array. Whether miRNAs hybridize to the array what miRNAs hybridize to the array and or how much total miRNA or any specific miRNAs hybridize to the array are ways of determining the extent of miRNA hybridization to the array. Methods of detecting measuring and quantifying hybridization are well known to those of skill in the art. In specific embodiments miRNA hybridization is quantified.

The present invention also concerns methods of generating an miRNA profile for a sample. The term miRNA profile refers to a set of data regarding the expression pattern for a plurality of miRNAs in the sample that was obtained using an miRNA array. In some embodiments of the invention an miRNA profile is generated by steps that include a labeling miRNA in the sample b hybridizing the miRNA to an miRNA array and c determining miRNA hybridization to the array wherein an miRNA profile is generated.

miRNA profiles can be generated to compare differences in miRNA expression between any two or more different samples. miRNA profiles can be compared for example between a sample with a particular disease disorder or condition and a sample that does not have the particular disease disorder or condition between samples that have a particular disease disorder or condition but a different stage of the disease disorder or condition between samples that have a particular disease disorder or condition but with a different prognosis with respect the disease disorder or condition between a sample that has been treated with a particular agent and a sample that has not been treated with that agent between samples that have responded differently to a particular substance or agent such as one responsive to the treatment and one not or one resistant to the treatment and one not samples that differ by gender of the sources samples that differ by age or stage of development of the source samples that differ by tissue type samples that differ by at least one known polymorphism between a sample that has a particular mutation and a sample that does not a sample that is defective in a particular pathway or has a defective protein and a sample that does not between a sample that is apparently resistant to a particular disease disorder or condition and a sample that is not expected to be resistant to that particular disease disorder or condition as well as a comparison involving any samples with a combination of characteristics as described above.

Samples from which miRNA profiles are generated include samples that can be characterized based on one or more of the following age developmental stage prognosis of a disease condition or disorder cell type tissue type organ type race or ethnicity gender susceptibility to or risk of a particular disease condition or disorder diet exposure to or treatment with a particular chemical agent or substance diagnosis of a particular a disease condition or disorder organism type genomic makeup etc.

Methods of the invention allow differences between two or more biological samples to be determined by generating an miRNA profile for each sample and comparing the profiles wherein a difference in the profiles identifies differentially expressed miRNA molecules. In specific embodiments a first sample is treated with a substance prior to generating the miRNA profile and a second sample is untreated. In other embodiments a first sample exhibits a disease or condition and a second sample exhibits the same disease or condition but at a different stage of progression. In further embodiments a first sample responds favorably to a therapeutic agent and a second sample is unresponsive to the therapeutic agent. Moreover in other embodiments a first sample is from a first subject who responds adversely to a therapeutic agent and a second sample is from a second subject does not respond adversely to the therapeutic agent.

Other methods of the invention concern identifying a correlation between miRNA expression and a disease or condition comprising comparing different miRNA profiles such as 1 an miRNA profile of a sample with the disease or condition or from a subject with the disease or condition and 2 an miRNA profile of a sample that is normal with respect to that disease or condition or that is from a subject that does not have the disease or condition. In specific embodiments methods include a isolating miRNA from a sample exhibiting the disease or condition b labeling the miRNA c hybridizing the miRNA to an miRNA array and d identifying miRNA differentially expressed in the sample compared to a normal sample. It is contemplated that the miRNA profiles may be generated in the process of performing the method alternatively they may be obtained from previously obtained results. Moreover it is contemplated that comparisons may be done by using a plurality of miRNA profiles multiple samples from the same source obtained at the same or different times and or samples from different sources . In this case a normalized miRNA profile may be generated and used for comparison purposes.

In certain embodiments methods concern identifying miRNAs indicative of a disease or condition by detecting a correlation between the expression of particular miRNAs and a sample believed to have a disease or condition.

In specific embodiments there are methods for analyzing a biological sample from a patient for a disease or condition comprising generating an miRNA profile for the sample and evaluating the miRNA profile to determine whether miRNA in the sample are differentially expressed compared to a normal sample. The comparison may involve using an array that has selective miRNA probes that are indicative of a disease or condition. Arrays of the invention include macroarrays and microarrays.

Arrays can contains miRNA sequences from any organism having miRNAs specifically including mammals such as humans mice and rats. However unless specifically indicated the naming of a particular miRNA refers to a human miRNA. Specifically contemplated are arrays having having at least or having at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264 265 266 267 268 269 270 271 272 273 274 275 276 277 278 279 280 281 282 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 419 420 421 422 423 424 425 426 427 428 429 430 431 432 433 434 435 436 437 438 439 440 441 442 443 444 445 446 447 448 449 450 451 452 453 454 455 456 457 458 459 460 461 462 463 464 465 466 467 468 469 470 471 472 473 474 475 476 477 478 479 480 481 482 483 484 485 486 487 488 489 490 491 492 493 494 495 496 497 498 499 500 501 502 503 504 505 506 507 508 509 510 511 512 513 514 515 516 517 518 519 520 521 522 523 524 525 526 527 528 529 530 531 532 533 534 535 536 537 538 539 540 541 542 543 544 545 546 547 548 549 550 551 552 553 554 555 556 557 558 559 560 561 562 563 564 565 566 567 568 569 570 571 572 573 574 575 576 577 578 579 580 581 582 583 584 585 586 587 588 589 590 591 592 593 594 595 596 597 598 599 600 700 800 900 1000 1100 1200 1300 1400 1500 1600 1700 1800 1900 2000 2100 2200 2300 2400 2500 2600 2700 2800 2900 3000 3100 3200 3300 3400 3500 3600 3700 3800 3900 4000 4100 4200 4300 4400 4500 4600 4700 4800 4900 5000 5100 5200 5300 5400 5500 5600 5700 5800 5900 6000 6100 6200 6300 6400 6500 6600 6700 6800 6900 7000 7100 7200 7300 7400 7500 7600 7700 7800 7900 8000 8100 8200 8300 8400 8500 8600 8700 8800 8900 9000 9100 9200 9300 9400 9500 9600 9700 9800 9900 10000 or more different miRNA probes that is miRNA probes having different sequences targeting the same or different miRNAs miRNA precursors or miRNA genes . Specifically contemplated are such arrays described in the previous sentence with probes for any of SEQ ID NOs 1 703 or with any other sequence provided herein including those identified in SEQ ID NOs 704 899. Moreover embodiments can specifically include or employ any or all of the probes identified in Table 1C. Probes may be identical or complementary to 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 or more contiguous nucleic acids or any range deriveable therein of SEQ ID NOs 704 899. Alternatively any probe used may be be at least or be at most 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 or 100 complementary or identical or any range derivable therein to any sequence in SEQ ID NOs 1 899. It is specifically contemplated that any embodiment discussed in the context of an miRNA array may be employed more generally in screening or profiling methods or kits of the invention. In other words any embodiments describing what may be included in a particular array can be practiced in the context of miRNA profiling more generally and need not involve an array per se.

In specific embodiments an array has has at least or has at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 probes or any range derivable therein for human miRNA selected from the group consisting of let 7 miR 7 let 7F 2 miR 9 miR 10a miR 16 miR 17 miR 21 miR 22 miR 23 miR 24 miR 26a miR 28 miR 29b miR 30a miR 31 miR 95 miR 105 miR 106 miR 125a miR 126 miR 130 miR 130a miR 133 miR 135A miR 137 miR 138 miR 139 miR 140 miR 143 miR 144 miR 145 miR 181a miR 182 miR 183 miR 186 miR 188 miR 192 miR 194 miR 195 miR 199a miR 200b mu miR 201 miR 205 miR 211 miR 215 miR 219 miR 223 miR 224 mu miR 290 mu miR 291 5P mu miR 298 miR 301 miR 328 miR 331 and miR 342. It will be understood that shorthand notations are employed such that a generic description of an miRNA refers to any of its gene family members distinguished by a number and related members distinguished by a letter unless otherwise indicated. For example mir 7 refers to miR 7 1 miR 7 2 and miR 7 3. Similarly let 7 for example refers to let 7a 1 let7 a 2 let 7b let 7c let 7d let 7e let 7f 1 and let 7f 2. Exceptions to this shorthand notations will be otherwise identified. A probe with at least 90 complementarity will allow hybridization to an miRNA. It is contemplated that a probe for a non human miRNA can be used in embodiments of the invention to target human homologs or sequences with sufficient complementarity to allow their detection with the non human miRNA probe. Such probes may be for miRNA identified in mouse or rat or any other organism including other mammals. For example mu miR 201 is a mouse miRNA yet when a probe for that sequence was employed with a human sample the human homolog for mu miRNA 201 was detected.

It is contemplated that any combination of these probes or the target miRNAs can be used in methods and compositions of the invention. Furthermore an increase and or decrease in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 or more of these miRNAs relative to a normal sample where normal sample means the sample does not have or is not afflicted with the disease condition or state that is being tested for. It is contemplated that such an array may also include a probe for an miRNA listed in Table 1.

Moreover in specific embodiments methods and compositions may involve samples that include cancer or tumor cells or are from a subject diagnosed with cancer. In particular embodiments the cancer is selected from the group consisting of astrocytoma bone cancer bladder cancer brain cancer breast cancer cervical cancer colon cancer gastrointestinal cancer glioblastoma cancer head cancer hepatocarcinoma leukemia lung cancer lymphoma melanoma mesothelioma neuroblastoma neck cancer non small cell lung cancer ovarian cancer pancreatic cancer prostate cancer renal cancer retinoblastoma small cell lung cancer stomach cancer testicular cancer thymus cancer and thyroid cancer.

In specific embodiments an array has has at least or has at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 or 40 probes or any range derivable therein for human miRNA selected from the group consisting let 7 let 7A let 7C let 7F 2 miR 7 miR 9a miR 9 as miR 10a miR 15A miR 16 miR 17 miR 20 miR 21 miR 22 miR 23a miR 23b miR 24 miR 25 miR 26a miR 28 miR 29b miR 30a miR 30a as miR 31 miR 92 miR 95 miR 99b miR 103 miR 105 miR 106a miR 125a miR 126 miR 126 as miR 130a miR 133 miR 137 miR 138 miR 139 miR 140 miR 141 miR 143 miR 144 miR 145 miR 152 miR 181a miR 181b miR 182 miR 183 miR 186 miR 188 miR 192 miR 194 miR 195 miR 199a miR 199a as miR 200b mu miR 201 miR 203 miR 205 miR 211 miR 215 miR 219 miR 221 miR 222 miR 223 miR 224 miR 290 miR 291 miR 291 5P miR 298 miR 301 miR 326 miR 328 mu mu miR 329 miR 331 miR 341 miR 342 miR 344 miR 361 and miR 425. In certain embodiments involving such an array a change in expression compared to a normal sample is indicative of cancer. Moreover it is contemplated that methods may involve such miRNA probes with or without the array.

In particular embodiments the array or methods involve at least one probe for one or more of the following miRNAs miR 21 miR 126 miR 143 miR 145 miR 188 miR 200B miR 219 and miR 331. In methods of the invention using such an array an increase in miR 17 miR 21 miR 182 miR 183 miR 200b miR 205 miR 223 and or miR 224 expression is indicative of cancer and or a decrease in the expression of let 7 miR 10a miR 16 miR 22 miR 23 miR 24 miR 26a miR 29b miR 30a miR 106 miR 125a miR 126 miR 130 miR130a miR 133 miR 143 miR 144 miR 145 miR 181a miR 188 miR 219 miR 192 miR 194 miR 195 miR 199a mu miR 201 miR 215 miR 328 miR 331 and or miR 342 is indicative of cancer.

This means that any of the following is indicative of cancer 1 a decrease in expression of one or more of the identified miRNAs or human homologs thereof 2 an increase in expression of one or more of the identified miRNAs or human homologs thereof or 3 both an increase and a decrease in expression of one or more of the identified miRNAs or human homologs thereof . It is contemplated in some embodiments that a difference in expression of at least 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264 265 266 267 268 269 270 271 272 273 274 275 276 277 278 279 280 281 282 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 419 420 421 422 423 424 425 426 427 428 429 430 431 432 433 434 435 436 437 438 439 440 441 442 443 444 445 446 447 448 449 450 451 452 453 454 455 456 457 458 459 460 461 462 463 464 465 466 467 468 469 470 471 472 473 474 475 476 477 478 479 480 481 482 483 484 485 486 487 488 489 490 491 492 493 494 495 496 497 498 499 500 different miRNAs can be used as indicators of a disease condition or other phenotype. Moreover it is contemplated that in some cases the indicators provide information about the risk of having or developing a particular disease condition or phenotype. These different miRNAs include any of the following let 7 let 7A let 7C let7D as let 7F 2 miR 7 miR 9a miR 9 as miR 10a miR 15a miR 15b miR 16 miR 17 miR 20 miR 21 miR 22 miR 23a miR 23b miR 24 miR 25 miR 26a miR 27a miR 28 miR 29 miR 29b miR 30a miR 30a as miR 31 miR 32 miR 92 miR 95 miR 99b miR 103 miR 105 miR 106a miR 125a miR 125b miR 126 miR 126 as miR 126a miR 128 miR 130a miR 133 miR 133a miR 135a miR 137 miR 138 miR 139 miR 140 miR 141 miR 143 miR 144 miR 145 miR 150 miR 152 miR 181a miR 181b miR 182 miR 183 miR 184 miR 186 miR 188 miR 189 miR 192 miR 194 miR 195 miR 199 miR 199a miR 199a as miR 200b miR 201 miR 203 miR 204 miR 205 miR 207 miR 211 miR 212 miR 215 miR 219 miR 221 miR 222 miR 223 miR 224 miR 239 miR 290 miR 291 miR 291 5P miR 298 miR 301 miR 326 miR 328 mu miR 329 miR 331 miR 338 miR 341 miR 342 miR 344 miR 361 and miR 425.

It is specifically contemplated that any kit array or other detection technique or tool or any method can involve profiling for any of these miRNAs.

Cancer includes but is not limited to malignant cancers tumors metastatic cancers unresectable cancers chemo and or radiation resistant cancers and terminal cancers. It is specifically contemplated that in any embodiments involving a possible decrease or increase in expression of certain miRNAs that only a decrease may be evaluated only an increase may be evaluated or that both an increase and decrease in expression of any of the miRNA mentioned in that context or any other discussed herein may be evaluated.

In specific embodiments an miRNA array or methods involving 1 2 3 4 5 6 7 8 9 10 11 12 or 13 probes or any range derivable therein for miRNA s selected from the group consisting of miR 21 miR 15a miR 16 miR 24 miR 25 miR 99 miR 100 miR 205 miR 197 miR 126 miR 143 and miR 145 or from the group consisting of miR 15A miR 21 miR 24 miR 135A miR 145 miR 200B miR 205 miR 211 miR 298 and mu miR 329. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 10 11 12 or 13 of the miRNA s or any range derivable therein compared to a normal sample is indicative of breast cancer. In methods of the invention an increase in miR 21 miR 15a miR 16 miR 24 and or miR 25 expression and or a decrease in miR 99 miR 100 miR 205 miR 197 miR 126 miR 143 and or miR 145 expression compared to a normal sample is indicative of breast cancer. In other embodiments a decrease in expression of miR 135A miR 145 miR 205 miR 211 miR 298 or mu miR 329 and or an increase in expression of miR 15A miR 21 miR 24 or miR 200B compared to a normal sample is indicative of breast cancer. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

In other embodiments the miRNA array or methods involving 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 probes or any range derivable therein for miRNA s selected from the group consisting of miR 10a miR 17 miR 21 mir 23 miR 26a miR 30a miR 106 miR 125a miR 126 miR 130 miR 130a miR 133 miR 143 miR 144 miR 145 miR 188 miR 192 miR 194 miR 195 miR 199a miR 200b miR 215 miR 223 miR 224 miR 331 and miR 342 or selected from the group consisting of miR 21 miR 31 miR 106A miR 125a mir 130a miR 133 miR 135 miR 143 miR 145 miR 200B miR 203 and miR 223. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 or 26 of the miRNA s or any range derivable therein compared to a normal sample is indicative of colon cancer.

In certain embodiments a decrease in expression of 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 or 21 or any range derivable therein of the following miRNA s is indicative of colon cancer miR 145 miR 143 miR 133 miR 342 miR 125a miR 195 miR 30a miR 10a miR 130 miR 130a miR 192 miR 194 miR 215 miR 144 miR 23 miR 26a miR 126 miR 199a miR 188 miR 331 and or miR 21 and an increase in expression of 1 2 3 4 5 6 or 7 or any range derivable therein of the following miRNA s is indicative of colon cancer miR 21 miR 106 miR 200b miR 223 miR 224 and or miR 17. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

In specific embodiments an miRNA array or methods involving 1 2 3 4 5 6 7 8 or 9 probes or any range derivable therein for miRNA s selected from the group consisting of miR 21 miR 31 miR 106A miR 125a mir 130a miR 133 miR 135A miR 143 miR 145 miR 200B miR 203 and miR 223. A difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 or 9 of the miRNA s or any range derivable therein compared to a normal sample is indicative of colon cancer. In certain methods of the invention an increase in miR 17 miR 21 miR 106 and or miR 223 expression and or a decrease in miR 130a miR 143 miR 145 miR 195 and or miR 331 expression compared to a normal sample is indicative of colon cancer. In other embodiments a decrease in expression of miR 125a mir 130a miR 133 miR 135A miR 143 or miR 145 and or an increase in expression of miR 21 miR 31 miR 106A miR 200B miR 203 or miR 223 compared to a normal sample is indicative of colon cancer. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

Throughout this application the term difference in expression means that the level of a particular miRNA in a sample is higher or lower than the level of that particular miRNA in a normal sample. Normal sample in the context of testing for cancer means a noncancerous sample.

In further embodiments the miRNA array or methods involving 1 2 or 3 miRNA s or any range derivable therein selected from the group consisting of miR 21 miR 23 and miR 143 or selected from the group consisting of miR 15A miR 21 miR 29 miR 141 miR 188 miR 290 and miR 331. In methods of the invention a decrease in miR 21 miR 23 and or miR 143 expression compared to a normal sample is indicative of prostate cancer. In certain embodiments a decrease in expression of miR 188 miR 290 or miR 331 and or an increase in expression of miR 15A miR 21 miR 29 or miR 141 compared to a normal sample is indicative of prostate cancer. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

Moreover in additional embodiments the miRNA array or methods involving 1 2 3 4 5 6 7 8 9 10 1 12 13 14 15 16 17 or 18 probes or any range derivable therein for miRNA s selected from the group consisting of miR 21 miR 125 miR 24 miR 200b miR 29b miR 221 miR 222 miR 224 miR 10a miR 183 miR 145 miR 22 miR 331 miR 126 miR 30a miR 199a and miR 223 or selected from the group consisting of miR 15A miR 21 miR 30A as miR 31 miR 135A miR 138 miR 152 miR 199A as miR 200B miR 203 and miR 331. In methods of the invention an increase in miR 21 miR 125 miR 24 miR 200b miR 29b miR 221 miR 222 miR 224 miR 10a and or miR 183 expression and or a decrease in miR 145 miR 22 miR 126 miR 30a miR 199a miR 223 and or miR 331 expression compared to a normal sample is indicative of thyroid cancer. In certain embodiments a decrease in expression of miR 30A as miR 31 miR 135A miR 138 miR 152 miR 199A as miR 203 or miR 331 and or an increase in expression of miR 15A miR 21 or miR 200B compared to a normal sample is indicative of thyroid cancer. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

In specific embodiments an miRNA array or methods involving 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 or 28 probes or any range derivable therein for miRNA s selected from the group consisting of let 7 miR 16 miR 17 miR 21 miR 22 miR 24 miR 26a miR 29b miR 30a miR 106 miR 125a miR 126 miR 143 miR 145 miR 181a miR 182 miR 183 miR 188 miR 195 miR 200b mu miR 201 miR 205 miR 223 miR 328 miR 331 and miR 342 or selected from the group consisting of let 7a let 7c miR 21 miR 26a miR 30A AS miR 95 miR 125a miR 126 miR 200b miR 205 miR 331 and miR 342. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 or 28 of the miRNA s or any range derivable therein compared to a normal sample is indicative of lung cancer. In certain embodiments a decrease in expression of 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 or 19 or any range derivable therein of the following miRNA s is indicative of lung cancer miR 130a miR 145 miR 126 miR 331 miR 342 miR 143 Let 7 miR 30a miR 16 miR 26a miR 125a miR 29b miR 24 miR 328 mu miR 201 miR 195 miR 22 miR 181a and miR 331 and or an increase in expression of 1 2 3 4 5 6 7 8 or 9 or any range derivable therein of the following miRNA s is indicative of lung cancer miR 223 miR 106 miR 21 miR 200b miR 182 miR 183 miR 17 and miR 205. In other embodiments a decrease in expression of let 7a let 7c miR 26a miR 30A AS miR 95 miR 125a miR 126 miR 331 or miR 342 and or an increase in expression of miR 21 miR 200B or miR 205 compared to a normal sample is indicative of lung cancer. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

In specific embodiments an miRNA array or methods involving 1 2 3 4 5 6 7 8 or 9 miRNA probes or any range derivable therein for miRNA s selected from the group consisting of miR 21 miR 30a miR 16 miR 126 miR 143 miR 145 miR 188 miR 200b and miR 331. In methods of the invention using such an array a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 or 9 miRNA s or any range derivable therein compared to a normal sample is indicative of lung cancer. In methods of the invention an increase in miR 21 and or miR 200b expression and or a decrease in miR 30a miR 126 miR 143 miR 145 miR 188 and or miR 331 expression compared to a normal sample is indicative of lung cancer. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

In specific embodiments an miRNA array or methods involving 1 2 3 4 5 6 7 8 9 10 or 11 probes or any range derivable therein for miRNA s selected from the group consisting of miR 145 miR 143 miR 126 miR 30a miR 125a miR 21 miR 195 miR 17 miR 182 and miR 183. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 10 or 11 of the miRNA s or any range derivable therein compared to a normal sample is indicative of cancer. In certain embodiments a decrease in expression of 1 2 3 4 5 6 7 8 or 9 or any range derivable therein of the following miRNA s is indicative of cancer miR 145 miR 143 miR 126 miR 30a miR 125a miR 21 miR 195 and or miR 17 and an increase in expression of 1 2 3 4 5 6 or 7 or any range derivable therein of the following miRNA s is indicative of cancer miR 125a miR 21 miR 195 miR 17 miR 182 and or miR 183. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

In further embodiments the miRNA array or methods involving 1 2 3 4 5 6 or 7 probe s or any range derivable therein for any miRNA selected from the group consisting of miR 9as miR 21 miR 133 miR 143 miR 145 miR 182 and miR 200B. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 or 7 miRNA s or any range derivable therein compared to a normal sample is indicative of bladder cancer. In certain embodiments wherein a decrease in expression of miR 133 miR 143 or miR 145 and or a increase in expression of miR 9as miR 21 miR 182 or miR 200B compared to a normal sample is indicative of bladder cancer. Arrays or other detection techniques may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe. Methods involving arrays or other detection techniques may similarly involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

The present invention also concerns an miRNA array or methods involving 1 2 3 or 4 probe s or any range derivable therein for any miRNA selected from the group consisting of miR 29B miR 326 miR 361 and miR 425. In methods of the invention a difference in expression of of at least or of at most 1 2 3 or 4 of the miRNA s or any range derivable therein compared to a normal sample is indicative of cervical cancer. In methods of the invention using such an array an increase in expression of miR 29B miR 326 miR 361 or miR 425 is indicative of cervical cancer. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

The present invention also concerns an miRNA array or methods involving 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 or 27 probe s or any range derivable therein for any miRNA selected from the group consisting of let 7 miR 15A miR 16 miR 20 miR 21 miR 23A miR 23B miR 25 miR 26A miR 92 miR 99B miR 103 miR 106A miR 126 miR 126AS miR 181A miR 181B miR 221 miR 222 miR 223 miR 291 miR 341 miR 361 and miR 425 miR 326 miR 361 or miR 425. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 or 27 miRNA s or any range derivable therein compared to a normal sample is indicative of leukemia.

In certain embodiments an miRNA array or method involves 1 2 3 4 5 6 7 8 or 9 probe s or any range derivable therein for any miRNA selected from the group consisting of miR 25 miR 126 miR 126AS miR 181B miR 221 miR 222 miR 291 miR 361 and miR 425. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 or 9 miRNA s or any range derivable therein compared to a normal sample is indicative of acute myelogenous leukemia. Moreover in some cases a decrease in expression of of miR 25 miR 291 miR 361 or miR 425 and or a increase in expression of miR 126 miR 126AS miR 181B miR 221 or miR 222 compared to a normal sample is indicative of acute myelogenous leukemia.

In other embodiments an miRNA array or method involves 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 or 20 probe s or any range derivable therein for any miRNA selected from the group consisting of of let 7 miR 15A miR 16 miR 20 miR 21 miR 23A miR 23B miR 26A miR 92 miR 99B miR 103 miR 106A miR 181A miR 181B miR 221 miR 222 miR 223 miR 341 miR 361 and miR 425. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 or 9 miRNA s or any range derivable therein compared to a normal sample is indicative of chronic lymphocytic leukemia. Moreover in some cases a decrease in expression of of let 7 miR 15A miR 16 miR 20 miR 21 miR 23A miR 23B miR 26A miR 92 miR 99B miR 103 miR 106A miR 181A miR 181B miR 221 miR 222 or miR 223 and or a increase in expression of miR 341 miR 361 or miR 425 compared to a normal sample is indicative of chronic lymphocytic leukemia.

Other embodiments of the invention include an miRNA array that comprises 1 2 3 4 5 6 7 8 9 10 11 12 13 or 14 probes or any range derivable therein for miRNA s selected from the group consisting of miR 21 miR 95 miR 105 miR 137 miR 186 miR 188 miR 199 miR 211 miR 215 miR 223 mu miR 290 miR 301 miR 331 and miR 342 or selected from the group consisting of let7D AS miR 21 miR 32 miR 95 miR 133A miR 137 miR 141 miR 144 miR 181A miR 184 miR 186 miR 188 miR 199 miR 201 miR 203 miR 204 miR 211 miR 212 miR 223 miR 224 mu miR 329 miR 331 and miR 344. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 10 11 12 13 or 14 miRNA s or any range derivable therein compared to a normal sample is indicative of lupus. In methods of the invention using such an array an increase in miR 21 miR 223 and or mir 342 expression and or a decrease in miR 95 miR 105 miR 137 miR 186 miR 188 miR 199 miR 211 miR 215 mu miR 290 miR 301 and or miR 331 expression compared to a normal sample is indicative of systemic lupus erythematosus SLE . Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

The present invention also involves an miRNA array or method that involves 1 2 3 4 5 6 7 8 9 or 10 probes or any range derivable therein for miRNA s selected from the group consisting of miR 7 miR 9 as miR 16 miR 24 miR 26A miR 27A miR 95 miR 130A and miR 135A or selected from the group consisting of miR 7 miR 9 as miR 16 miR 24 miR 26A miR 27A miR 95 miR 130A miR 135A and miR 239. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 10 11 12 13 or 14 miRNA s or any range derivable therein compared to a normal sample is indicative of a prion disease or susceptibility to a prion disease. In certain methods of the invention an increase in miR 7 miR 9 as miR 16 miR 24 miR 26A miR 27A and or miR 130A expression and or a decrease in miR 95 and or miR 135A expression compared to a normal sample uninfected and or insensitive to prions is indicative of a prion disease or susceptibility to a prion disease. In other embodiments a decrease in expression of miR 95 or miR 135A and or an increase in expression of miR 7 miR 9 as miR 16 miR 24 miR 26A miR 27A miR 130A or miR 239 compared to a normal sample is indicative of prion disease or susceptibility to prion disease. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

Another miRNA array of the invention is one comprising 1 2 3 4 5 6 7 8 9 or 10 probes or any range derivable therein for miRNA s selected from the group consisting of Let7F 2 miR 16 miR 28 miR 30A miR 31 miR 138 miR 139 miR 140 mu miR 291 5P and mu miR 298. In methods of the invention an increase in miR 28 miR 30A miR 31 miR 138 miR 139 miR 140 mu miR 291 5P and or mu miR 298expression and or a decrease in Let7F 2 and or miR 16 expression compared to a normal sample is indicative of ischemia or susceptibility to ischemia. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

The present invention also concerns an miRNA array or method that involves 1 2 3 4 5 6 or 7 probes or any range derivable therein for miRNA s selected from the group consisting of miR 125B miR 126a miR 150 miR 192 miR 194 miR 207 and miR 223. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 or 7 miRNA s or any range derivable therein compared to a normal sample is indicative of

Crohn s disease or susceptibility to Crohn s disease. In certain methods of the invention a decrease in expression of miR 126as miR 192 miR 194 or miR 207 and or an increase in expression of miR 125B miR 150 or miR 223 compared to a normal sample is indicative of Crohn s disease or susceptibility to Crohn s disease. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

Other embodiments of the invention relate to an miRNA array or method that involves 1 2 3 4 5 6 7 8 9 or 10 probes or any range derivable therein for miRNA s selected from the group consisting of let 7F2 miR 16 miR 126 miR 143 miR 145 miR 204 miR 223 miR 291 miR 338 and miR 425. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 8 9 or 10 miRNA s or any range derivable therein compared to a normal sample is indicative of Alzheimer s disease or susceptibility to Alzheimer s disease. In certain methods of the invention a decrease in expression of let 7F2 miR 16 miR 126 miR 143 miR 145 or miR 223 and or an increase in expression miR 204 miR 291 miR 338 or miR 425 compared to a normal sample is indicative of Alzheimer s disease or susceptibility to Alzheimer s disease. In other embodiments a difference in expression of miR 182 is indicative of Alzheimer s disease or susceptibility to Alzheimer s disease. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

The present invention also concerns an miRNA array or method that involves 1 2 3 4 5 6 7 or 8 probes or any range derivable therein for miRNA s selected from the group consisting of let 7A let 7C miR 15B miR 16 miR 17 miR 106 miR 128 miR181A and miR 326. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 6 7 or 8 miRNA s or any range derivable therein compared to a normal sample is indicative of T cell development. In certain methods of the invention a decrease in expression of miR 326 and or an increase in expression let 7A let 7C miR 15B miR 16 miR 17 miR 106A miR 128 or miR 181A compared to a normal sample is indicative of T cell development. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

The present invention also concerns an miRNA array or method that involves 1 2 3 4 5 or 6 probes or any range derivable therein for miRNA s selected from the group consisting of miR 23A miR 23B miR 99B miR 126 miR 133A and miR 326. In methods of the invention a difference in expression of of at least or of at most 1 2 3 4 5 or 6 miRNA s or any range derivable therein compared to a normal sample is indicative of cardiac hypertrophy or risk of cardiac hypertrophy. In certain methods of the invention a decrease in expression of miR 23A miR 23B miR 99B miR 126 miR 133A or miR 326 compared to a normal sample is indicative of cardiac hypertrophy or risk of cardiac hypertrophy. Arrays may specifically involve any combination of probes for the miRNAs described in this paragraph as well as any other miRNA probe.

It is specifically contemplated that miRNA profiles for patients particularly those suspected of having a particular disease or condition can be generated by evaluating any of the miRNAs discussed in this application. The miRNA profile that is generated from the patient will be one that provides information regarding the particular disease or condition. In many embodiments the miRNA profile is generated using the miRNA array discussed.

The present invention also concerns methods for identifying a candidate diagnostic marker or therapeutic target of a disease or condition comprising a generating a miRNA profile for a first sample exhibiting the disease or condition b identifying any difference in the first miRNA profile as compared to a second miRNA profile of a normal sample wherein any difference between the samples identifies differentially expressed miRNAs as a candidate diagnostic marker or therapeutic target for the disease or condition. In some embodiments the sample exhibiting the disease or condition and the normal sample are identified from a single patient. In other embodiments methods also include evaluating a second sample exhibiting the same disease or condition as the first sample for a difference in expression of at least one miRNA compared to its expression in a normal sample. Any of the methods or arrays discussed above can involve miRNA profiling.

Moreover it is specifically contemplated that any embodiment discussed in the context of an miRNA array can be implemented with or without the array format in methods of the invention in other words any miRNA in an miRNA array may be screened or evaluated in any method of the invention according to any techniques known to those of skill in the art. The array format is not required for the screening and diagnostic methods to be implemented.

In embodiments discussed above the expression of a set of miRNAs may be evaluated for information regarding a particular disease or condition. It is specifically contemplated that the invention can be implemented in which one or more miRNAs identified as part of the set is excluded either in terms of it being probed for or in terms of its expression data being excluded from the analysis or conclusion of the rest of the set. In other words individual miRNAs may be disclaimed in any set identified herein with respect to the embodiments provided.

Other methods of the invention include methods of screening for a candidate therapeutic agent for a disease or condition comprising a contacting a sample exhibiting the disease or condition with a substance b generating an miRNA profile for the sample and c comparing the miRNA profile for the sample with an miRNA profile of a sample not contacted with the substance wherein a difference in the miRNA profiles identifies a candidate therapeutic agent.

In certain methods one can identify or select a patient for a clinical trial or for a particular drug regimen based on his or her miRNA profile. Therefore in some embodiments of the invention a patient is identified as a suitable recipient of a drug or drug regimen based on his miRNA profile. Such methods involve generating an miRNA profile for the patient in which the profile involves one or more miRNAs that are indicative of efficacy or toxicity with respect to the drug or drug regimen in question. In some embodiments methods involve identifying a patient in need of treatment that can be provided by the drug or drug regimen.

Kits are also included as part of the invention. Kits for implementing methods of the invention described herein are specifically contemplated. In some embodiments there are kits for preparing miRNA for multi labeling and kits for preparing miRNA probes and or miRNA arrays. In these embodiments kit comprise in suitable container means 1 2 3 4 5 6 7 8 9 10 11 12 or more of the following 1 poly A polymerase 2 unmodified nucleotides G A T C and or U 3 a modified nucleotide labeled or unlabeled 4 poly A polymerase buffer and 5 at least one microfilter 6 label that can be attached to a nucleotide 7 at least one miRNA probe 8 reaction buffer 9 an miRNA array or components for making such an array 10 acetic acid 11 alcohol 12 solutions for preparing isolating enriching and purifying miRNAs or miRNA probes or arrays. Other reagents include those generally used for manipulating RNA such as formamide loading dye ribonuclease inhibitors and DNase. Buffers as well as other solutions are contemplated to have a pH of about at least about or at most about 5.0 5.1 5.2 5.3 5.4 5.5 5.6 5.7 5.8 5.9 6.0 6.1 6.2 6.3 6.4 6.5 6.6 6.7 6.8 6.9 7.0 7.1 7.2 7.3 7.4 7.5 7.6 7.7 7.8 7.9 8.0 8.1 8.2 8.3 8.4 8.5 8.6 8.7 8.8 8.9 9.0 or more or any range derivable therein in certain embodiments of the invention.

Poly A polymerase may be from any source but specifically contemplated is a poly A polymerase from yeast or which may be recombinant or purified from the organism. A reaction buffer for poly A polymerase may be included in any kit of the invention. Typically such a poly A polymerase reaction buffer includes a volume exclusion reagent such as PEG magnesium and sodium. In certain embodiments the poly A polymerase reaction buffer in the kit contains at least about 5 6 7 8 9 10 11 12 13 14 15 or more or any range derivable therein PEG about 5 10 15 20 25 30 35 40 45 50 mM or more MgCl or any range derivable therein about 100 200 300 400 500 600 700 800 900 mM NaCl or any range derivable therein about 50 60 70 80 90 100 110 120 130 140 150 mM or more MES or any range derivable therein and about 0.5 1 1.5 2 2.5 3 3.5 4 mM or more DTT or any range derivable therein . The kits may also include a manganese source which may be included as a separate component of a kit or in a solution or buffer with other components such as in the reaction buffer. It is contemplated that about 5 10 15 20 25 30 35 40 45 50 mM or more of MnClis included in the kit.

Nucleotides may also be included in kits of the invention. Nucleotides may be for DNA or RNA. Concentrations of a nucleotide or of a nucleotide mix total concentration of all nucleotides include but are not limited to about at least about or at most about 0.5 1.0 1.5 2.0 2.5 3.0 3.5 4.0 4.5 5.0 5.5 6.0 6.5 7.0 7.5 8.0 8.5 9.0 9.5 10.0 mM or more or any range derivable therein . Moreover they may be modified or not modified. If they are modified they may have a reactive group or they may have a label attached to it. In certain embodiments one or more nucleotides in a kit has a reactive group such as an amine reactive group. In other embodiments a nucleotide is already labeled. It may be labeled with a chemiluminescent or fluorecent label such as a dye. Specifically contemplated are amine reactive dyes. Moreover it is specifically contemplated that kits may or may not contain both modified and unmodified nucleotides. Also kits may contain the label that will be attached to the nucleotide. Any label that can be attached to a nucleotide as well as any specifically identified herein can be included in kits of the invention.

Other solutions that may be included in a kit are those solutions involved in isolating and or enriching miRNA from a mixed sample. A lysis solution may comprise a chaotropic salt a detergent a salt and or a reducing agent. In certain embodiments a lysis solution contains one or more of the following about 1 2 3 4 5 6 7 or 8 M guanidinium thiocyanate or any range derivable therein about 0.1 0.2 0.3 0.4 0.5 0.6 0.7 0.8 0.9 1.0 or more N lauryl sarcosine or any range derivable therein about 5 10 15 20 25 30 35 40 45 50 mM or more NaCitrate and or about 0.0.5 0.1 0.15 0.2 M or more 2 mercaptoethanol or any range derivable therein . Wash solutions that may be contained in kits include wash solutions having a chaotropic salt and ethanol and wash solutions having a salt and buffer. In specific embodiments wash solutions include about 1.0 1.1 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 2.0 2.1 2.2 2.3 2.4 2.5 M or more guanidinium thiocyanate or any range derivable therein and or 10 15 20 25 30 35 40 45 50 55 60 or more ethanol or any range derivable therein . Other wash solutions can include about 50 60 70 80 90 100 110 120 130 140 150 mM NaCl or any range derivable therein 3.0 3.5 4.0 4.5 5.0 5.5 6.0 6.5 7.0 mM or more EDTA or any range derivable therein about 5 10 15 20 25 30 35 40 mM or more Tris or any range derivable therein .

It is also contemplated that reagents for purifying miRNA using gel or tube electrophoresis can be included in kits of the invention.

A variety of microfilters can be used and specifically contemplated in some embodiments of the invention is a glass fiber or silica filter column. Solutions that can be used with such microfilters include a binding buffer and or wash buffer. In certain embodiments the binding buffer includes a salt and an alcohol. In particular cases the binding or washing buffer has about 0.2 0.3 0.4 0.5 0.6 0.7 0.8 0.9 1.0 M or more NaCl or any range derivable therein and or about 40 45 50 55 60 65 70 75 80 85 90 or more ethanol or any range derivable therein .

Reagents and materials for preparing miRNA arrays can be included in a kit. The solid support for an array can be included in a kit such as one or more slides. A kit may contain slide wash buffer which in some embodiments includes a salt buffer and or detergent. In certain embodiments the slide wash buffer has about 100 110 120 130 140 150 160 170 180 190 200 mM or more NaCl or any range derivable therein 50 60 70 80 90 100 110 120 130 140 150 mM or more Tris or any range derivable therein and or 0.05. 0.06. 0.07. 0.08. 0.09 0.1 0.15 0.02 or more Tween 20 or any range derivable therein . Kits may include one or more miRNA probes described herein for use on an array.

Reagents for using miRNA arrays are also contemplated to be included as kit components. In some embodiments there is a hybridization solution and or an array wash solution. Such solutions can contain about 1.5 2.0 2.3 3.0 3.5 4.0 or more SSC or any range derivable therein .

Kits may comprise components which may be individually packaged or placed in a container such as a tube bottle vial syringe or other suitable container means.

Individual components may also be provided in a kit in concentrated amounts in some embodiments a component is provided individually in the same concentration as it would be in a solution with other components. Concentrations of components may be provided as 1 2 5 10 or 20 or more.

Kits for using miRNA arrays of the invention for therapeutic prognostic or diagnostic applications are included as part of the invention. Such kits can include an miRNA array as well as information regarding a standard or normalized miRNA profile for the miRNAs on the array.

Control RNA or DNA is included in some kit embodiments. Control RNA is miRNA that can be used as a positive control for labeling and or array analysis.

Other embodiments of the invention involve a system and or apparatus for electrophoresis of a sample to isolate nucleic acids. In one embodiment the present invention provides a micro electrophoresis apparatus. The micro electrophoresis apparatus of the present invention may be used to isolate any nucleic acid molecules. In some embodiments the apparatus is used to isolate miRNA or other small nucleic acid molecules. In one embodiment the micro electrophoresis apparatus comprises an upper buffer chamber electronically coupled to a negative electrode a lower buffer collection chamber electronically coupled to a positive electrode and a gel matrix disposed between the upper buffer chamber and the lower buffer collection chamber. In one embodiment the micro electrophoresis apparatus is the flashPAGE Fractionator Ambion Inc. .

In certain embodiments the upper buffer chamber has a volume equal to or less than about 10 milliliters ml . In certain aspects the upper buffer chamber has a volume equal to or less than about 9 ml 8 ml 7 ml 6 ml 5 ml 4 ml 3 ml 2 ml 1 ml or 500 microliters l . In certain embodiments the lower buffer collection chamber has a volume equal to or less than about 10 ml. In certain aspects the lower buffer collection chamber has a volume equal to or less than about 9 ml 8 ml 7 ml 6 ml 5 ml 4 ml 3 ml 2 ml 1 ml or 500 l.

In certain embodiments of the invention the gel matrix is a polyacrylamide gel matrix. The gel matrix may be a pre cast gel matrix such as flashPAGE Pre Cast Gel cartridges Ambion Inc. .

The micro electrophoresis apparatus of the present invention may be configured to operate on any low current DC power source. For example the apparatus may operate on a 50 100 V and 2 5 mA DC source.

Any embodiments discussed with respect to compositions and methods of the invention as well as any embodiments in the Examples is specifically contemplated as being part of a kit.

It is contemplated that any method or composition described herein can be implemented with respect to any other method or composition described herein.

Throughout this application the term about is used to indicate that a value includes the standard deviation of error for the device or method being employed to determine the value.

The use of the word a or an when used in conjunction with the term comprising in the claims and or the specification may mean one but it is also consistent with the meaning of one or more at least one and one or more than one. 

It is specifically contemplated that any embodiments described in the Examples section are included as an embodiment of the invention.

Other objects features and advantages of the present invention will become apparent from the following detailed description. It should be understood however that the detailed description and the specific examples while indicating specific embodiments of the invention are given by way of illustration only since various changes and modifications within the spirit and scope of the invention will become apparent to those skilled in the art from this detailed description.

The present invention is directed to compositions and methods relating to preparation and characterization of miRNAs as well as use of miRNAs for therapeutic prognostic and diagnostic applications.

MicroRNA molecules miRNAs are generally 21 to 22 nucleotides in length though lengths of 19 and up to 23 nucleotides have been reported. The miRNAs are each processed from a longer precursor RNA molecule precursor miRNA . Precursor miRNAs are transcribed from non protein encoding genes. The precursor miRNAs have two regions of complementarity that enables them to form a stem loop or fold back like structure which is cleaved by an enzyme called Dicer in animals. Dicer is ribonuclease III like nuclease. The processed miRNA is typically a portion of the stem.

The processed miRNA also referred to as mature miRNA become part of a large complex to down regulate a particular target gene. Examples of animal miRNAs include those that imperfectly basepair with the target which halts translation Olsen et al. 1999 Seggerson et al. 2002 . SiRNA molecules also are processed by Dicer but from a long double stranded RNA molecule. SiRNAs are not naturally found in animal cells but they can function in such cells in a RNA induced silencing complex RISC to direct the sequence specific cleavage of an mRNA target Denli et al. 2003 .

The present invention concerns miRNAs that can be labeled used in array analysis or employed in diagnostic therapeutic or prognostic applications. The RNA may have been endogenously produced by a cell or been synthesized or produced chemically or recombinantly. They may be isolated and or purified. The term miRNA unless otherwise indicated refers to the processed RNA after it has been cleaved from its precursor. Table 1 indicates which SEQ ID NO corresponds to the particular precursor sequence of an miRNA and what sequences within the SEQ ID NO correspond to the mature sequence. The name of the miRNA is often abbreviated and referred to without the prefix and will be understood as such depending on the context. Unless otherwise indicated miRNAs referred to in the application are human sequences identified as mir X or let X where X is a number and or letter.

In certain experiments an miRNA probe designated by a suffix 5P or 3P can be used. 5P indicates that the mature miRNA derives from the 5 end of the precursor and a corresponding 3P indicates that it ferives from the 3 end of the precursor as described on the world wide web at sanger.ac.uk cgi bin rfam mirna. Moreover in some embodiments an miRNA probe is used that does not correspond to a known human miRNA. It is contemplated that these non human miRNA probes may be used in embodiments of the invention or that there may exist a human miRNA that is homologous to the non human miRNA. While the invention is not limited to human miRNA in certain embodiments miRNA from human cells or a human biological sample is evaluated. In other embodiments any mammalian cell or biological sample may be employed.

In some embodiments of the invention methods and compositions involving miRNA may concern miRNA and or other nucleic acids. Nucleic acids may be be at least or be at most 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 441 450 460 470 480 490 500 510 520 530 540 550 560 570 580 590 600 610 620 630 640 650 660 670 680 690 700 710 720 730 740 750 760 770 780 790 800 810 820 830 840 850 860 870 880 890 900 910 920 930 940 950 960 970 980 990 or 1000 nucleotides or any range derivable therein in length. Such lengths cover the lengths of processed miRNA miRNA probes precursor miRNA control nucleic acids and other probes and primers. In many embodiments miRNA are 19 24 nucleotides in length while miRNA probes are 19 35 nucleotides in length depending on the length of the processed miRNA and any flanking regions added. miRNA precursors are generally between 62 and 110 nucleotides in humans.

Nucleic acids of the invention may have regions of identity or complementarity to another nucleic acid. It is contemplated that the region of complementarity or identity can be at least 5 contiguous residues though it is specifically contemplated that the region is is at least or is at most 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 110 120 130 140 150 160 170 180 190 200 210 220 230 240 250 260 270 280 290 300 310 320 330 340 350 360 370 380 390 400 410 420 430 440 441 450 460 470 480 490 500 510 520 530 540 550 560 570 580 590 600 610 620 630 640 650 660 670 680 690 700 710 720 730 740 750 760 770 780 790 800 810 820 830 840 850 860 870 880 890 900 910 920 930 940 950 960 970 980 990 or 1000 contiguous nucleotides. It is further understood that the length of complementarity within a precursor miRNA or between an miRNA probe and an miRNA or an miRNA gene are such lengths. Moreover the complementarity may be expressed as a percentage meaning that the complementarity between a probe and its target is 90 or greater over the length of the probe. On some embodiments complementarity is or is at least 90 95 or 100 . In particular such lengths may be applied to any SEQ ID NO identified in any of SEQ ID NO 1 through SEQ ID NO 898 inclusive SEQ ID NO 1 SEQ ID NO 898 or any other sequence disclosed herein. Each of these SEQ ID NOs is disclosed in the tables below. The tables list known human miRNAs Table 1A mouse miRNAs Table 2 and rat miRNAs Table 3 . In the tables the commonly used name of the miRNA is given with its identifying source in the prefix for example hsa for human sequences the miRNA precursor sequence as identified in the corresponding SEQ ID NO and the processed miRNA sequence relative to the nucleotides identified in the miRNA precursor sequence. Unless otherwise indicated an miRNA without a prefix will understood to refer to a human miRNA. An miRNA designated for example as miR 1 2 in the application will be understood to refer to hsa mir 1 2 below. Moreover a lowercase letter in the table below may or may not be lowercase for example hsa mir 130b can also be referred to as miR 130B. In addition miRNA sequences with a mu or mmu sequence will be understood to refer to a mouse miRNA.

The term miRNA probe refers to a nucleic acid probe that can identify a particular miRNA or structurally related miRNAs. Table 1B shows what probe was used to identify or screen for which miRNA. Table 1C provides the sequence of the probe. It is contemplated that all or part of any of the probes disclosed in Tables 1B or 1C can be implemented in any embodiments of the invention involving miRNA probes. Moreover any of the miRNA identified in Table 1B may be screened for or profiled in embodiments of the invention.

It is understood that an miRNA is derived from genomic sequences or a gene. In this respect the term gene is used for simplicity to refer to the genomic sequence encoding the precursor miRNA for a given miRNA. However embodiments of the invention may involve genomic sequences of a miRNA that are involved in its expression such as a promoter or other regulatory sequences.

The term recombinant may be used and this generally refers to a molecule that has been manipulated in vitro or that is the replicated or expressed product of such a molecule.

The term nucleic acid is well known in the art. A nucleic acid as used herein will generally refer to a molecule one or more strands of DNA RNA or a derivative or analog thereof comprising a nucleobase. A nucleobase includes for example a naturally occurring purine or pyrimidine base found in DNA e.g. an adenine A a guanine G a thymine T or a cytosine C or RNA e.g. an A a G an uracil U or a C . The term nucleic acid encompass the terms oligonucleotide and polynucleotide each as a subgenus of the term nucleic acid. 

The term miRNA generally refers to a single stranded molecule but in specific embodiments molecules implemented in the invention will also encompass a region or an additional strand that is partially between 10 and 50 complementary across length of strand substantially greater than 50 but less than 100 complementary across length of strand or fully complementary to another region of the same single stranded molecule or to another nucleic acid. Thus nucleic acids may encompass a molecule that comprises one or more complementary or self complementary strand s or complement s of a particular sequence comprising a molecule. For example precursor miRNA may have a self complementary region which is up to 100 complementary. miRNA probes of the invention can be or be at least 60 65 70 75 80 85 90 95 or 100 complementary to their target.

As used herein hybridization hybridizes or capable of hybridizing is understood to mean the forming of a double or triple stranded molecule or a molecule with partial double or triple stranded nature. The term anneal as used herein is synonymous with hybridize. The term hybridization hybridize s or capable of hybridizing encompasses the terms stringent condition s or high stringency and the terms low stringency or low stringency condition s . 

As used herein stringent condition s or high stringency are those conditions that allow hybridization between or within one or more nucleic acid strand s containing complementary sequence s but precludes hybridization of random sequences. Stringent conditions tolerate little if any mismatch between a nucleic acid and a target strand. Such conditions are well known to those of ordinary skill in the art and are preferred for applications requiring high selectivity. Non limiting applications include isolating a nucleic acid such as a gene or a nucleic acid segment thereof or detecting at least one specific mRNA transcript or a nucleic acid segment thereof and the like.

Stringent conditions may comprise low salt and or high temperature conditions such as provided by about 0.02 M to about 0.5 M NaCl at temperatures of about 42 C. to about 70 C. It is understood that the temperature and ionic strength of a desired stringency are determined in part by the length of the particular nucleic acid s the length and nucleobase content of the target sequence s the charge composition of the nucleic acid s and to the presence or concentration of formamide tetramethylammonium chloride or other solvent s in a hybridization mixture.

It is also understood that these ranges compositions and conditions for hybridization are mentioned by way of non limiting examples only and that the desired stringency for a particular hybridization reaction is often determined empirically by comparison to one or more positive or negative controls. Depending on the application envisioned it is preferred to employ varying conditions of hybridization to achieve varying degrees of selectivity of a nucleic acid towards a target sequence. In a non limiting example identification or isolation of a related target nucleic acid that does not hybridize to a nucleic acid under stringent conditions may be achieved by hybridization at low temperature and or high ionic strength. Such conditions are termed low stringency or low stringency conditions and non limiting examples of low stringency include hybridization performed at about 0.15 M to about 0.9 M NaCl at a temperature range of about 20 C. to about 50 C. Of course it is within the skill of one in the art to further modify the low or high stringency conditions to suite a particular application.

As used herein a nucleobase refers to a heterocyclic base such as for example a naturally occurring nucleobase i.e. an A T G C or U found in at least one naturally occurring nucleic acid i.e. DNA and RNA and naturally or non naturally occurring derivative s and analogs of such a nucleobase. A nucleobase generally can form one or more hydrogen bonds anneal or hybridize with at least one naturally occurring nucleobase in manner that may substitute for naturally occurring nucleobase pairing e.g. the hydrogen bonding between A and T G and C and A and U .

 Purine and or pyrimidine nucleobase s encompass naturally occurring purine and or pyrimidine nucleobases and also derivative s and analog s thereof including but not limited to those a purine or pyrimidine substituted by one or more of an alkyl caboxyalkyl amino hydroxyl halogen i.e. fluoro chloro bromo or iodo thiol or alkylthiol moeity. Preferred alkyl e.g. alkyl caboxyalkyl etc. moieties comprise of from about 1 about 2 about 3 about 4 about 5 to about 6 carbon atoms. Other non limiting examples of a purine or pyrimidine include a deazapurine a 2 6 diaminopurine a 5 fluorouracil a xanthine a hypoxanthine a 8 bromoguanine a 8 chloroguanine a bromothymine a 8 aminoguanine a 8 hydroxyguanine a 8 methylguanine a 8 thioguanine an azaguanine a 2 aminopurine a 5 ethylcytosine a 5 methylcyosine a 5 bromouracil a 5 ethyluracil a 5 iodouracil a 5 chlorouracil a 5 propyluracil a thiouracil a 2 methyladenine a methylthioadenine a N N diemethyladenine an azaadenines a 8 bromoadenine a 8 hydroxyadenine a 6 hydroxyaminopurine a 6 thiopurine a 4 6 aminohexyl cytosine and the like. Other examples are well known to those of skill in the art.

A nucleobase may be comprised in a nucleoside or nucleotide using any chemical or natural synthesis method described herein or known to one of ordinary skill in the art. Such nucleobase may be labeled or it may be part of a molecule that is labeled and contains the nucleobase.

As used herein a nucleoside refers to an individual chemical unit comprising a nucleobase covalently attached to a nucleobase linker moiety. A non limiting example of a nucleobase linker moiety is a sugar comprising 5 carbon atoms i.e. a 5 carbon sugar including but not limited to a deoxyribose a ribose an arabinose or a derivative or an analog of a 5 carbon sugar. Non limiting examples of a derivative or an analog of a 5 carbon sugar include a 2 fluoro 2 deoxyribose or a carbocyclic sugar where a carbon is substituted for an oxygen atom in the sugar ring.

Different types of covalent attachment s of a nucleobase to a nucleobase linker moiety are known in the art. By way of non limiting example a nucleoside comprising a purine i.e. A or G or a 7 deazapurine nucleobase typically covalently attaches the 9 position of a purine or a 7 deazapurine to the 1 position of a 5 carbon sugar. In another non limiting example a nucleoside comprising a pyrimidine nucleobase i.e. C T or U typically covalently attaches a 1 position of a pyrimidine to a 1 position of a 5 carbon sugar Kornberg and Baker 1992 .

As used herein a nucleotide refers to a nucleoside further comprising a backbone moiety . A backbone moiety generally covalently attaches a nucleotide to another molecule comprising a nucleotide or to another nucleotide to form a nucleic acid. The backbone moiety in naturally occurring nucleotides typically comprises a phosphorus moiety which is covalently attached to a 5 carbon sugar. The attachment of the backbone moiety typically occurs at either the 3 or 5 position of the 5 carbon sugar. However other types of attachments are known in the art particularly when a nucleotide comprises derivatives or analogs of a naturally occurring 5 carbon sugar or phosphorus moiety.

A nucleic acid may comprise or be composed entirely of a derivative or analog of a nucleobase a nucleobase linker moiety and or backbone moiety that may be present in a naturally occurring nucleic acid. RNA with nucleic acid analogs may also be labeled according to methods of the invention. As used herein a derivative refers to a chemically modified or altered form of a naturally occurring molecule while the terms mimic or analog refer to a molecule that may or may not structurally resemble a naturally occurring molecule or moiety but possesses similar functions. As used herein a moiety generally refers to a smaller chemical or molecular component of a larger chemical or molecular structure. Nucleobase nucleoside and nucleotide analogs or derivatives are well known in the art and have been described see for example Scheit 1980 incorporated herein by reference .

Additional non limiting examples of nucleosides nucleotides or nucleic acids comprising 5 carbon sugar and or backbone moiety derivatives or analogs include those in U.S. Pat. No. 5 681 947 which describes oligonucleotides comprising purine derivatives that form triple helixes with and or prevent expression of dsDNA U.S. Pat. Nos. 5 652 099 and 5 763 167 which describe nucleic acids incorporating fluorescent analogs of nucleosides found in DNA or RNA particularly for use as fluorescent nucleic acids probes U.S. Pat. No. 5 614 617 which describes oligonucleotide analogs with substitutions on pyrimidine rings that possess enhanced nuclease stability U.S. Pat. Nos. 5 670 663 5 872 232 and 5 859 221 which describe oligonucleotide analogs with modified 5 carbon sugars i.e. modified 2 deoxyfuranosyl moieties used in nucleic acid detection U.S. Pat. No. 5 446 137 which describes oligonucleotides comprising at least one 5 carbon sugar moiety substituted at the 4 position with a substituent other than hydrogen that can be used in hybridization assays U.S. Pat. No. 5 886 165 which describes oligonucleotides with both deoxyribonucleotides with 3 5 internucleotide linkages and ribonucleotides with 2 5 internucleotide linkages U.S. Pat. No. 5 714 606 which describes a modified internucleotide linkage wherein a 3 position oxygen of the internucleotide linkage is replaced by a carbon to enhance the nuclease resistance of nucleic acids U.S. Pat. No. 5 672 697 which describes oligonucleotides containing one or more 5 methylene phosphonate internucleotide linkages that enhance nuclease resistance U.S. Pat. Nos. 5 466 786 and 5 792 847 which describe the linkage of a substituent moiety which may comprise a drug or label to the 2 carbon of an oligonucleotide to provide enhanced nuclease stability and ability to deliver drugs or detection moieties U.S. Pat. No. 5 223 618 which describes oligonucleotide analogs with a 2 or 3 carbon backbone linkage attaching the 4 position and 3 position of adjacent 5 carbon sugar moiety to enhanced cellular uptake resistance to nucleases and hybridization to target RNA U.S. Pat. No. 5 470 967 which describes oligonucleotides comprising at least one sulfamate or sulfamide internucleotide linkage that are useful as nucleic acid hybridization probe U.S. Pat. Nos. 5 378 825 5 777 092 5 623 070 5 610 289 and 5 602 240 which describe oligonucleotides with three or four atom linker moiety replacing phosphodiester backbone moiety used for improved nuclease resistance cellular uptake and regulating RNA expression U.S. Pat. No. 5 858 988 which describes hydrophobic carrier agent attached to the 2 O position of oligonucleotides to enhanced their membrane permeability and stability U.S. Pat. No. 5 214 136 which describes oligonucleotides conjugated to anthraquinone at the 5 terminus that possess enhanced hybridization to DNA or RNA enhanced stability to nucleases U.S. Pat. No. 5 700 922 which describes PNA DNA PNA chimeras wherein the DNA comprises 2 deoxy erythro pentofuranosyl nucleotides for enhanced nuclease resistance binding affinity and ability to activate RNase H and U.S. Pat. No. 5 708 154 which describes RNA linked to a DNA to form a DNA RNA hybrid U.S. Pat. No. 5 728 525 which describes the labeling of nucleoside analogs with a universal fluorescent label.

Additional teachings for nucleoside analogs and nucleic acid analogs are U.S. Pat. No. 5 728 525 which describes nucleoside analogs that are end labeled U.S. Pat. No. 5 637 683 6 251 666 L nucleotide substitutions and U.S. Pat. No. 5 480 980 7 deaza 2 deoxyguanosine nucleotides and nucleic acid analogs thereof .

Labeling methods and kits of the invention specifically contemplate the use of nucleotides that are both modified for attachment of a label and can be incorporated into an miRNA molecule. Such nucleotides include those that can be labeled with a dye including a fluorescent dye or with a molecule such as biotin. Labeled nucleotides are readily available they can be acquired commercially or they can be synthesized by reactions known to those of skill in the art.

Modified nucleotides for use in the invention are not naturally occurring nucleotides but instead refer to prepared nucleotides that have a reactive moiety on them. Specific reactive functionalities of interest include amino sulfhydryl sulfoxyl aminosulfhydryl azido epoxide isothiocyanate isocyanate anhydride monochlorotriazine dichlorotriazine mono or dihalogen substituted pyridine mono or disubstituted diazine maleimide epoxide aziridine sulfonyl halide acid halide alkyl halide aryl halide alkylsulfonate N hydroxysuccinimide ester imido ester hydrazine azidonitrophenyl azide 3 2 pyridyl dithio propionamide glyoxal aldehyde iodoacetyl cyanomethyl ester p nitrophenyl ester o nitrophenyl ester hydroxypyridine ester carbonyl imidazole and the other such chemical groups. In some embodiments the reactive functionality may be bonded directly to a nucleotide or it may be bonded to the nucleotide through a linking group. The functional moiety and any linker cannot substantially impair the ability of the nucleotide to be added to the miRNA or to be labeled. Representative linking groups include carbon containing linking groups typically ranging from about 2 to 18 usually from about 2 to 8 carbon atoms where the carbon containing linking groups may or may not include one or more heteroatoms e.g. S O N etc. and may or may not include one or more sites of unsaturation. Of particular interest in many embodiments are alkyl linking groups typically lower alkyl linking groups of 1 to 16 usually 1 to 4 carbon atoms where the linking groups may include one or more sites of unsaturation. The functionalized nucleotides or primers used in the above methods of functionalized target generation may be fabricated using known protocols or purchased from commercial vendors e.g. Sigma Roche Ambion and NEN. Functional groups may be prepared according to ways known to those of skill in the art including the representative information found in U.S. Pat. Nos. 4 404 289 4 405 711 4 337 063 and 5 268 486 and Br. Pat. No. 1 529 202 which are all incorporated by reference.

Amine modified nucleotides are used in several embodiments of the invention. The amine modified nucleotide is a nucleotide that has a reactive amine group for attachment of the label. It is contemplated that any ribonucleotide G A U or C or deoxyribonucleotide G A T or C can be modified for labeling. Examples include but are not limited to the following modified ribo and deoxyribo nucleotides 5 3 aminoallyl UTP 8 4 amino butyl amino ATP and 8 6 amino butyl amino ATP N 4 amino butyl ATP N 6 amino butyl ATP N 2 2 oxy bis ethylamine CTP N 6 Amino hexyl ATP 8 6 Amino hexyl amino ATP 5 propargylamino CTP 5 propargylamino UTP 5 3 aminoallyl dUTP 8 4 amino butyl amino dATP and 8 6 amino butyl amino dATP N 4 amino butyl dATP N 6 amino butyl dATP N 2 2 oxy bis ethylamine dCTP N 6 Amino hexyl dATP 8 6 Amino hexyl amino dATP 5 propargylamino dCTP and 5 propargylamino dUTP. Such nucleotides can be prepared according to methods known to those of skill in the art. Moreover a person of ordinary skill in the art could prepare other nucleotide entities with the same amine modification such as a 5 3 aminoallyl CTP GTP ATP dCTP dGTP dTTP or dUTP in place of a 5 3 aminoallyl UTP.

A nucleic acid may be made by any technique known to one of ordinary skill in the art such as for example chemical synthesis enzymatic production or biological production. It is specifically contemplated that miRNA probes of the invention are chemically synthesized.

In some embodiments of the invention miRNAs are recovered from a biological sample. The miRNA may be recombinant or it may be natural or endogenous to the cell produced from the cell s genome . It is contemplated that a biological sample may be treated in a way so as to enhance the recovery of small RNA molecules such as miRNA. U.S. patent application Ser. No. 10 667 126 describes such methods and it is specifically incorporated by reference herein. Generally methods involve lysing cells with a solution having guanidinium and a detergent as described in Example 1.

Alternatively nucleic acid synthesis is performed according to standard methods. See for example Itakura and Riggs 1980 . Additionally U.S. Pat. No. 4 704 362 U.S. Pat. No. 5 221 619 and U.S. Pat. No. 5 583 013 each describe various methods of preparing synthetic nucleic acids. Non limiting examples of a synthetic nucleic acid e.g. a synthetic oligonucleotide include a nucleic acid made by in vitro chemically synthesis using phosphotriester phosphite or phosphoramidite chemistry and solid phase techniques such as described in EP 266 032 incorporated herein by reference or via deoxynucleoside H phosphonate intermediates as described by Froehler et al. 1986 and U.S. Pat. No. 5 705 629 each incorporated herein by reference. In the methods of the present invention one or more oligonucleotide may be used. Various different mechanisms of oligonucleotide synthesis have been disclosed in for example U.S. Pat Nos. 4 659 774 4 816 571 5 141 813 5 264 566 4 959 463 5 428 148 5 554 744 5 574 146 5 602 244 each of which is incorporated herein by reference.

A non limiting example of an enzymatically produced nucleic acid include one produced by enzymes in amplification reactions such as PCR see for example U.S. Pat. No. 4 683 202 and U.S. Pat. No. 4 682 195 each incorporated herein by reference or the synthesis of an oligonucleotide described in U.S. Pat. No. 5 645 897 incorporated herein by reference. A non limiting example of a biologically produced nucleic acid includes a recombinant nucleic acid produced i.e. replicated in a living cell such as a recombinant DNA vector replicated in bacteria see for example Sambrook et al. 1989 incorporated herein by reference .

Oligonucleotide synthesis is well known to those of skill in the art. Various different mechanisms of oligonucleotide synthesis have been disclosed in for example U.S. Pat. Nos. 4 659 774 4 816 571 5 141 813 5 264 566 4 959 463 5 428 148 5 554 744 5 574 146 5 602 244 each of which is incorporated herein by reference.

Basically chemical synthesis can be achieved by the diester method the triester method polynucleotides phosphorylase method and by solid phase chemistry. These methods are discussed in further detail below.

Diester method. The diester method was the first to be developed to a usable state primarily by Khorana and co workers. Khorana 1979 . The basic step is the joining of two suitably protected deoxynucleotides to form a dideoxynucleotide containing a phosphodiester bond. The diester method is well established and has been used to synthesize DNA molecules Khorana 1979 .

Triester method. The main difference between the diester and triester methods is the presence in the latter of an extra protecting group on the phosphate atoms of the reactants and products Itakura et al. 1975 . The phosphate protecting group is usually a chlorophenyl group which renders the nucleotides and polynucleotide intermediates soluble in organic solvents. Therefore purification s are done in chloroform solutions. Other improvements in the method include i the block coupling of trimers and larger oligomers ii the extensive use of high performance liquid chromatography for the purification of both intermediate and final products and iii solid phase synthesis.

Polynucleotide phosphorylase method. This is an enzymatic method of DNA synthesis that can be used to synthesize many useful oligonucleotides Gillam et al. 1978 Gillam et al. 1979 . Under controlled conditions polynucleotide phosphorylase adds predominantly a single nucleotide to a short oligonucleotide. Chromatographic purification allows the desired single adduct to be obtained. At least a trimer is required to start the procedure and this primer must be obtained by some other method. The polynucleotide phosphorylase method works and has the advantage that the procedures involved are familiar to most biochemists.

Solid phase methods. Drawing on the technology developed for the solid phase synthesis of polypeptides it has been possible to attach the initial nucleotide to solid support material and proceed with the stepwise addition of nucleotides. All mixing and washing steps are simplified and the procedure becomes amenable to automation. These syntheses are now routinely carried out using automatic nucleic acid synthesizers.

Phosphoramidite chemistry Beaucage and Lyer 1992 has become by far the most widely used coupling chemistry for the synthesis of oligonucleotides. As is well known to those skilled in the art phosphoramidite synthesis of oligonucleotides involves activation of nucleoside phosphoramidite monomer precursors by reaction with an activating agent to form activated intermediates followed by sequential addition of the activated intermediates to the growing oligonucleotide chain generally anchored at one end to a suitable solid support to form the oligonucleotide product.

Recombinant methods. Recombinant methods for producing nucleic acids in a cell are well known to those of skill in the art. These include the use of vectors viral and non viral plasmids cosmids and other vehicles for delivering a nucleic acid to a cell which may be the target cell or simply a host cell to produce large quantities of the desired RNA molecule . Alternatively such vehicles can be used in the context of a cell free system so long as the reagents for generating the RNA molecule are present. Such methods include those described in Sambrook 2003 Sambrook 2001 and Sambrook 1989 which are hereby incorporated by reference.

In certain embodiments the present invention concerns nucleic acid molecules that are not synthetic. In some embodiments the nucleic acid molecule has a chemical structure of a naturally occuring nucleic acid and a sequence of a naturally occuring nucleic acid such as the exact and entire sequence of a single stranded primary miRNA see Lee 2002 a single stranded precursor miRNA or a single stranded mature miRNA. In addition to the use of recombinant technology such non synthetic nucleic acids may be generated chemically such as by employing technology used for creating oligonucleotides.

Nucleic acids may be isolated using techniques well known to those of skill in the art though in particular embodiments methods for isolating small nucleic acid molecules and or isolating RNA molecules can be employed. Chromatography is a process often used to separate or isolate nucleic acids from protein or from other nucleic acids. Such methods can involve electrophoresis with a gel matrix filter columns alcohol precipitation and or other chromatography. If miRNA from cells is to be used or evaluated methods generally involve lysing the cells with a chaotropic e.g. guanidinium isothiocyanate and or detergent e.g. N lauroyl sarcosine prior to implementing processes for isolating particular populations of RNA.

In particular methods for separating miRNA from other nucleic acids a gel matrix is prepared using polyacrylamide though agarose can also be used. The gels may be graded by concentration or they may be uniform. Plates or tubing can be used to hold the gel matrix for electrophoresis. Usually one dimensional electrophoresis is employed for the separation of nucleic acids. Plates are used to prepare a slab gel while the tubing glass or rubber typically can be used to prepare a tube gel. The phrase tube electrophoresis refers to the use of a tube or tubing instead of plates to form the gel. Materials for implementing tube electrophoresis can be readily prepared by a person of skill in the art or purchased such as from C.B.S. Scientific Co. Inc. or Scie Plas.

Methods may involve the use of organic solvents and or alcohol to isolate nucleic acids particularly miRNA used in methods and compositions of the invention. Some embodiments are described in U.S. patent application Ser. No. 10 667 126 which is hereby incorporated by reference. Generally this disclosure provides methods for efficiently isolating small RNA molecules from cells comprising adding an alcohol solution to a cell lysate and applying the alcohol lysate mixture to a solid support before eluting the RNA molecules from the solid support. In some embodiments the amount of alcohol added to a cell lysate achieves an alcohol concentration of about 55 to 60 . While different alcohols can be employed ethanol works well. A solid support may be any structure and it includes beads filters and columns which may include a mineral or polymer support with electronegative groups. A glass fiber filter or column has worked particularly well for such isolation procedures.

In specific embodiments miRNA isolation processes include a lysing cells in the sample with a lysing solution comprising guanidinium wherein a lysate with a concentration of at least about 1 M guanidinium is produced b extracting miRNA molecules from the lysate with an extraction solution comprising phenol c adding to the lysate an alcohol solution for form a lysate alcohol mixture wherein the concentration of alcohol in the mixture is between about 35 to about 70 d applying the lysate alcohol mixture to a solid support e eluting the miRNA molecules from the solid support with an ionic solution and f capturing the miRNA molecules. Typically the sample is dried down and resuspended in a liquid and volume appropriate for subsequent manipulation.

In some embodiments the present invention concerns miRNA that are labeled. It is contemplated that miRNA may first be isolated and or purified prior to labeling. This may achieve a reaction that more efficiently labels the miRNA as opposed to other RNA in a sample in which the miRNA is not isolated or purified prior to labeling. In many embodiments of the invention the label is non radioactive. Generally nucleic acids may be labeled by adding labeled nucleotides one step process or adding nucleotides and labeling the added nucleotides two step process .

In some embodiments nucleic acids are labeled by catalytically adding to the nucleic acid an already labeled nucleotide or nucleotides. One or more labeled nucleotides can be added to miRNA molecules. See U.S. Pat. No. 6 723 509 which is hereby incorporated by reference.

In other embodiments an unlabeled nucleotide or nucleotides is catalytically added to an miRNA and the unlabeled nucleotide is modified with a chemical moiety that enables it to be subsequently labeled. In embodiments of the invention the chemical moiety is a reactive amine such that the nucleotide is an amine modified nucleotide.

Examples of amine modified nucleotides are well known to those of skill in the art many being commercially available such as from Ambion Sigma Jena Bioscience and TriLink.

In contrast to labeling of cDNA during its synthesis the issue for labeling miRNA is how to label the already existing molecule. The present invention concerns the use of an enzyme capable of using a di or tri phosphate ribonucleotide or deoxyribonucleotide as a substrate for its addition to an miRNA a small RNA molecule. Moreover in specific embodiments it involves using a modified di or tri phosphate ribonucleotide which is added to the 3 end of an miRNA. The source of the enzyme is not limiting. Examples of sources for the enzymes include yeast gram negative bacteria such as and sheep pox virus.

Enzymes capable of adding such nucleotides include but are not limited to poly A polymerase terminal transferase and polynucleotide phosphorylase. In specific embodiments of the invention ligase is contemplated as NOT being the enzyme used to add the label and instead a non ligase enzyme is employed.

Poly A polymerase has been cloned from a number of organisms from plants to humans. It has been shown to catalyze the addition of homopolymer tracts to RNA Martin et al. 1998 .

Labels on miRNA or miRNA probes may be colorimetric includes visible and UV spectrum including fluorescent luminescent enzymatic or positron emitting including radioactive . The label may be detected directly or indirectly. Radioactive labels include I P P and S. Examples of enzymatic labels include alkaline phosphatase luciferase horseradish peroxidase and galactosidase. Labels can also be proteins with luminescent properties e.g. green fluorescent protein and phicoerythrin.

The colorimetric and fluorescent labels contemplated for use as conjugates include but are not limited to Alexa Fluor dyes BODIPY dyes such as BODIPY FL Cascade Blue Cascade Yellow coumarin and its derivatives such as 7 amino 4 methylcoumarin aminocoumarin and hydroxycoumarin cyanine dyes such as Cy3 and Cy5 eosins and erythrosins fluorescein and its derivatives such as fluorescein isothiocyanate macrocyclic chelates of lanthanide ions such as Quantum Dye Marina Blue Oregon Green rhodamine dyes such as rhodamine red tetramethylrhodamine and rhodamine 6G Texas Red fluorescent energy transfer dyes such as thiazole orange ethidium heterodimer and TOTAB.

Specific examples of dyes include but are not limited to those identified above and the following Alexa Fluor 350 Alexa Fluor 405 Alexa Fluor 430 Alexa Fluor 488 Alexa Fluor 500. Alexa Fluor 514 Alexa Fluor 532 Alexa Fluor 546 Alexa Fluor 555 Alexa Fluor 568 Alexa Fluor 594 Alexa Fluor 610 Alexa Fluor 633 Alexa Fluor 647 Alexa Fluor 660 Alexa Fluor 680 Alexa Fluor 700 and Alexa Fluor 750 amine reactive BODIPY dyes such as BODIPY 493 503 BODIPY 530 550 BODIPY 558 568 BODIPY 564 570 BODIPY 576 589 BODIPY 581 591 BODIPY 630 650 BODIPY 650 655 BODIPY FL BODIPY R6G BODIPY TMR and BODIPY TR Cy3 Cy5 6 FAM Fluorescein Isothiocyanate HEX 6 JOE Oregon Green 488 Oregon Green 500 Oregon Green 514 Pacific Blue REG Rhodamine Green Rhodamine Red Renographin ROX SYPRO TAMRA 2 4 5 7 Tetrabromosulfonefluorescein and TET.

Probes and these include Alexa Fluor 488 5 UTP Fluorescein 12 UTP BODIPY FL 14 UTP BODIPY TMR 14 UTP Tetramethylrhodamine 6 UTP Alexa Fluor 546 14 UTP Texas Red 5 UTP and BODIPY TR 14 UTP. Other fluorescent ribonucleotides are available from Amersham Biosciences such as Cy3 UTP and Cy5 UTP.

Examples of fluorescently labeled deoxyribonucleotides include Dinitrophenyl DNP 11 dUTP Cascade Blue 7 dUTP Alexa Fluor 488 5 dUTP Fluorescein 12 dUTP Oregon Green 488 5 dUTP BODIPY FL 14 dUTP Rhodamine Green 5 dUTP Alexa Fluor 532 5 dUTP BODIPY TMR 14 dUTP Tetramethylrhodamine 6 dUTP Alexa Fluor 546 14 dUTP Alexa Fluor 568 5 dUTP Texas Red 12 dUTP Texas Red 5 dUTP BODIPY TR 14 dUTP Alexa Fluor 594 5 dUTP BODIPY 630 650 14 dUTP BODIPY 650 665 14 dUTP Alexa Fluor 488 7 OBEA dCTP Alexa Fluor 546 16 OBEA dCTP Alexa Fluor 594 7 OBEA dCTP Alexa Fluor 647 12 OBEA dCTP.

It is contemplated that nucleic acids may be labeled with two different labels. Furthermore fluorescence resonance energy transfer FRET may be employed in methods of the invention e.g. Klostermeier et al. 2002 Emptage 2001 Didenko 2001 each incorporated by reference .

Alternatively the label may not be detectable per se but indirectly detectable or allowing for the isolation or separation of the targeted nucleic acid. For example the label could be biotin digoxigenin polyvalent cations chelator groups and the other ligands include ligands for an antibody.

A number of techniques for visualizing or detecting labeled nucleic acids are readily available. The reference by Stanley T. Crooke 2000 has a discussion of such techniques Chapter 6 which is incorporated by reference. Such techniques include microscopy arrays Fluorometry Light cyclers or other real time PCR machines FACS analysis scintillation counters Phosphoimagers Geiger counters MRI CAT antibody based detection methods Westerns immunofluorescence immunohistochemistry histochemical techniques HPLC Griffey et al. 1997 spectroscopy capillary gel electrophoresis Cummins et al. 1996 spectroscopy mass spectroscopy radiological techniques and mass balance techniques.

When two or more differentially colored labels are employed fluorescent resonance energy transfer FRET techniques may be employed to characterize the dsRNA. Furthermore a person of ordinary skill in the art is well aware of ways of visualizing identifying and characterizing labeled nucleic acids and accordingly such protocols may be used as part of the invention. Examples of tools that may be used also include fluorescent microscopy a BioAnalyzer a plate reader Storm Molecular Dynamics Array Scanner FACS fluorescent activated cell sorter or any instrument that has the ability to excite and detect a fluorescent molecule.

The present invention concerns the preparation and use of miRNA arrays which are ordered macroarrays or microarrays of nucleic acid molecules probes that are fully or nearly complementary or identical to a plurality of miRNA molecules or precursor miRNA molecules and that are positioned on a support material in a spatially separated organization. Macroarrays are typically sheets of nitrocellulose or nylon upon which probes have been spotted. Microarrays position the nucleic acid probes more densely such that up to 10 000 nucleic acid molecules can be fit into a region typically 1 to 4 square centimeters. Microarrays can be fabricated by spotting nucleic acid molecules e.g. genes oligonucleotides etc. onto substrates or fabricating oligonucleotide sequences in situ on a substrate. Spotted or fabricated nucleic acid molecules can be applied in a high density matrix pattern of up to about 30 non identical nucleic acid molecules per square centimeter or higher e.g. up to about 100 or even 1000 per square centimeter. Microarrays typically use coated glass as the solid support in contrast to the nitrocellulose based material of filter arrays. By having an ordered array of miRNA complementing nucleic acid samples the position of each sample can be tracked and linked to the original sample. A variety of different array devices in which a plurality of distinct nucleic acid probes are stably associated with the surface of a solid support are known to those of skill in the art. Useful substrates for arrays include nylon glass and silicon Such arrays may vary in a number of different ways including average probe length sequence or types of probes nature of bond between the probe and the array surface e.g. covalent or non covalent and the like. The labeling and screening methods of the present invention and the arrays are not limited in its utility with respect to any parameter except that the probes detect miRNA consequently methods and compositions may be used with a variety of different types of miRNA arrays.

Representative methods and apparatus for preparing a microarray have been described for example in U.S. Pat. Nos. 5 143 854 5 202 231 5 242 974 5 288 644 5 324 633 5 384 261 5 405 783 5 412 087 5 424 186 5 429 807 5 432 049 5 436 327 5 445 934 5 468 613 5 470 710 5 472 672 5 492 806 5 525 464 5 503 980 5 510 270 5 525 464 5 527 681 5 529 756 5 532 128 5 545 531 5 547 839 5 554 501 5 556 752 5 561 071 5 571 639 5 580 726 5 580 732 5 593 839 5 599 695 5 599 672 5 610 287 5 624 711 5 631 134 5 639 603 5 654 413 5 658 734 5 661 028 5 665 547 5 667 972 5 695 940 5 700 637 5 744 305 5 800 992 5 807 522 5 830 645 5 837 196 5 871 928 5 847 219 5 876 932 5 919 626 6 004 755 6 087 102 6 368 799 6 383 749 6 617 112 6 638 717 6 720 138 as well as WO 93 17126 WO 95 11995 WO 95 21265 WO 95 21944 WO 95 35505 WO 96 31622 WO 97 10365 WO 97 27317 WO 99 35505 WO 09923256 WO 09936760 WO0138580 WO 0168255 WO 03020898 WO 03040410 WO 03053586 WO 03087297 WO 03091426 WO03100012 WO 04020085 WO 04027093 EP 373 203 EP 785 280 EP 799 897 and UK 8 803 000 the disclosures of which are all herein incorporated by reference.

It is contemplated that the arrays can be high density arrays such that they contain 100 or more different probes. It is contemplated that they may contain 1000 16 000 65 000 250 000 or 1 000 000 or more different probes. The probes can be directed to targets in one or more different organisms. The oligonucleotide probes range from 5 to 50 5 to 45 10 to 40 or 15 to 40 nucleotides in length in some embodiments. In certain embodiments the oligonucleotide probes are 20 to 25 nucleotides in length.

The location and sequence of each different probe sequence in the array are generally known. Moreover the large number of different probes can occupy a relatively small area providing a high density array having a probe density of generally greater than about 60 100 600 1000 5 000 10 000 40 000 100 000 or 400 000 different oligonucleotide probes per cm. The surface area of the array can be about or less than about 1 1.6 2 3 4 5 6 7 8 9 or 10 cm.

Moreover a person of ordinary skill in the art could readily analyze data generated using an array. Such protocols are disclosed above and include information found in WO 9743450 WO 03023058 WO 03022421 WO 03029485 WO 03067217 WO 03066906 WO 03076928 WO 03093810 WO 03100448A1 all of which are specifically incorporated by reference.

It is contemplated that the miRNA of a wide variety of samples can be analyzed using the array technology of the invention. While endogenous miRNA is contemplated for use with compositions and methods of the invention recombinant miRNA including nucleic acids that are complementary or identical to endogenous miRNA or precursor miRNA can also be handled and analyzed as described herein. Samples may be biological samples in which case they can be from blood tissue organs semen saliva tears other bodily fluid hair follicles skin or any sample containing or constituting biological cells. Alternatively the sample may not be a biological sample but be a chemical mixture such as a cell free reaction mixture which may contain one or more biological enzymes .

After the array is prepared and the miRNA in the sample is labeled the population of target nucleic acids is contacted with the array under hybridization conditions where such conditions can be adjusted as desired to provide for an optimum level of specificity in view of the particular assay being performed. Suitable hybridization conditions are well known to those of skill in the art and reviewed in Sambrook et al. 1989 and WO 95 21944. Of particular interest in many embodiments is the use of stringent conditions during hybridization. Stringent conditions are known to those of skill in the art.

It is specifically contemplated that a single array may be contacted with multiple samples. The samples may be labeled with different labels to distinguish the samples. For example a single array can be contacted with a tumor tissue sample labeled with Cy3 and normal tissue sample labeled with Cy5. Differences between the samples for particular miRNAs corresponding to probes on the array can be readily ascertained and quantified.

The small surface area of the array permits uniform hybridization conditions such as temperature regulation and salt content. Moreover because of the small area occupied by the high density arrays hybridization may be carried out in extremely small fluid volumes e.g. about 250 l or less including volumes of about or less than about 5 10 25 50 60 70 80 90 100 l or any range derivable therein . In small volumes hybridization may proceed very rapidly.

Arrays of the invention can be used to detect differences between two samples. Specifically contemplated applications include identifying and or quantifying differences between miRNA from a sample that is normal and from a sample that is not normal or between two differently treated samples. Also miRNA may be compared between a sample believed to be susceptible to a particular disease or condition and one believed to be not susceptible or resistant to that disease or condition. A sample that is not normal is one exhibiting phenotypic trait s of a disease or condition or one believed to be not normal with respect to that disease or condition. It may be compared to a cell that is normal with respect to that disease or condition. Phenotypic traits include symptoms of or susceptibility to a disease or condition of which a component is or may or may not be genetic.

Particularly arrays can be used to evaluate samples with respect to diseases or conditions that include but are not limited to the following AIDS autoimmune diseases rheumatoid arthritis multiple sclerosis diabetes insulin dependent and non independent systemic lupus erythematosus and Graves disease cancer e.g. malignant benign metastatic precancer cardiovascular diseases heart disease or coronary artery disease stroke ischemic and hemorrhagic and rheumatic heart disease diseases of the nervous system and infection by pathogenic microorganisms Athlete s Foot Chickenpox Common cold Diarrheal diseases Flu Genital herpes Malaria Meningitis Pneumonia Sinusitis Skin diseases Strep throat Tuberculosis Urinary tract infections Vaginal infections Viral hepatitis inflammation allergy asthma prion diseases e.g. CJD kuru GSS FFI .

Moreover miRNA can be evaluated with respect to the following diseases conditions and disorders Abdominal Adhesions Anal Abscess Brain Abscess Peritonsillar Abscess Absence Seizures Achalasia Acne Acoustic Neuroma Acquired Immunodeficiency Syndrome AIDS Acrochordon Actinic Keratosis Adenocarcinoma of the Lung ADHD Adult Onset Diabetes Aero Otitis Age Spots Age Related Hearing Loss Age Related Macular Degeneration Age Related Vision Change Presbyopia Agoraphobia Alcohol Withdrawal Alcoholism Allergen Immunotherapy Allergic Rhinitis Allergies Alopecia Areata Hereditary Patterned and Traumatic Altitude Sickness Alzheimer s Disease Amaurotic Familial Infantile Idiocy Amblyopia Amenorrhea Amyloidosis Amyotrophic Lateral Sclerosis ALS Anaphylaxis Androgenetic Alopecia Anemia Aplastic Hemolytic Pernicious and Sickle Cell Angina Angiomas Spider Angioplasty Ankylosing Spondylitis Anorexia Nervosa Anovulatory Bleeding Antibiotic Associated Diarrhea Antiphospholipid Antibody Syndrome Antisocial Personality Disorder Anus Fissure Fistula Hemorrhoids Anus Itch Stricture Anxiety Disorders Generalized Obsessive Compulsive Disorder Panic Disorder Phobia and Post Traumatic Stress Disorder Aortic Aneurysm Aortic Arch Syndrome Appendicitis Arrhythmias Cardiac Arteritis Takayasu s Arthritic Diseases Ankylosing Spondylitis Gout Infectious Juvenile Osteoarthritis Pseudogout Psoriatic Arthritis and Rheumatoid Asbestosis Ascending Cholangitis Asteatotic Eczema Asthma Astigmatism Asymptomatic Bacteriuria Ataxia Friedreich s Atherosclerosis Athlete s Foot Atopic Dermatitis Atrial Fibrillation Atrophic Vaginitis Attention Deficit Hyperactivity Disorder Autism Autoimmune Diseases Celiac Disease Crohn s Disease Diabetes Mellitus Type 1 Insulin Dependent Juvenile Onset Diabetes Mellitus Type 2 Non Insulin Dependent Adult Onset Graves Disease Hyperthyroidism Immune Thrombocytopenic Purpura Lupus Myasthenia Gravis Polyarteritis Nodosa Rheumatoid Arthritis Scleroderma Takayasu s Arteritis and Ulcerative Colitis B12 Deficiency Bacillary Dysentery Bacterial Gastroenteritis Bacterial Vaginosis Balanitis Baldness Hereditary Patterned Barber s Itch Barotitis Barotrauma Bartholin s Gland Cyst Basal Cell Carcinoma Bed Wetting Bedsores Behcet s Syndrome Bell s Palsy Bends Benign Prostatic Hyperplasia Bile Duct Diseases Biliary Colic Biopsy Bipolar Disorder Bladder conditions Infection Interstitial Cystitis Prolapse Urethritis Urinary Incontinence Urinary Tract Infection Blepharitis Blepharoptosis Blighted Ovum Friction Blisters Blood Pressure High Boils Bone diseases and conditions Osteoporosis Paget s Disease Bone Yaws Borderline Personality Disorder Bornholm Disease Botulism Bowel Obstruction Bradycardia Bronchitis Bulimia Nervosa Bunion Bursitis C. Difficile Colitis Calcaneal Apophysitis Calcium Pyrophosphate Deposition Disease Campylobacteriosis Cancer Candidiasis Carbon Monoxide Poisoning Carbuncles Cardiac Arrhythmias Atrial Fibrillation Bradycardia Cardiomyopathy Carpal Tunnel Syndrome Cataracts Cellulitis Central Serous Retinopathy Cerebral Palsy Cerebromacular Degeneration Cerumen Impaction Cervicitis Nabothian Cysts Cervical Polyps Cervical Warts Chalazion Chickenpox Chlamydia Chloasma Cholangitis Cholecystitis Cholesteatoma Chondromalacia Chorea Choroidal Melanoma Chronic Bronchitis Chronic Fatigue Syndrome Chronic Hepatitis Chronic Leukemia Chronic Obstructive Pulmonary Disease Chronic Otitis Media Cirrhosis Cluster Headache Cogan s Syndrome Cold Common Colic Biliary Pseudomembranous Colitis Ulcerative Colitis Collapsed Lung Collarbone Fracture Coma Complex Regional Pain Syndrome Congestive Heart Failure Conjunctivitis Constipation Contact Dermatitis Conversion Disorder COPD Cornea Abrasion Cornea Keratitis Corns Coronary Artery Disease Creutzfeldt Jakob Disease Crossed Eyes Croup Cryptorchidism Cystic Fibrosis Interstitial Cystitis Cystocele Cysts Cytomegalovirus infection Dacryocystitis Dandruff Decompression Sickness Decubitus Ulcers Delirium Tremens Delusional Disorder Dementia Depressive Disorders Bipolar Disorder Dysthymia Major Depression Manic Depression Postpartum Depression Dermatitis Dermatofibroma Dermatomyositis Detached Retina Developmental Dysplasia of the Hip Deviated Septum Devil s Grip Diabetes Gestational Diabetes Type 1 Diabetes Insulin Dependent Juvenile Type 2 Diabetes Non Insulin Dependent Adult Onset Hypoglycemia Ketoacidosis Nephropathy Neuropathies Retinopathy Antibiotic Associated Diarrhea Diplopia Herniated Disk Dislocated Lens Hip Dislocation Developmental Diverticulitis Diverticulosis Dizziness Doerderland s Vaginitis Double Vision Down Syndrome Drooping Eyelid Dry Skin Sun Damaged Skin Dry Eye Syndrome Duck Foot Dysautonomia Familial Dysfunctional Uterine Bleeding Dyslexia Dyspareunia Dysthymia Dysuria Eating Disorders Anorexia Nervosa Bulimia Nervosa Eclampsia Eczema Edema Emphysema Encephalitis Encopresis End Stage Renal Disease Endocarditis Endometriosis Endophthalmitis Endoscopy Enlarged Prostate Enuresis Epidemic Benign Dry Pleurisy Epididymitis Epiglottitis Epilepsy Epistaxis Erectile Dysfunction Erythema Infectiosum Esophagitis Esophagus Achalasia Esophagitis Essential Hypertension Essential Tremor Ewing s Sarcoma Familial Dysautonomia Farsightedness Febrile Seizures Fecal Incontinence Fever Fever Induced Seizures Fibroids Fibromyalgia Fifth Disease Filiform Warts Flat Warts Flatulence Flu Focal Seizures Food Allergy Food Poisoning Forefoot Neuroma Fragile X Syndrome Friction Blisters Friedreich s Ataxia Frostbite Fungal Infections Athlete s Foot Brain Abscess Infectious Arthritis Jock Itch Onychomycosis Ringworm Swimmer s Ear Tinea Cruris Tinea Unguium Tinea Versicolor Furuncle Gallstones Gardnerella Vaginitis Gastritis Gastrocnemius Strain Gastroenteritis Gastroesophageal Reflux Disease Gastrointestinal Amebiasis Generalized Anxiety Disorder Generalized Barotrauma Genital Herpes Genital Warts GERD Germ Cell Tumors Extragonadal Giant Cell Arteritis Giardiasis Glaucoma Glomerulonephritis Gluten Sensitive Enteropathy GM2 Gangliosidosis Gonorrhea Gout Grand Mal Seizures Graves Disease Graves Ophthalmopathy Guillain Barr Syndrome Hammertoe Hay Fever Headache Hearing Loss Heart Attack Heat Stroke Heel Spur Heloma Spider Hemangiomas Hematoma Hematuria Hemochromatosis Hemolytic Anemia Hemophilia Hemorrhagic Stroke Subarachnoid Hemorrhagic Stroke Hemorrhoids Hepatitis A Hepatitis B Hepatitis C Hereditary Patterned Baldness Hernia Herniated Disk High Blood Pressure High Cholesterol Hirsutism Histiocytosis X HIV AIDS Hordeolum Human Papilloma Virus HPV Huntington s Disease Hydatidiform Mole Hydrocephalus Hyperactivity Hypercholesterolemia Hyperkeratosis Hyperopia Hypertension Ocular Hypertension Secondary Hypertension Hypertensive Retinopathy Hyperthermia Hyperthyroidism Hypochondriasis Hypoglycemia Hypoparathyroidism Hypothyroidism IBS ICD Ichthyosis Immune Thrombocytopenic Purpura Impetigo Impotence Incontinence Infantile Ganglioside Lipidosis Infectious Arthritis Infectious Mononucleosis Infertility Inflammatory Bowel Disease Inguinal Hernia Insomnia Intercerebral Hemorrhage Interdigital Neuroma Intermetatarsal Neuroma Intermittent Claudication Interstitial Cystitis Intestinal Obstruction Iron Deficiency Irritable Bowel Syndrome Juvenile Arthritis Kaposi s Sarcoma Kawasaki Syndrome Keloids Keratitis Actinic Keratosis Labyrinthitis Lactose Intolerance Lacunar Stroke Langerhans Cell Histiocytosis Laryngitis Laryngotracheitis Lateral Epicondylitis Latex Allergy Lazy Eye Lead Poisoning Intermittent Claudication Restless Legs Syndrome Shin Splints Leg Strain Cataract Dislocated Lens Leukemia Lice Lichen Simplex Chronicus Cirrhosis Hepatitis Liver Spots Lockjaw Lou Gehrig s Disease Lupus Erythematosus Systemic Lyme Disease Lymphedema Lymphoma Macular Degeneration Malabsorption Syndromes Malaria Male Pattern Baldness Malignant Hyperthermia Manic Depression Marfan s Syndrome Mastoiditis Measles Meckel s Diverticulum Melasma Meniere s Disease Meningitis Menopause Mental Retardation Phenylketonuria Migraine Miscarriage Mitral Valve Prolapse Mittelschmerz Molar Pregnancy Molluscum Contagiosum Mononucleosis Morton s Neuroma Mosaic Warts Motor Tics Mucocutaneous Lymph Node Syndrome Multiple Sclerosis Mumps Muscular Dystrophy Musculoskeletal Disorders Fibromyalgia Giant Cell Arteritis Gout Infectious Arthritis Muscular Dystrophy Myositis Osteoarthritis Osteoporosis Paget s Disease Of Bone Polymyalgia Rheumatica Pseudogout Reflex Sympathetic Dystrophy Rheumatoid Arthritis Scleroderma Systemic Lupus Erythematosus Tendonitis Myasthenia Gravis Myocardial Infarction Myocarditis Myopia Myositis Nail Felon Onycholysis Onychomycosis Paronychia Subungual Hematoma Narcolepsy Nasal Polyps Nausea Nearsightedness Needle Biopsy Nephrectomy Nephroblastoma Nephrolithiasis Nephropathy Diabetic Neuritis Retrobulbar Neuroblastoma Neuromuscular Disorders Neuropathies Guillain Barre Syndrome Retrobulbar Nevi Nevus Flammeus Nevus Simplex Nocturnal Enuresis Non Tropical Sprue Obesity Obsessive Compulsive Disorder Occupational Hearing Loss Ocular Hypertension Ocular Rosacea Onycholysis Onychomycosis Glaucoma Retrobulbar Neuritis Optic Nerve Swelling Orbit Fracture Orchitis Osgood Schlatter Disease Osteoarthritis Osteoporosis Osteosarcoma Otitis Externa Otitis Media Chronic Otitis Media Otosclerosis Ototoxicity Pelvic Inflammatory Disease Polycystic Ovary Syndrome Painful Bladder Syndrome Pancreatitis Panic Disorder Papilledema Paraphimosis Parkinson s Disease Paronychia Partial Seizures PCL Injuries Pedunculated Warts Pelvic Relaxation Paraphimosis Peyronie s Disease Peptic Ulcer Perforated Eardrum Pericarditis Perimenopause Peripheral Vascular Disease Peritonsillar Abscess Persistent Vegetative State Personality Disorders Petit Mal Seizures Peyronie s Disease Pharyngitis Pharynx Cancer Phenylketonuria Phimosis Phobia Photosensitivity Pigmentation Disorders Chloasma Melasma Vitiligo Piles Pinkeye Pityriasis Rosea PKU Plague Plantar Fasciitis Plantar Warts Plantaris Strain Pleurisy Pleurodynia PMS Pneumoconiosis Pneumonectomy Pneumonia Pneumothorax Lead Poisoning Polio Poliomyelitis Polyarteritis Nodosa Polychondritis Polymyalgia Rheumatica Polymyositis Colonic Polyps Nasal Polyps Vocal Cord Polyps Port Wine Stain Post Polio Syndrome Postinfectious Thrombocytopenia Postpartum Depression Preeclampsia Pregnancy Induced Hypertension Premenstrual Syndrome Pressure Sores Primary Sclerosing Cholangitis Prolapse Enlarged Prostate Acute Prostatitis Chronic Prostatitis Pruritus Ani Pseudogout Psoriasis Psoriatic Arthritis Ptosis Pulseless Disease Pyelonephritis Quadriceps Strain Quinsy Rash Raynaud s Phenomenon Rectal Itch Rectocele Reflex Sympathetic Dystrophy Renal Failure Respiratory Disorders Respiratory Syncytial Virus Retina Detachment Retinitis Pigmentosa Retinopathy Retrobulbar Neuritis Reye s Syndrome Rhabdomyosarcoma Rheumatoid Arthritis Allergic Rhinitis Viral Rhinitis Common Cold Riley Day Syndrome Ringworm Rocky Mountain Spotted Fever Rosacea Rubeola Mumps Salivary Gland Disorders Salmon Patch Sarcoidosis Scabies Scarlet Fever Scars Schizophrenia Schizotypal Personality Disorder Sciatica Scleritis Scleroderma Scoliosis Sebaceous Cysts Seborrhea Seborrheic Keratoses Secondary Hypertension Seizures Sexual Dysfunction Sexually Transmitted Diseases Shigellosis Shingles Sialadenitis Sialadenosis Sialolithiasis Sickle Cell Anemia Siderosis Silicosis Sinus Cancer Sj gren s Syndrome Sleep Disorders Smallpox Social Anxiety Disorder Solar Lentigo Somatoform Disorders Hypochondriasis Somatization Disorder Somnambulism Spastic Colon Spider Veins Spina Bifida Spinal Cord Trauma Spontaneous Abortion Stasis Dermatitis Strabismus Strep Throat Streptococcal Toxic Shock Syndrome Stroke Subarachnoid Hemorrhage Transient Ischemic Attack Stuttering Subungual Hematoma Sun Allergy Sun Damaged Skin Sylvest s Disease Systemic Lupus Erythematosus Systemic Sclerosis Tachycardia Takayasu s Arteritis Tay Sachs Disease Tear Duct Infection Telogen Effluvium Temporal Arteritis Tendonitis Tennis Elbow Tension Headache Testicular Torsion Undescended Testicles Tetanus Thrombocytopenia Thrombophlebitis Thrombotic Stroke Tinea Tinnitus Tonsillitis Torsional Deformities Toxemia Of Pregnancy Toxic Shock Syndrome Streptococcal Toxoplasmosis Trichomoniasis Trigeminal Neuralgia Tic Douloureux Tuberculosis Tylosis Ulcer Urethritis Urinary Tract disorders and conditions Uroliniasis Urticaria Uterine disorders Uterine Prolapse Uveitis Vaginitis Bacterial Gardnerella Vaginosis Varicella Varices Esophageal Varicose Veins Vascular Disorders Hypertension Intermittent Claudication Peripheral Vascular Disease Polyarteritis Nodosa Raynaud s Phenomenon Takayasu s Arteritis Thrombophlebitis Vasculitis Wegener s Granulomatosis Vein Inflammation Varicose Veins Vertigo Vestibular Schwannoma Viral Rhinitis Vitamin B12 Deficiency Vitiligo Vocal Tics Vocal Cord Disorders Common Warts Genital Warts Plantar Warts Water On The Brain Wax Blockage Of Ear Canal Esophageal Webs Werlhofs Disease Wrinkles Yersinia Pestis Infection.

Cancers that may be evaluated by methods and compositions of the invention include cancer cells from the bladder blood bone bone marrow brain breast colon esophagus gastrointestine gum head kidney liver lung nasopharynx neck ovary prostate skin stomach testis tongue or uterus. In addition the cancer may specifically be of the following histological type though it is not limited to these neoplasm malignant carcinoma carcinoma undifferentiated giant and spindle cell carcinoma small cell carcinoma papillary carcinoma squamous cell carcinoma lymphoepithelial carcinoma basal cell carcinoma pilomatrix carcinoma transitional cell carcinoma papillary transitional cell carcinoma adenocarcinoma gastrinoma malignant cholangiocarcinoma hepatocellular carcinoma combined hepatocellular carcinoma and cholangiocarcinoma trabecular adenocarcinoma adenoid cystic carcinoma adenocarcinoma in adenomatous polyp adenocarcinoma familial polyposis coli solid carcinoma carcinoid tumor malignant branchiolo alveolar adenocarcinoma papillary adenocarcinoma chromophobe carcinoma acidophil carcinoma oxyphilic adenocarcinoma basophil carcinoma clear cell adenocarcinoma granular cell carcinoma follicular adenocarcinoma papillary and follicular adenocarcinoma nonencapsulating sclerosing carcinoma adrenal cortical carcinoma endometroid carcinoma skin appendage carcinoma apocrine adenocarcinoma sebaceous adenocarcinoma ceruminous adenocarcinoma mucoepidermoid carcinoma cystadenocarcinoma papillary cystadenocarcinoma papillary serous cystadenocarcinoma mucinous cystadenocarcinoma mucinous adenocarcinoma signet ring cell carcinoma infiltrating duct carcinoma medullary carcinoma lobular carcinoma inflammatory carcinoma paget s disease mammary acinar cell carcinoma adenosquamous carcinoma adenocarcinoma w squamous metaplasia thymoma malignant ovarian stromal tumor malignant thecoma malignant granulosa cell tumor malignant androblastoma malignant sertoli cell carcinoma leydig cell tumor malignant lipid cell tumor malignant paraganglioma malignant extra mammary paraganglioma malignant pheochromocytoma glomangiosarcoma malignant melanoma amelanotic melanoma superficial spreading melanoma malig melanoma in giant pigmented nevus epithelioid cell melanoma blue nevus malignant sarcoma fibrosarcoma fibrous histiocytoma malignant myxosarcoma liposarcoma leiomyosarcoma rhabdomyosarcoma embryonal rhabdomyosarcoma alveolar rhabdomyosarcoma stromal sarcoma mixed tumor malignant mullerian mixed tumor nephroblastoma hepatoblastoma carcinosarcoma mesenchymoma malignant brenner tumor malignant phyllodes tumor malignant synovial sarcoma mesothelioma malignant dysgerminoma embryonal carcinoma teratoma malignant struma ovarii malignant choriocarcinoma mesonephroma malignant hemangiosarcoma hemangioendothelioma malignant kaposi s sarcoma hemangiopericytoma malignant lymphangiosarcoma osteosarcoma juxtacortical osteosarcoma chondrosarcoma chondroblastoma malignant mesenchymal chondrosarcoma giant cell tumor of bone ewing s sarcoma odontogenic tumor malignant ameloblastic odontosarcoma ameloblastoma malignant ameloblastic fibrosarcoma pinealoma malignant chordoma glioma malignant ependymoma astrocytoma protoplasmic astrocytoma fibrillary astrocytoma astroblastoma glioblastoma oligodendroglioma oligodendroblastoma primitive neuroectodermal cerebellar sarcoma ganglioneuroblastoma neuroblastoma retinoblastoma olfactory neurogenic tumor meningioma malignant neurofibrosarcoma neurilemmoma malignant granular cell tumor malignant malignant lymphoma Hodgkin s disease Hodgkin s lymphoma paragranuloma malignant lymphoma small lymphocytic malignant lymphoma large cell diffuse malignant lymphoma follicular mycosis fungoides other specified non Hodgkin s lymphomas malignant histiocytosis multiple myeloma mast cell sarcoma immunoproliferative small intestinal disease leukemia lymphoid leukemia plasma cell leukemia erythroleukemia lymphosarcoma cell leukemia myeloid leukemia basophilic leukemia eosinophilic leukemia monocytic leukemia mast cell leukemia megakaryoblastick leukemia myeloid sarcoma and hairy cell leukemia. Moreover miRNA can be evaluated in precancers such as metaplasia dysplasia and hyperplasia.

It is specifically contemplated that the invention can be used to evaluate differences between stages of disease such as between pre cancer and cancer or between a primary tumor and a metastasized tumor.

Moreover it is contemplated that samples that have differences in the activity of certain pathways may also be compared. These pathways include the following and those involving the following factors antibody response apoptosis calcium NFAT signaling cell cycle cell migration cell adhesion cell division cytokines and cytokine receptors drug metabolism growth factors and growth factor receptors inflammatory response insulin signaling NFx B signaling angiogenesis adipogenesis cell adhesion viral infecton bacterial infection senescence motility glucose transport stress response oxidation aging telomere extension telomere shortening neural transmission blood clotting stem cell differentiation G Protein Coupled Receptor GPCR signaling and p53 activation.

Cellular pathways that may be profiled also include but are not limited to the following any adhesion or motility pathway including but not limited to those involving cyclic AMP protein kinase A G protein couple receptors adenylyl cyclase L selectin E selectin PECAM VCAM 1 actinin paxillin cadherins AKT integrin integrin RAF 1 ERK PI 3 kinase vinculin matrix metalloproteinases Rho GTPases p85 trefoil factors profilin FAK MAP kinase Ras caveolin calpain 1 calpain 2 epidermal growth factor receptor ICAM 1 ICAM 2 cofilin actin gelsolin RhoA RAC1 myosin light chain kinase platelet derived growth factor receptor or ezrin any apoptosis pathway including but not limited to those involving AKT Fas ligand NFB caspase 9 PI3 kinase caspase 3 caspase 7 ICAD CAD EndoG Granzyme B Bad Bax Bid Bak APAF 1 cytochrome C p53 ATM Bc1 2 PARP Chk1 Chk2 p21 c Jun p73 Rad51 Mdm2 Rad50 c Abl BRCA 1 perforin caspase 4 caspase 8 caspase 6 caspase 1 caspase 2 caspase 10 Rho Jun kinase Jun kinase kinase Rip2 lamin A lamin B1 lamin B2 Fas receptor HO Granzyme A NADPH oxidase HMG2 CD4 CD28 CD3 TRADD IKK FADD GADD45 DR3 death receptor DR4 5 death receptor FLIPs APO 3 GRB2 SHC ERK MEK RAF 1 cyclic AMP protein kinase A E2F retinoblastoma protein Smac Diablo ACH receptor 14 3 3 FAK SODD TNF receptor RIP cyclin D1 PCNA Bcl XL PIP2 PIP3 PTEN ATM Cdc2 protein kinase C calcineurin 1KK IKK IKK SOS 1 c FOS Traf 1 Traf 2 IB or the proteasome any cell activation pathway including but not limited to those involving protein kinase A nitric oxide caveolin 1 actin calcium protein kinase C Cdc2 cyclin B Cdc25 GRB2 SRC protein kinase ADP ribosylation factors ARFs phospholipase D AKAP95 p68 Aurora B CDK1 Eg7 histone H3 PKAc CD80 PI3 kinase WASP Arp2 Arp3 p16 p34 p20 PP2A angiotensin angiotensin converting enzyme protease activated receptor 1 protease activated receptor 4 Ras RAF 1 PLC PLC COX 1 G protein coupled receptors phospholipase A2 IP3 SUMO1 SUMO 2 3 ubiquitin Ran Ran GAP Ran GEF p53 glucocorticoids glucocorticoid receptor components of the SWI SNF complex RanBP1 RanBP2 importins exportins RCC1 CD40 CD40 ligand p38 IKK IKK NFB TRAF2 TRAF3 TRAF5 TRAF6 IL 4 IL 4 receptor CDK5 AP 1 transcription factor CD45 CD4 T cell receptors MAP kinase nerve growth factor nerve growth factor receptor c Jun c Fos Jun kinase GRB2 SOS 1 ERK 1 ERK JAK2 STAT4 IL 12 IL 12 receptor nitric oxide synthase TYK2 IFN elastase IL 8 epithelins IL 2 IL 2 receptor CD28 SMAD3 SMAD4 TGF or TGF receptor any cell cycle regulation signaling or differentiation pathway including but not limited to those involving TNFs SRC protein kinase Cdc2 cyclin B Grb2 Sos 1 SHC p68 Aurora kinases protein kinase A protein kinase C Eg7 p53 cyclins cyclin dependent kinases neural growth factor epidermal growth factor retinoblastoma protein ATF 2 ATM ATR AKT CHK1 CHK2 14 3 3 WEE1 CDC25 CDC6 Origin Recognition Complex proteins p15 p16 p27 p21 ABL c ABL SMADs ubiquitin SUMO heat shock proteins Wnt GSK 3 angiotensin p73 any PPAR TGF TGF p300 MDM2 GADD45 Notch cdc34 BRCA 1 BRCA 2 SKP1 the proteasome CUL1 E2F p107 steroid hormones steroid hormone receptors IB IB Sin3A heat shock proteins Ras Rho ERKs IKKs PI3 kinase Bcl 2 Bax PCNA MAP kinases dynein RhoA PKAc cyclin AMP FAK PIP2 PIP3 integrins thrombopoietin Fas Fas ligand PLK3 MEKs JAKs STATs acetylcholine paxillin calcineurin p38 importins exportins Ran Rad50 Rad51 DNA polymerase RNA polymerase Ran GAP Ran GEF NuMA Tpx2 RCC1 Sonic Hedgehog Crm1 Patched Ptc 1 MPF CaM kinases tubulin actin kinetochore associated proteins centromere binding proteins telomerase TERT PP2A c MYC insulin T cell receptors B cell receptors CBP IKB NFB RAC1 RAF1 EPO diacylglycerol c Jun c Fos Jun kinase hypoxia inducible factors GATA4 catenin catenin calcium arrestin survivin caspases procaspases CREB CREM cadherins PECAMs corticosteroids colony stimulating factors calpains adenylyl cyclase growth factors nitric oxide transmembrane receptors retinoids G proteins ion channels transcriptional activators transcriptional coactivators transcriptional repressors interleukins vitamins interferons transcriptional corepressors the nuclear pore nitrogen toxins proteolysis or phosphorylation or any metabolic pathway including but not limited to those involving the biosynthesis of amino acids oxidation of fatty acids biosynthesis of neurotransmitters and other cell signaling molecules biosynthesis of polyamines biosynthesis of lipids and sphingolipids catabolism of amino acids and nutrients nucleotide synthesis eicosanoids electron transport reactions ER associated degradation glycolysis fibrinolysis formation of ketone bodies formation of phagosomes cholesterol metabolism regulation of food intake energy homeostasis prothrombin activation synthesis of lactose and other sugars multi drug resistance biosynthesis of phosphatidylcholine the proteasome amyloid precursor protein Rab GTPases starch synthesis glycosylation synthesis of phoshoglycerides vitamins the citric acid cycle IGF 1 receptor the urea cycle vesicular transport or salvage pathways. It is further contemplated that nucleic acids molecules of the invention can be employed in diagnostic and therapeutic methods with respect to any of the above pathways or factors. Thus in some embodiments of the invention an miRNA may be differentially expressed with respect to one or more of the above pathways or factors.

Phenotypic traits also include characteristics such as longevity morbidity appearance e.g. baldness obesity strength speed endurance fertility susceptibility or receptivity to particular drugs or therapeutic treatments drug efficacy and risk of drug toxicity. Samples that differ in these phenotypic traits may also be evaluated using the arrays and methods described.

In certain embodiments miRNA profiles may be generated to evaluate and correlate those profiles with pharmacokinetics. For example miRNA profiles may be created and evaluated for patient tumor and blood samples prior to the patient s being treated or during treatment to determine if there are miRNAs whose expression correlates with the outcome of the patient. Identification of differential miRNAs can lead to a diagnostic assay involving them that can be used to evaluate tumor and or blood samples to determine what drug regimen the patient should be provided. In addition it can be used to identify or select patients suitable for a particular clinical trial. If an miRNA profile is determined to be correlated with drug efficacy or drug toxicity that may be relevant to whether that patient is an appropriate patient for receiving the drug or for a particular dosage of the drug.

In addition to the above prognostic assay blood samples from patients with a variety of diseases can be evaluated to determine if different diseases can be identified based on blood miRNA levels. A diagnostic assay can be created based on the profiles that doctors can use to identify individuals with a disease or who are at risk to develop a disease. Alternatively treatments can be designed based on miRNA profiling. Examples of such methods and compositions are described in the U.S. Provisional Patent Application entitled Methods and Compositions Involving miRNA and miRNA Inhibitor Molecules filed on May 23 2005 in the names of David brown Lance Ford Angie Cheng and Rich Jarvis which is hereby incorporated by reference in its entirety.

In addition to the use of arrays and microarrays it is contemplated that a number of difference assays could be employed to analyze miRNAs their activities and their effects. Such assays include but are not limited to RT PCR in situ hybridization hybridization protection assay HPA GenProbe branched DNA bDNA assay Chiron rolling circle amplification RCA single molecule hybridization detection US Genomics Invader assay ThirdWave Technologies Bridge Litigation Assay Genaco .

Any of the compositions described herein may be comprised in a kit. In a non limiting example reagents for isolating miRNA labeling miRNA and or evaluating an miRNA population using an array are included in a kit. The kit may further include reagents for creating or synthesizing miRNA probes. The kits will thus comprise in suitable container means an enzyme for labeling the miRNA by incorporating labeled nucleotide or unlabeled nucleotides that are subsequently labeled. It may also include one or more buffers such as reaction buffer labeling buffer washing buffer or a hybridization buffer compounds for preparing the miRNA probes and components for isolating miRNA. Other kits of the invention may include components for making a nucleic acid array comprising miRNA and thus may include for example a solid support.

Kits are also included as part of the invention. Kits for implementing methods of the invention described herein are specifically contemplated. In some embodiments there are kits for preparing miRNA for multi labeling and kits for preparing miRNA probes and or miRNA arrays. In these embodiments kit comprise in suitable container means 1 2 3 4 5 6 7 8 9 10 11 12 or more of the following 1 poly A polymerase 2 unmodified nucleotides G A T C and or U 3 a modified nucleotide labeled or unlabeled 4 poly A polymerase buffer and 5 at least one microfilter 6 label that can be attached to a nucleotide 7 at least one miRNA probe 8 reaction buffer 9 an miRNA array or components for making such an array 10 acetic acid 11 alcohol 12 solutions for preparing isolating enriching and purifying miRNAs or miRNA probes or arrays. Other reagents include those generally used for manipulating RNA such as formamide loading dye ribonuclease inhibitors and DNase.

In specific embodiments kits of the invention include an array containing miRNA probes as described in the application. An array may have probes corresponding to all known miRNAs of an organism or to a subset of such probes. The subset of probes on arrays of the invention may be or include those identified as relevant to a particular diagnostic therapeutic or prognostic application. For example the array may contain one or more probes that is indicative or suggestive of 1 a disease or condition 2 susceptibility or resistance to a particular drug or treatment 3 susceptibility to toxicity from a drug or substance 4 the stage of development or severity of a disease or condition prognosis and 5 genetic predisposition to a disease or condition.

For any kit embodiment including an array there can be nucleic acid molecules that contain a sequence that is identical or complementary to all or part of any of SEQ ID NOS 1 899. In certain embodiments a kit or array of the invention can contain one or more probes for SEQ ID NOS 1 703 and SEQ ID NO 899 and or include one or more probes having all or part of the sequence in any of SEQ ID NOs 704 898. Any nucleic acid discussed above may be implemented as part of a kit.

The components of the kits may be packaged either in aqueous media or in lyophilized form. The container means of the kits will generally include at least one vial test tube flask bottle syringe or other container means into which a component may be placed and preferably suitably aliquoted. Where there is more than one component in the kit labeling reagent and label may be packaged together the kit also will generally contain a second third or other additional container into which the additional components may be separately placed. However various combinations of components may be comprised in a vial. The kits of the present invention also will typically include a means for containing the nucleic acids and any other reagent containers in close confinement for commercial sale. Such containers may include injection or blow molded plastic containers into which the desired vials are retained.

When the components of the kit are provided in one and or more liquid solutions the liquid solution is an aqueous solution with a sterile aqueous solution being particularly preferred.

However the components of the kit may be provided as dried powder s . When reagents and or components are provided as a dry powder the powder can be reconstituted by the addition of a suitable solvent. It is envisioned that the solvent may also be provided in another container means. In some embodiments labeling dyes are provided as a dried power. It is contemplated that 10 20 30 40 50 60 70 80 90 100 120 120 130 140 150 160 170 180 190 200 300 400 500 600 700 800 900 1000 g or at least or at most those amounts of dried dye are provided in kits of the invention. The dye may then be resuspended in any suitable solvent such as DMSO.

The container means will generally include at least one vial test tube flask bottle syringe and or other container means into which the nucleic acid formulations are placed preferably suitably allocated. The kits may also comprise a second container means for containing a sterile pharmaceutically acceptable buffer and or other diluent.

The kits of the present invention will also typically include a means for containing the vials in close confinement for commercial sale such as e.g. injection and or blow molded plastic containers into which the desired vials are retained.

Such kits may also include components that facilitate isolation of the labeled miRNA. It may also include components that preserve or maintain the miRNA or that protect against its degradation. Such components may be RNAse free or protect against RNAses. Such kits generally will comprise in suitable means distinct containers for each individual reagent or solution.

A kit will also include instructions for employing the kit components as well the use of any other reagent not included in the kit. Instructions may include variations that can be implemented.

Kits of the invention may also include one or more of the following Control RNA nuclease free water RNase free containers such as 1.5 ml tubes RNase free elution tubes PEG or dextran ethanol acetic acid sodium acetate ammonium acetate guanidinium detergent nucleic acid size marker RNase free tube tips and RNase or DNase inhibitors.

It is contemplated that such reagents are embodiments of kits of the invention. Such kits however are not limited to the particular items identified above and may include any reagent used for the manipulation or characterization of miRNA.

The following examples are included to demonstrate preferred embodiments of the invention. It should be appreciated by those of skill in the art that the techniques disclosed in the examples which follow represent techniques discovered by the inventor to function well in the practice of the invention and thus can be considered to constitute preferred modes for its practice. However those of skill in the art should in light of the present disclosure appreciate that many changes can be made in the specific embodiments which are disclosed and still obtain a like or similar result without departing from the spirit and scope of the invention.

Unless otherwise designated catalog numbers refer to products available by that number from Ambion Inc. The RNA Company.

miRNA was obtained from cell or tissue samples using a two step process. In the first all of the RNA in the sample is isolated away from the other contents of the sample. In the second step the small RNAs that include miRNAs are preferentially extracted from the sample to create an RNA fraction that is significantly enriched for miRNAs.

Although there are a variety of methods that can be used to recover all of the RNA in a sample silica binding and RNA elution were used as described below 

For tissue samples 1 ml of Lysis Binding Solution 4 M Guanidinium Thiocyanate 0.5 N Lauryl Sarcosine 25 mM NaCitrate pH 7.2 0.1 M 2 Mercaptoethanol per 0.1 g of tissue was added. The samples were kept cold and the tissue was thoroughly disrupted in Lysis Binding Solution using a motorized rotor stator homogenizer. For cell samples 0.6 ml of Lysis Binding Solution was added per 10cells. The cells were lysed by vortexing for 30 60 seconds.

A 1 10 volume of glacial acetic acid was added to the homogenate. For example if the lysate volume was 300 l 30 l glacial acetic acid was added. The sample was then mixed well by vortexing or inverting the tube several times before leaving it on ice for 10 min.

A volume of Acid Phenol Chloroform equal to the sample lysate volume was added to the sample and then vortexed for 30 60 sec to mix it. The mixture was centrifuged for 5 min at maximum speed 10 000 g at room temp to separate the aqueous and organic phases. The aqueous upper phase was carefully removed without disturbing the lower phase and transferred to a fresh tube. 1.25 volumes of 100 ethanol was added to the aqueous phase e.g. if 300 l was recovered from the aqueous phase 375 l of ethanol was added . The lysate ethanol mixture from the previous step was pipetted onto a glass fiber filter column. The column was then centrifuged at RCF relative centrifugal force 10 000 g for approximately 15 sec to pass the mixture through the filter. After discarding the flow through the process was repeated until all of the lysate ethanol mixture passed through the filter.

700 l of miRNA Wash Solution 1 1.6 M Guanidinium Thiocyanate 30 EtOH was applied to the filter column which was subsequently centrifuged for 5 10 sec or placed under a vacuum to pull the solution through the filter. The flow through was discarded. 500 l of Wash Solution 2 3 100 mM NaCl 4.5 mM EDTA 10 mM Tris pH 7.5 was applied to the filter column and drawn through it as in the previous step. The wash was repeated with a second 500 l aliquot of Wash Solution 2 3. After discarding the flow through from the last wash the assembly was spun for 1 min to remove residual fluid from the filter. 100 l of room temp nuclease free water was added to the center of the filter and the cap was closed. It was spun for approximately 20 30 seconds at maximum speed to recover the RNA.

The miRNA labeling process is most effective with samples that lack most of the larger mRNAs and rRNAs that are in a total RNA sample. To remove these RNAs from the samples one of the following was employed an enrichment process to remove the vast majority of the RNA in a sample that is greater than 70 nucleotides or a gel purification process to obtain a population of RNAs that were 15 30 nucleotides in length.

To the isolated total RNA 300 l of Lysis Binding Solution was added. 100 l of 100 ethanol was then added. It was mixed thoroughly by vortexing or inverting the tube several times.

The lysate ethanol mixture was pipetted onto a glass fiber filter column and centrifuged for 15 sec at RCF 10 000 g typically 10 000 rpm . The filtrate was collected and A 2 3volume of 100 ethanol was added to the filtrate i.e. flow through and it was mixed thoroughly.

The filtrate ethanol mixture from the previous step was pipetted onto a new glass fiber filter column. The column was centrifuged at RCF relative centrifugal force 10 000 g for approximately 15 sec to pass the mixture through the filter. 700 l of miRNA Wash Solution 1 1.6 M Guanidinium Thiocyanate 30 EtOH was applied to the filter column which was subsequently centrifuged for 5 10 sec or placed under a vacuum to pull the solution through the filter. The flow through was discarded. 500 l of Wash Solution 2 3 100 mM NaCl 4.5 mM EDTA 10 mM Tris pH 7.5 was applied to the filter column and drawn through it as in the previous step. The wash was repeated with a second 500 l aliquot of Wash Solution 2 3. After discarding the flow through from the last wash the assembly was spun for 1 min to remove residual fluid from the filter. 100 l of room temp nuclease free water was added to the center of the filter and the cap was closed. It was spun for approximately 20 30 seconds at maximum speed to recover the miRNA. The miRNA was lyophilized and stored until labeled for analysis.

Isolating the miRNAs in a sample away from contaminating mRNAs tRNAs and rRNAs can be done using gel electrophoresis. Two methods for gel electrophoresis were used. The first features standard gel conditions and excision of the miRNA fraction of the gel following electrophoresis. The second features a method called tube electrophoresis wherein gel filled tubes are used to fractionate RNA samples and facilitate recovery of the miRNA fraction.

An equal volume of Gel Loading Buffer 95 formamide 18 mM EDTA 0.01 xylene cyanol and 0.01 bromophenol blue was added to as much as 150 g of total RNA. The mixture was then heated for 5 minutes at 95 C. in a water bath and applied to a 15 denaturing polyacrylamide gel to fractionate the RNA.

After the gel was run the gel slice corresponding to 19 30 nt from under xylene cyanol to between bromophenol blue and xylene cyanol was excised. The miRNA was extracted by incubating the gel slice in 10 volumes of Gel Elution Buffer 0.5 mM NaCl 10 mM EDTA at 4 C. overnight. This was then centrifuged at 2500 rpm 1000 G and the Gel Elution Buffer was removed and placed into a new tube. Two volumes of Gel Elution Buffer was added to the gel and incubated for 60 min at room temperature. Again this was centrifuged and the Gel Elution Buffer samples were pooled.

The recovered and pooled buffer samples were mixed with enough 100 ethanol to create a 60 final ethanol concentration. The sample ethanol was loaded onto glass fiber filter column and a vacuum was applied. Two volumes of miRNA Binding Washing Buffer 0.5 mM NaCl 55 ethanol was applied to the column. The column was then placed in a sample collection tube. The column was washed with 0.6 ml of miRNA Binding Washing Buffer. The column was then centrifuged at 10 000 G for at least 1 minute. The filtrate was removed and the column was centrifuged again at 10 000 G for 1 minute. 50 l of hot 95 C. nuclease free water was added to the column and centrifuged at 10 000 G for 30 seconds. The previous two steps were repeated with a second 50 l wash. The miRNA was lyophilized and stored until labeling.

Tube electrophoresis is a method whereby nylon tubing is filled with a polyacrylamide gel matrix and the gel tube is placed in a special electrophoresis device. The total RNA sample is added to the top of the tube and an electric current is applied to fractionate the RNA based on size. The miRNA fraction is captured when it flows from the end of the tube into the electrophoresis buffer. The miRNA is then recovered from the buffer. This process is described in greater detail below.

The gel cartridge was prepared by cutting nylon 6 tubing into lengths of approximately 10 15 inches. Gentle pressure was applied during the cutting to avoid crushing and or cracking it. The tubing was placed on a glass plate and heated in an oven for 6 12 minutes at 180 C. After the tubing was removed from the oven each end of the tubing was held to straighten the length until the plastic cooled.

A 16 Denaturing Acrylamide solution was prepared by mixing 4 mL 20 acrylamide 7M Urea 1 TBE with 1 mL 7M Urea 1 TBE. Added to this was 50 L of 10 APS 5 L TEMED. After mixing the solution the gel mix was drawn up into the tubing using a Drummond pipettor. The gel was allowed to polymerize for 30 minutes.

The tubing was then cut to fit in a 50 mL conical. The tubing was soaked overnight in 1 TBE solution 10 TBE Cat 9866 . Upon use 1.2 cm tube lengths were cut using gentle pressure. The cartridge was loaded into the tube electrophoresis apparatus using the lower buffer chamber platform. The gel cartridge was placed on its end and the upper plate cartridge receptacle hole was aligned. With firm and even pressure the tube cartridge was slid into place. 200 L 1 TBE was added to the bottom cartridge so that it surrounded the bottom of the gel cartridge and the top platform was put in place. 125 L 1 TBE was put into the upper buffer chamber which was checked for leaks before the sample was added.

The sample was prepared by adding an equal volume of deionized formamide Cat 9342 to total RNA total volume 50 L . 2 L of filtered serva blue R dye 5 mg mL was added to the sample which was then heated at 95 C. for 2 minutes. The sample was then applied to the upper buffer chamber just above the gel surface. The gel was run at 70V for 7 10 minutes. The miRNA fraction 40 bp was allowed to migrate off the gel into the lower buffer chamber. The gel was stopped when the serva blue dye which designates a 40 nucleotide species reached the bottom of the gel. The 200 L buffer was collected from the lower chamber and placed in an eppendorf tube.

15 L 0.5 mg mL linear acrylamide 5 mg mL Cat 7118 60 L 5 M NaCl and 400 L absolute EtOH were added to the sample. A glass filter fiber column Cat 10066G was prepared by prewetting with Wash Solution 1 0.5 M NaCl 55 EtOH . The sample was added to the column. The column was spun for 1 min at 5000 rpm. 500 L of wash solution 2 80 EtOH was added to the column which was spun again for 1 min at 5000 rpm. The column was spun again for 1 min at 10000 rpm to get rid of any excess EtOH. 15 L of HO heated at 95 C. was applied to the column which was allowed to sit at room temperature RT for 2 min. The column was then spun for 1 min at 10000 rpm. Another 15 L of heated elution solution was added. After sitting at RT for 2 min the column was spun for 1 min at 10000 rpm. The sample was lyophilized and stored until labeling.

PolyA tails of several hundred nucleotides were efficiently added to the 3 ends of miRNAs using ATP and Poly A Polymerase. Other nucleotides could also serve as substrates for the enzyme most notably UTP. Modified nucleotides like 5 3 aminoallyl UTP adenosine 5 1 thiotriphosphate uridine 5 1 thiotriphosphate 8 4 amino butyl amino ATP and 8 6 amino butyl amino ATP were substrates for Poly A Polymerase but the lengths of the 3 tails were significantly shorter than those generated with ATP and UTP. When only modified nucleotides were in the Poly A Polymerase reaction the tails were typically 3 10 nucleotides long.

Various mixtures of standard and modified NTP ATP or UTP were tested for their capacity to be incorporated by polyA polymerase. The length of the tail could be extended tremendously adding as little as 10 M standard NTP to the reaction. The labeling efficiency of miRNAs samples tailed with fluorescent NTP only was compared to the labeling efficiency of ones with fluorescent NTP to various ratios of standard NTP. The optimal modified NTP standard NTP ratio was 5 1 under these conditions.

miRNAs were labeled in a variety of ways. For radiolabeling P ATP or P UTP at 0.5 5 M was used in a Poly A Polymerase reaction. Several hundred nucleotides can be appended to each miRNA providing substantial radioactive signal for each miRNA. For fluorescent labeling either a dye labeled nucleotide in addition to a carrier standard nucleotide is incorporated or a nucleotide with a reactive group to which can be appended a dye molecule following the tailing reaction is incorporated. The latter method generated longer tails with greater fluorescence. The use of aminoallyl UTP at a final concentration of 500 M and UTP at a final concentration of 100 M were conditions that provided favorable results. This combination typically yielded tails of 10 50 nucleotides and presumably as many as 20 dye molecules per miRNA.

The Poly A Polymerase reaction conditions described in the literature were not very efficient when using the ng and sub nanogram amounts of miRNA that were isolated from samples of total RNA. The addition of volume exclusion reagents like polyethylene glycol PEG significantly improved the labeling reaction. 10 PEG provides as much as ten times more labeled material in samples with 0.1 ng of miRNA.

Optimization of the pH of the miRNA poly A tailing reactions revealed that pH 6 7 provides much better tailing efficiency than the pH 7.9 that is recommended in the literature Sillero et al. 2001 . miRNAs tailed with poly A polymerase at pH 6.5 provided as much as ten times more signal than the pH 7.9 that is described in the literature. Our final reaction buffer for miRNA labeling was 10 Polyethylene glycol MW 3 350 10 mM MgCl 250 mM NaCl 50 mM MES pH 6.5 and 1 mM DTT.

Isolated miRNA was resuspended in 4.0 L H2O. The following were added to the miRNA sample 10 L 2 Poly A Polymerase PAP Buffer 20 Polyethylene glycol MW 3 350 20 mM MgCl2 500 mM NaCl 100 mM MES pH 6.5 2 mM DTT 2.0 L 25 mM MnCl2 2.0 L NTP mix 1 mM UTP 5 mM aminoallyl UTP 2.0 L PAP 2 U L Cat 2030G . The reaction mixture was incubated at 37 C. for 2 hours.

To begin the process of obtaining the tailed miRNA from the mixture 10 g of sheared yeast RNA and 80 L of miRNA binding buffer 0.6 M NaCl 70 EtOH were added to the reaction mix and incubated at RT for 5 minutes. The mix was subsequently applied to a glass fiber filter column which was then spun at 5000 rpm for 1 min. 300 L of miRNA Wash Buffer 0.5 M NaCl 55 EtOH was added and the column spun again at 5000 rpm for 1 min. The Wash spin step was repeated before the column was spun again at 10000 rpm for 1 min. 25 L of ddH2O heated at 95 C. was added to the column which was then incubated at 65 C. for 5 min. The column was spun at 10000 rpm for 1 minute and the eluted fraction was dried down and resuspended typically in 4 or 7 L.

Amine modified miRNA was reacted with amine modified labeled moieties like NHS ester Cy3 Cy5 AlexaFluor 555 AlexaFluor 647 and biotin. This was accomplished by the following procedure.

Amine modified dye was resuspended in 8 L dimethyl sulfoxide DMSO . Two samples were labeled per dye tube. The labeling reaction was set up by mixing 7 L of RNA with 9 L 0.2 M Sodium Bicarbonate pH 9.0 and 4 L of the amine reactive dye in DMSO. This reaction was incubated in the dark at RT for 1 hr. 4.5 L of 4M Hydroxylamine was added and the mixture was incubated in the dark at RT for 15 min.

To isolate the labeled miRNA 80 L of miRNA binding buffer was added to the mixture. This was incubated at RT for 5 min. This was then applied to a glass fiber filter column which was spun at 5000 rpm for 1 min. 300 L of miRNA Wash Buffer 0.5 M NaCl 55 EtOH was applied to the microfilter which was spun at 5000 rpm for 1 min. The wash spin step was repeated before spinning it again at 10 000 rpm for 1 min. 25 L of 95 C. ddHO was incubated with the microfilter at 65 C. for 5 min. It was then spun at 10 000 rpm for 1 min.

The miRNA was resuspended in 4.0 L HO. The reaction contained 10 L 2 PAP Buffer 20 Polyethylene glycol MW 3 350 20 mM MgCl 500 mM NaCl 100 mM MES pH 6.5 2 mM DTT 2.0 L 25 mM MnCl 2.0 L NTP mix 1 mM UTP 5 mM Cy3 UTP and 2.0 L PAP 2 U L Cat 2030G . The reaction mixture was incubated at 37 C. for 2 hrs. Then 10 g sheared yeast RNA Cat 7118 and 80 L miRNA binding buffer were added and the mixture was incubated at RT for 5 min. The mixture was applied to a microfilter silica filter column which was then spun at 5000 rpm for 1 min. 300 L of miRNA Wash Buffer 0.5 M NaCl 55 EtOH was applied to the microfilter which was spun at 5000 rpm for 1 min. The wash spin step was repeated before spinning it again at 10 000 rpm for 1 min. 25 L of 95 C. ddHO was incubated with the microfilter at 65 C. for 5 min. It was then spun at 10 000 rpm for 1 min. The eluted fraction was dried down and resuspended.

The miRNA was resuspended in 4.0 L HO. The reaction contained 10 L 2 PAP Buffer 20 Polyethylene glycol MW 3 350 20 mM MgCl 500 mM NaCl 100 mM MES pH 6.5 2 mM DTT 2.0 L 25 mM MnCl 2.0 L P ATP mix and 2.0 L PAP 2 U L Cat 2030G . The reaction mixture was incubated at 37 C. for 2 hrs. Then 10 g sheared yeast RNA Cat 7118 and 80 L miRNA binding buffer were added and the mixture was incubated at RT for 5 min. The mixture was applied to a microfilter silica filter column which was then spun at 5000 rpm for 1 min. 300 L of miRNA Wash Buffer 0.5 M NaCl 55 EtOH was applied to the microfilter which was spun at 5000 rpm for 1 min. The wash spin step was repeated before spinning it again at 10 000 rpm for 1 min. 25 L of 95 C. ddHO was incubated with the microfilter at 65 C. for 5 min. It was then spun at 10 000 rpm for 1 min. The eluted fraction was dried down and resuspended in 7 L.

Various probe designs were tested in order to get the best level of detection from the labeled miRNAs. Amine modification position of amine modification and the benefits of extraneous nucleotides were factors that were examined. Based on hybridization with both chemically synthesized and endogenous miRNAs it was determined that a linker having 21 nts resulted in the best signal however it was also concluded that the linker length can be decreased to 15 nts if necessary while still getting a comparable signal. The amine group required for slide attachment located at the 3 end rather than the 5 end gave better results and the addition of 4 bps on each side of the miRNA probe sequence which are complementary to the miRNA precursor increased the signal in some miRNAs.

MicroRNA probes with 3 and 5 amines and linker lengths of 3 21 nucleotides between the terminal amine and the microRNA probe sequence were prepared. The probes were spotted on glass slides to create microarrays. miRNAs that had been 3 end labeled using the procedure described above were hybridized to the microarrays. In general the probes with longer linker lengths provided greater signal. The miRNA probes with 3 amines performed much better than the equivalent probes with 5 amines .

The sequence of the linker between the terminal amine and the miRNA probe sequence was varied without impacting the hybridization signal. Five repeats of the sequence ACC were used at the 3 end of the miRNA probes.

The performance of probes that had only the 19 24 nt predicted to be in the final mature miRNAs was compared to the performance of probes that included 4 nucleotides of flanking region from both sides of the miRNA coding region. The theory was that miRNA processing is not always precise and thus some miRNAs might include sequence from either side of the predicted mature sequence.

miRNA probes were prepared and spotted using a SpotBot TeleChem according to the manufacturer s recommendations. The arrays were then processed using the method described below.

Up to five slides were placed in a 50 ml slide washer. 40 mls of slide wash buffer 150 mM NaCl 100 mM Tris pH 7.5 0.1 Tween 20 were added and the buffer and slides were mixed by inversion for 60 seconds. The first wash buffer was discarded. Another 40 mls of fresh slide buffer were added and inverted with the slides for 60 seconds before they were allowed to set without agitation for 5 minutes. The buffer was poured off and the slides were rinsed with de ionized water. The water was poured off and another 40 mls of de ionized water were added and mixed vigorously with the slides by inversion for 60 seconds. The wash process was repeated. The arrays were removed one at a time. They were then dried in either a microfuge or 50 ml conical. If they were dried by picofuge up to 4 slides were included and the picofuge was spun for 10 seconds.

Hybridization Solution 3 SSC was heated at 65 C. for 10 minutes and mixed thoroughly several times throughout the incubation. 35 l of pre warmed were added to the labeled miRNA and vortexed. The mixture was heated to 95 C. for 3 minutes in a heat temperature block. While the miRNA Hybridization Solution mixture was being heated 35 l of Hybridization Solution was added to the top and bottom humidity reservoirs of the hybridization chamber and an additional 150 l spread throughout the central span of the chamber. The array was placed in the chamber and precautions were taken to ensure the slide did not come into contact with the sides of the chamber. A coverslip was placed over the sub arrays outlined by the etched regions. The 35 l of heated miRNA mixture was added to the array and it was allowed to travel by capillary action to the other side of the coverslip. The hybridization chamber was sealed and the array incubated at 42 C. for 4 16 hours but no longer than 16 hours.

The slides were removed from the hybridization chamber and submerged in 2 SSC 0.1 SDS to remove the coverslip. Once the cover slip was removed the slides were moved up and down 10 times and immediately placed in 2 SSC buffer. They were washed for 2 minutes with gentle stirring 150 200 rpm in a wash chamber. The slides were transferred to a new slide holder in a new container containing fresh 2 SSC and washed for 1 min with up and down motion. The holder was transferred to a new container with fresh buffer and washed for 1 additional minute. The slides were then dried using a centrifuge or picofuge.

For any method that is used to compare the abundance of biomolecules like RNA in one sample to the biomolecules in another sample it is critical that the process be highly reproducible. The reproducibility of the miRNArray system was tested by repeatedly comparing the miRNA profiles of human prostate and colon samples.

Total RNA from a single human prostate sample was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 using the two step fluorescent labeling process described above. Total RNA from a single human colon sample was isolated using the glass fiber filter method described above. The total RNA was split into six independent samples and each sample was fractionated by tube electrophoresis to recover the miRNAs. The six miRNA samples were fluorescently labeled with Cy5 using the two step fluorescent labeling process described above.

The Cy3 labeled prostate miRNAs were split into six equal samples and each was mixed with one of the Cy5 labeled colon samples. The six independently labeled miRNA mixtures were hybridized to the miRNA probes arrayed on six different glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the prostate and colon samples. Signal ratios at each element are expressed as a log ratio and shown in . The average correlation between the six independent reactions was 98 indicating that the miRNArray process is highly reproducible.

Molecular methods that distinguish the tissue origin of unknown samples are needed. An example is where a metastatic tumor has been found and its origin is unknown. Identifying the originating tissue allows a physician to identify other affected organs as well as create a therapeutic regimen with a higher likelihood of success.

Total RNA from 24 human tissues was isolated using the glass fiber filter method described above. Each of the total RNAs were fractionated by tube electrophoresis to recover the miRNAs from each sample. One half of each sample was placed in a single tube to create a miRNA pool comprising the miRNAs from all 24 samples. The miRNAs from the pooled sample as well as each of the 24 single tissue samples were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signal from each single tissue sample was compared to the signal from the pooled sample. The relative signal intensities of the single and pooled samples were compared for each miRNA. Shown in the table below is the list of miRNAs that were exclusively or primarily expressed in one or a few related tissues. The expression of these miRNAs could be used to determine the origin of a human tissues sample. Similarly age sex race etc might also be determined using miRNA profiling.

The ultimate test of array results is confirming microarray expression profiles with a second method of RNA quantification. A ribonuclease protection assay was used to compare the expression of three different miRNAs in 9 human tissue samples to the miRNArray data generated in the previous example.

Total RNA from human samples of adrenal gland colon kidney liver lung skeletal muscle pancreas prostate spleen and thymus was isolated using the glass fiber filter method described above. Each of the total RNAs were analyzed using the procedure below to measure the expression of let 7 miR 16 and miR 200b.

Radiolabeled RNAs with sequences complementary to let 7 miR 16 and miR 200b were produced according to the miRNA Probe Labeling kit Ambion . 104 cpm of each probe was mixed in separate tubes with 1 ug of each of the nine human tissue sample RNAs listed above. 5 g of yeast RNA Ambion and hybridization buffer 40 formamide 50 mM Na Citrate pH 6.4 150 mM Na Acetate pH 6.4 1 mM EDTA 2.5 PEG 8000 were added to each sample. The samples were heated to 95 C. for three minutes and then incubated overnight at 42 C. To each sample 150 l of RNAse Digestion solution 300 mM Na Acetate pH 7.5 10 mM HEPEs pH 7.5 5 mM EDTA 20 ug ml Salmon Sperm DNA 150 ug ml Glyco Blue 2.5 U ml RNase A and 100 U ml RNase T1 was added. The resulting mixtures were incubated for 30 minutes at 37 C. 220 l of RNAse Inactivation Precipitation buffer 1 M Guanidinium Thiocyanate 0.167 N Lauryl Sarcosine 10 mM DTT 83 Isopropanol was added to each sample. The samples were incubated at 20 C. for fifteen minutes. The sample tubes were centrifuged at 12 000 RPM for fifteen minutes. The pellets were washed with 70 ethanol dried and dissolved in gel loading buffer 95 formamide 18 mM EDTA pH 8.0 0.025 bromophenol blue 0.025 xylene cyanol . The samples were heated to 95 C. and fractionated using a 15 denaturing polyacrylamide gel.

Radiolabeled probes that were hybridized to target miRNAs were protected from nuclease digestion and were seen as bands on autoradiographs that were exposed to the gels upon which the miRNA samples were fractionated. The intensities of the bands in the various samples correlates with the abundances of the miRNAs in the samples. As shown in the Ribonuclease Protection Assay data mirror the miRNArray data for the same 9 samples that were generated in the experiment described in the previous example. This confirms that the miRNArray system is accurate in estimating the abundance of miRNAs in samples.

In a one set of experiments tumor and normal adjacent tissue NAT samples were obtained from seven lung cancer patients. In a second set of experiments tumor and NAT samples were evaluated from 16 additional lung cancer patients. The final data set comprise the first seven samples shown in the first table 16 additional samples.

Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the tumor and NAT samples from each patient. Signal ratios of tumor NAT for some of the miRNA elements are shown in . A table listing the miRNAs that were differentially expressed in at least four of the seven lung tumor NAT samples from the first set of experiments is provided in Table 5A below. 16 additional lung cancer samples were analyzed using arrays. Those miRNAs that are significantly and reproducibly up or down regulated in the tumors relative to normal tissues from the combined 23 samples are shown in Table 5B. The miRNAs shown in Table 5B could be used to determine if a given sample were cancerous. These miRNAs represent potential targets for therapeutic development. Likewise the genes that are regulated by these miRNAs might provide effective targets for therapeutic development. The miRNAs that are differentially expressed in only a subset of the samples likely represent molecular markers of the sub classes of the cancer samples. These will likely prove to be valuable as prognostic indicators.

As with the normal tissue samples described above a second method of RNA quantification was used to confirm that the miRNAs that were observed to be differentially expressed by the miRNArray process in the first set of experiments were indeed differentially expressed. Tumor NAT samples from two of the lung cancer patients were analyzed for expression of miR 16 miR 21 miR 143 miR 145 and let 7. As was observed in the comparisons of the miRNA expression profiles in the normal samples the quantitative data from the miRNArray system were very similar to the secondary analysis . As above this is validation of the miRNArray system. Furthermore it shows that an alternative RNA analysis system could be used to analyze the expression of defined miRNAs in a clinical setting.

Tumor and normal adjacent tissue NAT samples were obtained from six colon cancer patients in a first set of experiments. In a second set of experiments tumor and NAT samples were evaluated from 18 additional colon cancer patients. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the tumor and NAT samples from each patient. Signal ratios of tumor NAT for some of the miRNA elements are shown in . A table listing the miRNAs that were differentially expressed in at least four of the first six colon tumor NAT samples from the first set of experiments is provided below in Table 6A Data from the combination of the first and second set of experiments is provided in Table 6B. Those miRNAs that are consistently seen to be down or up regulated in tumor samples could be used to determine if a given sample were cancerous. Likewise these miRNAs represent potential targets for therapeutic development. Also the genes that are regulated by these miRNAs might provide effective targets for therapeutic development. The miRNAs that are differentially expressed in only a subset of the samples likely represent molecular markers of the sub classes of the cancer samples. These will likely prove to be valuable as prognostic indicators.

Tumor and normal adjacent tissue NAT samples were obtained from three breast cancer patients two thyroid cancer patients and one bladder cancer patient. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the tumor and NAT samples from each patient.

Interestingly many of the miRNAs that were differentially expressed in the lung and colon tumor samples were likewise differentially expressed in the breast thyroid and bladder samples. A heat map showing a subset of the differentially expressed miRNAs is shown in . Note that green squares represent miRNAs that are expressed at least 50 lower in the tumor than the normal adjacent tissue and red represents miRNAs that are expressed at least 50 higher in tumor than in normal adjacent tissues. A table listing the miRNAs that were differentially expressed in samples from each of cancer types is provided below. These miRNAs appear to be the functional equivalents of the known protein expressing oncogenes and are likely involved in regulating cell proliferation and cell death as well as angiogenesis. These miRNAs represent potential targets for therapeutic development. Likewise the genes that are regulated by these miRNAs might provide effective targets for therapeutic development.

An assumption of many molecular diagnostic methods is that a tumor sample is relatively the same throughout the cancerous tissue. The validity of this assumption was tested by measuring the miRNA profiles of two isolates of a single thyroid tumor from one patient.

Total RNA from the two thyroid tumor and normal adjacent samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to the miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the isolates of the same tumor. Signal ratios at each element are expressed as a log ratio and shown in . The average correlation between the two isolates was 92 indicating that there is little variation between different regions of the same sample.

Formaldehyde and paraformaldehyde fixed tissues represent a huge resource of clinically interesting samples. In clinical setting most tumors and other disease tissues are fixed analyzed and stored indefinitely. Analyzing these historical samples provide scientists with the opportunity to match molecular characteristics of the fixed samples to clinical outcomes for the patients from which the samples derived. Although researchers have long recognized the benefits of performing RNA analysis on fixed tissues the fixed tissues have not been used because it is essentially impossible to isolate high quality RNA from the samples. This is due to the fact that the fixatives introduce chemical moieties into the RNA molecules that make them incompatible with essentially all quantitative RNA techniques.

It was tested whether miRNAs could be extracted and analyzed from fixed tissues. Mouse kidney and brain samples were either fixed in 4 paraformaldehyde for three weeks or frozen and stored at 80 C. for three weeks. Following the three week incubations the fixed samples were processed using the Optimum kit Ambion according to manufacturer s recommendation. Total RNA was isolated from both the fixed and frozen samples using the glass fiber filter method described above. Each of the total RNAs was fractionated by tube electrophoresis to recover the miRNAs from each sample. The miRNAs from each sample were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signal from each fixed sample was compared to the signal from its corresponding frozen sample. The relative signal intensities of the fixed and frozen samples were compared for each miRNA. The correlations between the fixed and frozen samples approached 99 indicating that miRNAs can be used to analyze fixed tissues. A representative array profile for one of the sample comparisons is provided in .

Another problem that is encountered by researchers who are comparing the gene expression profiles of different samples is that RNA is fragile. Nucleases in tissues as well as those introduced by mishandling can degrade mRNAs and other interesting RNA molecules in samples. Most methods of RNA analysis are affected by RNA degradation making it essentially impossible to get reliable data when one or more RNA samples in the analysis are at least partially degraded. Given the surprising results with miRNAs in fixed tissues it was decided to test the impact of RNA degradation in tissues on miRNA expression profiles.

A series of RNA samples was created with variable levels of degradation by harvesting the colon of a mouse and dividing the sample into two equal parts. RNA from one of the parts was isolated using the glass fiber filter method described above. This RNA represented the non degraded control. The second part was sub divided into five equal samples. The five samples were placed in phosphate buffered saline and incubated for 0 120 minutes at room temperature. RNA was isolated from one of the five samples at 0 5 10 20 and 120 minutes using the glass fiber filter method described above. Each of the total RNA samples was analyzed using the Bio Analyzer Agilent according to the manufacturer s recommended protocol. Profiles of the various samples are shown in . Note that the non degraded noted as frozen sample has two distinct bands corresponding to the 18S rRNAs and 28S rRNAs in the sample. The 18S rRNA and 28S rRNA bands disappear as the nucleases in the colon samples are allowed to degrade the RNA in the tissues during the room temperature incubation.

Each of the total RNAs was fractionated by tube electrophoresis to recover the miRNAs from each sample. The miRNAs from each sample were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signal from each degraded sample was compared to the signal from the non degraded sample. The relative signal intensities of the degraded and non degraded samples were compared for each miRNA . The correlations between the degraded and non degraded samples approached 95 indicating that miRNAs can be used to analyze degraded RNA samples. The benefits to using miRNAs profiling to compare samples is that it reduces the stringency required for sample handing this is especially important for clinical and field settings and it allows many historical samples that were prepared poorly to be used for comparative analyses.

Many diagnostic assays are designed to assay blood samples since blood is readily available and often representative of the health of the patient. To confirm that miRNA can be analyzed in blood samples samples from ten different individuals were obtained. Total RNA from each of the samples was isolated by pouring whole blood into a 10 ml syringe attached to a LK4 leukocyte depletion filter Pall in a reusable syringe device. The blood was passed thru the LK4 filter over a time period of 75 sec. per sample. The filter device was then transferred to a 3 ml syringe containing 3 ml of Ambion s RNAlater which was passed thru the filter containing the captured WBCs in a drop wise fashion. The plunger was then retracted and residual drops of RNAlater were expelled but the filters were left damp. The filters with treated WBCs were then removed from the devices and transferred to 15 ml conicals. Filters were stored at room temp. for approximately 1 hour before beginning the RNA extraction.

2.5 ml Lysis Solution. from RiboPure Blood kit Ambion and 0.25 ml of 2 M NaOAc was added to each 15 ml conical containing an LK4 filter with captured treated cells and these were shaken vigorously for 30 sec. and then stored for 5 min. on ice. 2.5 ml acid phenol ChClwas added to each tube and contents mixed by vigorous shaking for 30 sec. Preps were centrifuged at 3 200 rpm for 10 min. in swinging bucket table top centrifuge then the aqueous phase was removed to a second 15 ml conical. 1.25 volumes of 100 ethanol was added to each prep using calibration marks on the 15 ml tubes to determine volume of prep prior to adding EtOH and contents mixed by vortexing. Preps were filtered thru silica filters in RiboPure Blood kit Ambion using vacuum suction vacuum manifold. Silica filters were transferred to 2 ml microfuge tubes and washed by adding 0.75 ml of Wash 1 1.6 M GuSCN 70 EtOH and centrifuging briefly to pass the solution thru the filters. Silica filters were washed twice with 0.75 ml of Wash 2 3from RiboPure Blood kit Ambion as above then centrifuged for 1 min. at 13.2 K rpm to remove residual fluid. RNA was eluted by transferring the silica filters to fresh 2 ml collection tubes adding 200 ul of nuclease free water heated to 78 C. to each silica filter storing for 1 min. at room temp. then centrifuging for 1 min. at 13.2 K rpm. The miRNAs from each sample were isolated using standard gel purification as described above. The miRNAs from each sample were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon . The signal intensities of the various miRNAs were plotted. provides miRNA profiles for three of the donors. The signals for many of the miRNAs are on par with what we observe for tissue samples indicating that miRNAs are indeed present in blood samples and might be used for diagnostic assays. Interestingly the miRNA profiles were very similar for the ten different donors suggesting that there is a global norm for miRNAs in blood. This could make it easier to identify individuals who are sick given that minor miRNA perturbations would be seen against the relatively non variable background of normal individuals.

As noted in previous examples a number of miRNAs have been identified that are differentially expressed between tumor and normal adjacent tissue samples from the same cancer patients. Interestingly there is significant overlap in the miRNAs that are differentially expressed between different cancers suggesting there is a core set of miRNAs that influence cellular processes that when altered lead to cancer. The following describes experiments aimed at developing a link between miRNA mis regulation and cancer.

Twenty two tumor and normal adjacent tissue NAT samples from lung cancer patients were analyzed using the miRNA array system described above. The arrays were analyzed and the relative expression of each miRNA was compared between the tumor and normal adjacent tissues from each patient. The various miRNAs were clustered based on their relative expression in tumors across different patients . Six miRNAs miR 126 30a 143 145 188 and 331 were expressed at significantly lower levels in the tumors of more than 70 of the patients. Three miRNAs miR 21 and 200b were expressed at significantly higher levels in the tumors of more than 70 of the patients. The differential expression of a number of these miRNAs was verified by Northern analysis .

Twenty five tumor and NAT samples from colon cancer patients were analyzed using our miRNA array process. Like the lung cancer comparisons the various miRNAs were clustered based on their relative expression in tumors across the different colon cancer patients . Five miRNAs miR 143 145 195 130a and miR 331 were expressed at significantly lower levels in the tumors of more than 70 of the patients. Five miRNAs miR 223 31 21 17 and 106 were expressed at significantly higher levels in the tumors of more than 70 of the patients.

It is interesting that eight different miRNAs were differentially expressed between the tumor and normal adjacent samples for most of the lung and colon patient samples that we analyzed . These same miRNAs were also found to be differentially expressed in the breast thymus bladder pancreatic and prostate cancer patients that we analyzed suggesting that these miRNAs might control cellular processes that when altered lead to cancer.

To address whether specific miRNAs might be participating in cancer through the mis regulation of oncogenes we scanned the 3 untranslated regions UTRs of 150 well known oncogenes for sequences with significant homology to the miRNAs identified in our microarray analysis. Potential target sites were selected based on two criteria 

As seen in Table 8 potential target sites in the 3 UTRs of known oncogenes were identified for all of the miRNAs that were observed to be routinely differentially expressed in tumor samples. Interestingly KRAS2 MYCL1 and CBL have multiple predicted miRNA binding sites which could provide the cooperative miRNA binding that has been implicated as an important factor in miRNA regulation Doench et al. 2003 Zeng et al. 2003 . Many of the genes listed in Table 8 become oncogenic when they are over expressed thus it is conceivable that reduced expression of a miRNA could lead to up regulation of one or more oncogenes and subsequently lead to oncogenesis.

Confirming miRNA target site predictions can be done in a variety of ways. In and genetic approaches have been applied wherein mutations in the miRNA and the putative miRNA target site s are made and shown to result in similar phenotypes Ha et al. 1996 Vella et al. 2004 . In mammalian cells where genetic approaches are far more difficult reporter constructs have been used to show that the 3 UTRs of putative target genes are regulated in cells at levels that are disproportionate to reporter vector controls that contain mutations in the putative miRNA binding sites Lewis et al. 2003 . In addition vectors and oligonucleotides have been used to introduce or inhibit miRNAs in cells to determine the effects on endogenous levels of putative target genes Lewis et al. 2003 Kiriakidou et al. 2004 . The latter approach has been undertaken to validate the miRNA target site predictions.

Synthetic miRNAs and miRNA inhibitors have been developed that can be transfected into mammalian cells to either introduce miRNAs into cells or inhibit the activity of miRNAs in cells respectively. See U.S. Nos. 60 627 171 and 60 649 634 both of which are hereby incorporated by reference. A synthetic miRNA and a miRNA inhibitor corresponding to let 7 were used to determine if the target site predictions were correct. In these experiments cultured cells that express undetectable levels of the miRNA were transfected with the synthetic miRNA using siPORT NeoFX Transfection Agent Ambion . Immunofluorescence assays were used to RAS and C MYC in the transfected cells. The proteins from both oncogenes were expressed at almost three fold lower levels in cells transfected with the synthetic miRNA than cells transfected with a Negative Control miRNA Ambion . In a reciprocal experiment cells that naturally express high levels of the miRNA were transfected with the let 7 miRNA inhibitor. As expected the proteins from both oncogenes were higher in cells transfected with the miRNA inhibitor than in cells transfected with the Negative Control inhibitor Ambion . These results are consistent with the model that the miRNA regulates the expression of the two oncogenes. These data suggest that mis regulation of a key miRNA could participate in cancer progression by failing to regulate the expression of one or more oncogenes.

Systemic lupus erythematosus SLE Lupus is a chronic inflammatory auto immune disease that ultimately leads to immune complex mediated end organ failure. It is characterized by an over activation of CD4 T helper cells and repression of CD8 T cytotoxic activity leading to an overproduction of natural antibodies and pathogenic autoantibodies. Recently several histone modifications were reported in peripheral blood mononuclear cells PBMCs isolated from lupus patients. Diagnosis of lupus is still frequently incorrect mainly because the symptoms vary so widely and they come and go frequently and because the disease mimics so many other disorders. Furthermore diagnosis does not indicate the particular therapy to be used. In the absence of a cure present day treatment of lupus is still primarily tailored to symptomatic relief and not to the diagnosis. A diagnostic assay with high specificity and sensitivity would be very important.

Samples were analyzed from 16 individuals 8 with clinically verified lupus and 8 non lupus patients that were age and gender matched with the lupus patients. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the lupus and normal sample signals were compared to identify differentially expressed miRNAs. Each array experiment included duplicate arrays.

Fourteen miRNAs were differentially expressed in all of the lupus samples relative to the matched samples. miR 301 miR 199 miR 95 miR 105 mu miR 290 miR 215 miR 188 miR 186 miR 211 miR 331 and miR 137 were expressed at 50 or less in the lupus samples than the corresponding normal samples. miR 21 miR 223 and miR 342 were expressed at 50 or greater in the lupus samples than the corresponding normal samples. Several of the miRNAs were differentially expressed by as much as ten fold between the lupus and normal samples. These miRNAs represent targets for diagnostic assay of therapeutic development.

In a further analysis of these samples twenty three miRNAs were differentially expressed 1.5 fold or greater in at least 50 of the lupus samples relative to the matched samples. Among these miRNAs miR 95 miR 144 miR 184 and miR 186 were differentially express in all the lupus samples. Together with let7D AS miR 21 miR 32 miR 133A miR 137 miR 141 miR 181A miR 188 miR 199 miR 201 miR 203 miR 204 miR 211 miR 212 miR 223 miR 224 mu miR 329 miR 331 and miR 344 these miRNAs represent targets for diagnostic assay or therapeutic development.

Novel infectious particles termed prions composed largely and perhaps solely of a single protein are the likely causative agents of a group of transmissible spongiform encephalopathies that produce lethal decline of cognitive and motor function. Evidence indicates that the responsible protein arrives at a pathogenic state by misfolding from a normal form that has ubiquitous tissue distribution.

Using two cell based prion model systems the identification of miRNAs that might be associated with the process was pursued. One model system comprises two cell lines one of which is susceptible to prion formation and one that is not. The second model system involves cells before and after they have been infected with prions. Total RNA from prion sensitive cells prion insensitive cells and prion infected cells was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signal from each of the samples was compared to identify differentially expressed miRNAs.

As seen in ten miRNAs were significantly up or down regulated in both prion sensitive and prion infected cells relative to prion resistant uninfected cells. Arrays on multiple biological replicates for both model systems have confirmed these results. Based on their expression profiles miR 95 135A 7 9 as 27A 130A 16 26A and 24 likely are involved directly or indirectly in prion infection and might represent diagnostic or therapeutic targets for prion disease.

Additional analyses revealed that about 40 miRNAs were found differentially expressed between cells sensitive and resistant to infection and about 20 miRNAs were found differentially expressed during infection by prion. Ten miRNAs were significantly up or down regulated in both model systems. miR 7 miR 9 as miR 16 miR 24 miR 26A miR 27A miR 130A and miR 239 are induced during infection and expressed at higher levels in cells sensitive to infection. These miRNAs might be involved in the mechanism of prion infection. miR 95 and miR 135A are repressed during infection and expressed at higher levels in cells not sensitive to infection and therefore might confer some type of resistance to prim infection. Arrays on multiple biological replicates for both model systems have confirmed these results. Based on their expression profiles miR 95 135A 7 9 as 27A 239 130A 16 26A and 24 likely are involved directly or indirectly in prion infection and might represent diagnostic or therapeutic targets for prion disease.

Stroke is a major cause of death and permanent disability in humans. They occur when blood flow to a region of the brain is obstructed and may result in death of brain tissue. There are two main types of stroke ischemic and hemorrhagic. Ischemic stroke is caused by blockage in an artery that supplies blood to the brain resulting in a deficiency in blood flow ischemia . Hemorrhagic stroke is caused by the bleeding of ruptured blood vessels hemorrhage in the brain. Understanding miRNAs involved in stroke might enhance detection and or treatment.

A stroke model system was used wherein mice are preconditioned by reducing oxygen flow to the brain Kitagawa 1991 . An equivalent set of six mice were used three were preconditioned and three were untreated. 24 hours after pre conditioning the mice were sacrificed. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the preconditioned and normal sample signals were compared to identify differentially expressed miRNAs.

Analysis of the miRNA profiles of the preconditioned animals labeled P1 P2 and P4 revealed 10 miRNAs that were expressed at significantly different levels in all three pre conditioned animals relative to the three non treated animals . These miRNAs resulted from ischemic pre conditioning and represent potential targets for stroke diagnosis prevention or treatment.

Tumor and normal adjacent tissue NAT samples were obtained from six breast cancer patients. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the tumor and NAT samples from each patient. A table listing the miRNAs that were differentially expressed in at least 80 of the breast tumor NAT samples is provided in Table 9. Those miRNAs that are consistently seen to be down or up regulated in tumor samples could be used to determine if a given sample were cancerous. Likewise these miRNAs represent potential targets for therapeutic development. Also the genes that are regulated by these miRNAs might provide effective targets for therapeutic development. The miRNAs that are differentially expressed in only a subset of the samples likely represent molecular markers of the sub classes of the cancer samples. These will likely prove to be valuable as prognostic indicators.

Tumor and normal adjacent tissue NAT samples were obtained from six thyroid cancer patients. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the tumor and NAT samples from each patient. A table listing the miRNAs that were differentially expressed in at least 80 of the thyroid tumor NAT samples is provided in Table 10. Those miRNAs that are consistently seen to be down or up regulated in tumor samples could be used to determine if a given sample were cancerous. Likewise these miRNAs represent potential targets for therapeutic development. Also the genes that are regulated by these miRNAs might provide effective targets for therapeutic development. The miRNAs that are differentially expressed in only a subset of the samples likely represent molecular markers of the sub classes of the cancer samples. These will likely prove to be valuable as prognostic indicators.

Tumor and normal adjacent tissue NAT samples were obtained from prostate cancer patients. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the tumor and NAT samples from each patient. A table listing the miRNAs that were differentially expressed in the prostate tumor NAT samples is provided in Table 11. Those miRNAs that are consistently seen to be down or up regulated in tumor samples could be used to determine if a given sample were cancerous. Likewise these miRNAs represent potential targets for therapeutic development. Also the genes that are regulated by these miRNAs might provide effective targets for therapeutic development. The miRNAs that are differentially expressed in only a subset of the samples likely represent molecular markers of the sub classes of the cancer samples. These will likely prove to be valuable as prognostic indicators.

Tumor and normal adjacent tissue NAT samples were obtained from bladder cancer patients. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the tumor and NAT samples from each patient. A table listing the miRNAs that were differentially expressed in the bladder tumor NAT samples is provided in Table 12. Those miRNAs that are consistently seen to be down or up regulated in tumor samples could be used to determine if a given sample were cancerous. Likewise these miRNAs represent potential targets for therapeutic development. Also the genes that are regulated by these miRNAs might provide effective targets for therapeutic development. The miRNAs that are differentially expressed in only a subset of the samples likely represent molecular markers of the sub classes of the cancer samples. These will likely prove to be valuable as prognostic indicators.

Crohn s disease is also called regional ileitis it is a chronic progressive inflammatory disease of the bowel. The symptoms are most commonly that of diarrhea and pain. Weight loss fatigue and irritability are characteristic of the disease. The bowel movements often include mucus blood and pus because of the infection. Fat may occur in the bowel movements making them bulky and foul smelling. The root cause of Crohn s disease is unknown and there are commercially available therapeutic options available for patients with the disease. Identifying the molecular cause of Crohn s could lead to enhanced diagnostics as well as therapeutics.

Intestinal tissue was recovered from Crohn s patients and normal individuals. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the Crohn s disease and normal sample signals were compared to identify differentially expressed miRNAs. Each array experiment included duplicate arrays.

Seven miRNAs were differentially expressed in all of the Crohn s patient samples relative to the normal controls. miR 126as miR 194 miR 192 and mu miR 207 were expressed at more than two fold lower levels in the Crohn s patient intestinal samples while miR 150 miR 125B and miR 223 were expressed at more than two fold higher levels in Crohn s patients. These miRNAs represent targets for diagnostic assay or therapeutic development for Crohn s disease.

Cancer of the uterine cervix is the second leading cause of cancer related deaths among women worldwide. Epidemiological and molecular studies have demonstrated that human papillomaviruses HPVs are the etiological agents of the majority 99 of cervical cancers. More than a hundred of different HPV types have been characterized so far and new types are regularly added to this list. Among the HPVs identified about the third 46 infect the genital tract. These HPV types are sexually transmitted and can be further categorized into low risk or high risk groups according to the propensity for malignant progression of the lesions that they cause. The high risk mucosal HPV types cause intraepithelial lesions that can progress to invasive carcinomas. Although routine cytology screening and reflexive HPV testing has improved cervical cancer detection and reduced mortality significant problems and barriers remain including the low predictive value of current testing which leads to costly and invasive testing and emotional stress in patients.

To identify host human miRNAs signatures in patients carrying high risk HPV type s total RNA from HPV negative or high risk HPV positive liquid based cervical Pap smears was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signal from each of the samples was compared to identify differentially expressed miRNAs. Four miRNAs were consistently expressed at levels at least 1.5 fold higher in the high risk HPV positive patients relative to the HPV negative controls. Based on their expression profiles miR 29B miR 326 miR 361 and miR 425 likely are involved directly or indirectly in viral infection and or cervical cancer onset and might represent diagnostic or therapeutic targets for cervical cancer.

Leukemia is cancer of T cells. The two most prevalent forms of leukemia are acute myelogenous leukemia and chronic lymphocytic leukemia. Acute myelogenous leukemia is a rapidly progressing disease that affects mostly cells that are unformed or primitive not yet fully developed or differentiated . These immature cells cannot carry out their normal functions. Chronic leukemia progresses slowly and permits the growth of greater numbers of more developed cells. In general these more mature cells can carry out some of their normal functions. To understand how miRNAs are affected by leukemia or might be contributing to leukemia samples from leukemia patients were compared to samples from normal individuals.

Total RNA from white blood cells of patients with acute myelogenous leukemia AML chronic lymphocytic leukemia CLL and no disease was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the tumor and NAT sample signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the leukemia and normal samples from each patient.

The following miRNAs were expressed at lower levels in all of the AML samples than the corresponding normal samples miR 425 miR 361 miR 25 and mu miR 291 5P. The following miRNAs were expressed at higher levels in all of the AML samples than the corresponding normal samples miR 126 miR 126 AS miR 222 miR 221 and miR 181B. Those miRNAs that are consistently seen to be down or up regulated in AML patients could be used to diagnose patients. Likewise these miRNAs represent potential targets for therapeutic development. Likewise the genes that are regulated by these miRNAs might provide effective targets for therapeutic development.

The following miRNAs were expressed at lower levels in all of the CLL samples than the corresponding normal samples miR 361 miR 425 and mu miR 341. The following miRNAs were expressed at higher levels in all of the CLL samples than the corresponding normal samples miR 92 miR 99B miR 23A miR 23B miR 223 miR 26A miR 221 miR 222 miR 21 miR 20 miR 181A miR 181B miR 16 miR 15 miR 106A let 7 and miR 103. Those miRNAs that are consistently seen to be down or up regulated in CLL patients could be used to diagnose patients. Likewise these miRNAs represent potential targets for therapeutic development. Likewise the genes that are regulated by these miRNAs might provide effective targets for therapeutic development.

Alzheimer s disease is a progressive brain disorder that gradually destroys a person s memory and ability to learn reason make judgments communicate and carry out daily activities. As Alzheimer s progresses individuals may also experience changes in personality and behavior such as anxiety suspiciousness or agitation as well as delusions or hallucinations.

There is currently no cure for Alzheimer s and positive diagnosis of the disease can only be achieved post mortem by analyzing the patient brain for two abnormal microscopic structures called plaques and tangles. Amyloid plaques are clumps of protein that accumulate outside the brain s nerve cells. Tangles are twisted strands of another protein that form inside cells. Understanding the molecular basis of Alzheimer s will be critical to the development of effective drugs and diagnostic assays. We analyze the miRAN expression profiles of the brains of Alzheimer s patients to identify those miRNAs that are differentially expressed.

Alzheimer s and normal amygdala tissue samples were obtained from eight individuals. Total RNA from these samples was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The amygdala miRNAs were fluorescently labeled with Cy3 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signals from the Alzheimer s and normal amygdale samples were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the tumor and NAT samples from each patient. The following miRNAs were down regulated in all of the Alzheimer samples than corresponding normal samples miR 425 mu miR 291 5P miR 204 and miR 338. The following miRNAs were up regulated in all of the Alzheimer samples than corresponding normal samples miR 145 miR 16 miR 223 miR 126 let 7F2 and miR 143. Those miRNAs that are consistently seen to be down or up regulated in Alzheimer s samples could be used to diagnose Alzheimer s patients. Likewise these miRNAs represent potential targets for therapeutic development. The genes that are regulated by these miRNAs might provide effective targets for therapeutic development.

Although Alzheimer Disease AD is the most common form of dementia there is currently no molecular test for AD and definitive diagnosis can only be made by post mortem histological examination to identify extracellular amyloid plaques and intracellular neurofibillary tangles in human brain sections. Promising new therapies offer the prospect of better management of AD provided methods for definitive diagnosis can be developed. Both early and late onset human AD have been linked to specific haplotypes of the sequence encoding the ApoE apolipoprotein transporter. Apo E mice JAX strain 02052 serve as an animal model for human Alzheimer Disease 1 2 3 . These mice are homozygous for the Apoeknockout mutation in the apolipoprotein E gene which leads to defects in lipid transport defects in cholesterol metabolism and xanthomatous lesions in the brain. These mice also show altered responses to stress impaired learning and memory and synaptic damage. In order to identify biomarkers associated with AD we carried out global expression profiling of mRNA and microRNA miRNA transcripts in brain regions of ApoE and control mice using a novel microarray system developed at Ambion followed by qRT PCR assays to validate differentially expressed miRNAs. These studies resulted in identification of a miRNA miR 182 that is differentially expressed in specific brain regions of the ApoE compared to control mice. The miR 182 molecule was dysregulated in all the brain regions tested cortex hypothalamus cerebellum and hippocampus with the most dramatic difference being 10 fold down regulation in cortex of the AD mouse as determined by qRT PCR . Computational analysis was used to identify candidate mRNA targets and the relevant gene product levels were assessed in AD mice. These efforts may lead to discovery of new RNA based biomarkers useful for diagnosis and treatment of AD.

Total RNA from the brains of patients with multiple sclerosis along with normal brains was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signals from the Multiple Sclerosis and normal samples were compared to identify differentially expressed miRNAs. The following miRNAs were expressed at higher levels in each of the multiple sclerosis brain samples miR 370 miR 30A miR 132 mu miR 298 miR 422A miR 31 and miR 155. The following miRNAs were expressed at lower levels in each of the multiple sclerosis brain samples mu miR 322 miR 326 miR 328 miR 9as miR 137 miR 126 miR 21 miR 181B and miR 218. Those miRNAs that are consistently seen to be down or up regulated in multiple sclerosis samples could be used to determine if a person has multiple sclerosis. Likewise these miRNAs represent potential targets for therapeutic development. Also the genes that are regulated by these miRNAs might provide effective targets for therapeutic development.

T cells are the primary component of both leukemia and lymphoma as well as HIV infection. Understanding these diseases requires understanding the molecular components of T cells. A significant amount is already known about T cell development. T cell development occurs in the thymus the thymic microenvironment directs differentiation as well as positive and negative selection. Lymphoid progenitors which have developed from hematopoietic stem cells in the bone marrow migrate to the thymus to complete their antigen independent maturation into functional T cells. In the thymus T cells develop their specific T cell markers including TCR CD3 CD4 or CD8 and CD2. When progenitor cells begin to express CD2 but have not yet rearranged their TCR genes CD2 CD3 they are double negative for CD4 and CD8 CD4 CD8 the markers for Th and Tc lineages. Of the double negative cells in the thymus about 20 have rearranged gd TCR about 20 have very homogenous ab TCR and 60 are committed to becoming the majority of mature ab T cells. These cells next express the adhesion molecule CD44 then the alpha chain of the IL 2 receptor CD25 . CD44CD25 double negative T cells rearrange TCR beta chain. beta chain rearrangement begins with D J joining followed by V DJ joining. The chances of successful b chain rearrangement are increased by the presence of two DJCb gene clusters. If rearrangement in the first cluster fails rearrangement in the second can occur

Productive rearrangement of beta chain is followed by its expression on the T cell membrane with CD3 and surrogate a chain pTa analogous to 15 in B cells . Signaling through the preT receptor causes the cells to stop rearranging b chain undergo a period of proliferation and begin to express both CD4 and CD8 becoming double positive T cells.

Using a mouse model of T cell development we compared double positive T cells to more mature CD4 cells. Total RNA from double positive and CD4 T cells was isolated using the glass fiber filter method described above. The total RNA was fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the double positive and CD4 samples. miRNAs that are expressed at more than two fold higher levels in the more mature CD4 cells are miR 181A miR 16 miR 15B miR 128A miR 17 5P let 7A let 7C and miR 106. One miRNA was expressed at more than two fold lower levels in the more mature CD4 cells miR 326.

Patients with chronic heart failure develop an enlargement of the heart called cardiac hypertrophy. The causes and effects of cardiac hypertrophy have been extensively documented but the underlying molecular mechanisms that link the molecular signals to cell changes or cardiac signaling pathways remain poorly understood. miRNAs that induce cardiac hypertrophy would supply targets for diagnostic or therapeutic intervention.

The miRNA expression profiles of two different hypertrophic mouse models was examined. The first mouse model was a knockout that eliminated the Gq coupled receptor signaling pathway which has been implicated in the cardiac hypertrophic response to stress. The second mouse model was a double knockout that eliminated both the Gq coupled receptor signaling pathway and the STE20 HGK pathway both of which have been implicated in cardiac hypertrophy.

Total RNA from the hearts of Gq and Gq HGK transgenics as well as normal mouse hearts was isolated using the glass fiber filter method described above. The total RNAs were fractionated by tube electrophoresis to recover the miRNAs. The miRNAs were fluorescently labeled with Cy3 or Cy5 using the two step fluorescent labeling process described above. The labeled miRNAs were hybridized to miRNA probes arrayed on glass slides as described above. The signal from the hybridized miRNAs was quantified using a GenePix 4000B Scanner Axon and the signals were compared to identify differentially expressed miRNAs. The relative signal intensities for each element were compared between the transgenic mouse samples and the normal heart samples. miRNAs that are expressed at more than two fold lower levels in the Gq transgenic hearts are miR 23B miR 326 miR 126 miR 133A miR 23A and miR 99B. Interestingly only miR 326 was also down regulated in the Gq HGK transgenic mice.

The previous examples involved purification of miRNA using a micro electrophoresis system that contains small electrode buffer chambers with a small volume of separating gel matrix each less than 0.5 ml . This purification relies on the fact that all desired species are at the small end of the size spectrum. This allows the collection of the sample simply by the removal of all lower electrode buffer after these species have migrated completely through the gel matrix. Species can be concentrated either by alcohol precipitation or immobilization on glass fiber filters in high alcohol concentrations covered in Ambion Patent App . Although this is similar to passing the sample through a filter with pores of a particulae nm size it is distinguishable in that the driving force is electrophoretic movement of the species rather than passage of the carrier fluid under atmospheric or centrifugal pressure. Also since the total thickness of the separation matrix being passed through is substantially greater 150 vs. 

To acomplish this the sample was mixed with an equal volume of denaturing solution containing 95 formamide to a volume of up to 100 l and applied under the upper electrode buffer 250 l to the top of the separating gel matrix. The gel was a polyacrylamide matrix in a urea solution. The use of denaturing conditions ensured both that the small RNAs were separated from any hybrids they form and that their migration rates reflected their size rather than their structure. The gel buffers were a standard Tris Borate EDTA system. By using a 10 polyacrylamide 9.33 monomer 0.67 methylene bis acrylamide gel matrix xylene cyanol dye can be used to track the position of RNA chains of about 40 nt so that standard procedure is to stop the gel as the dye begins to emerge. The samples were run at 60 80 V drawing a current of 3 mA for 12 15 min. The entire contents of the lower electrode chamber 250 l were then collected and the RNA species purified from it using either precipitation in 80 ethanol or capture on glass fiber filters in the presence of 75 ethanol 0.5 M guanidinium thiocyanate 3.75 mM CaCl2 16 mM NaAcetate pH 4.0. After ethanolic washes the sample were eluted in water at a temperature of 65 C.

All of the compositions and methods disclosed and claimed herein can be made and executed without undue experimentation in light of the present disclosure. While the compositions and methods of this invention have been described in terms of embodiments it will be apparent to those of skill in the art that variations may be applied to the compositions and methods and in the steps or in the sequence of steps of the methods described herein without departing from the concept spirit and scope of the invention. More specifically it will be apparent that certain agents that are both chemically and physiologically related may be substituted for the agents described herein while the same or similar results would be achieved. All such similar substitutes and modifications apparent to those skilled in the art are deemed to be within the spirit scope and concept of the invention as defined by the appended claims.

