---

title: Methods and compositions for diagnosing or monitoring autoimmune and chronic inflammatory diseases
abstract: Methods of diagnosing or monitoring an autoimmune or chronic inflammatory disease, particularly SLE in a patient by detecting the expression level of one or more genes or surrogates derived therefrom in the patient are described. Diagnostic oligonucleotides for diagnosing or monitoring chronic inflammatory disease, particularly SLE infection and kits or systems containing the same are also described.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08110364&OS=08110364&RS=08110364
owner: XDx, Inc.
number: 08110364
owner_city: Brisbane
owner_country: US
publication_date: 20101201
---
This application is a continuation of application Ser. No. 12 235 969 filed Sep. 23 2008 which is a divisional of application Ser. No. 10 990 298 filed Nov. 15 2004 now U.S. Pat. No. 7 579 148 which is a divisional of application Ser. No. 10 131 827 filed Apr. 24 2002 now U.S. Pat. No. 6 905 827 which is a continuation in part of application Ser. No. 10 006 290 filed Oct. 22 2001 now abandoned which claims the benefit of U.S. Provisional Application No. 60 296 764 filed Jun. 8 2001 all of which are hereby incorporated by reference in their entirety.

The content of the following submission on ASCII text file is incorporated herein by reference in its entirety a computer readable form CRF of the Sequence Listing file name 506612500101SeqList.txt date recorded Dec. 1 2010 size 1 709 KB .

This application is in the field of chronic inflammatory diseases. In particular this invention relates to methods and compositions for diagnosing or monitoring chronic inflammatory diseases.

Many of the current shortcomings in diagnosis prognosis risk stratification and treatment of disease can be approached through the identification of the molecular mechanisms underlying a disease and through the discovery of nucleotide sequences or sets of nucleotide sequences whose expression patterns predict the occurrence or progression of disease states or predict a patient s response to a particular therapeutic intervention. In particular identification of nucleotide sequences and sets of nucleotide sequences with such predictive value from cells and tissues that are readily accessible would be extremely valuable. For example peripheral blood is attainable from all patients and can easily be obtained at multiple time points at low cost. This is a desirable contrast to most other cell and tissue types which are less readily accessible or accessible only through invasive and aversive procedures. In addition the various cell types present in circulating blood are ideal for expression profiling experiments as the many cell types in the blood specimen can be easily separated if desired prior to analysis of gene expression. While blood provides a very attractive substrate for the study of diseases using expression profiling techniques and for the development of diagnostic technologies and the identification of therapeutic targets the value of expression profiling in blood samples rests on the degree to which changes in gene expression in these cell types are associated with a predisposition to and pathogenesis and progression of a disease.

There is an extensive literature supporting the role of leukocytes e.g. T and B lymphocytes monocytes and granulocytes including neutrophils in a wide range of disease processes including such broad classes as cardiovascular diseases inflammatory autoimmune and rheumatic diseases infectious diseases transplant rejection cancer and malignancy and endocrine diseases.

Of particular interest is the role of leukocytes and leukocyte gene expression in chronic inflammatory diseases such as Systemic Lupus Erythematosis and Rheumatoid Arthritis. Systemic lupus erythematosis SLE and Rheumatoid Arthritis RA are chonic autoimmune and inflammatory disorders characterized by dysregulation of the immune system which causes damage to a variety of organs. These diseases clearly involve differential expression of genes in leukocytes. Diagnostic and disease monitoring tools are severly lacking for these patients and their physicians. Leukocyte expression profiling can be applied to discover expression markers for SLE and RA and apply them as patient management tools in the clinical setting. In addition osteoarthirtis is a degenerative joint disease that can be confused with RA. This disease also involves leukocytes and expression profiling of leukocytes associated with osteoarthritis may lead to the discovery of new diagnostic and therapeutic approaches to the disease.

The accuracy of technologies based on expression profiling for the diagnosis prognosis and monitoring of disease would be dramatically increased if numerous differentially expressed nucleotide sequences each with a measure of sensitivity and specificity for a disease in question could be identified and assayed in a concerted manner. Using the expression of multiple genes gene sets for diagnostic applications helps overcome assay and population variability. In order to achieve this improved accuracy the appropriate sets of nucleotide sequences need to be identified and validated against numerous samples in combination with relevant clinical data.

In order to meet these needs the present invention identifies genes and gene sets that have clinical utility as diagnostic tools for the management of transplant recipients lupus patients and patients with a variety of chronic inflammatory and autoimmune diseases. The present invention is thus directed to a method of diagnosing or monitoring chronic autoimmune or inflammatory disease in a patient. The method of the invention involves detecting in a patient expression of one or more genes such as those genes depicted in Table 8 and Table 10 A and surrogates derived therefrom. Exemplary surrogates are provided in Table 10C. The present invention is further directed to a method of diagnosing or monitoring an autoimmune or chronic inflammatory disease in a patient by detecting the expression level of one or more genes or surrogates derived therefrom in said patient to diagnose or monitor the autoimmune or chronic inflammatory disease in the patient wherein said genes include a nucleotide sequence selected from SEQ ID NO 41 SEQ ID NO 328 SEQ ID NO 668 SEQ ID NO 855 SEQ ID NO 981 SEQ ID NO 1001 SEQ ID NO 1003 SEQ ID NO 1025 SEQ ID NO 1035 SEQ ID NO 1227 SEQ ID NO 1341 SEQ ID NO 1390 SEQ ID NO 1436 SEQ ID NO 1535 SEQ ID NO 1750 SEQ ID NO 2102 SEQ ID NO 2331 SEQ ID NO 2386 SEQ ID NO 2412 SEQ ID NO 2560 SEQ ID NO 2648 SEQ ID NO 2895 SEQ ID NO 3249 SEQ ID NO 3305 SEQ ID NO 3541 SEQ ID NO 3692 SEQ ID NO 3701 SEQ ID NO 3741 SEQ ID NO 3825 SEQ ID NO 3827 SEQ ID NO 3832 SEQ ID NO 4149 SEQ ID NO 4400 SEQ ID NO 4601 SEQ ID NO 4604 SEQ ID NO 4631 SEQ ID NO 4637 SEQ ID NO 5067 SEQ ID NO 5074 SEQ ID NO 5468 SEQ ID NO 5531 SEQ ID NO 5607 SEQ ID NO 6382 SEQ ID NO 6956 SEQ ID NO 7238 SEQ ID NO 7330 SEQ ID NO 7641 SEQ ID NO 8015 and SEQ ID NO 8095.

In the method of the invention the chronic inflammatory disease or autoimmune disease may be systemic lupus erythematosis SLE .

In the method of the invention RNA may be isolated from the patient prior to detecting expression of a gene such as those depicted in Table 10A. RNA levels may be detected by PCR hybridization. such as hybridization to an oligonucleotide. The nucleotide sequence may include comprises DNA cDNA PNA genomic DNA or synthetic oligonucleotides.

In the methods of the invention the RNA may be detected by hybridization to an oligonucleotide having a nucleotide sequence selected from SEQ ID NO 41 SEQ ID NO 328 SEQ ID NO 668 SEQ ID NO 855 SEQ ID NO 981 SEQ ID NO 1001 SEQ ID NO 1003 SEQ ID NO 1025 SEQ ID NO 1035 SEQ ID NO 1227 SEQ ID NO 1341 SEQ ID NO 1390 SEQ ID NO 1436 SEQ ID NO 1535 SEQ ID NO 1750 SEQ ID NO 2102 SEQ ID NO 2331 SEQ ID NO 2386 SEQ ID NO 2412 SEQ ID NO 2560 SEQ ID NO 2648 SEQ ID NO 2895 SEQ ID NO 3249 SEQ ID NO 3305 SEQ ID NO 3541 SEQ ID NO 3692 SEQ ID NO 3701 SEQ ID NO 3741 SEQ ID NO 3825 SEQ ID NO 3827 SEQ ID NO 3832 SEQ ID NO 4149 SEQ ID NO 4400 SEQ ID NO 4601 SEQ ID NO 4604 SEQ ID NO 4631 SEQ ID NO 4637 SEQ ID NO 5067 SEQ ID NO 5074 SEQ ID NO 5468 SEQ ID NO 5531 SEQ ID NO 5607 SEQ ID NO 6382 SEQ ID NO 6956 SEQ ID NO 7238 SEQ ID NO 7330 SEQ ID NO 7641 SEQ ID NO 8015 and SEQ ID NO 8095.

The present invention is further directed to a diagnostic oligonucleotide for detecting chronic or inflammatory disease wherein the oligonucleotide has a nucleotide sequence selected from SEQ ID NO 4637 The diagnostic oligonucleotide of may include DNA cDNA PNA genomic DNA or synthetic oligonucleotides.

The present invention is further directed to a system or kit for diagnosing or monitoring chronic inflammatory or autoimmune disease in a patient comprising an isolated DNA molecule wherein the isolated DNA molecule detects expression of a gene listed in Table 10A. In the system of the invention the DNA molecules may be synthetic DNA genomic DNA PNA or cDNA. The isolated DNA molecule may be immobilized on an array. Such arrays may include a chip array a plate array a bead array a pin array a membrane array a solid surface array a liquid array an oligonucleotide array polynucleotide array or a cDNA array a microtiter plate a membrane and a chip.

The present invention is further directed to a system or detecting differential gene expression. In one format the system has one or more isolated DNA molecules wherein each isolated DNA molecule detects expression of a gene selected from the group of genes corresponding to the oligonucleotides depicted in the Sequence Listing. It is understood that the DNA sequences and oligonucleotides of the invention may have slightly different sequences than those identified herein. Such sequence variations are understood to those of ordinary skill in the art to be variations in the sequence which do not significantly affect the ability of the sequences to detect gene expression.

The sequences encompassed by the invention have at least 40 50 50 60 70 80 80 85 85 90 90 95 or 95 100 sequence identity to the sequences disclosed herein. In some embodiments DNA molecules are less than about any of the following lengths in bases or base pairs 10 000 5 000 2500 2000 1500 1250 1000 750 500 300 250 200 175 150 125 100 75 50 25 10. In some embodiments DNA molecule is greater than about any of the following lengths in bases or base pairs 10 15 20 25 30 40 50 60 75 100 125 150 175 200 250 300 350 400 500 750 1000 2000 5000 7500 10000 20000 50000. Alternately a DNA molecule can be any of a range of sizes having an upper limit of 10 000 5 000 2500 2000 1500 1250 1000 750 500 300 250 200 175 150 125 100 75 50 25 or 10 and an independently selected lower limit of 10 15 20 25 30 40 50 60 75 100 125 150 175 200 250 300 350 400 500 750 1000 2000 5000 7500 wherein the lower limit is less than the upper limit.

The gene expression system may be a candidate library a diagnostic agent a diagnostic oligonucleotide set or a diagnostic probe set. The DNA molecules may be genomic DNA protein nucleic acid PNA cDNA or synthetic oligonucleotides.

In one format the gene expression system is immobilized on an array. The array may be a chip array a plate array a bead array a pin array a membrane array a solid surface array a liquid array an oligonucleotide array a polynucleotide array a cDNA array a microfilter plate a membrane or a chip.

A brief description of the sequence listing is given below. There are 9090 entries. The Sequence Listing presents 50mer oligonucleotide sequences derived from human leukocyte plant and viral genes. These are listed as SEQ IDs 1 8143. The 50mer sequences and their sources are also displayed in Table 8. Most of these 50mers were designed from sequences of genes in Tables 2 3A B and C and the Sequence listing.

SEQ IDs 8144 8766 are the cDNA sequences derived from human leukocytes that were not homologous to UniGene sequences or sequences found in dbEST at the time they were searched. Some of these sequences match human genomic sequences and are listed in Tables 3B and C. The remaining clones are putative cDNA sequences that contained less than 50 masked nucleotides when submitted to RepeatMasker were longer than 147 nucleotides and did not have significant similarity to the UniGene Unique database dbEST the NR nucleotide database of Genbank or the assembled human genome of Genbank.

SEQ IDs 8767 8770 8828 8830 and 8832 are sequences that appear in the specification primer masked sequences exemplary sequences etc. .

SEQ IDs 8845 8893 are the full length gene sequences for the genes identified by an accession number in Table 10A.

Table 2 Table 2 describes genes and other nucleotide sequences identified using data mining of publically available publication databases and nucleotide sequence databases. Corresponding Unigene build 133 cluster numbers are listed with each gene or other nucleotide sequence.

Table 3A Table 3A describes differentially expressed nucleotide sequences useful for the prediction of clinical outcomes. This table contains 4517 identified cDNAs and cDNA regions of genes that are members of a leukocyte candidate library for use in measuring the expression of nucleotide sequences that could subsequently be correlated with human clinical conditions. The regions of similarity were found by searching three different databases for pair wise similarity using blastn. The three databases were UniGene Unique build Mar. 30 2001 file Hs.seq.uniq.Z the downloadable database located at the website ftp.ncbi.nlm.nih.com blast db est human.Z with date Apr. 8 2001 which is a section of Genbank version 122 and the non redundant section of Genbank ver 123. The Hs.XXXXX numbers represent UniGene accession numbers from the Hs.seq.uniq.Z file of Mar. 30 2001. The clone sequences are not in the sequence listing.

Table 3B Table 3B describes Identified Genomic Regions that code for novel mRNAs. The table contains 591 identified genomic regions that are highly similar to the cDNA clones. Those regions that are within 100 to 200 Kb of each other on the same contig are likely to represent exons of the same gene. The indicated clone is exemplary of the cDNA clones that match the indicated genomic region. The number clones column indicates how many clones were isolated from the libraries that are similar to the indicated region of the chromosome. The probability number is the likelihood that region of similarity would occur by chance on a random sequence. The Accession numbers are from the Mar. 15 2001 build of the human genome. The file date for the downloaded data was Apr. 17 2001. These sequences may prove useful for the prediction of clinical outcomes.

Table 3C Table 3C describes 48 clones whose sequences align to two or more non contiguous sequences on the same assembled human contig of genomic sequence. The Accession numbers are from the Mar. 15 2001 build of the human genome. The file date for the downloaded data was Apr. 17 2001. The alignments of the clone and the contig are indicated in the table. The start and stop offset of each matching region is indicated in the table. The sequence of the clones themselves is included in the sequence listing. The alignments of these clones strongly suggest that they are novel nucleotide sequences. Furthermore no EST or mRNA aligning to the clone was found in the database. These sequences may prove useful for the prediction of clinical outcomes.

Table 4 Database mining. The Library Browser at the NCBI UniGene web site was used to identify genes that are specifically expressed in leukocyte cell populations. The table lists the library name and type the number of sequences in each library and the number used for the array.

Table 5 Table 5 describes the nucleotide sequence databases used in the sequence analysis described herein.

Table 6 Table 6 describes the algorithms and software packages used for exon and polypeptide prediction used in the sequence analysis described herein.

Table 7 Table 7 describes the databases and algorithms used for the protein sequence analysis described herein.

Table 8 Table 8 provides a listing of all oligonucleotides designed for the arrays and their associated genes. In this table the sequence ID is given which corresponds to the sequence listing. The origin of the sequence for inclusion on the array is noted as coming from one of the cDNA libraries described in example 1 mining from databases as described in examples 2 and 11 or identification from the published literature. The unigene number genebank accession and GI number are also given for each sequence when known. These data were obtained from the Unigene unique database build 137. The name of the gene associated with the accession number is noted. The sequence of these genes as available from the databases are hereby incorporated by reference in their entirety. The strand is noted as 1 or 1 meaning that the probe was designed from the complement of the sequence 1 or directly from the sequence 1 . The nucleotide sequence of each probe is also given in the Sequence Listing.

Table 9 Table 9 shows viral genes for arrays. Viral genomes were used to design oligonucleotides for the microarrays. The accession numbers for the viral genomes used are given along with the gene name and location of the region used for oligonucleotide design.

Table 10A. Table 10A shows Lupus gene expression markers. This table lists the oligonucleotides and associated genes identified as having value for the diagnosis and monitoring of lupus. The first column gives the SEQ ID that corresponds to the oligonucleotide in the sequence listing. The origin of the sequence for inclusion on the array is noted as coming from one of the cDNA libraries described in example 1 mining from databases as described in examples 2 and 11 or identification from the published literature. The unigene number genebank accession and GI number are also given for each sequence when known. The SEQ ID for the sequence listing for the full length genes corresponding to the accession numbers in the table are also given SEQ ID ACC . These data were obtained from the Unigene unique database build . The name of the gene associated with the accession number is noted. The strand is noted as 1 or 1 meaning that the probe was designed from the complement of the sequence 1 or directly from the sequence 1 . Next the nucleotide sequence of each probe is also given. For each gene the false detection rate FDR from the significance analsysis described in example 10 is given if applicable. Also those genes that were identified by CART as a diagnostic gene are noted with the model and position in the model see example 10 and .

Table 10B. Table 10B shows primers for PCR. For each of the lupus gene expression markers identified in Table 10A 2 sets of PCR primer pairs are shown that were derived by the methods described in example 15. The melting temperature Tm for each primer is shown as is the corresponding SEQ ID number for the primer in the sequence listing.

Table 10C. Table 10C shows surrogates for the lupus gene expression markers disclosed herein. For some of the lupus marker genes identified in Table 10A genes are identified by the SEQ ID number as surrogates. The surrogates are identified as such by the CART algorithm or by hierarchical clustering as detailed below.

Unless defined otherwise all scientific and technical terms are understood to have the same meaning as commonly used in the art to which they pertain. For the purpose of the present invention the following terms are defined below.

In the context of the invention the term gene expression system refers to any system device or means to detect gene expression and includes diagnostic agents candidate libraries oligonucleotide oligonucleotide sets or probe sets.

The terms diagnostic oligonucleotide or diagnostic oligonucleotide set generally refers to an oligonucleotide or to a set of two or more oligonucleotides that when evaluated for differential expression their corresponding diagnostic genes collectively yields predictive data. Such predictive data typically relates to diagnosis prognosis monitoring of therapeutic outcomes and the like. In general the components of a diagnostic oligonucleotide or a diagnostic oligonucleotide set are distinguished from oligonucleotide sequences that are evaluated by analysis of the DNA to directly determine the genotype of an individual as it correlates with a specified trait or phenotype such as a disease in that it is the pattern of expression of the components of the diagnostic oligonucleotide set rather than mutation or polymorphism of the DNA sequence that provides predictive value. It will be understood that a particular component or member of a diagnostic oligonucleotide set can in some cases also present one or more mutations or polymorphisms that are amenable to direct genotyping by any of a variety of well known analysis methods e.g. Southern blotting RFLP AFLP SSCP SNP and the like.

A diagnostic gene is a gene whose expression is detected by a diagnostic oligonucleotide or diagnostic oligonucleotide set.

A disease specific target oligonucleotide sequence is a gene or other oligonucleotide that encodes a polypeptide most typically a protein or a subunit of a multi subunit protein that is a therapeutic target for a disease or group of diseases.

A candidate library or a candidate oligonucleotide library refers to a collection of oligonucleotide sequences or gene sequences that by one or more criteria have an increased probability of being associated with a particular disease or group of diseases. The criteria can be for example a differential expression pattern in a disease state or in activated or resting leukocytes in vitro as reported in the scientific or technical literature tissue specific expression as reported in a sequence database differential expression in a tissue or cell type of interest or the like. Typically a candidate library has at least 2 members or components more typically the library has in excess of about 10 or about 100 or about 1000 or even more members or components.

The term disease criterion is used herein to designate an indicator of a disease such as a diagnostic factor a prognostic factor a factor indicated by a medical or family history a genetic factor or a symptom as well as an overt or confirmed diagnosis of a disease associated with several indicators such as those selected from the above list. A disease criterian includes data describing a patient s health status including retrospective or prospective health data e.g. in the form of the patient s medical history laboratory test results diagnostic test result clinical events medications lists response s to treatment and risk factors etc.

An autoimmune disorder is defined as a disease state in which a patient s immune system recognizes an antigen in that patient s organs or tissues as foreign and becomes activated. The activated immune cells can then cause damage to the inciting organ or tissue or can damage other organs or tissues. In some cases the disorder may be caused by a dysregulation of the immune system cells rather than by the recognition as a self antigen as foreign. Dysregulated immune cells can secrete inflammatory cytokines that cause systemic inflammation or they can recognize self antigens as foreign.

Examples of autoimmune diseases include Autoimmune hepatitis Multiple Sclerosis Myasthenia Gravis Type I diabetes Rheumatoid Arthritis Psoriasis Systemic Lupus Erythematosis Hashimoto s Thyroiditis Grave s disease Ankylosing Spondylitis Sjogrens Disease CREST syndrome Scleroderma and many more.

Most of the autoimmune diseases are also chronic inflammatory diseases. This is defined as a disease process associated with long term 6 months activation of inflammatory cells leukocytes . The chronic inflammation leads to damage of patient organs or tissues. Many diseases are chronic inflammatory disorders but are not know to have an autoimmune basis. For example Atherosclerosis Congestive Heart Failure Crohn s disease Ulcerative Colitis Polyarteritis nodosa Whipple s Disease Primary Sclerosing Cholangitis and many more.

The terms molecular signature or expression profile refers to the collection of expression values for a plurality e.g. at least 2 but frequently about 10 about 100 about 1000 or more of members of a candidate library. In many cases the molecular signature represents the expression pattern for all of the nucleotide sequences in a library or array of candidate or diagnostic nucleotide sequences or genes. Alternatively the molecular signature represents the expression pattern for one or more subsets of the candidate library. The term oligonucleotide refers to two or more nucleotides. Nucleotides may be DNA or RNA naturally occurring or synthetic.

The term healthy individual as used herein is relative to a specified disease or disease criterion. That is the individual does not exhibit the specified disease criterion or is not diagnosed with the specified disease. It will be understood that the individual in question can of course exhibit symptoms or possess various indicator factors for another disease.

Similarly an individual diagnosed with a disease refers to an individual diagnosed with a specified disease or disease criterion . Such an individual may or may not also exhibit a disease criterion associated with or be diagnosed with another related or unrelated disease.

The term monitoring is used herein to describe the use of gene sets to provide useful information about an individual or an individual s health or disease status. Monitoring can include determination of prognosis risk stratification selection of drug therapy assessment of ongoing drug therapy prediction of outcomes determining response to therapy diagnosis of a disease or disease complication following progression of a disease or providing any information relating to a patients health status.

An array is a spatially or logically organized collection e.g. of oligonucleotide sequences or nucleotide sequence products such as RNA or proteins encoded by an oligonucleotide sequence. In some embodiments an array includes antibodies or other binding reagents specific for products of a candidate library.

When referring to a pattern of expression a qualitative difference in gene expression refers to a difference that is not assigned a relative value. That is such a difference is designated by an all or nothing valuation. Such an all or nothing variation can be for example expression above or below a threshold of detection an on off pattern of expression . Alternatively a qualitative difference can refer to expression of different types of expression products e.g. different alleles e.g. a mutant or polymorphic allele variants including sequence variants as well as post translationally modified variants etc.

In contrast a quantitative difference when referring to a pattern of gene expression refers to a difference in expression that can be assigned a value on a graduated scale e.g. a 0 5 or 1 10 scale a scale a grade 1 grade 5 scale or the like it will be understood that the numbers selected for illustration are entirely arbitrary and in no way are meant to be interpreted to limit the invention .

The invention is directed to methods of detecting gene expression with a gene expression system having one or more DNA molecules wherein the one or more DNA molecules has a nucleotide sequence which detects expression of a gene corresponding to the oligonucleotides depicted in the Sequence Listing. In one format the oligonucleotide detects expression of a gene that is differentially expressed in leukocytes. The gene expression system may be a candidate library a diagnostic agent a diagnostic oligonucleotide set or a diagnostic probe set. The DNA molecules may be genomic DNA RNA protein nucleic acid PNA cDNA or synthetic oligonucleotides. Following the procedures taught herein one can identity sequences of interest for analyzing gene expression in leukocytes. Such sequences may be predictive of a disease state.

The invention relates to diagnostic oligonucleotides and diagnostic oligonucleotide set s comprising members of the leukocyte candidate library listed in Table 2 Table 3 and Tables 8 10 in the Sequence Listing for which a correlation exists between the health status of an individual and the individual s expression of RNA or protein products corresponding to the nucleotide sequence. In some instances only one oligonucleotide is necessary for such detection. Members of a diagnostic oligonucleotide set may be identified by any means capable of detecting expression of RNA or protein products including but not limited to differential expression screening PCR RT PCR SAGE analysis high throughput sequencing microarrays liquid or other arrays protein based methods e.g. western blotting proteomics and other methods described herein and data mining methods as further described herein.

In one embodiment a diagnostic oligonucleotide set comprises at least two oligonucleotide sequences listed in Table 2 Table 3 and Tables 8 10 or the Sequence Listing which are differentially expressed in leukocytes in an individual with at least one disease criterion for at least one leukocyte implicated disease relative to the expression in individual without the at least one disease criterion wherein expression of the two or more nucleotide sequences is correlated with at least one disease criterion as described below.

In another embodiment a diagnostic oligonucleotide set comprises at least one oligonucleotide having an oligonucleotide sequence listed in Table 2 3 and Tables 8 10 or the Sequence Listing which is differentially expressed and further wherein the differential expression correlation has not previously been described. In some embodiments the diagnostic oligonucleotide set is immobilized on an array.

In another embodiment diagnostic oligonucleotides or oligonucleotide sets are related to the members of the leukocyte candidate library listed in Table 2 Table 3 Tables 8 10 and in the Sequence Listing for which a correlation exists between the health status or disease criterion of an individual. The diagnostic oligonucleotides are partially or totally contained in or derived from full length gene sequences or predicted full length gene sequences for the members of the candidate library listed in Table 2 3 and the Sequence Listing. This includes sequences from accession numbers and unigene numbers from Table 8. Table 8 shows the accession and unigene number when known for each oligonucleotide used on the 8134 gene leukocyte array described in examples 11 13. In some cases oligonucleotide sequences are designed from EST or Chromosomal sequences from a public database. In these cases the full length gene sequences may not be known. Full length sequences in these cases can be predicted using gene prediction algorithms Examples 4 6 . Alternatively the full length can be determined by cloning and sequencing the full length gene or genes that contain the sequence of interest using standard molecular biology approaches described here. The same is true for olignonucleotides designed from our sequencing of cDNA libraries see Examples 1 4 where the cDNA does not match any sequence in the public databases.

The diagnostic oligonucleotides may also be derived from other genes that are coexpressed with the correlated sequence or full length gene. Genes may share expression patterns because they are regulated in the same molecular pathway. Because of the similarity of expression behavior genes are identified as surrogates in that they can substitute for a diagnostic gene in a diagnostic gene set. Example 10 demonstrates the discovery of surrogates from the data and Table 10C and the sequence listing identify and give the sequence for surrogates for lupus diagnostic genes. Surrogate oligonucleotide and surrogate oligonucleotide sets can be utilized to detect expression of surrogate genes and thereby diagnose or monitor patients with a disease.

As used herein the term gene cluster or cluster refers to a group of genes related by expression pattern. In other words a cluster of genes is a group of genes with similar regulation across different conditions such as a patient having a chronic autoimmune or inflammatory disease or a patient without chronic autoimmune or inflammatory disease. The expression profile for each gene in a cluster should be correlated with the expression profile of at least one other gene in that cluster. Correlation may be evaluated using a variety of statistical methods. As used herein the term surrogate refers to a gene with an expression profile such that it can substitute for a diagnostic gene in a diagnostic assay. Such genes are often members of the same gene cluster as the diagnostic gene. For each member of a diagnostic gene set a set of potential surrogates can be identified through identification of genes with similar expression patterns as described below.

Many statistical analyses produce a correlation coefficient to describe the relatedness between two gene expression patterns. Patterns may be considered correlated if the correlation coefficient is greater than or equal to 0.8. In preferred embodiments the correlation coefficient should be greater than 0.85 0.9 or 0.95. Other statistical methods produce a measure of mutual information to describe the relatedness between two gene expression patterns. Patterns may be considered correlated if the normalized mutual information value is greater than or equal to 0.7. In preferred embodiments the normalized mutual information value should be greater than 0.8 0.9 or 0.95. Patterns may also be considered similar if they cluster closely upon hierarchical clustering of gene expression data Eisen et al. 1998 . Similar patterns may be those genes that are among the 1 2 5 10 20 50 or 100 nearest neighbors in a hierarchical clustering or have a similarity score Eisen et al. 1998 of 0.5 0.7 0.8 0.9 0.95 or 0.99. Similar patterns may also be identified as those genes found to be surrogates in a classification tree by CART Breiman et al. 1994 . Often but not always members of a gene cluster have similar biological functions in addition to similar gene expression patterns.

Correlated genes clusters and surrogates are identified for the diagnostic genes of the invention. These surrogates may be used as diagnostic genes in an assay instead of or in addition to the diagnostic genes for which they are surrogates.

The invention also provides diagnostic probe sets. It is understood that a probe includes any reagent capable of specifically identifying a nucleotide sequence of the diagnostic nucleotide set including but not limited to amplified DNA amplified RNA cDNA synthetic oligonucleotide partial or full length nucleic acid sequences. In addition the probe may identify the protein product of a diagnostic nucleotide sequence including for example antibodies and other affinity reagents.

It is also understood that each probe can correspond to one gene or multiple probes can correspond to one gene or both or one probe can correspond to more than one gene.

Homologs and variants of the disclosed nucleic acid molecules may be used in the present invention. Homologs and variants of these nucleic acid molecules will possess a relatively high degree of sequence identity when aligned using standard methods. The sequences encompassed by the invention have at least 40 50 50 60 70 80 80 85 85 90 90 95 or 95 100 sequence identity to the sequences disclosed herein.

It is understood that for expression profiling variations in the disclosed sequences will still permit detection of gene expression. The degree of sequence identity required to detect gene expression varies depending on the length of the oligomer. For a 60 mer an oligonucleotide with 60 nucleotides 6 8 random mutations or 6 8 random deletions in a 60 mer do not affect gene expression detection. Hughes T R et al. Expression profiling using microarrays fabricated by an ink jet oligonucleotide synthesizer. Nature Biotechnology 19 343 347 2001 . As the length of the DNA sequence is increased the number of mutations or deletions permitted while still allowing gene expression detection is increased.

As will be appreciated by those skilled in the art the sequences of the present invention may contain sequencing errors. That is there may be incorrect nucleotides frameshifts unknown nucleotides or other types of sequencing errors in any of the sequences however the correct sequences will fall within the homology and stringency definitions herein.

The minimum length of an oligonucleotide probe necessary for specific hybridization in the human genome can be estimated using two approaches. The first method uses a statistical argument that the probe will be unique in the human genome by chance. Briefly the number of independent perfect matches Po expected for an oligonucleotide of length L in a genome of complexity C can be calculated from the equation Laird C D Chromosoma 32 378 1971 2

In the case of mammalian genomes 2C 3.6 10 and an oligonucleotide of 14 15 nucleotides is expected to be represented only once in the genome. However the distribution of nucleotides in the coding sequence of mammalian genomes is nonrandom Lathe R. J. Mol. Biol. 183 1 1985 and longer oligonucleotides may be preferred in order to in increase the specificity of hybridization. In practical terms this works out to probes that are 19 40 nucleotides long Sambrook J et al. infra . The second method for estimating the length of a specific probe is to use a probe long enough to hybridize under the chosen conditions and use a computer to search for that sequence or close matches to the sequence in the human genome and choose a unique match. Probe sequences are chosen based on the desired hybridization properties as described in Chapter 11 of Sambrook et al infra The PRIMER3 program is useful for designing these probes S. Rozen and H. Skaletsky 1996 1997 Primer3 code available at genome.wi.mitedu genome software other primer3.html the website . The sequences of these probes are then compared pair wise against a database of the human genome sequences using a program such as BLAST or MEGABLAST Madden T. L et al. 1996 Meth. Enzymol. 266 131 141 . Since most of the human genome is now contained in the database the number of matches will be determined. Probe sequences are chosen that are unique to the desired target sequence.

In some embodiments a diagnostic oligonucleotide or oligonucleotide probe set is immobilized on an array. The array is optionally comprises one or more of a chip array a plate array a bead array a pin array a membrane array a solid surface array a liquid array an oligonucleotide array a polynucleotide array or a cDNA array a microtiter plate a pin array a bead array a membrane or a chip.

In some embodiments the leukocyte implicated disease is selected from the diseases listed in Table 1. In other embodiments the disease is atherosclerosis or cardiac allograft rejection. In other embodiments the disease is congestive heart failure angina myocardial infarction chronic autoimmune and inflammatory diseases systemic lupus erythematosis SLE and rheumatoid arthritis.

In some embodiments diagnostic oligonucleotides of the invention are used as a diagnostic gene set in combination with genes that are know to be associated with a disease state known markers . The use of the diagnostic oligonucleotides in combination with the known markers can provide information that is not obtainable through the known markers alone. The known markers include those identified by the prior art listing provided.

In the context of the invention nucleic acids and or proteins are manipulated according to well known molecular biology techniques. Detailed protocols for numerous such procedures are described in e.g. in Ausubel et al. supplemented through 2000 John Wiley Sons New York Ausubel Sambrook et al. 2nd Ed. Vol. 1 3 Cold Spring Harbor Laboratory Cold Spring Harbor N.Y. 1989 Sambrook and Berger and Kimmel volume 152 Academic Press Inc. San Diego Calif. Berger .

In addition to the above references protocols for in vitro amplification techniques such as the polymerase chain reaction PCR the ligase chain reaction LCR Q replicase amplification and other RNA polymerase mediated techniques e.g. NASBA useful e.g. for amplifying cDNA probes of the invention are found in Mullis et al. 1987 U.S. Pat. No. 4 683 202 Innis et al. eds Academic Press Inc. San Diego Calif. 1990 Innis Arnheim and Levinson 1990 36 1991 3 81 Kwoh et al. 1989 86 1173 Guatelli et al. 1990 87 1874 Lomell et al. 1989 35 1826 Landegren et al. 1988 241 1077 Van Brunt 1990 8 291 Wu and Wallace 1989 4 560 Barringer et al. 1990 89 117 and Sooknanan and Malek 1995 13 563. Additional methods useful for cloning nucleic acids in the context of the present invention include Wallace et al. U.S. Pat. No. 5 426 039. Improved methods of amplifying large nucleic acids by PCR are summarized in Cheng et al. 1994 369 684 and the references therein.

Certain polynucleotides of the invention e.g. oligonucleotides can be synthesized utilizing various solid phase strategies involving mononucleotide and or trinucleotide based phosphoramidite coupling chemistry. For example nucleic acid sequences can be synthesized by the sequential addition of activated monomers and or trimers to an elongating polynucleotide chain. See e.g. Caruthers M. H. et al. 1992 211 3.

In lieu of synthesizing the desired sequences essentially any nucleic acid can be custom ordered from any of a variety of commercial sources such as The Midland Certified Reagent Company mcrc oligos.com The Great American Gene Company genco.com ExpressGen Inc. expressgen.com Operon Technologies Inc. operon.com and many others.

Similarly commercial sources for nucleic acid and protein microarrays are available and include e.g. Agilent Technologies Palo Alto Calif. and Affymetrix Santa Clara Calif.

Libraries of candidate genes that are differentially expressed in leukocytes are substrates for the identification and evaluation of diagnostic oligonucleotides and oligonucleotide sets and disease specific target nucleotide sequences.

The term leukocyte is used generically to refer to any nucleated blood cell that is not a nucleated erythrocyte. More specifically leukocytes can be subdivided into two broad classes. The first class includes granulocytes including most prevalently neutrophils as well as eosinophils and basophils at low frequency. The second class the non granular or mononuclear leukocytes includes monocytes and lymphocytes e.g. T cells and B cells . There is an extensive literature in the art implicating leukocytes e.g. neutrophils monocytes and lymphocytes in a wide variety of disease processes including inflammatory and rheumatic diseases neurodegenerative diseases such as Alzheimer s dementia cardiovascular disease endocrine diseases transplant rejection malignancy and infectious diseases and other diseases listed in Table 1. Mononuclear cells are involved in the chronic immune response while granulocytes which make up approximately 60 of the leukocytes have a non specific and stereotyped response to acute inflammatory stimuli and often have a life span of only 24 hours.

In addition to their widespread involvement and or implication in numerous disease related processes leukocytes are particularly attractive substrates for clinical and experimental evaluation for a variety of reasons. Most importantly they are readily accessible at low cost from essentially every potential subject. Collection is minimally invasive and associated with little pain disability or recovery time. Collection can be performed by minimally trained personnel e.g. phlebotomists medical technicians etc. in a variety of clinical and non clinical settings without significant technological expenditure. Additionally leukocytes are renewable and thus available at multiple time points for a single subject.

At least two conceptually distinct approaches to the assembly of candidate libraries exist. Either or both or other approaches can be favorably employed. The method of assembling or identifying candidate libraries is secondary to the criteria utilized for selecting appropriate library members. Most importantly library members are assembled based on differential expression of RNA or protein products in leukocyte populations. More specifically candidate nucleotide sequences are induced or suppressed or expressed at increased or decreased levels in leukocytes from a subject with one or more disease or disease state a disease criterion relative to leukocytes from a subject lacking the specified disease criterion. Alternatively or in addition library members can be assembled from among nucleotide sequences that are differentially expressed in activated or resting leukocytes relative to other cell types.

Firstly publication and sequence databases can be mined using a variety of search strategies. For example currently available scientific and medical publication databases such as Medline Current Contents OMIM online Mendelian inheritance in man various Biological and Chemical Abstracts Journal indexes and the like can be searched using term or key word searches or by author title or other relevant search parameters. Many such databases are publicly available and one of skill is well versed in strategies and procedures for identifying publications and their contents e.g. genes other nucleotide sequences descriptions indications expression pattern etc. Numerous databases are available through the internet for free or by subscription see e.g. the websites ncbi.nlm.nih.gov PubMed 3.infotrieve.com isinet.com sciencemag.org . Additional or alternative publication or citation databases are also available that provide identical or similar types of information any of which are favorably employed in the context of the invention. These databases can be searched for publications describing differential gene expression in leukocytes between patients with and without diseases or conditions listed in Table 1. We identified the nucleotide sequences listed in Table 2 and some of the sequences used to design oligonucleotides for microarrays sequence listing using data mining methods.

Alternatively a variety of publicly available and proprietary sequence databases including GenBank dbEST UniGene and TIGR and SAGE databases including sequences corresponding to expressed nucleotide sequences such as expressed sequence tags ESTs are available. For example the Genbank website located at ncbi.nlm.nih.gov Genbank among others can be readily accessed and searched via the internet. These and other sequence and clone database resources are currently available however any number of additional or alternative databases comprising nucleotide sequence sequences EST sequences clone repositories PCR primer sequences and the like corresponding to individual nucleotide sequence sequences are also suitable for the purposes of the invention. Nucleotide sequences can be identified that are only found in libraries derived from leukocytes or sub populations of leukocytes for example see Table 2 and Example 2.

Alternatively the representation or relative frequency of a nucleotide sequence may be determined in a leukocyte derived nucleic acid library and compared to the representation of the sequence in non leukocyte derived libraries. The representation of a nucleotide sequence correlates with the relative expression level of the nucleotide sequence in leukocytes and non leukocytes. An oligonucleotide sequence that has increased or decreased representation in a leukocyte derived nucleic acid library relative to a non leukocyte derived libraries is a candidate for a leukocyte specific gene.

Nucleotide sequences identified as having specificity to activated or resting leukocytes or to leukocytes from patients or patient samples with a variety of disease types can be isolated for use in a candidate library for leukocyte expression profiling through a variety of mechanisms. These include but are not limited to the amplification of the nucleotide sequence from RNA or DNA using nucleotide sequence specific primers for PCR or RT PCR isolation of the nucleotide sequence using conventional cloning methods the purchase of an IMAGE consortium cDNA clone EST with complimentary sequence or from the same expressed nucleotide sequence design of oligonucleotides preparation of synthetic nucleic acid sequence or any other nucleic acid based method. In addition the protein product of the nucleotide sequence can be isolated or prepared and represented in a candidate library using standard methods in the art as described further below.

While the above discussion related primarily to genomics approaches it is appreciated that numerous analogous proteomics approaches are suitable to the present invention. For example a differentially expressed protein product can for example be detected using western analysis two dimensional gel analysis chromatographic separation mass spectrometric detection protein fusion reporter constructs colorometric assays binding to a protein array or by characterization of polysomal mRNA. The protein is further characterized and the nucleotide sequence encoding the protein is identified using standard techniques e.g. by screening a cDNA library using a probe based on protein sequence information.

The second approach involves the construction of a differential expression library by any of a variety of means. Any one or more of differential screening differential display or subtractive hybridization procedures or other techniques that preferentially identify isolate or amplify differentially expressed nucleotide sequences can be employed to produce a library of differentially expressed candidate nucleotide sequences a subset of such a library a partial library or the like. Such methods are well known in the art. For example peripheral blood leukocytes i.e. a mixed population including lymphocytes monocytes and neutrophils from multiple donor samples are pooled to prevent bias due to a single donor s unique genotype. The pooled leukocytes are cultured in standard medium and stimulated with individual cytokines or growth factors e.g. with IL 2 IL 1 MCP1 TNF and or IL8 according to well known procedures see e.g. Tough et al. 1999 Winston et al. 1999 Hansson et al. 1989 . Typically leukocytes are recovered from Buffy coat preparations produced by centrifugation of whole blood. Alternatively mononuclear cells monocytes and lymphocytes can be obtained by density gradient centrifugation of whole blood or specific cell types such as a T lymphocyte can be isolated using affinity reagents to cell specific surface markers. When affinity reagents are used to isolate specific cell types it is desirable to isolate the cells using negative selection to avoid activation of the desired cell type by binding of the antibody. Leukocytes may also be stimulated by incubation with ionomycin and phorbol myristate acetate PMA . This stimulation protocol is intended to non specifically mimic activation of numerous pathways due to variety of disease conditions rather than to simulate any single disease condition or paradigm.

Using well known subtractive hybridization procedures as described in e.g. U.S. Pat. Nos. 5 958 738 5 589 339 5 827 658 5 712 127 5 643 761 each of which are hereby incorporated by reference a library is produced that is enriched for RNA species messages that are differentially expressed between test and control leukocyte populations. In some embodiments the test population of leukocytes are simply stimulated as described above to emulate non specific activation events while in other embodiments the test population can be selected from subjects or patients with a specified disease or class of diseases. Typically the control leukocyte population lacks the defining test condition e.g. stimulation disease state diagnosis genotype etc. Alternatively the total RNA from control and test leukocyte populations are prepared by established techniques treated with DNAseI and selected for messenger RNA with an intact 3 end i.e. polyA messenger RNA e.g. using commercially available kits according to the manufacturer s instructions e.g. Clontech. Double stranded cDNA is synthesized utilizing reverse transcriptase. Double stranded cDNA is then cut with a first restriction enzyme e.g. NlaIII that cuts at the recognition site CATG and cuts the cDNA sequence at approximately 256 bp intervals that cuts the cDNA molecules into conveniently sized fragments.

The cDNAs prepared from the test population of leukocytes are divided into typically 2 tester pools while cDNAs prepared from the control population of leukocytes are designated the driver pool. Typically pooled populations of cells from multiple individual donors are utilized and in the case of stimulated versus unstimulated cells the corresponding tester and driver pools for any single subtraction reaction are derived from the same donor pool.

A unique double stranded adapter is ligated to each of the tester cDNA populations using unphosphorylated primers so that only the sense strand is covalently linked to the adapter. An initial hybridization is performed consisting of each of the tester pools of cDNA each with its corresponding adapter and an excess of the driver cDNA. Typically an excess of about 10 100 fold driver relative to tester is employed although significantly lower or higher ratios can be empirically determined to provide more favorable results. The initial hybridization results in an initial normalization of the cDNAs such that high and low abundance messages become more equally represented following hybridization due to a failure of driver tester hybrids to amplify.

A second hybridization involves pooling un hybridized sequences from initial hybridizations together with the addition of supplemental driver cDNA. In this step the expressed sequences enriched in the two tester pools following the initial hybridization can hybridize. Hybrids resulting from the hybridization between members of each of the two tester pools are then recovered by amplification in a polymerase chain reaction PCR using primers specific for the unique adapters. Again sequences originating in a tester pool that form hybrids with components of the driver pool are not amplified. Hybrids resulting between members of the same tester pool are eliminated by the formation of panhandles between their common 5 and 3 ends. For additional details see e.g. Lukyanov et al. 1997 230 285 8.

Typically the tester and driver pools are designated in the alternative such that the hybridization is performed in both directions to ensure recovery of messenger RNAs that are differentially expressed in either a positive or negative manner i.e. that are turned on or turned off up regulated or down regulated . Accordingly it will be understood that the designation of test and control populations is to some extent arbitrary and that a test population can just as easily be compared to leukocytes derived from a patient with the same of another disease of interest.

If so desired the efficacy of the process can be assessed by such techniques as semi quantitative PCR of known i.e. control nucleotide sequences of varying abundance such as actin. The resulting PCR products representing partial cDNAs of differentially expressed nucleotide sequences are then cloned i.e. ligated into an appropriate vector e.g. a commercially available TA cloning vector such as pGEM from Promega and optionally transformed into competent bacteria for selection and screening.

Either of the above approaches or both in combination or indeed any procedure which permits the assembly of a collection of nucleotide sequences that are expressed in leukocytes is favorably employed to produce the libraries of candidates useful for the identification of diagnostic nucleotide sets and disease specific target nucleotides of the invention. Additionally any method that permits the assembly of a collection of nucleotides that are expressed in leukocytes and preferentially associated with one or more disease or condition whether or not the nucleotide sequences are differentially expressed is favorably employed in the context of the invention. Typically libraries of about 2 000 10 000 members are produced although libraries in excess of 10 000 are not uncommon . Following additional evaluation procedures as described below the proportion of unique clones in the candidate library can approximate 100 .

A candidate oligonucleotide sequence may be represented in a candidate library by a full length or partial nucleic acid sequence deoxyribonucleic acid DNA sequence cDNA sequence RNA sequence synthetic oligonucleotides etc. The nucleic acid sequence can be at least 19 nucleotides in length at least 25 nucleotides at least 40 nucleotides at least 100 nucleotides or larger. Alternatively the protein product of a candidate nucleotide sequence may be represented in a candidate library using standard methods as further described below.

The sequence of individual members e.g. clones partial sequence listing in a database such as an EST etc. of the candidate oligonucleotide libraries is then determined by conventional sequencing methods well known in the art e.g. by the dideoxy chain termination method of Sanger et al. 1977 74 5463 7 by chemical procedures e.g. Maxam and Gilbert 1977 74 560 4 or by polymerase chain reaction cycle sequencing methods e.g. Olsen and Eckstein 1989 17 9613 20 DNA chip based sequencing techniques or variations including automated variations e.g. as described in Hunkapiller et al. 1991 254 59 67 Pease et al. 1994 91 5022 6 thereof. Numerous kits for performing the above procedures are commercially available and well known to those of skill in the art. Character strings corresponding to the resulting nucleotide sequences are then recorded i.e. stored in a database. Most commonly the character strings are recorded on a computer readable medium for processing by a computational device.

Generally to facilitate subsequent analysis a custom algorithm is employed to query existing databases in an ongoing fashion to determine the identity expression pattern and potential function of the particular members of a candidate library. The sequence is first processed by removing low quality sequence. Next the vector sequences are identified and removed and sequence repeats are identified and masked. The remaining sequence is then used in a Blast algorithm against multiple publicly available and or proprietary databases e.g. NCBI nucleotide EST and protein databases Unigene and Human Genome Sequence. Sequences are also compared to all previously sequenced members of the candidate libraries to detect redundancy.

In some cases sequences are of high quality but do not match any sequence in the NCBI nr human EST or Unigene databases. In this case the sequence is queried against the human genomic sequence. If a single chromosomal site is matched with a high degree of confidence that region of genomic DNA is identified and subjected to further analysis with a gene prediction program such as GRAIL. This analysis may lead to the identification of a new gene in the genomic sequence. This sequence can then be translated to identify the protein sequence that is encoded and that sequence can be further analyzed using tools such as Pfam Blast P or other protein structure prediction programs as illustrated in Table 7. Typically the above analysis is directed towards the identification of putative coding regions e.g. previously unidentified open reading frames confirming the presence of known coding sequences and determining structural motifs or sequence similarities of the predicted protein i.e. the conceptual translation product in relation to known sequences. In addition it has become increasingly possible to assemble virtual cDNAs containing large portions of coding region simply through the assembly of available expressed sequence tags ESTs . In turn these extended nucleic acid and amino acid sequences allow the rapid expansion of substrate sequences for homology searches and structural and functional motif characterization. The results of these analysis permits the categorization of sequences according to structural characteristics e.g. as structural proteins proteins involved in signal transduction cell surface or secreted proteins etc.

It is understood that full length nucleotide sequences may also be identified using conventional methods for example library screening RT PCR chromosome walking etc. as described in Sambrook and Ausubel infra.

We identified members of a candidate nucleotide library that are differentially expressed in activated leukocytes and resting leukocytes. Accordingly the invention provides the candidate leukocyte nucleotide library comprising the nucleotide sequences listed in Table 2 Table 3 Tables 8 10 and in the Sequence Listing. In another embodiment the invention provides a candidate library comprising at least two nucleotide sequences listed in Table 2 Table 3 Tables 8 10 and the Sequence Listing. In another embodiment at least two nucleotide sequences are 18 nucleotides in length at least 35 nucleotides at least 40 nucleotides or at least 100 nucleotides. In some embodiments the nucleotide sequences comprises deoxyribonucleic acid DNA sequence ribonucleic acid RNA sequence synthetic oligonucleotide sequence or genomic DNA sequence. It is understood that the nucleotide sequences may each correspond to one gene or that several nucleotide sequences may correspond to one gene or that a single nucleotide sequence may correspond to multiple genes.

The invention also provides probes to the candidate nucleotide library. In one embodiment of the invention the probes comprise at least two nucleotide sequences listed in Table 2 Table 3 Tables 8 10 or the Sequence Listing which are differentially expressed in leukocytes in an individual with a least one disease criterion for at least one leukocyte related disease and in leukocytes in an individual without the at least one disease criterion wherein expression of the two or more nucleotide sequences is correlated with at least one disease criterion. It is understood that a probe may detect either the RNA expression or protein product expression of the candidate nucleotide library. Alternatively or in addition a probe can detect a genotype associated with a candidate nucleotide sequence as further described below. In another embodiment the probes for the candidate nucleotide library are immobilized on an array.

The candidate nucleotide library of the invention is useful in identifying diagnostic nucleotide sets of the invention as described below. The candidate nucleotide sequences may be further characterized and may be identified as a disease target nucleotide sequence and or a novel nucleotide sequence as described below. The candidate nucleotide sequences may also be suitable for use as imaging reagents as described below.

Following identification or assembly of a library of differentially expressed candidate nucleotide sequences leukocyte expression profiles corresponding to multiple members of the candidate library are obtained. Leukocyte samples from one or more subjects are obtained by standard methods. Most typically these methods involve trans cutaneous venous sampling of peripheral blood. While sampling of circulating leukocytes from whole blood from the peripheral vasculature is generally the simplest least invasive and lowest cost alternative it will be appreciated that numerous alternative sampling procedures exist and are favorably employed in some circumstances. No pertinent distinction exists in fact between leukocytes sampled from the peripheral vasculature and those obtained e.g. from a central line from a central artery or indeed from a cardiac catheter or during a surgical procedure which accesses the central vasculature. In addition other body fluids and tissues that are at least in part composed of leukocytes are also desirable leukocyte samples. For example fluid samples obtained from the lung during bronchoscopy may be rich in leukocytes and amenable to expression profiling in the context of the invention e.g. for the diagnosis prognosis or monitoring of lung transplant rejection inflammatory lung diseases or infectious lung disease. Fluid samples from other tissues e.g. obtained by endoscopy of the colon sinuses esophagus stomach small bowel pancreatic duct biliary tree bladder ureter vagina cervix or uterus etc. are also suitable. Samples may also be obtained other sources containing leukocytes e.g. from urine bile cerebrospinal fluid feces gastric or intestinal secretions semen or solid organ or joint biopsies.

Most frequently mixed populations of leukocytes such as are found in whole blood are utilized in the methods of the present invention. A crude separation e.g. of mixed leukocytes from red blood cells and or concentration e.g. over a sucrose percoll or ficoll gradient or by other methods known in the art can be employed to facilitate the recovery of RNA or protein expression products at sufficient concentrations and to reduce non specific background. In some instances it can be desirable to purify sub populations of leukocytes and methods for doing so such as density or affinity gradients flow cytometry Fluorescence Activated Cell Sorting FACS immuno magnetic separation panning and the like are described in the available literature and below.

A variety of techniques are available for the isolation of RNA from whole blood. Any technique that allows isolation of mRNA from cells in the presence or absence of rRNA and tRNA can be utilized. In brief one method that allows reliable isolation of total RNA suitable for subsequent gene expression analysis is described as follows. Peripheral blood either venous or arterial is drawn from a subject into one or more sterile endotoxin free tubes containing an anticoagulant e.g. EDTA citrate heparin etc. . Typically the sample is divided into at least two portions. One portion e.g. of 5 8 ml of whole blood is frozen and stored for future analysis e.g. of DNA or protein. A second portion e.g. of approximately 8 ml whole blood is processed for isolation of total RNA by any of a variety of techniques as described in e.g Sambook Ausubel below as well as U.S. Pat. Nos. 5 728 822 and 4 843 155.

Typically a subject sample of mononuclear leukocytes obtained from about 8 ml of whole blood a quantity readily available from an adult human subject under most circumstances yields 5 20 g of total RNA. This amount is ample e.g. for labeling and hybridization to at least two probe arrays. Labeled probes for analysis of expression patterns of nucleotides of the candidate libraries are prepared from the subject s sample of RNA using standard methods. In many cases cDNA is synthesized from total RNA using a polyT primer and labeled e.g. radioactive or fluorescent nucleotides. The resulting labeled cDNA is then hybridized to probes corresponding to members of the candidate nucleotide library and expression data is obtained for each nucleotide sequence in the library. RNA isolated from subject samples e.g. peripheral blood leukocytes or leukocytes obtained from other biological fluids and samples is next used for analysis of expression patterns of nucleotides of the candidate libraries.

In some cases however the amount of RNA that is extracted from the leukocyte sample is limiting and amplification of the RNA is desirable. Amplification may be accomplished by increasing the efficiency of probe labeling or by amplifying the RNA sample prior to labeling. It is appreciated that care must be taken to select an amplification procedure that does not introduce any bias with respect to gene expression levels during the amplification process.

Several methods are available that increase the signal from limiting amounts of RNA e.g. use of the Clontech Glass Fluorescent Labeling Kit or Stratagene Fairplay Microarray Labeling Kit or the Micromax kit New England Nuclear Inc. . Alternatively cDNA is synthesized from RNA using a T7 polyT primer in the absence of label and DNA dendrimers from Genisphere 3 DNA Submicro are hybridized to the poly T sequence on the primer or to a different capture sequence which is complementary to a fluorescently labeled sequence. Each 3DNA molecule has 250 fluorescent molecules and therefore can strongly label each cDNA.

Alternatively the RNA sample is amplified prior to labeling. For example linear amplification may be performed as described in U.S. Pat. No. 6 132 997. A T7 polyT primer is used to generate the cDNA copy of the RNA. A second DNA strand is then made to complete the substrate for amplification. The T7 promoter incorporated into the primer is used by a T7 polymerase to produce numerous antisense copies of the original RNA. Fluorescent dye labeled nucleotides are directly incorporated into the RNA. Alternatively amino allyl labeled nucleotides are incorporated into the RNA and then fluorescent dyes are chemically coupled to the amino allyl groups as described in Hughes et al. 2001. Other exemplary methods for amplification are described below.

It is appreciated that the RNA isolated must contain RNA derived from leukocytes but may also contain RNA from other cell types to a variable degree. Additionally the isolated RNA may come from subsets of leukocytes e.g. monocytes and or T lymphocytes as described above. Such consideration of cell type used for the derivation of RNA depends on the method of expression profiling used.

DNA samples may be obtained for analysis of the presence of DNA mutations single nucleotide polymorphisms SNPs or other polymorphisms. DNA is isolated using standard techniques e.g. Maniatus supra.

Expression of products of candidate nucleotides may also be assessed using proteomics. Protein s are detected in samples of patient serum or from leukocyte cellular protein. Serum is prepared by centrifugation of whole blood using standard methods. Proteins present in the serum may have been produced from any of a variety of leukocytes and non leukocyte cells and may include secreted proteins from leukocytes. Alternatively leukocytes or a desired sub population of leukocytes are prepared as described above. Cellular protein is prepared from leukocyte samples using methods well known in the art e.g. Trizol Invitrogen Life Technologies cat 15596108 Chomczynski P. and Sacchi N. 1987 Anal. Biochem. 162 156 Simms D. Cizdziel P. E. and Chomczynski P. 1993 Focus 15 99 Chomczynski P. Bowers Finn R. and Sabatini L. 1987 J. of NIH Res. 6 83 Chomczynski P. 1993 Bio Techniques 15 532 Bracete A. M. Fox D. K. and Simms D. 1998 Focus 20 82 Sewall A. and McRae S. 1998 Focus 20 36 Anal Biochem 1984 April 138 1 141 3 A method for the quantitative recovery of protein in dilute solution in the presence of detergents and lipids Wessel D Flugge U I. 1984 Anal Biochem. 1984 April 138 1 141 143.

Expression patterns or profiles of a plurality of nucleotides corresponding to members of the candidate library are then evaluated in one or more samples of leukocytes. Typically the leukocytes are derived from patient peripheral blood samples although as indicated above many other sample sources are also suitable. These expression patterns constitute a set of relative or absolute expression values for some number of RNAs or protein products corresponding to the plurality of nucleotide sequences evaluated which is referred to herein as the subject s expression profile for those nucleotide sequences. While expression patterns for as few as one independent member of the candidate library can be obtained it is generally preferable to obtain expression patterns corresponding to a larger number of nucleotide sequences e.g. about 2 about 5 about 10 about 20 about 50 about 100 about 200 about 500 or about 1000 or more. The expression pattern for each differentially expressed component member of the library provides a finite specificity and sensitivity with respect to predictive value e.g. for diagnosis prognosis monitoring and the like.

For the purpose of discussion the term subject or subject sample of leukocytes refers to an individual regardless of health and or disease status. A subject can be a patient a study participant a control subject a screening subject or any other class of individual from whom a leukocyte sample is obtained and assessed in the context of the invention. Accordingly a subject can be diagnosed with a disease can present with one or more symptom of a disease or a predisposing factor such as a family genetic or medical history medical factor for a disease or the like. Alternatively a subject can be healthy with respect to any of the aforementioned factors or criteria. It will be appreciated that the term healthy as used herein is relative to a specified disease or disease factor or disease criterion as the term healthy cannot be defined to correspond to any absolute evaluation or status. Thus an individual defined as healthy with reference to any specified disease or disease criterion can in fact be diagnosed with any other one or more disease or exhibit any other one or more disease criterion.

Furthermore while the discussion of the invention focuses and is exemplified using human sequences and samples the invention is equally applicable through construction or selection of appropriate candidate libraries to non human animals such as laboratory animals e.g. mice rats guinea pigs rabbits domesticated livestock e.g. cows horses goats sheep chicken etc. and companion animals e.g. dogs cats etc.

Numerous methods for obtaining expression data are known and any one or more of these techniques singly or in combination are suitable for determining expression profiles in the context of the present invention. For example expression patterns can be evaluated by northern analysis PCR RT PCR Tag Man analysis FRET detection monitoring one or more molecular beacon hybridization to an oligonucleotide array hybridization to a cDNA array hybridization to a polynucleotide array hybridization to a liquid microarray hybridization to a microelectric array molecular beacons cDNA sequencing clone hybridization cDNA fragment fingerprinting serial analysis of gene expression SAGE subtractive hybridization differential display and or differential screening see e.g. Lockhart and Winzeler 2000 405 827 836 and references cited therein .

For example specific PCR primers are designed to a member s of a candidate nucleotide library. cDNA is prepared from subject sample RNA by reverse transcription from a poly dT oligonucleotide primer and subjected to PCR. Double stranded cDNA may be prepared using primers suitable for reverse transcription of the PCR product followed by amplification of the cDNA using in vitro transcription. The product of in vitro transcription is a sense RNA corresponding to the original member s of the candidate library. PCR product may be also be evaluated in a number of ways known in the art including real time assessment using detection of labeled primers e.g. TaqMan or molecular beacon probes. Technology platforms suitable for analysis of PCR products include the ABI 7700 5700 or 7000 Sequence Detection Systems Applied Biosystems Foster City Calif. the MJ Research Opticon MJ Research Waltham Mass. the Roche Light Cycler Roche Diagnositics Indianapolis Ind. the Stratagene MX4000 Stratagene La Jolla Calif. and the Bio Rad iCycler Bio Rad Laboratories Hercules Calif. . Alternatively molecular beacons are used to detect presence of a nucleic acid sequence in an unamplified RNA or cDNA sample or following amplification of the sequence using any method e.g. IVT In Vitro transcription or NASBA nucleic acid sequence based amplification . Molecular beacons are designed with sequences complementary to member s of a candidate nucleotide library and are linked to fluorescent labels. Each probe has a different fluorescent label with non overlapping emission wavelengths. For example expression of ten genes may be assessed using ten different sequence specific molecular beacons.

Alternatively or in addition molecular beacons are used to assess expression of multiple nucleotide sequences at once. Molecular beacons with sequence complimentary to the members of a diagnostic nucleotide set are designed and linked to fluorescent labels. Each fluorescent label used must have a non overlapping emission wavelength. For example 10 nucleotide sequences can be assessed by hybridizing 10 sequence specific molecular beacons each labeled with a different fluorescent molecule to an amplified or un amplified RNA or cDNA sample. Such an assay bypasses the need for sample labeling procedures.

Alternatively or in addition bead arrays can be used to assess expression of multiple sequences at once See e.g LabMAP 100 Luminex Corp Austin Tex. . Alternatively or in addition electric arrays are used to assess expression of multiple sequences as exemplified by the e Sensor technology of Motorola Chicago Ill. or Nanochip technology of Nanogen San Diego Calif. 

Of course the particular method elected will be dependent on such factors as quantity of RNA recovered practitioner preference available reagents and equipment detectors and the like. Typically however the elected method s will be appropriate for processing the number of samples and probes of interest. Methods for high throughput expression analysis are discussed below.

Alternatively expression at the level of protein products of gene expression is performed. For example protein expression in a sample of leukocytes can be evaluated by one or more method selected from among western analysis two dimensional gel analysis chromatographic separation mass spectrometric detection protein fusion reporter constructs colorimetric assays binding to a protein array and characterization of polysomal mRNA. One particularly favorable approach involves binding of labeled protein expression products to an array of antibodies specific for members of the candidate library. Methods for producing and evaluating antibodies are widespread in the art see e.g. Coligan supra and Harlow and Lane 1989 Cold Spring Harbor Press NY Harlow and Lane . Additional details regarding a variety of immunological and immunoassay procedures adaptable to the present invention by selection of antibody reagents specific for the products of candidate nucleotide sequences can be found in e.g. Stites and Ten eds. 1991 7ed. and Paul supra. Another approach uses systems for performing desorption spectrometry. Commercially available systems e.g. from Ciphergen Biosystems Inc. Fremont Calif. are particularly well suited to quantitative analysis of protein expression. Indeed Protein Chip arrays see e.g. the website ciphergen.com used in desorption spectrometry approaches provide arrays for detection of protein expression. Alternatively affinity reagents e.g. antibodies small molecules etc. are developed that recognize epitopes of the protein product. Affinity assays are used in protein array assays e.g. to detect the presence or absence of particular proteins. Alternatively affinity reagents are used to detect expression using the methods described above. In the case of a protein that is expressed on the cell surface of leukocytes labeled affinity reagents are bound to populations of leukocytes and leukocytes expressing the protein are identified and counted using fluorescent activated cell sorting FACS .

It is appreciated that the methods of expression evaluation discussed herein although discussed in the context of discovery of diagnostic nucleotide sets are also applicable for expression evaluation when using diagnostic nucleotide sets for e.g. diagnosis of diseases as further discussed below.

A number of suitable high throughput formats exist for evaluating gene expression. Typically the term high throughput refers to a format that performs at least about 100 assays or at least about 500 assays or at least about 1000 assays or at least about 5000 assays or at least about 10 000 assays or more per day. When enumerating assays either the number of samples or the number of candidate nucleotide sequences evaluated can be considered. For example a northern analysis of e.g. about 100 samples performed in a gridded array e.g. a dot blot using a single probe corresponding to a candidate nucleotide sequence can be considered a high throughput assay. More typically however such an assay is performed as a series of duplicate blots each evaluated with a distinct probe corresponding to a different member of the candidate library. Alternatively methods that simultaneously evaluate expression of about 100 or more candidate nucleotide sequences in one or more samples or in multiple samples are considered high throughput.

Numerous technological platforms for performing high throughput expression analysis are known. Generally such methods involve a logical or physical array of either the subject samples or the candidate library or both. Common array formats include both liquid and solid phase arrays. For example assays employing liquid phase arrays e.g. for hybridization of nucleic acids binding of antibodies or other receptors to ligand etc. can be performed in multiwell or microtiter plates. Microtiter plates with 96 384 or 1536 wells are widely available and even higher numbers of wells e.g 3456 and 9600 can be used. In general the choice of microtiter plates is determined by the methods and equipment e.g. robotic handling and loading systems used for sample preparation and analysis. Exemplary systems include e.g. the ORCA system from Beckman Coulter Inc. Fullerton Calif. and the Zymate systems from Zymark Corporation Hopkinton Mass. .

Alternatively a variety of solid phase arrays can favorably be employed in to determine expression patterns in the context of the invention. Exemplary formats include membrane or filter arrays e.g nitrocellulose nylon pin arrays and bead arrays e.g. in a liquid slurry . Typically probes corresponding to nucleic acid or protein reagents that specifically interact with e.g. hybridize to or bind to an expression product corresponding to a member of the candidate library are immobilized for example by direct or indirect cross linking to the solid support. Essentially any solid support capable of withstanding the reagents and conditions necessary for performing the particular expression assay can be utilized. For example functionalized glass silicon silicon dioxide modified silicon any of a variety of polymers such as poly tetrafluoroethylene poly vinylidenedifluoride polystyrene polycarbonate or combinations thereof can all serve as the substrate for a solid phase array.

In a preferred embodiment the array is a chip composed e.g. of one of the above specified materials. Polynucleotide probes e.g. RNA or DNA such as cDNA synthetic oligonucleotides and the like or binding proteins such as antibodies that specifically interact with expression products of individual components of the candidate library are affixed to the chip in a logically ordered manner i.e. in an array. In addition any molecule with a specific affinity for either the sense or anti sense sequence of the marker nucleotide sequence depending on the design of the sample labeling can be fixed to the array surface without loss of specific affinity for the marker and can be obtained and produced for array production for example proteins that specifically recognize the specific nucleic acid sequence of the marker ribozymes peptide nucleic acids PNA or other chemicals or molecules with specific affinity.

Detailed discussion of methods for linking nucleic acids and proteins to a chip substrate are found in e.g. U.S. Pat. No. 5 143 854 Large Scale Photolithographic Solid Phase Synthesis Of Polypeptides And Receptor Binding Screening Thereof to Pirrung et al. issued Sep. 1 1992 U.S. Pat. No. 5 837 832 Arrays Of Nucleic Acid Probes On Biological Chips to Chee et al. issued Nov. 17 1998 U.S. Pat. No. 6 087 112 Arrays With Modified Oligonucleotide And Polynucleotide Compositions to Dale issued Jul. 11 2000 U.S. Pat. No. 5 215 882 Method Of Immobilizing Nucleic Acid On A Solid Substrate For Use In Nucleic Acid Hybridization Assays to Bahl et al. issued Jun. 1 1993 U.S. Pat. No. 5 707 807 Molecular Indexing For Expressed Gene Analysis to Kato issued Jan. 13 1998 U.S. Pat. No. 5 807 522 Methods For Fabricating Microarrays Of Biological Samples to Brown et al. issued Sep. 15 1998 U.S. Pat. No. 5 958 342 Jet Droplet Device to Gamble et al. issued Sep. 28 1999 U.S. Pat. No. 5 994 076 Methods Of Assaying Differential Expression to Chenchik et al. issued Nov. 30 1999 U.S. Pat. No. 6 004 755 Quantitative Microarray Hybridization Assays to Wang issued Dec. 21 1999 U.S. Pat. No. 6 048 695 Chemically Modified Nucleic Acids And Method For Coupling Nucleic Acids To Solid Support to Bradley et al. issued Apr. 11 2000 U.S. Pat. No. 6 060 240 Methods For Measuring Relative Amounts Of Nucleic Acids In A Complex Mixture And Retrieval Of Specific Sequences Therefrom to Kamb et al. issued May 9 2000 U.S. Pat. No. 6 090 556 Method For Quantitatively Determining The Expression Of A Gene to Kato issued Jul. 18 2000 and U.S. Pat. No. 6 040 138 Expression Monitoring By Hybridization To High Density Oligonucleotide Arrays to Lockhart et al. issued Mar. 21 2000 each of which are hereby incorporated by reference in their entirety.

For example cDNA inserts corresponding to candidate nucleotide sequences in a standard TA cloning vector are amplified by a polymerase chain reaction for approximately 30 40 cycles. The amplified PCR products are then arrayed onto a glass support by any of a variety of well known techniques e.g. the VSLIPS technology described in U.S. Pat. No. 5 143 854. RNA or cDNA corresponding to RNA isolated from a subject sample of leukocytes is labeled e.g. with a fluorescent tag and a solution containing the RNA or cDNA is incubated under conditions favorable for hybridization with the probe chip. Following incubation and washing to eliminate non specific hybridization the labeled nucleic acid bound to the chip is detected qualitatively or quantitatively and the resulting expression profile for the corresponding candidate nucleotide sequences is recorded. It is appreciated that the probe used for diagnostic purposes may be identical to the probe used during diagnostic nucleotide sequence discovery and validation. Alternatively the probe sequence may be different than the sequence used in diagnostic nucleotide sequence discovery and validation. Multiple cDNAs from a nucleotide sequence that are non overlapping or partially overlapping may also be used.

In another approach oligonucleotides corresponding to members of a candidate nucleotide library are synthesized and spotted onto an array. Alternatively oligonucleotides are synthesized onto the array using methods known in the art e.g. Hughes et al. supra. The oligonucleotide is designed to be complementary to any portion of the candidate nucleotide sequence. In addition in the context of expression analysis for e.g. diagnostic use of diagnostic nucleotide sets an oligonucleotide can be designed to exhibit particular hybridization characteristics or to exhibit a particular specificity and or sensitivity as further described below.

Oligonucleotide probes are also prepared using the DNA sequence information for the candidate genes identified by differential hybridization screening listed in Tables 3 8 10 and the Sequence Listing and or the sequence information for the genes identified by database mining listed in Table 2 is used to design complimentary oligonucleotide probes. Oligo probes are designed on a contract basis by various companies for example Compugen Mergen Affymetrix Telechem or designed from the candidate sequences using a variety of parameters and algorithms as indicated at the website genome.wi.mit.edu cgi bin primer primer3.cgi. Briefly the length of the oligonucleotide to be synthesized is determined preferably at least 16 nucleotides generally 18 24 nucleotides 24 70 nucleotides and in some circumstances more than 70 nucleotides. The sequence analysis algorithms and tools described above are applied to the sequences to mask repetitive elements vector sequences and low complexity sequences. Oligonucleotides are selected that are specific to the candidate nucleotide sequence based on a Blast n search of the oligonucleotide sequence in question against gene sequences databases such as the Human Genome Sequence UniGene dbEST or the non redundant database at NCBI and have 

Hybridization signal may be amplified using methods known in the art and as described herein for example use of the Clontech kit Glass Fluorescent Labeling Kit Stratagene kit Fairplay Microarray Labeling Kit the Micromax kit New England Nuclear Inc. the Genisphere kit 3DNA Submicro linear amplification e.g. as described in U.S. Pat. No. 6 132 997 or described in Hughes T R et al. Nature Biotechnology 19 343 347 2001 and or Westin et al. 18 199 204. In some cases amplification techniques do not increase signal intensity but allow assays to be done with small amounts of RNA.

Alternatively fluorescently labeled cDNA are hybridized directly to the microarray using methods known in the art. For example labeled cDNA are generated by reverse transcription using Cy3 and Cy5 conjugated deoxynucleotides and the reaction products purified using standard methods. It is appreciated that the methods for signal amplification of expression data useful for identifying diagnostic nucleotide sets are also useful for amplification of expression data for diagnostic purposes.

Microarray expression may be detected by scanning the microarray with a variety of laser or CCD based scanners and extracting features with numerous software packages for example Imagene Biodiscovery Feature Extraction Software Agilent Scanalyze Eisen M. 1999. SCANALYZE User Manual Stanford Univ. Stanford Calif. Ver 2.32. GenePix Axon Instruments .

In another approach hybridization to microelectric arrays is performed e.g. as described in Umek et al 2001 3 74 84. An affinity probe e.g. DNA is deposited on a metal surface. The metal surface underlying each probe is connected to a metal wire and electrical signal detection system. Unlabelled RNA or cDNA is hybridized to the array or alternatively RNA or cDNA sample is amplified before hybridization e.g. by PCR. Specific hybridization of sample RNA or cDNA results in generation of an electrical signal which is transmitted to a detector. See Westin 2000 18 199 204 describing anchored multiplex amplification of a microelectronic chip array Edman 1997 25 4907 14 Vignali 2000 243 243 55.

In another approach a microfluidics chip is used for RNA sample preparation and analysis. This approach increases efficiency because sample preparation and analysis are streamlined. Briefly microfluidics may be used to sort specific leukocyte sub populations prior to RNA preparation and analysis. Microfluidics chips are also useful for e.g. RNA preparation and reactions involving RNA reverse transcription RT PCR . Briefly a small volume of whole anti coagulated blood is loaded onto a microfluidics chip for example chips available from Caliper Mountain View Calif. or Nanogen San Diego Calif. A microfluidics chip may contain channels and reservoirs in which cells are moved and reactions are performed. Mechanical electrical magnetic gravitational centrifugal or other forces are used to move the cells and to expose them to reagents. For example cells of whole blood are moved into a chamber containing hypotonic saline which results in selective lysis of red blood cells after a 20 minute incubation. Next the remaining cells leukocytes are moved into a wash chamber and finally moved into a chamber containing a lysis buffer such as guanidine isothyocyanate. The leukocyte cell lysate is further processed for RNA isolation in the chip or is then removed for further processing for example RNA extraction by standard methods. Alternatively the microfluidics chip is a circular disk containing ficoll or another density reagent. The blood sample is injected into the center of the disc the disc is rotated at a speed that generates a centrifugal force appropriate for density gradient separation of mononuclear cells and the separated mononuclear cells are then harvested for further analysis or processing.

It is understood that the methods of expression evaluation above although discussed in the context of discovery of diagnostic nucleotide sets are also applicable for expression evaluation when using diagnostic nucleotide sets for e.g. diagnosis of diseases as further discussed below.

Expression patterns can be evaluated by qualitative and or quantitative measures. Certain of the above described techniques for evaluating gene expression as RNA or protein products yield data that are predominantly qualitative in nature. That is the methods detect differences in expression that classify expression into distinct modes without providing significant information regarding quantitative aspects of expression. For example a technique can be described as a qualitative technique if it detects the presence or absence of expression of a candidate nucleotide sequence i.e. an on off pattern of expression. Alternatively a qualitative technique measures the presence and or absence of different alleles or variants of a gene product.

In contrast some methods provide data that characterize expression in a quantitative manner. That is the methods relate expression on a numerical scale e.g. a scale of 0 5 a scale of 1 10 a scale of from grade 1 to grade 5 a grade from a to z or the like. It will be understood that the numerical and symbolic examples provided are arbitrary and that any graduated scale or any symbolic representation of a graduated scale can be employed in the context of the present invention to describe quantitative differences in nucleotide sequence expression. Typically such methods yield information corresponding to a relative increase or decrease in expression.

Any method that yields either quantitative or qualitative expression data is suitable for evaluating expression of candidate nucleotide sequence in a subject sample of leukocytes. In some cases e.g. when multiple methods are employed to determine expression patterns for a plurality of candidate nucleotide sequences the recovered data e.g. the expression profile for the nucleotide sequences is a combination of quantitative and qualitative data.

In some applications expression of the plurality of candidate nucleotide sequences is evaluated sequentially. This is typically the case for methods that can be characterized as low to moderate throughput. In contrast as the throughput of the elected assay increases expression for the plurality of candidate nucleotide sequences in a sample or multiple samples of leukocytes is assayed simultaneously. Again the methods and throughput are largely determined by the individual practitioner although typically it is preferable to employ methods that permit rapid e.g. automated or partially automated preparation and detection on a scale that is time efficient and cost effective.

It is understood that the preceding discussion while directed at the assessment of expression of the members of candidate libraries is also applies to the assessment of the expression of members of diagnostic nucleotide sets as further discussed below.

In addition to or in conjunction with the correlation of expression profiles and clinical data it is often desirable to correlate expression patterns with the subject s genotype at one or more genetic loci or to correlate both expression profiles and genetic loci data with clinical data. The selected loci can be for example chromosomal loci corresponding to one or more member of the candidate library polymorphic alleles for marker loci or alternative disease related loci not contributing to the candidate library known to be or putatively associated with a disease or disease criterion . Indeed it will be appreciated that where a polymorphic allele at a locus is linked to a disease or to a predisposition to a disease the presence of the allele can itself be a disease criterion.

Numerous well known methods exist for evaluating the genotype of an individual including southern analysis restriction fragment length polymorphism RFLP analysis polymerase chain reaction PCR amplification length polymorphism AFLP analysis single stranded conformation polymorphism SSCP analysis single nucleotide polymorphism SNP analysis e.g. via PCR Taqman or molecular beacons among many other useful methods. Many such procedures are readily adaptable to high throughput and or automated or semi automated sample preparation and analysis methods. Most can be performed on nucleic acid samples recovered via simple procedures from the same sample of leukocytes as yielded the material for expression profiling. Exemplary techniques are described in e.g. Sambrook and Ausubel supra.

Identification of diagnostic nucleotides and nucleotide sets and disease specific target nucleotide sequence proceeds by correlating the leukocyte expression profiles with data regarding the subject s health status to produce a data set designated a molecular signature. Examples of data regarding a patient s health status also termed disease criteria ion is described below and in the Section titled selected diseases below. Methods useful for correlation analysis are further described elsewhere in the specification.

Generally relevant data regarding the subject s health status includes retrospective or prospective health data e.g. in the form of the subject s medical history as provided by the subject physician or third party such as medical diagnoses laboratory test results diagnostic test results clinical events or medication lists as further described below. Such data may include information regarding a patient s response to treatment and or a particular medication and data regarding the presence of previously characterized risk factors. For example cigarette smoking and obesity are previously identified risk factors for heart disease. Further examples of health status information including diseases and disease criteria is described in the section titled Selected diseases below.

Typically the data describes prior events and evaluations i.e. retrospective data . However it is envisioned that data collected subsequent to the sampling i.e. prospective data can also be correlated with the expression profile. The tissue sampled e.g. peripheral blood bronchial lavage etc. can be obtained at one or more multiple time points and subject data is considered retrospective or prospective with respect to the time of sample procurement.

Data collected at multiple time points called longitudinal data is often useful and thus the invention encompasses the analysis of patient data collected from the same patient at different time points. Analysis of paired samples such as samples from a patient at different times allows identification of differences that are specifically related to the disease state since the genetic variability specific to the patient is controlled for by the comparison. Additionally other variables that exist between patients may be controlled for in this way for example the presence or absence of inflammatory diseases e.g. rheumatoid arthritis the use of medications that may effect leukocyte gene expression the presence or absence of co morbid conditions etc. Methods for analysis of paired samples are further described below. Moreover the analysis of a pattern of expression profiles generated by collecting multiple expression profiles provides information relating to changes in expression level over time and may permit the determination of a rate of change a trajectory or an expression curve. Two longitudinal samples may provide information on the change in expression of a gene over time while three longitudinal samples may be necessary to determine the trajectory of expression of a gene. Such information may be relevant to the diagnosis of a disease. For example the expression of a gene may vary from individual to individual but a clinical event for example a heart attack may cause the level of expression to double in each patient. In this example clinically interesting information is gleaned from the change in expression level as opposed to the absolute level of expression in each individual.

When a single patient sample is obtained it may still be desirable to compare the expression profile of that sample to some reference expression profile. In this case one can determine the change of expression between the patient s sample and a reference expression profile that is appropriate for that patient and the medical condition in question. For example a reference expression profile can be determined for all patients without the disease criterion in question who have similar characteristics such as age sex race diagnoses etc.

Generally small sample sizes of 10 40 samples from 10 20 individuals are used to identify a diagnostic nucleotide set. Larger sample sizes are generally necessary to validate the diagnostic nucleotide set for use in large and varied patient populations as further described below. For example extension of gene expression correlations to varied ethnic groups demographic groups nations peoples or races may require expression correlation experiments on the population of interest.

Expression profiles derived from a patient i.e. subjects diagnosed with or exhibiting symptoms of or exhibiting a disease criterion or under a doctor s care for a disease sample are compared to a control or standard expression RNA to facilitate comparison of expression profiles e.g. of a set of candidate nucleotide sequences from a group of patients relative to each other i.e. from one patient in the group to other patients in the group or to patients in another group .

The reference RNA used should have desirable features of low cost and simplicity of production on a large scale. Additionally the reference RNA should contain measurable amounts of as many of the genes of the candidate library as possible.

For example in one approach to identifying diagnostic nucleotide sets expression profiles derived from patient samples are compared to a expression reference standard. Standard expression reference can be for example RNA derived from resting cultured leukocytes or commercially available reference RNA such as Universal reference RNA from Stratagene. See Nature V406 8 17 00 p. 747 752. Use of an expression reference standard is particularly useful when the expression of large numbers of nucleotide sequences is assayed e.g. in an array and in certain other applications e.g. qualitative PCR RT PCR etc. where it is desirable to compare a sample profile to a standard profile and or when large numbers of expression profiles e.g. a patient population are to be compared. Generally an expression reference standard should be available in large quantities should be a good substrate for amplification and labeling reactions and should be capable of detecting a large percentage of candidate nucleic acids using suitable expression profiling technology.

Alternatively or in addition the expression profile derived from a patient sample is compared with the expression of an internal reference control gene for example actin or CD4. The relative expression of the profiled genes and the internal reference control gene from the same individual is obtained. An internal reference control may also be used with a reference RNA. For example an expression profile for gene 1 and the gene encoding CD4 can be determined in a patient sample and in a reference RNA. The expression of each gene can be expressed as the relative ratio of expression the gene in the patient sample compared with expression of the gene in the reference RNA. The expression ratio sample reference for gene 1 may be divided by the expression ration for CD4 sample reference and thus the relative expression of gene 1 to CD4 is obtained.

The invention also provides a buffy coat control RNA useful for expression profiling and a method of using control RNA produced from a population of buffy coat cells the white blood cell layer derived from the centrifugation of whole blood. Buffy coat contains all white blood cells including granulocytes mononuclear cells and platelets. The invention also provides a method of preparing control RNA from buffy coat cells for use in expression profile analysis of leukocytes. Buffy coat fractions are obtained e.g. from a blood bank or directly from individuals preferably from a large number of individuals such that bias from individual samples is avoided and so that the RNA sample represents an average expression of a healthy population. Buffy coat fractions from about 50 or about 100 or more individuals are preferred. 10 ml buffy coat from each individual is used. Buffy coat samples are treated with an erthythrocyte lysis buffer so that erthythrocytes are selectively removed. The leukocytes of the buffy coat layer are collected by centrifugation. Alternatively the buffy cell sample can be further enriched for a particular leukocyte sub populations e.g. mononuclear cells T lymphocytes etc. To enrich for mononuclear cells the buffy cell pellet above is diluted in PBS phosphate buffered saline and loaded onto a non polystyrene tube containing a polysucrose and sodium diatrizoate solution adjusted to a density of 1.077 0.001 g ml. To enrich for T lymphocytes 45 ml of whole blood is treated with RosetteSep Stem Cell Technologies and incubated at room temperature for 20 minutes. The mixture is diluted with an equal volume of PBS plus 2 FBS and mixed by inversion. 30 ml of diluted mixture is layered on top of 15 ml DML medium Stem Cell Technologies . The tube is centrifuged at 1200 g and the enriched cell layer at the plasma medium interface is removed washed with PBS 2 FBS and cells collected by centrifugation at 1200 g. The cell pellet is treated with 5 ml of erythrocyte lysis buffer EL buffer Qiagen for 10 minutes on ice and enriched T lymphoctes are collected by centrifugation.

In addition or alternatively the buffy cells whole buffy coat or sub population e.g. mononuclear fraction can be cultured in vitro and subjected to stimulation with cytokines or activating chemicals such as phorbol esters or ionomycin. Such stimuli may increase expression of nucleotide sequences that are expressed in activated immune cells and might be of interest for leukocyte expression profiling experiments.

Following sub population selection and or further treatment e.g. stimulation as described above RNA is prepared using standard methods. For example cells are pelleted and lysed with a phenol guanidinium thiocyanate and RNA is prepared. RNA can also be isolated using a silica gel based purification column or the column method can be used on RNA isolated by the phenol guanidinium thiocyanate method. RNA from individual buffy coat samples can be pooled during this process so that the resulting reference RNA represents the RNA of many individuals and individual bias is minimized or eliminated. In addition a new batch of buffy coat reference RNA can be directly compared to the last batch to ensure similar expression pattern from one batch to another using methods of collecting and comparing expression profiles described above below. One or more expression reference controls are used in an experiment. For example RNA derived from one or more of the following sources can be used as controls for an experiment stimulated or unstimulated whole buffy coat stimulated or unstimulated peripheral mononuclear cells or stimulated or unstimulated T lymphocytes.

Alternatively the expression reference standard can be derived from any subject or class of subjects including healthy subjects or subjects diagnosed with the same or a different disease or disease criterion. Expression profiles from subjects in two or more distinct classes are compared to determine which subset of nucleotide sequences in the candidate library can best distinguish between the subject classes as further discussed below. It will be appreciated that in the present context the term distinct classes is relevant to at least one distinguishable criterion relevant to a disease of interest a disease criterion. The classes can of course demonstrate significant overlap or identity with respect to other disease criteria or with respect to disease diagnoses prognoses or the like. The mode of discovery involves e.g. comparing the molecular signature of different subject classes to each other such as patient to control patients with a first diagnosis to patients with a second diagnosis etc. or by comparing the molecular signatures of a single individual taken at different time points. The invention can be applied to a broad range of diseases disease criteria conditions and other clinical and or epidemiological questions as further discussed above below.

It is appreciated that while the present discussion pertains to the use of expression reference controls while identifying diagnostic nucleotide sets expression reference controls are also useful during use of diagnostic nucleotide sets e.g. use of a diagnostic nucleotide set for diagnosis of a disease as further described below.

In order to facilitate ready access e.g. for comparison review recovery and or modification the molecular signatures expression profiles are typically recorded in a database. Most typically the database is a relational database accessible by a computational device although other formats e.g. manually accessible indexed files of expression profiles as photographs analogue or digital imaging readouts spreadsheets etc. can be used. Further details regarding preferred embodiments are provided below. Regardless of whether the expression patterns initially recorded are analog or digital in nature and or whether they represent quantitative or qualitative differences in expression the expression patterns expression profiles collective expression patterns and molecular signatures correlated expression patterns are stored digitally and accessed via a database. Typically the database is compiled and maintained at a central facility with access being available locally and or remotely.

As additional samples are obtained and their expression profiles determined and correlated with relevant subject data the ensuing molecular signatures are likewise recorded in the database. However rather than each subsequent addition being added in an essentially passive manner in which the data from one sample has little relation to data from a second prior or subsequent sample the algorithms optionally additionally query additional samples against the existing database to further refine the association between a molecular signature and disease criterion. Furthermore the data set comprising the one or more molecular signatures is optionally queried against an expanding set of additional or other disease criteria. The use of the database in integrated systems and web embodiments is further described below.

Expression data is analyzed using methods well known in the art including the software packages Imagene Biodiscovery Marina del Rey Calif. Feature Extraction Software Agilent Palo Alto Calif. and Scanalyze Stanford University . In the discussion that follows a feature refers to an individual spot of DNA on an array. Each gene may be represented by more than one feature. For example hybridized microarrays are scanned and analyzed on an Axon Instruments scanner using GenePix 3.0 software Axon Instruments Union City Calif. . The data extracted by GenePix is used for all downstream quality control and expression evaluation. The data is derived as follows. The data for all features flagged as not found by the software is removed from the dataset for individual hybridizations. The not found flag by GenePix indicates that the software was unable to discriminate the feature from the background. Each feature is examined to determine the value of its signal. The median pixel intensity of the background B is subtracted from the median pixel intensity of the feature F to produce the background subtracted signal hereinafter BGSS . The BGSS is divided by the standard deviation of the background pixels to provide the signal to noise ratio hereinafter S N . Features with a S N of three or greater in both the Cy3 channel corresponding to the sample RNA and Cy5 channel corresponding to the reference RNA are used for further analysis hereinafter denoted useable features . Alternatively different S Ns are used for selecting expression data for an analysis. For example only expression data with signal to noise ratios 3 might be used in an analysis. Alternatively features with S N values 

For each usable feature i the expression level e is expressed as the logarithm of the ratio R of the Background Subtracted Signal hereinafter BGSS for the Cy3 sample RNA channel divided by the BGSS for the Cy5 channel reference RNA . This log ratio value is used for comparison to other experiments.

Variation in signal across hybridizations may be caused by a number of factors affecting hybridization DNA spotting wash conditions and labeling efficiency.

A single reference RNA may be used with all of the experimental RNAs permitting multiple comparisons in addition to individual comparisons. By comparing sample RNAs to the same reference the gene expression levels from each sample are compared across arrays permitting the use of a consistent denominator for our experimental ratios.

The data may be scaled normalized to control for labeling and hybridization variability within the experiment using methods known in the art. Scaling is desirable because it facilitates the comparison of data between different experiments patients etc. Generally the BGSS are scaled to a factor such as the median the mean the trimmed mean and percentile. Additional methods of scaling include to scale between 0 and 1 to subtract the mean or to subtract the median.

Scaling is also performed by comparison to expression patterns obtained using a common reference RNA as described in greater detail above. As with other scaling methods the reference RNA facilitates multiple comparisons of the expression data e.g. between patients between samples etc. Use of a reference RNA provides a consistent denominator for experimental ratios.

In addition to the use of a reference RNA individual expression levels may be adjusted to correct for differences in labeling efficiency between different hybridization experiments allowing direct comparison between experiments with different overall signal intensities for example. A scaling factor a may be used to adjust individual expression levels as follows. The median of the scaling factor a for example BGSS is determined for the set of all features with a S N greater than three. Next the BGSS the BGSS for each feature i is divided by the median for all features a generating a scaled ratio. The scaled ration is used to determine the expression value for the feature e or the log ratio.

In addition or alternatively control features are used to normalize the data for labeling and hybridization variability within the experiment. Control feature may be cDNA for genes from the plant that are included when spotting the mini array. Equal amounts of RNA complementary to control cDNAs are added to each of the samples before they were labeled. Using the signal from these control genes a normalization constant L is determined according to the following formula 

Using the formula above the mean for all control features of a particular hybridization and dye e.g. Cy3 is calculated. The control feature means for all Cy3 hybridizations are averaged and the control feature mean in one hybridization divided by the average of all hybridizations to generate a normalization constant for that particular Cy3 hybridization L which is used as a in equation 0.3 . The same normalization steps may be performed for Cy3 and Cy5 values.

Many additional methods for normalization exist and can be applied to the data. In one method the average ratio of Cy3 BGSS Cy5 BGSS is determined for all features on an array. This ratio is then scaled to some arbitrary number such as 1 or some other number. The ratio for each probe is then multiplied by the scaling factor required to bring the average ratio to the chosen level. This is performed for each array in an analysis. Alternatively the ratios are normalized to the average ratio across all arrays in an analysis.

If multiple features are used per gene sequence or oligonucleotide these repeats can be used to derive an average expression value for each gene. If some of the replicate features are of poor qualitay and don t meet requirements for analysis the remaining features can be used to represent the gene or gene sequence.

Correlation analysis is performed to determine which array probes have expression behavior that best distinguishes or serves as markers for relevant groups of samples representing a particular clinical condition. Correlation analysis or comparison among samples representing different disease criteria e.g. clinical conditions is performed using standard statistical methods. Numerous algorithms are useful for correlation analysis of expression data and the selection of algorithms depends in part on the data analysis to be performed. For example algorithms can be used to identify the single most informative gene with expression behavior that reliably classifies samples or to identify all the genes useful to classify samples. Alternatively algorithms can be applied that determine which set of 2 or more genes have collective expression behavior that accurately classifies samples. The use of multiple expression markers for diagnostics may overcome the variability in expression of a gene between individuals or overcome the variability intrinsic to the assay. Multiple expression markers may include redundant markers surrogates in that two or more genes or probes may provide the same information with respect to diagnosis. This may occur for example when two or more genes or gene probes are coordinately expressed. For diagnostic application it may be appropriate to utilize a gene and one or more of its surrogates in the assay. This redundancy may overcome failures technical or biological of a single marker to distinguish samples. Alternatively one or more surrogates may have properties that make them more suitable for assay development such as a higher baseline level of expression better cell specificity a higher fold change between sample groups or more specific sequence for the design of PCR primers or complimentary probes. It will be appreciated that while the discussion above pertains to the analysis of RNA expression profiles the discussion is equally applicable to the analysis of profiles of proteins or other molecular markers.

Prior to analysis expression profile data may be formatted or prepared for analysis using methods known in the art. For example often the log ratio of scaled expression data for every array probe is calculated using the following formula 

log Cy 3 BGSS Cy5 BGSS where Cy 3 signal corresponds to the expression of the gene in the clinical sample and Cy5 signal corresponds to expression of the gene in the reference RNA.

Data may be further filtered depending on the specific analysis to be done as noted below. For example filtering may be aimed at selecting only samples with expression above a certain level or probes with variability above a certain level between sample sets.

The following non limiting discussion consider several statistical methods known in the art. Briefly the t test and ANOVA are used to identify single genes with expression differences between or among populations respectively. Multivariate methods are used to identify a set of two or more genes for which expression discriminates between two disease states more specifically than expression of any single gene.

The simplest measure of a difference between two groups is the Student s t test. See e.g. Welsh et al. 2001 98 1176 81 demonstrating the use of an unpaired Student s t test for the discovery of differential gene expression in ovarian cancer samples and control tissue samples . The t test assumes equal variance and normally distributed data. This test identifies the probability that there is a difference in expression of a single gene between two groups of samples. The number of samples within each group that is required to achieve statistical significance is dependent upon the variation among the samples within each group. The standard formula for a t test is 

The combination of the t statistic and the degrees of freedom min n n 1 provides a p value the probability of rejecting the null hypothesis. A p value of 0.01 signifying a 99 percent probability the mean expression levels are different between the two groups a 1 chance that the mean expression levels are in fact not different and that the observed difference occurred by statistical chance is often considered acceptable.

When performing tests on a large scale for example on a large dataset of about 8000 genes a correction factor must be included to adjust for the number of individual tests being performed. The most common and simplest correction is the Bonferroni correction for multiple tests which divides the p value by the number of tests run. Using this test on an 8000 member dataset indicates that a p value of 0.00000125 is required to identify genes that are likely to be truly different between the two test conditions.

Significance analysis for microarrays SAM Tusher 2001 is a method through which genes with a correlation between their expression values and the response vector are statistically discovered and assigned a statistical significance. The ratio of false significant to significant genes is the False Discovery Rate FDR . This means that for each threshold there are a set of genes which are called significant and the FDR gives a confidence level for this claim. If a gene is called differentially expressed between 2 classes by SAM with a FDR of 5 there is a 95 chance that the gene is actually differentially expressed between the classes. SAM takes into account the variability and large number of variables of microarrays. SAM will identity genes that are most globally differentially expressed between the classes. Thus important genes for identifying and classifying outlier samples or patients may not be identified by SAM.

This method is non parametric and is utilized for paired comparisons. See e.g. Sokal and Rohlf 1987 2edition WH Freeman New York. At least 6 pairs are necessary to apply this statistic. This test is useful for analysis of paired expression data for example a set of patients who have had samples taken before and after administration of a pharmacologic agent .

Differences in gene expression across multiple related groups may be assessed using an Analysis of Variance ANOVA a method well known in the art Michelson and Schofield 1996 .

Many algorithms suitable for multivariate analysis are known in the art Katz 1999 . Generally a set of two or more genes for which expression discriminates between two disease states more specifically than expression of any single gene is identified by searching through the possible combinations of genes using a criterion for discrimination for example the expression of gene X must increase from normal 300 percent while the expression of genes Y and Z must decrease from normal by 75 percent. Ordinarily the search starts with a single gene then adds the next best fit at each step of the search. Alternatively the search starts with all of the genes and genes that do not aid in the discrimination are eliminated step wise.

Paired samples or samples collected at different time points from the same patient are often useful as described above. For example use of paired samples permits the reduction of variation due to genetic variation among individuals. In addition the use of paired samples has a statistical significance in that data derived from paired samples can be calculated in a different manner that recognizes the reduced variability. For example the formula for a t test for paired samples is 

Once a discriminating set of genes is identified the diagnostic classifier a mathematical function that assigns samples to diagnostic categories based on expression data is applied to unknown sample expression levels.

CLEAVER is an algorithm used for classification of useful expression profile data. See Raychaudhuri et al. 2001 19 189 193. CLEAVER uses positive training samples e.g. expression profiles from samples known to be derived from a particular patient or sample diagnostic category disease or disease criteria negative training samples e.g. expression profiles from samples known not to be derived from a particular patient or sample diagnostic category disease or disease criteria and test samples e.g. expression profiles obtained from a patient and determines whether the test sample correlates with the particular disease or disease criteria or does not correlate with a particular disease or disease criteria. CLEAVER also generates a list of the 20 most predictive genes for classification.

Artificial neural networks hereinafter ANN can be used to recognize patterns in complex data sets and can discover expression criteria that classify samples into more than 2 groups. The use of artificial neural networks for discovery of gene expression diagnostics for cancers using expression data generated by oligonucleotide expression microarrays is demonstrated by Khan et al. 2001 7 673 9. Khan found that 96 genes provided 0 error rate in classification of the tumors. The most important of these genes for classification was then determined by measuring the sensitivity of the classification to a change in expression of each gene. Hierarchical clustering using the 96 genes results in correct grouping of the cancers into diagnostic categories.

Golub uses cDNA microarrays and a distinction calculation to identify genes with expression behavior that distinguishes myeloid and lymphoid leukemias. See Golub et al. 1999 286 531 7. Self organizing maps were used for new class discovery. Cross validation was done with a leave one out analysis. 50 genes were identified as useful markers. This was reduced to as few as 10 genes with equivalent diagnostic accuracy.

Hierarchical and non hierarchical clustering methods are also useful for identifying groups of genes that correlate with a subset of clinical samples such as those with and without Lupus. Alizadeh used hierarchical clustering as the primary tool to distinguish different types of diffuse B cell lymphomas based on gene expression profile data. See Alizadeh et al. 2000 403 503 11. Alizadeh used hierarchical clustering as the primary tool to distinguish different types of diffuse B cell lymphomas based on gene expression profile data. A cDNA array carrying 17856 probes was used for these experiments 96 samples were assessed on 128 arrays and a set of 380 genes was identified as being useful for sample classification.

Perou demonstrates the use of hierarchical clustering for the molecular classification of breast tumor samples based on expression profile data. See Perou et al. 2000 406 747 52. In this work a cDNA array carrying 8102 gene probes was used. 1753 of these genes were found to have high variation between breast tumors and were used for the analysis.

Hastie describes the use of gene shaving for discovery of expression markers. Hastie et al. 2000 1 2 RESEARCH 0003.1 0003.21. The gene shaving algorithm identifies sets of genes with similar or coherent expression patterns but large variation across conditions RNA samples sample classes patient classes . In this manner genes with a tight expression pattern within a diagnostic group but also with high variability across the diagnoses are grouped together. The algorithm takes advantage of both characteristics in one grouping step. For example gene shaving can identify useful marker genes with co regulated expression. Sets of useful marker genes can be reduced to a smaller set with each gene providing some non redundant value in classification. This algorithm was used on the data set described in Alizadeh et al. supra and the set of 380 informative gene markers was reduced to 234.

Supervised harvesting of expression trees Hastie 2001 identifies genes or clusters that best distinguish one class from all the others on the data set. The method is used to identify the genes clusters that can best separate one class versus all the others for datasets that include two or more classes or all classes from each other. This algorithm can be used for discovery or testing of a diagnostic gene set.

CART is a decision tree classification algorithm Breiman 1984 . From gene expression and or other data CART can develop a decision tree for the classification of samples. Each node on the decision tree involves a query about the expression level of one or more genes or variables. Samples that are above the threshold go down one branch of the decision tree and samples that are not go down the other branch. See examples 10 and 16 for further description of its use in classification analysis and examples of its usefulness in discovering and implementing a diagnostic gene set. CART identifies surrogates for each splitter genes that are the next best substitute for a useful gene inclassification.

Once a set of genes and expression criteria for those genes have been established for classification cross validation is done. There are many approaches including a 10 fold cross validation analysis in which 10 of the training samples are left out of the analysis and the classification algorithm is built with the remaining 90 . The 10 are then used as a test set for the algorithm. The process is repeated 10 times with 10 of the samples being left out as a test set each time. Through this analysis one can derive a cross validation error which helps estimate the robustness of the algorithm for use on prospective test samples.

Clinical data are gathered for every patient sample used for expression analysis. Clinical variables can be quantitative or non quantitative. A clinical variable that is quantitiative can be used as a variable for significance or classification analysis. Non quantitative clinical variables such as the sex of the patient can also be used in a significance analysis or classification analysis with some statistical tool. It is appreciated that the most useful diagnostic gene set for a condition may be optimal when considered along with one or more predictive clinical variables. Clinical data can also be used as supervising vectors for a correlation analysis. That is to say that the clinical data associated with each sample can be used to divide the samples into meaningful diagnostic categories for analysis. For example samples can be divided into 2 or more groups based on the presence or absence of some diagnostic criterion a . In addition clinical data can be utilized to select patients for a correlation analysis or to exclude them based on some undesirable characteristic such as an ongoing infection a medicine or some other issue. Clincial data can also be used to assess the pre test probability of an outcome. For example patients who are female are much more likely to be diagnosed as having systemic lupus erythematosis than patients who are male.

Once a set of genes are identified that classify samples with acceptable accuracy. These genes are validated as a set using new samples that were not used to discover the gene set. These samples can be taken from frozen archieves from the discovery clinical study or can be taken from new patients prospectively. Validation using a test set of samples can be done using expression profiling of the gene set with microarrays or using real time PCR for each gene on the test set samples. Alternatively a different expression profiling technology can be used.

Prior to widespread application of the diagnostic probe sets of the invention the predictive value of the probe set is validated. When the diagnostic probe set is discovered by microarray based expression analysis the differential expression of the member genes may be validated by a less variable and more quantitive and accurate technology such as real time PCR. In this type of experiment the amplification product is measured during the PCR reaction. This enables the researcher to observe the amplification before any reagent becomes rate limiting for amplification. In kinetic PCR the measurement is of C threshold cycle or C crossing point . This measurement C C is the point at which an amplification curve crosses a threshold fluorescence value. The threshold is set to a point within the area where all of the reactions were in their linear phase of amplification. When measuring C a lower Cvalue is indicative of a higher amount of starting material since an earlier cycle number means the threshold was crossed more quickly.

Several fluorescence methodologies are available to measure amplification product in real time PCR. Taqman Applied BioSystems Foster City Calif. uses fluorescence resonance energy transfer FRET to inhibit signal from a probe until the probe is degraded by the sequence specific binding and Taq 3 exonuclease activity. Molecular Beacons Stratagene La Jolla Calif. also use FRET technology whereby the fluorescence is measured when a hairpin structure is relaxed by the specific probe binding to the amplified DNA. The third commonly used chemistry is Sybr Green a DNA binding dye Molecular Probes Eugene Oreg. . The more amplified product that is produced the higher the signal. The Sybr Green method is sensitive to non specific amplification products increasing the importance of primer design and selection. Other detection chemistries can also been used such as ethedium bromide or other DNA binding dyes and many modifications of the fluorescent dye quencher dye Taqman chemistry for example scorpions.

Typically the oligonucleotide sequence of each probe is confirmed e.g. by DNA sequencing using an oligonucleotide specific primer. Partial sequence obtained is generally sufficient to confirm the identity of the oligonucleotide probe. Alternatively a complementary polynucleotide is fluorescently labeled and hybridized to the array or to a different array containing a resynthesized version of the oligo nucleotide probe and detection of the correct probe is confirmed.

Typically validation is performed by statistically evaluating the accuracy of the correspondence between the molecular signature for a diagnostic probe set and a selected indicator. For example the expression differential for a nucleotide sequence between two subject classes can be expressed as a simple ratio of relative expression. The expression of the nucleotide sequence in subjects with selected indicator can be compared to the expression of that nucleotide sequence in subjects without the indicator as described in the following equations. the average expression of nucleotide sequence in the members of group the average expression of nucleotide sequence in the members of group the average differential expression of nucleotide sequence

where indicates a sum Ex is the expression of nucleotide sequence x relative to a standard ai are the individual members of group A group A has N members bi are the individual members of group B group B has M members.

Individual components of a diagnostic probe set each have a defined sensitivity and specificity for distinguishing between subject groups. Such individual nucleotide sequences can be employed in concert as a diagnostic probe set to increase the sensitivity and specificity of the evaluation. The database of molecular signatures is queried by algorithms to identify the set of nucleotide sequences i.e. corresponding to members of the probe set with the highest average differential expression between subject groups. Typically as the number of nucleotide sequences in the diagnostic probe set increases so does the predictive value that is the sensitivity and specificity of the probe set. When the probe sets are defined they may be used for diagnosis and patient monitoring as discussed below. The diagnostic sensitivity and specificity of the probe sets for the defined use can be determined for a given probe set with specified expression levels as demonstrated above. By altering the expression threshold required for the use of each nucleotide sequence as a diagnostic the sensitivity and specificity of the probe set can be altered by the practitioner. For example by lowering the magnitude of the expression differential threshold for each nucleotide sequence in the set the sensitivity of the test will increase but the specificity will decrease. As is apparent from the foregoing discussion sensitivity and specificity are inversely related and the predictive accuracy of the probe set is continuous and dependent on the expression threshold set for each nucleotide sequence. Although sensitivity and specificity tend to have an inverse relationship when expression thresholds are altered both parameters can be increased as nucleotide sequences with predictive value are added to the diagnostic nucleotide set. In addition a single or a few markers may not be reliable expression markers across a population of patients. This is because of the variability in expression and measurement of expression that exists between measurements individuals and individuals over time. Inclusion of a large number of candidate nucleotide sequences or large numbers of nucleotide sequences in a diagnostic nucleotide set allows for this variability as not all nucleotide sequences need to meet a threshold for diagnosis. Generally more markers are better than a single marker. If many markers are used to make a diagnosis the likelihood that all expression markers will not meet some thresholds based upon random variability is low and thus the test will give fewer false negatives. Surrogate markers are useful for these purposes. These are markers or genes that are coordinately expressed. Surrogate markers essential provide redundant information but this redundancy can improve accuracy by decreasing errors due to assay variability.

It is appreciated that the desired diagnostic sensitivity and specificity of the diagnostic nucleotide set may vary depending on the intended use of the set. For example in certain uses high specificity and high sensitivity are desired. For example a diagnostic nucleotide set for predicting which patient population may experience side effects may require high sensitivity so as to avoid treating such patients. In other settings high sensitivity is desired while reduced specificity may be tolerated. For example in the case of a beneficial treatment with few side effects it may be important to identify as many patients as possible high sensitivity who will respond to the drug and treatment of some patients who will not respond is tolerated. In other settings high specificity is desired and reduced sensitivity may be tolerated. For example when identifying patients for an early phase clinical trial it is important to identify patients who may respond to the particular treatment. Lower sensitivity is tolerated in this setting as it merely results in reduced patients who enroll in the study or requires that more patients are screened for enrollment.

To discover and validate a gene set that can be applied to accurately diagnose or classify patients across the country or around the world it is necessary to ensure that the gene set was developed and validated using samples that represent the types of patients that will be encountered in the clinical setting. For example diverse ethnicity drug usage and clinical practice patterns must all be represented in the discovery and validation to ensure that the test works on this variety of patients.

In principle individual oligonucleotides and diagnostic oligonucleotide sets of the invention may be developed and applied to essentially any disease or disease criterion as long as at least one subset of oligonucleotide sequences is differentially expressed in samples derived from one or more individuals with a disease criteria or disease and one or more individuals without the disease criteria or disease wherein the individual may be the same individual sampled at different points in time or the individuals may be different individuals or populations of individuals . For example the subset of oligonucleotide sequences may be differentially expressed in the sampled tissues of subjects with the disease or disease criterion e.g. a patient with a disease or disease criteria as compared to subjects without the disease or disease criterion e.g. patients without a disease control patients . Alternatively or in addition the subset of oligonucleotide sequence s may be differentially expressed in different samples taken from the same patient e.g at different points in time at different disease stages before and after a treatment in the presence or absence of a risk factor etc.

Expression profiles corresponding to oligonucleotides and sets of oligonucleotide sequences that correlate not with a diagnosis but rather with a particular aspect of a disease can also be used to identify the diagnostic oligonucleotide sets and disease specific target oligonucleotide sequences of the invention. For example such an aspect or disease criterion can relate to a subject s medical or family history e.g. occurance of an autoimmune disease childhood illness cause of death of a parent or other relative prior surgery or other intervention medications laboratory values and results of diagnostic testing radiology pathology etc. symptoms including onset and or duration of symptoms etc. Alternatively the disease criterion can relate to a diagnosis e.g. chronic inflammatory disease such as lupus rheumatoid arthritis osteoarthritis or prognosis e.g. prediction of future diagnoses events or complications e.g. renal failure from lupus joint replacement surgery for rheumatoid arthritis rheumatoid arthritis or systemic lupus erythematosis disease activity or the like. In other cases the disease criterion corresponds to a therapeutic outcome e.g. response to a medication response to a surgery or physical therapy for a joint. Alternatively the disease criteria correspond with previously identified or classic risk factors and may correspond to prognosis or future disease diagnosis. As indicated above a disease criterion can also correspond to genotype for one or more loci. Disease criteria including patient data may be collected and compared from the same patient at different points in time from different patients between patients with a disease criterion and patients respresenting a control population etc. Longitudinal data i.e. data collected at different time points from an individual or group of individuals may be used for comparisons of samples obtained from an individual group of individuals at different points in time to permit identification of differences specifically related to the disease state and to obtain information relating to the change in expression over time including a rate of change or trajectory of expression over time. The usefulness of longitudinal data is further discussed in the section titled Identification of diagnostic nucleotide sets of the invention .

It is further understood that diagnostic oligonucleotides and oligonucleotide sets may be developed for use in diagnosing conditions for which there is no present means of diagnosis. For example in rheumatoid arthritis joint destruction is often well under way before a patient experience symptoms of the condition. A diagnostic nucleotide or nucleotide set may be developed that diagnoses rheumatic joint destruction at an earlier stage than would be possible using present means of diagnosis which rely in part on the presentation of symptoms by a patient. Diagnostic nucleotide sets may also be developed to replace or augment current diagnostic procedures. For example the use of a diagnostic nucleotide or nucleotide set to diagnose lupus may replace or supplement the current diagnostic tests and strategies.

It is understood that the following discussion of diseases is exemplary and non limiting and further that the general criteria discussed above e.g. use of family medical history are generally applicable to the specific diseases discussed below.

In addition to leukocytes as described throughout the general method is applicable to oligonucleotide sequences that are differentially expressed in any subject tissue or cell type by the collection and assessment of samples of that tissue or cell type. However in many cases collection of such samples presents significant technical or medical problems given the current state of the art.

SLE is a chronic systemic inflammatory disease characterized by dysregulation of the immune system which effects up to 2 million patients in the US. Symptoms of SLE include rashes joint pain abnormal blood counts renal dysfunction and damage infections CNS disorders arthralgias and autoimmunity. Patients may also have early onset atherosclerosis. The diagnosis of SLE is difficult to make with certainty using current diagnostic tests and algorithms. Antibody tests can be specific for the disease but often lack sensitivity. Clinical diagnosis may lack both high sensitivity and specificity. SLE is a disease that clearly involves differential gene expression in leukocytes compared to patients without the disease.

Diagnostic oligonucleotides and oligonucleotide sets are identified and validated for use in diagnosis and monitoring of SLE activity and progression. Disease criteria correspond to clinical data e.g. symptom rash joint pain malaise rashes blood counts white and red tests of renal function e.g. creatinine blood urea nitrogen hereinafter bun creative clearance data obtained from laboratory tests including complete blood counts with differentials CRP ESR ANA Serum IL6 Soluble CD40 ligand LDL HDL Anti DNA antibodies rheumatoid factor C3 C4 serum creatinine and any medication levels the need for pain medications cumulative doses or immunosuppressive therapy symptoms or any manifestation of carotid atherosclerosis e.g. ultrasound diagnosis or any other manifestations of the disease data from surgical procedures such as gross operative findings and pathological evaluation of resected tissues and biopsies e.g. renal CNS information on pharmacological therapy and treatment changes clinical diagnoses of disease flare hospitalizations death response to medications quantitative joint exams results from health assessment questionnaires HAQs and other clinical measures of patient symptoms and disability. In addition disease criteria correspond to the clinical score known as SLEDAI Bombadier C Gladman D D Urowitz M B Caron D Chang C H and the Committee on Prognosis Studies in SLE Derivation of the SLEDAI for Lupus Patients. 35 630 640 1992. . Diagnostic nucleotide sets may be useful for diagnosis of SLE monitoring disease progression including progressive renal dysfunction carotid atherosclerosis and CNS dysfunction and predicting occurrence of side effects for example.

Rheumatoid arthritis RA effects about two million patients in the US and is a chronic and debilitating inflammatory arthritis particularly involving pain and destruction of the joints. RA often goes undiagnosed because patients may have no pain but the disease is actively destroying the joint. Other patients are known to have RA and are treated to alleviate symptoms but the rate of progression of joint destruction can t easily be monitored. Drug therapy is available but the most effective medicines are toxic e.g. steroids methotrexate and thus need to be used with caution. A new class of medications TNF blockers is very effective but the drugs are expensive have side effects and not all patients respond. Side effects are common and include immune suppression toxicity to organ systems allergy and metabolic disturbances.

Diagnostic oligonucleotides and oligonucleotide sets of the invention are developed and validated for use in diagnosis and treatment of RA. Disease criteria correspond to disease symptoms e.g. joint pain joint swelling and joint stiffness and any of the American College for Rheumatology criteria for the diagnosis of RA see Arnett et al 1988 31 315 24 progression of joint destruction e.g. as measured by serial hand radiographs assessment of joint function and mobility surgery need for medication additional diagnoses of inflammatory and non inflammatory conditions and clinical laboratory measurements including complete blood counts with differentials CRP ESR ANA Serum IL6 Soluble CD40 ligand LDL HDL Anti DNA antibodies rheumatoid factor C3 C4 serum creatinine death hospitalization and disability due to joint destruction. In addition or alternatively disease criteria correspond to response to drug therapy and presence or absence of side effects or measures of improvement exemplified by the American College of Rheumatology 20 and 50 response improvement rates. See Felson et al 1995 38 531 37. Diagnostic nucleotide sets are identified that monitor and predict disease progression including flaring acute worsening of disease accompanied by joint pain or other symptoms response to drug treatment and likelihood of side effects.

In addition to peripheral leukocytes surgical specimens of rheumatoid joints can be used for leukocyte expression profiling experiments. Members of diagnostic nucleotide sets are candidates for leukocyte target nucleotide sequences e.g. as a candidate drug target for rheumatoid arthritis. Synovial specimens can be used for expression profiling or cells derived and sorted from that tissue such as subsets of leukocytes can be used. Cells can be separated by fluorescence activated cell sorting or magnetic affinity reagent techniques or some other technique. Synovial specimens and blood can be obtained from the same patient and gene expression can be compared between these 2 sample types.

20 40 million patients in the US have osteoarthritis. Patient groups are heterogeneous with a subset of patients having earlier onset more aggressive joint damage involving more inflammation leukocyte infiltration . Leukocyte diagnostics can be used to distinguish osteoarthritis from rheumatoid arthritis and other differntial diagnoses define likelihood and degree of response to NSAID therapy non steroidal anti inflammatory drugs or other anti inflammatory therapies. Rate of progression of joint damage can also be assessed. Diagnostic nucleotide sets may be developed for use in selection and titration of treatment therapies. Disease criteria correspond to response to therapy and disease progression using certain therapies response to medications need for joint surgery joint pain and disability.

In addition to peripheral leukocytes surgical specimens of osteoarthritic joints can be used for leukocyte expression profiling experiments. Diagnostic oligonucleotides and diagnostic oligonucleotide sets are candidates for leukocyte target nucleotide sequences e.g. as a candidate drug target for osteoarthritis. Synovial specimens can be used for expression profiling or cells derived and sorted from that tissue such as subsets of leukocytes can be used. Cells can be separated by fluorescence activated cell sorting or magnetic affinity reagent techniques or some other technique. Synovial specimens and blood can be obtained from the same patient and gene expression can be compared between these 2 sample types.

In another example diagnostic nucleotide sets are developed and validated for use in diagnosis and therapy of peri prosthetic osteolysis. In this disease a prosthetic joint such as a knee or hip is found to loosen over time and requires repeat surgery. Loosening may occur in some patients due to an inflammatory response incited by the foreign material of the prosthesis. Disease criteria include joint loosening radiographic evidence of peri prosthetic osteolysis need for repeat surgery response to pharmacological therapy and or histological from biopsy or surgery or biochemical markers of bone metabolism such as alkaline phosphatase evidence of osteolysis. Tissues used for expression profiling can include peripheral leukocytes or leukocyte subsets periprosthetic tissue or synovial fluid. In addition gene sets can be discovered using an in vitro model of the disease in which immune cells are exposed to prosthesis materials such as cement or titanium.

Pharmocogenomics is the study of the individual propensity to respond to a particular drug therapy combination of therapies . In this context response can mean whether a particular drug will work on a particular patient e.g. some patients respond to one drug but not to another drug. Response can also refer to the likelihood of successful treatment or the assessment of progress in treatment. Titration of drug therapy to a particular patient is also included in this description e.g. different patients can respond to different doses of a given medication. This aspect may be important when drugs with side effects or interactions with other drug therapies are contemplated.

Diagnostic oligonucleotides and oligonucleotide sets are developed and validated for use in assessing whether a patient will respond to a particular therapy and or monitoring response of a patient to drug therapy therapies . Disease criteria correspond to presence or absence of clinical symptoms or clinical endpoints presence of side effects or interaction with other drug s . The diagnostic nucleotide set may further comprise nucleotide sequences that are targets of drug treatment or markers of active disease.

Diagnostic oligonucleotides and oligonucleotide sets are developed and validated for use in assessing whether a patient has a particular drug toxicity or toxicity due to an environmental work related or other agent. Such exposures of the patient may also be related to biological or biochemical agents used in warfare. Diagnostic oligonucleotides and oligonucleotide sets may allow early diagnosis of a toxicity or exposure or may monitor the severity and course of toxic responses.

The invention also provide methods of using the diagnostic oligonucleotides and oligonucleotide sets to diagnose or monitor disease assess severity of disease predict future occurrence of disease predict future complications of disease determine disease prognosis evaluate the patient s risk or stratify a group of patients assess response to current drug therapy assess response to current non pharmacological therapy determine the most appropriate medication or treatment for the patient predict whether a patient is likely to respond to a particular drug and determine most appropriate additional diagnostic testing for the patient among other clinically and epidemiologically relevant applications.

The oligonucleotides and oligonucleotide sets of the invention can be utilized for a variety of purposes by physicians healthcare workers hospitals laboratories patients companies and other institutions. As indicated previously essentially any disease condition or status for which at least one nucleotide sequence is differentially expressed in leukocyte populations or sub populations can be evaluated e.g. diagnosed monitored etc. using the diagnostic nucleotide sets and methods of the invention. In addition to assessing health status at an individual level the diagnostic nucleotide sets of the present invention are suitable for evaluating subjects at a population level e.g. for epidemiological studies or for population screening for a condition or disease.

RNA protein and or DNA are prepared using methods well known in the art as further described herein. It is appreciated that subject samples collected for use in the methods of the invention are generally collected in a clinical setting where delays may be introduced before RNA samples are prepared from the subject samples of whole blood e.g. the blood sample may not be promptly delivered to the clinical lab for further processing. Further delay may be introduced in the clinical lab setting where multiple samples are generally being processed at any given time. For this reason methods that feature lengthy incubations of intact leukocytes at room temperature are not preferred because the expression profile of the leukocytes may change during this extended time period. For example RNA can be isolated from whole blood using a phenol guanidine isothiocyanate reagent or another direct whole blood lysis method as described in e.g. U.S. Pat. Nos. 5 346 994 and 4 843 155. This method may be less preferred under certain circumstances because the large majority of the RNA recovered from whole blood RNA extraction comes from erythrocytes since these cells outnumber leukocytes 1000 1. Care must be taken to ensure that the presence of erythrocyte RNA and protein does not introduce bias in the RNA expression profile data or lead to inadequate sensitivity or specificity of probes.

Alternatively intact leukocytes may be collected from whole blood using a lysis buffer that selectively lyses erythrocytes but not leukocytes as described e.g. in U.S. Pat. Nos. 5 973 137 and 6 020 186 . Intact leukocytes are then collected by centrifugation and leukocyte RNA is isolated using standard protocols as described herein. However this method does not allow isolation of sub populations of leukocytes e.g. mononuclear cells which may be desired. In addition the expression profile may change during the lengthy incubation in lysis buffer especially in a busy clinical lab where large numbers of samples are being prepared at any given time.

Alternatively specific leukocyte cell types can be separated using density gradient reagents Boyum A 1968. . For example mononuclear cells may be separated from whole blood using density gradient centrifugation as described e.g. in U.S. Pat. Nos. 4 190 535 4 350 593 4 751 001 4 818 418 and 5 053 134. Blood is drawn directly into a tube containing an anticoagulant and a density reagent such as Ficoll or Percoll . Centrifugation of this tube results in separation of blood into an erythrocyte and granulocyte layer a mononuclear cell suspension and a plasma layer. The mononuclear cell layer is easily removed and the cells can be collected by centrifugation lysed and frozen. Frozen samples are stable until RNA can be isolated. Density centrifugation however must be conducted at room temperature and if processing is unduly lengthy such as in a busy clinical lab the expression profile may change.

The quality and quantity of each clinical RNA sample is desirably checked before amplification and labeling for array hybridization using methods known in the art. For example one microliter of each sample may be analyzed on a Bioanalyzer Agilent 2100 Palo Alto Calif. USA using an RNA 6000 nano LabChip Caliper Mountain View Calif. USA . Degraded RNA is identified by the reduction of the 28S to 18S ribosomal RNA ratio and or the presence of large quantities of RNA in the 25 100 nucleotide range.

It is appreciated that the RNA sample for use with a diagnostic oligonucleotide or oligonucleotide set may be produced from the same or a different cell population sub population and or cell type as used to identify the diagnostic nucleotide set. For example a diagnostic oligonucleotide or oligonucleotide set identified using RNA extracted from mononuclear cells may be suitable for analysis of RNA extracted from whole blood or mononuclear cells depending on the particular characteristics of the members of the diagnostic nucleotide set. Generally diagnostic oligonucleotides or oligonucleotide sets must be tested and validated when used with RNA derived from a different cell population sub population or cell type than that used when obtaining the diagnostic gene set. Factors such as the cell specific gene expression of diagnostic nucleotide set members redundancy of the information provided by members of the diagnostic nucleotide set expression level of the member of the diagnostic nucleotide set and cell specific alteration of expression of a member of the diagnostic nucleotide set will contribute to the usefullness of a different RNA source than that used when identifying the members of the diagnostic nucleotide set. It is appreciated that it may be desirable to assay RNA derived from whole blood obviating the need to isolate particular cell types from the blood.

Expression profiles for the oligonucleotides or the set of diagnostic oligonucleotide sequences in a subject sample can be evaluated by any technique that determines the expression of each component oligonucleotide sequence. Methods suitable for expression analysis are known in the art and numerous examples are discussed in the Sections titled Methods of obtaining expression data and high throughput expression Assays above.

In many cases evaluation of expression profiles is most efficiently and cost effectively performed by analyzing RNA expression. Alternatively the proteins encoded by each component of the diagnostic nucleotide set are detected for diagnostic purposes by any technique capable of determining protein expression e.g. as described above. Expression profiles can be assessed in subject leukocyte sample using the same or different techniques as those used to identify and validate the diagnostic oligonucleotide or oligonucleotide set. For example a diagnostic nucleotide set identified as a subset of sequences on a cDNA microarray can be utilized for diagnostic or prognostic or monitoring etc. purposes on the same array from which they were identified. Alternatively the diagnostic nucleotide sets for a given disease or condition can be organized onto a dedicated sub array for the indicated purpose. It is important to note that if diagnostic nucleotide sets are discovered using one technology e.g. RNA expression profiling but applied as a diagnostic using another technology e.g. protein expression profiling the nucleotide or gene or protein sets must generally be validated for diagnostic purposes with the new technology. In addition it is appreciated that diagnostic nucleotide sets that are developed for one use e.g. to diagnose a particular disease may later be found to be useful for a different application e.g. to predict the likelihood that the particular disease will occur. Generally the diagnostic nucleotide set will need to be validated for use in the second circumstance. As discussed herein the sequence of diagnostic nucleotide set members may be amplified from RNA or cDNA using methods known in the art providing specific amplification of the nucleotide sequences.

Novel nucleotide sequences that are differentially expressed in leukocytes are also part of the invention. Previously unidentified open reading frames may be identified in a library of differentially expressed candidate nucleotide sequences as described above and the DNA and predicted protein sequence may be identified and characterized as noted above. We identified unnamed not previously described as corresponding to a gene or an expressed gene nucleotide sequences in our candidate nucleotide library depicted in Table 3A 3B AND 3C and the sequence listing. Accordingly further embodiments of the invention are the isolated nucleic acids described in Tables 3A and 3B AND 3C and in the sequence listing. The novel differentially expressed nucleotide sequences of the invention are useful in the diagnostic nucleotide set of the invention described above and are further useful as members of a diagnostic nucleotide set immobilized on an array. The novel partial nucleotide sequences may be further characterized using sequence tools and publically or privately accessible sequence databases as is well known in the art Novel differentially expressed nucleotide sequences may be identified as disease target nucleotide sequences described below. Novel nucleotide sequences may also be used as imaging reagent as further described below.

As used herein novel nucleotide sequence refers to a a nucleotide sequence containing at least one of the DNA sequences disclosed herein as shown in FIGS. Table 3A 3B and the sequence listing b any DNA sequence that encodes the amino acid sequence encoded by the DNA sequences disclosed herein c any DNA sequence that hybridizes to the complement of the coding sequences disclosed herein contained within the coding region of the nucleotide sequence to which the DNA sequences disclosed herein as shown in Table 3A 3B AND 3C and the sequence listing belong under highly stringent conditions e.g. hybridization to filter bound DNA in 0.5 M NaHPO 7 sodium dodecyl sulfate SDS 1 mM EDTA at 65 C. and washing in 0.1 SSC 0.1 SDS at 68 C. Ausubel F. M. et al. eds. 1989 Current Protocols in Molecular Biology Vol. I Green Publishing Associates Inc. and John Wiley sons Inc. New York at p. 2.10.3 d any DNA sequence that hybridizes to the complement of the coding sequences disclosed herein as shown in Table 3A 3B AND 3C and the sequence listing contained within the coding region of the nucleotide sequence to which DNA sequences disclosed herein as shown in TABLES 3A 3B and the sequence listing belong under less stringent conditions such as moderately stringent conditions e.g. washing in 0.2 SSC 0.1 SDS at 42 C. Ausubel et al. 1989 supra yet which still encodes a functionally equivalent gene product and or e any DNA sequence that is at least 90 identical at least 80 identical or at least 70 identical to the coding sequences disclosed herein as shown in TABLES 3A 3B AND 3C and the sequence listing wherein identity is determined using standard algorithms known in the art.

The invention also includes nucleic acid molecules preferably DNA molecules that hybridize to and are therefore the complements of the DNA sequences a through c in the preceding paragraph. Such hybridization conditions may be highly stringent or less highly stringent as described above. In instances wherein the nucleic acid molecules are deoxyoligonucleotides oligos highly stringent conditions may refer e.g. to washing in 6 SSC 0.05 sodium pyrophosphate at 37 C. for 14 base oligos 48 C. for 17 base oligos 55 C. for 20 base oligos and 60 C. for 23 base oligos . These nucleic acid molecules may act as target nucleotide sequence antisense molecules useful for example in target nucleotide sequence regulation and or as antisense primers in amplification reactions of target nucleotide sequence nucleic acid sequences. Further such sequences may be used as part of ribozyme and or triple helix sequences also useful for target nucleotide sequence regulation. Still further such molecules may be used as components of diagnostic methods whereby the presence of a disease causing allele may be detected.

The Invention Also Encompasses Nucleic Acid Molecules Contained in Full Length Gene Sequences That Are Related to Or Derived From Sequences In Tables 2 3 8 10 and the Sequence Listing. One Sequence May Map to More Than One Full Length Gene.

The invention also encompasses a DNA vectors that contain any of the foregoing coding sequences and or their complements i.e. antisense b DNA expression vectors that contain any of the foregoing coding sequences operatively associated with a regulatory element that directs the expression of the coding sequences and c genetically engineered host cells that contain any of the foregoing coding sequences operatively associated with a regulatory element that directs the expression of the coding sequences in the host cell. As used herein regulatory elements include but are not limited to inducible and non inducible promoters enhancers operators and other elements known to those skilled in the art that drive and regulate expression. The invention includes fragments of any of the DNA sequences disclosed herein. Fragments of the DNA sequences may be at least 5 at least 10 at least 15 at least 19 nucleotides at least 25 nucleotides at least 50 nucleotides at least 100 nucleotides at least 200 at least 500 or larger.

In addition to the oligonucleotide sequences described above homologues and orthologs of such sequences as may for example be present in other species may be identified and may be readily isolated without undue experimentation by molecular biological techniques well known in the art as well as use of gene analysis tools described above and e.g. in Example 4. Further there may exist nucleotide sequences at other genetic loci within the genome that encode proteins which have extensive homology to one or more domains of such gene products. These nucleotide sequences may also be identified via similar techniques.

For example the isolated differentially expressed nucleotide sequence may be labeled and used to screen a cDNA library constructed from mRNA obtained from the organism of interest. Hybridization conditions will be of a lower stringency when the cDNA library was derived from an organism different from the type of organism from which the labeled sequence was derived. Alternatively the labeled fragment may be used to screen a genomic library derived from the organism of interest again using appropriately stringent conditions. Such low stringency conditions will be well known to those of skill in the art and will vary predictably depending on the specific organisms from which the library and the labeled sequences are derived. For guidance regarding such conditions see for example Sambrook et al. 1989 Molecular Cloning A Laboratory Manual Cold Springs Harbor Press N.Y. and Ausubel et al. 1989 Current Protocols in Molecular Biology Green Publishing Associates and Wiley Interscience N.Y.

Novel nucleotide products include those proteins encoded by the novel nucleotide sequences described above. Specifically novel gene products may include polypeptides encoded by the novel nucleotide sequences contained in the coding regions of the nucleotide sequences to which DNA sequences disclosed herein in TABLES 3A 3B and the sequence listing .

In addition novel protein products of novel nucleotide sequences may include proteins that represent functionally equivalent gene products. Such an equivalent novel gene product may contain deletions additions or substitutions of amino acid residues within the amino acid sequence encoded by the novel nucleotide sequences described above but which result in a silent change thus producing a functionally equivalent novel nucleotide sequence product. Amino acid substitutions may be made on the basis of similarity in polarity charge solubility hydrophobicity hydrophilicity and or the amphipathic nature of the residues involved.

For example nonpolar hydrophobic amino acids include alanine leucine isoleucine valine proline phenylalanine tryptophan and methionine polar neutral amino acids include glycine serine threonine cysteine tyrosine asparagine and glutamine positively charged basic amino acids include arginine lysine and histidine and negatively charged acidic amino acids include aspartic acid and glutamic acid. Functionally equivalent as utilized herein refers to a protein capable of exhibiting a substantially similar in vivo activity as the endogenous novel gene products encoded by the novel nucleotide described above.

The novel gene products protein products of the novel nucleotide sequences may be produced by recombinant DNA technology using techniques well known in the art. Methods which are well known to those skilled in the art can be used to construct expression vectors containing novel nucleotide sequence protein coding sequences and appropriate transcriptional translational control signals. These methods include for example in vitro recombinant DNA techniques synthetic techniques and in vivo recombination genetic recombination. See for example the techniques described in Sambrook et al. 1989 supra and Ausubel et al. 1989 supra. Alternatively RNA capable of encoding novel nucleotide sequence protein sequences may be chemically synthesized using for example synthesizers. See for example the techniques described in Oligonucleotide Synthesis 1984 Gait M. J. ed. IRL Press Oxford which is incorporated by reference herein in its entirety. A variety of host expression vector systems may be utilized to express the novel nucleotide sequence coding sequences of the invention. Ruther et al. 1983 EMBO J. 2 1791 Inouye Inouye 1985 Nucleic Acids Res. 13 3101 3109 Van Heeke Schuster 1989 J. Biol. Chem. 264 5503 Smith et al. 1983 J. Virol. 46 584 Smith U.S. Pat. No. 4 215 051 Logan Shenk 1984 Proc. Natl. Acad. Sci. USA 81 3655 3659 Bittner et al. 1987 Methods in Enzymol. 153 516 544 Wigler et al. 1977 Cell 11 223 Szybalska Szybalski 1962 Proc. Natl. Acad. Sci. USA 48 2026 Lowy et al. 1980 Cell 22 817 Wigler et al. 1980 Natl. Acad. Sci. USA 77 3567 O Hare et al. 1981 Proc. Natl. Acad. Sci. USA 78 1527 Mulligan Berg 1981 Proc. Natl. Acad. Sci. USA 78 2072 Colberre Garapin et al. 1981 J. Mol. Biol. 150 1 Santerre et al. 1984 Gene 30 147 Janknecht et al. 1991 Proc. Natl. Acad. Sci. USA 88 8972 8976

Where recombinant DNA technology is used to produce the protein encoded by the novel nucleotide sequence for such assay systems it may be advantageous to engineer fusion proteins that can facilitate labeling immobilization and or detection.

Indirect labeling involves the use of a protein such as a labeled antibody which specifically binds to the protein encoded by the novel nucleotide sequence. Such antibodies include but are not limited to polyclonal monoclonal chimeric single chain Fab fragments and fragments produced by an Fab expression library.

The invention also provides for antibodies to the protein encoded by the novel nucleotide sequences. Described herein are methods for the production of antibodies capable of specifically recognizing one or more novel nucleotide sequence epitopes. Such antibodies may include but are not limited to polyclonal antibodies monoclonal antibodies mAbs humanized or chimeric antibodies single chain antibodies Fab fragments F ab 2 fragments fragments produced by a Fab expression library anti idiotypic anti Id antibodies and epitope binding fragments of any of the above. Such antibodies may be used for example in the detection of a novel nucleotide sequence in a biological sample or alternatively as a method for the inhibition of abnormal gene activity for example the inhibition of a disease target nucleotide sequence as further described below. Thus such antibodies may be utilized as part of cardiovascular or other disease treatment method and or may be used as part of diagnostic techniques whereby patients may be tested for abnormal levels of novel nucleotide sequence encoded proteins or for the presence of abnormal forms of the such proteins.

For the production of antibodies to a novel nucleotide sequence various host animals may be immunized by injection with a novel protein encoded by the novel nucleotide sequence or a portion thereof. Such host animals may include but are not limited to rabbits mice and rats to name but a few. Various adjuvants may be used to increase the immunological response depending on the host species including but not limited to Freund s complete and incomplete mineral gels such as aluminum hydroxide surface active substances such as lysolecithin pluronic polyols polyanions peptides oil emulsions keyhole limpet hemocyanin dinitrophenol and potentially useful human adjuvants such as BCG bacille Calmette Guerin and

Polyclonal antibodies are heterogeneous populations of antibody molecules derived from the sera of animals immunized with an antigen such as novel gene product or an antigenic functional derivative thereof. For the production of polyclonal antibodies host animals such as those described above may be immunized by injection with novel gene product supplemented with adjuvants as also described above.

Monoclonal antibodies which are homogeneous populations of antibodies to a particular antigen may be obtained by any technique which provides for the production of antibody molecules by continuous cell lines in culture. These include but are not limited to the hybridoma technique of Kohler and Milstein 1975 Nature 256 495 497 and U.S. Pat. No. 4 376 110 the human B cell hybridoma technique Kosbor et al. 1983 Immunology Today 4 72 Cole et al. 1983 Proc. Natl. Acad. Sci. USA 80 2026 2030 and the EBV hybridoma technique Cole et al. 1985 Monoclonal Antibodies And Cancer Therapy Alan R. Liss Inc. pp. 77 96 . Such antibodies may be of any immunoglobulin class including IgG IgM IgE IgA IgD and any subclass thereof. The hybridoma producing the mAb of this invention may be cultivated in vitro or in vivo.

In addition techniques developed for the production of chimeric antibodies Morrison et al. 1984 Proc. Natl. Acad. Sci. 81 6851 6855 Neuberger et al. 1984 Nature 312 604 608 Takeda et al. 1985 Nature 314 452 454 by splicing the genes from a mouse antibody molecule of appropriate antigen specificity together with genes from a human antibody molecule of appropriate biological activity can be used. A chimeric antibody is a molecule in which different portions are derived from different animal species such as those having a variable region derived from a murine mAb and a human immunoglobulin constant region.

Alternatively techniques described for the production of single chain antibodies U.S. Pat. No. 4 946 778 Bird 1988 Science 242 423 426 Huston et al. 1988 Proc. Natl. Acad. Sci. USA 85 5879 5883 and Ward et al. 1989 Nature 334 544 546 can be adapted to produce novel nucleotide sequence single chain antibodies. Single chain antibodies are formed by linking the heavy and light chain fragments of the Fv region via an amino acid bridge resulting in a single chain polypeptide.

Antibody fragments which recognize specific epitopes may be generated by known techniques For example such fragments include but are not limited to the F ab 2 fragments which can be produced by pepsin digestion of the antibody molecule and the Fab fragments which can be generated by reducing the disulfide bridges of the F ab 2 fragments. Alternatively Fab expression libraries may be constructed Huse et al. 1989 Science 246 1275 1281 to allow rapid and easy identification of monoclonal Fab fragments with the desired specificity.

The invention also provides disease specific target oligonucleotide sequences and sets of disease specific target oligonucleotide sequences. The diagnostic oligonucleotide sets subsets thereof novel oligonucleotide sequences and individual members of the diagnostic oligonucleotide sets identified as described above are also disease specific target oligonucleotide sequences. In particular individual oligonucleotide sequences that are differentially regulated or have predictive value that is strongly correlated with a disease or disease criterion are especially favorable as disease specific target oligonucleotide sequences. Sets of genes that are co regulated may also be identified as disease specific target oligonucleotide sets. Such oligonucleotide sequences and or oligonucleotide sequence products are targets for modulation by a variety of agents and techniques. For example disease specific target oligonucleotide sequences or the products of such oligonucleotide sequences or sets of disease specific target oligonucleotide sequences can be inhibited or activated by e.g. target specific monoclonal antibodies or small molecule inhibitors or delivery of the oligonucleotide sequence or gene product of the oligonucleotide sequence to patients. Also sets of genes can be inhibited or activated by a variety of agents and techniques. The specific usefulness of the target oligonucleotide sequence s depends on the subject groups from which they were discovered and the disease or disease criterion with which they correlate.

The present invention is optionally provided to a user as a kit. Typically a kit contains one or more diagnostic nucleotide sets of the invention. Alternatively the kit contains the candidate nucleotide library of the invention. Most often the kit contains a diagnostic nucleotide probe set or other subset of a candidate library e.g. as a cDNA oligonucleotide or antibody microarray or reagents for performing an assay on a diagnostic gene set using any expression profiling technology packaged in a suitable container. The kit may further comprise one or more additional reagents e.g. substrates labels primers for labeling expression products tubes and or other accessories reagents for collecting blood samples buffers e.g. erythrocyte lysis buffer leukocyte lysis buffer hybridization chambers cover slips etc. as well as a software package e.g. including the statistical methods of the invention e.g. as described above and a password and or account number for accessing the compiled database. The kit optionally further comprises an instruction set or user manual detailing preferred methods of using the diagnostic nucleotide sets in the methods of the invention.

Example 4 Further sequence analysis of novel nucleotide sequences identified by subtractive hybridization screening

Example 9 Identification of diagnostic oligonucleotide sets for use in diagnosis of rheumatoid arthritis.

Example 10 Identification of diagnostic oligonucleotide sets for diagnosis of Systemic Lupus Erythematosis

To produce a candidate nucleotide library with representatives from the spectrum of nucleotide sequences that are differentially expressed in leukocytes subtracted hybridization libraries were produced from the following cell types and conditions 

Cells were obtained from multiple individuals to avoid introduction of bias by using only one person as a cell source.

Buffy coats platelets and leukocytes that are isolated from whole blood were purchased from Stanford Medical School Blood Center. Four buffy coats were used each of which was derived from about 350 ml of whole blood from one donor individual 10 ml of buffy coat sample was drawn from the sample bag using a needle and syringe. 40 ml of Buffer EL Qiagen was added per 10 ml of buffy coat to lyse red blood cells. The sample was placed on ice for 15 minutes and cells were collected by centrifugation at 2000 rpm for 10 minutes. The supernatant was decanted and the cell pellet was re suspended in leukocyte growth media supplemented with DNase LGM 3 from Clonetics supplemented with Dnase at a final concentration of 30 U ml . Cell density was determined using a hemocytometer. Cells were plated in media at a density of 1 10cells ml in a total volume of 30 ml in a T 75 flask Corning . Half of the cells were stimulated with ionomycin and phorbol myristate acetate PMA at a final concentration of 1 g ml and 62 ng ml respectively. Cells were incubated at 37 C. and at 5 COfor 3 hours then cells were scraped off the flask and collected into 50 ml tubes. Stimulated and resting cell populations were kept separate. Cells were centrifuged at 2000 rpm for 10 minutes and the supernatant was removed. Cells were lysed in 6 ml of phenol guanidine isothyocyanate Trizol reagent GibcoBRL homogenized using a rotary homogenizer and frozen at 80 . Total RNA and mRNA were isolated as described below.

Two frozen vials of 5 10pooled human peripheral blood mononuclear cells PBMCs were purchased from Clonetics catalog number cc 2702 . The cells were rapidly thawed in a 37 C. water bath and transferred to a 15 ml tube containing 10 ml of leukocyte growth media supplemented with DNase prepared as described above . Cells were centrifuged at 200 g for 10 minutes. The supernatant was removed and the cell pellet was resuspended in LGM 3 media supplemented with DNase. Cell density was determined using a hemocytometer. Cells were plated at a density of 1 10cells ml in a total volume of 30 ml in a T 75 flask Corning . Half of the cells were stimulated with ionomycin and PMA at a final concentration of 1 g ml and 62 ng ml respectively. Cells were incubated at 37 C. and at 5 COfor 3 hours then cells were scraped off the flask and collected into 50 ml tubes. Stimulated and resting cell populations were kept separate. Cells were centrifuged at 2000 rpm and the supernatant was removed. Cells were lysed in 6 ml of phenol guanidine isothyocyanate solution TRIZOL reagent GibcoBRL homogenized using a rotary homogenizer and frozen at 80 . Total RNA and mRNA were isolated from these samples using the protocol described below.

45 ml of whole blood was drawn from a peripheral vein of four healthy human subjects into tubes containing anticoagulant. 50 l RosetteSep Stem Cell Technologies T cell isolation cocktail per ml of blood was added mixed well and incubated for 20 minutes at room temperature. The mixture was diluted with an equal volume of PBS 2 fetal bovine serum FBS and mixed by inversion. 30 ml of diluted mixture sample was layered on top of 15 ml DML medium Stem Cell Technologies . The sample tube was centrifuged for 20 minutes at 1200 g at room temperature. The enriched T lymphocyte cell layer at the plasma medium interface was removed. Enriched cells were washed with PBS 2 FBS and centrifuged at 1200 g. The cell pellet was treated with 5 ml of erythrocyte lysis buffer EL buffer Qiagen for 10 minutes on ice. The sample was centrifuged for 5 min at 1200 g. Cells were plated at a density of 1 10cells ml in a total volume of 30 ml in a T 75 flask Corning . Half of the cells were stimulated with ionomycin and PMA at a final concentration of 1 g ml and 62 ng ml respectively. Cells were incubated at 37 C. and at 5 COfor 3 hours then cells were scraped off the flask and collected into 50 ml tubes. Stimulated and resting cell populations were kept separate. Cells were centrifuged at 2000 rpm and the supernatant was removed. Cells were lysed in 6 ml of phenol guanidine isothyocyanate solution TRIZOL reagent GibcoBRL homogenized using a rotary homogenizer and frozen at 80 . Total RNA and mRNA were isolated as described below.

Total RNA and mRNA were isolated using the following procedure the homogenized samples were thawed and mixed by vortexing. Samples were lysed in a 1 0.2 mixture of Trizol and chloroform respectively. For some samples 6 ml of Trizol chloroform was added. Variable amounts of Trizol chloroform was added to other samples. Following lysis samples were centrifuged at 3000 g for 15 min at 4 C. The aqueous layer was removed into a clean tube and 4 volumes of Buffer RLT Qiagen was added for every volume of aqueous layer. The samples were mixed thoroughly and total RNA was prepared from the sample by following the Qiagen Rneasy midi protocol for RNA cleanup October 1999 protocol Qiagen . For the final step the RNA was eluted from the column twice with 250 l Rnase free water. Total RNA was quantified using a spectrophotometer. Isolation of mRNA from total RNA sample was done using The Oligotex mRNA isolation protocol Qiagen was used to isolate mRNA from total RNA according to the manufacturer s instructions Qiagen 7 99 version . mRNA was quantified by spectrophotometry.

Subtracted cDNA libraries were prepared using Clontech s PCR Select cDNA Subtraction Kit protocol number PT 1117 1 as described in the manufacturer s protocol. The protocol calls for two sources of RNA per library designated Driver and Tester. The following 6 libraries were made 

The Clontech protocol results in the PCR amplification of cDNA products. The PCR products of the subtraction protocol were ligated to the pGEM T easy bacterial vector as described by the vector manufacturer Promega 6 99 version . Ligated vector was transformed into competent bacteria using well known techniques plated and individual clones are picked grown and stored as a glycerol stock at 80 C. Plasmid DNA was isolated from these bacteria by standard techniques and used for sequence analysis of the insert. Unique cDNA sequences were searched in the Unigene database build 133 and Unigene cluster numbers were identified that corresponded to the DNA sequence of the cDNA. Unigene cluster numbers were recorded in an Excel spreadsheet.

Existing and publicly available gene sequence databases were used to identify candidate nucleotide sequences for leukocyte expression profiling. Genes and nucleotide sequences with specific expression in leukocytes for example lineage specific markers or known differential expression in resting or activated leukocytes were identified. Such nucleotide sequences are used in a leukocyte candidate nucleotide library alone or in combination with nucleotide sequences isolated through cDNA library construction as described above.

Leukocyte candidate nucleotide sequences were identified using three primary methods. First the publically accessible publication database PubMed was searched to identify nucleotide sequences with known specific or differential expression in leukocytes. Nucleotide sequences were identified that have been demonstrated to have differential expression in peripheral blood leukocytes between subjects with and without particular disease s selected from Table 1. Additionally genes and gene sequences that were known to be specific or selective for leukocytes or sub populations of leukocytes were identified in this way.

Next two publicly available databases of DNA sequences Unigene located on the website at ncbi.nlm.nih.gov UniGene and BodyMap located on the website at bodymap.ims.u tokyo.ac.jp were searched for sequenced DNA clones that showed specificity to leukocyte lineages or subsets of leukocytes or resting or activated leukocytes.

The human Unigene database build 133 was used to identify leukocyte candidate nucleotide sequences that were likely to be highly or exclusively expressed in leukocytes. We used the Library Differential Display utility of Unigene located on the website at ncbi.nlm.nih.gov UniGene info ddd.html which uses statistical methods The Fisher Exact Test to identify nucleotide sequences that have relative specificity for a chosen library or group of libraries relative to each other. We compared the following human libraries from Unigene release 133 

Sequences from these libraries were compared to sequences from non leukocyte derived libraries in the Unigene database to identify genes that had some specificity for the leukcoyte derived libraries.

BodyMap like Unigene contains cell specific libraries that contain potentially useful information about genes that may serve as lineage specific or leukocyte specific markers Okubo et al. 1992 . We compared three leukocyte specific libraries Granulocyte CD4 T cell and CD8 T cell with the other libraries. Nucleotide sequences that were found in one or more of the leukocyte specific libraries but absent in the others were identified. Clones that were found exclusively in one of the three leukocyte libraries were also included in a list of nucleotide sequences that could serve as lineage specific markers.

Next the sequence of the nucleotide sequences identified in PubMed or BodyMap were searched in Unigene version 133 and a human Unigene cluster number was identified for each nucleotide sequence. The cluster number was recorded in a Microsoft Excel spreadsheet and a non redundant list of these clones was made by sorting the clones by UniGene number and removing all redundant clones using Microsoft Excel tools. The non redundant list of UniGene cluster numbers was then compared to the UniGene cluster numbers of the cDNAs identified using subtractive cDNA hybridization as described above in Example 1 listed in Table 3 and the sequence listing . Only UniGene clusters that were not contained in the cDNA libraries were retained. Unigene clusters corresponding to 1911 candidate nucleotide sequences for leukocyte expression profiling were identified in this way and are listed in Table 3 and the sequence listing.

DNA clones corresponding to each UniGene cluster number are obtained in a variety of ways. First a cDNA clone with identical sequence to part of or all of the identified UniGene cluster is bought from a commercial vendor or obtained from the IMAGE consortium located on the web at image.llnl.gov the Integrated Molecular Analysis of Genomes and their Expression. Alternatively PCR primers are designed to amplify and clone any portion of the nucleotide sequence from cDNA or genomic DNA using well known techniques. Alternatively the sequences of the identified UniGene clusters are used to design and synthesize oligonucleotide probes for use in oligonucleotide microarray based expression profiling.

Clones of differentially expressed cDNAs identified by subtractive hybridization described above were sequenced on an MJ Research BaseStation slab gel based fluorescent detection system using BigDye Applied Biosystems Foster City Calif. terminator chemistry was used Heiner et al. Genome Res 1998 May 8 5 557 61 .

The fluorescent profiles were analyzed using the Phred sequence analysis program Ewing et al 1998 Genome Research 8 175 185 . Analysis of each clone results in a one pass nucleotide sequence and a quality file containing a number for each base pair with a score based on the probability that the determined base is correct. Each of the sequence files and its respective quality files were initially combined into single fasta format Pearson W R. Methods Mol Biol. 2000 132 185 219 multi sequence file with the appropriate labels for each clone in the headers for subsequent automated analysis.

Initially known sequences were analyzed by pair wise similarity searching using the blastn option of the blastall program obtained from the National Center for Biological Information National Library of Medicine National Institutes of Health NCBI to determine the quality score that produced accurate matching Altschul S F et al. J Mol Biol. 1990 Oct. 5 215 3 403 10. . Empirically it was determined that a raw score of 8 was the minimum that contained useful information. Using a sliding window average for 16 base pairs an average score was determined. The sequence was removed trimmed when the average score fell below 8. Maximum reads were 950 nucleotides long.

Next the sequences were compared by similarity matching against a database file containing the flanking vector sequences used to clone the cDNA using the blastall program with the blastn option. All regions of vector similarity were removed or trimmed from the sequences of the clones using scripts in the GAWK programming language a variation of AWK Aho A V et al The Awk Programming Language Addison Wesley Reading Mass. 1988 Robbins AD Effective AWK Programming Free Software Foundation Boston Mass. 1997 . It was found that the first 45 base pairs of all the sequences were related to vector these sequences were also trimmed and thus removed from consideration. The remaining sequences were then compared against the NCBI vector database Kitts P. A. et al. National Center for Biological Information National Library of Medicine National Institutes of Health Manuscript in preparation 2001 using blastall with the blastn option. Any vector sequences that were found were removed from the sequences.

Messenger RNA contains repetitive elements that are found in genomic DNA. These repetitive elements lead to false positive results in similarity searches of query mRNA sequences versus known mRNA and EST databases. Additionally regions of low information content long runs of the same nucleotide for example also result in false positive results. These regions were masked using the program RepeatMasker2 found on the website at repeatmasker.genome.washington.edu Smit AFA Green P RepeatMasker at the website at genome.washington.edu RM RepeatMasker.html. The trimmed and masked files were then subjected to further sequence analysis.

cDNA sequences were further characterized using BLAST analysis. The BLASTN program was used to compare the sequence of the fragment to the UniGene dbEST and nr databases at NCBI GenBank release 123.0 see Table 5 . In the BLAST algorithm the expect value for an alignment is used as the measure of its significance. First the cDNA sequences were compared to sequences in Unigene on the web at ncbi.nlm.nih.gov UniGene. If no alignments were found with an expect value less than 10 the sequence was compared to the sequences in the dbEST database using BLASTN. If no alignments were found with an expect value less than 10 the sequence was compared to sequences in the nr database.

The BLAST analysis produced the following categories of results a a significant match to a known or predicted human gene b a significant match to a nonhuman DNA sequence such as vector DNA or DNA c a significant match to an unidentified GenBank entry a sequence not previously identified or predicted to be an expressed sequence or a gene such as a cDNA clone mRNA or cosmid or d no significant alignments. If a match to a known or predicted human gene was found analysis of the known or predicted protein product was performed as described below. If a match to an unidentified GenBank entry was found or if no significant alignments were found the sequence was searched against all known sequences in the human genome database located on the web at ncbi.nlm.nih.gov genome seq page.cgi F HsBlast.html ORG Hs see Table 5.

If many unknown sequences were to be analyzed with BLASTN the clustering algorithm CAP2 Contig Assembly Program version 2 was used to cluster them into longer contiguous sequences before performing a BLAST search of the human genome. Sequences that can be grouped into contigs are likely to be cDNA from expressed genes rather than vector DNA DNA or human chromosomal DNA from a noncoding region any of which could have been incorporated into the library. Clustered sequences provide a longer query sequence for database comparisons with BLASTN increasing the probability of finding a significant match to a known gene. When a significant alignment was found further analysis of the putative gene was performed as described below. Otherwise the sequence of the original cDNA fragment or the CAP2 contig is used to design a probe for expression analysis and further approaches are taken to identify the gene or predicted gene that corresponds to the cDNA sequence including similarity searches of other databases molecular cloning and Rapid Amplification of cDNA Ends RACE .

In some cases the process of analyzing many unknown sequences with BLASTN was automated by using the BLAST network client program blastcl3 which was downloaded from ftp ncbi.nlm.nih.gov blast network netblast.

When a cDNA sequence aligned to the sequence of one or more chromosomes a large piece of the genomic region around the loci was used to predict the gene containing the cDNA. To do this the contig corresponding to the mapped locus as assembled by the RefSeq project at NCBI was downloaded and cropped to include the region of alignment plus 100 000 bases preceding it and 100 000 bases following it on the chromosome. The result was a segment 200 kb in length plus the length of the alignment. This segment designated a putative gene was analyzed using an exon prediction algorithm to determine whether the alignment area of the unknown sequence was contained within a region predicted to be transcribed see Table 6 .

This putative gene was characterized as follows all of the exons comprising the putative gene and the introns between them were taken as a unit by noting the residue numbers on the 200 kb segment that correspond to the first base of the first exon and the last base of the last exon as given in the data returned by the exon prediction algorithm. The truncated sequence was compared to the UniGene dbEST and nr databases to search for alignments missed by searching with the initial fragment.

The predicted amino acid sequence of the gene was also analyzed. The peptide sequence of the gene predicted from the exons was used in conjunction with numerous software tools for protein analysis see Table 7 . These were used to classify or identify the peptide based on similarities to known proteins as well as to predict physical chemical and biological properties of the peptides including secondary and tertiary structure flexibility hydrophobicity antigenicity hydrophilicity common domains and motifs and localization within the cell or tissues. The peptide sequence was compared to protein databases including SWISS PROT TrEMBL GenPept PDB PIR PROSITE ProDom PROSITE Blocks PRINTS and Pfam using BLASTP and other algorithms to determine similarities to known proteins or protein subunits.

This sequence was used as input for a series of BLASTN searches. First it was used to search the UniGene database build 132 located on the web at ncbi.nlm.nih.gov BLAST. No alignments were found with an expect value less than the threshold value of 10. A BLASTN search of the database dbEST release 041001 was then performed on the sequence and 21 alignments were found http www.ncbi.nlm.nih.gov BLAST . Ten of these had expect values less than 10 but all were matches to unidentified cDNA clones. Next the sequence was used to run a BLASTN search of the nr database release 123.0. No significant alignment to any sequence in nr was found. Finally a BLASTN search of the human genome was performed on the sequence located on the web at ncbi.nlm.nih.gov genome seq page.cgi F HsBlast.html ORG Hs.

A single alignment to the genome was found on contig NT004698.3 e 0.0 . The region of alignment on the contig was from base 1 821 298 to base 1 822 054 and this region was found to be mapped to chromosome 1 from base 105 552 694 to base 105 553 450. The sequence containing the aligned region plus 100 kilobases on each side of the aligned region was downloaded. Specifically the sequence of chromosome 1 from base 105 452 694 to 105 653 450 was downloaded from the website at ncbi.nlm.nih.gov cgi bin Entrez seq reg.cgi chr 1 from 105452694 to 105653450.

This 200 757 bp segment of the chromosome was used to predict exons and their peptide products as follows. The sequence was used as input for the Genscan algorithm located on the web at genes.mit.edu GENSCAN.html using the following Genscan settings 

The region matching the sequence of clone 596H6 was known to span base numbers 100 001 to 100 757 of the input sequence. An exon was predicted by the algorithm with a probability of 0.695 covering bases 100 601 to 101 094 designated exon 4.14 of the fourth predicted gene . This exon was part of a predicted cistron that is 24 195 bp in length. The sequence corresponding to the cistron was noted and saved separately from the 200 757 bp segment. BLASTN searches of the Unigene dbEST and nr databases were performed on it.

At least 100 significant alignments to various regions of the sequence were found in the dbEST database although most appeared to be redundant representations of a few exons. All matches were to unnamed cDNAs and mRNAs unnamed cDNAs and mRNAs are cDNAs and mRNAs not previously identified or shown to correspond to a known or predicted human gene from various tissue types. Most aligned to a single region on the sequence and spanned 500 bp or less but several consisted of five or six regions separated by gaps suggesting the locations of exons in the gene. Several significant matches to entries in the UniGene database were found as well even after masking low complexity regions and short repeats in the sequence. All matches were to unnamed cDNA clones.

At least 100 significant alignments were found in the nr database as well. A similarity to hypothetical protein FLJ22457 UniGene cluster Hs.238707 was found e 0.0 . The cDNA of this predicted protein has been isolated from B lymphocytes located on the web at ncbi.nlm.nih.gov entrez viewer.cgi save 0 cmd cfm on f 1 view gp txt 0 val 13637988.

Multiple analyses were performed using this prediction. First a pairwise comparison of the sequence above and the sequence of FLJ22457 the hypothetical protein mentioned above using BLASTP version 2.1.2 located on the web at ncbi.nlm.nih.gov BLAST resulted in a match with an expect value of 0.0. The peptide sequence predicted from clone 596H6 was longer and 19 of the region of alignment between the two resulted from gaps in hypothetical protein FLJ22457. The cause of the discrepancy might be alternative mRNA splicing alternative post translational processing or differences in the peptide predicting algorithms used to create the two sequences but the homology between the two is significant.

BLASTP and TBLASTN were also used to search for sequence similarities in the SWISS PROT TrEMBL GenBank Translated and PDB databases. Matches to several proteins were found among them a tumor cell suppression protein HTS1. No matches aligned to the full length of the peptide sequence however suggesting that similarity is limited to a few regions of the peptide.

TBLASTN produced matches to several proteins both identified and theoretical but again no matches aligned to the full length of the peptide sequence. The best alignment was to the same hypothetical protein found in GenBank before FLJ22457 .

To discover similarities to protein families comparisons of the domains described above were carried out using the Pfam and Blocks databases. A search of the Pfam database identified two regions of the peptide domains as belonging the DENN protein family e 2.1 10 . The human DENN protein possesses an RGD cellular adhesion motif and a leucine zipper like motif associated with protein dimerization and shows partial homology to the receptor binding domain of tumor necrosis factor alpha. DENN is virtually identical to MADD a human MAP kinase activating death domain protein that interacts with type I tumor necrosis factor receptor located on the web at srs.ebi.ac.uk srs6bin cgi bin wgetz id fS5n1GQsHf e INTERPRO IPR001194 . The search of the Blocks database also revealed similarities between regions of the peptide sequence and known protein groups but none with a satisfactory degree of confidence. In the Blocks scoring system scores over 1 100 are likely to be relevant. The highest score of any match to the predicted peptide was 1 058.

The Prosite ProDom PRINTS databases all publicly available were used to conduct further domain and motif analysis. The Prosite search generated many recognized protein domains. A BLASTP search was performed to identify areas of similarity between the protein query sequence and PRINTS a protein database of protein fingerprints groups of motifs that together form a characteristic signature of a protein family. In this case no groups were found to align closely to any section of the submitted sequence. The same was true when the ProDom database was searched with BLASTP.

A prediction of protein structure was done by performing a BLAST search of the sequence against PDB a database in which every member has tertiary structure information. No significant alignments were found by this method. Secondary and super secondary structure was examined using the Garnier algorithm. Although it is only considered to be 60 65 accurate the algorithm provided information on the locations and lengths of alpha helices beta sheets turns and coils.

The antigenicity of the predicted peptide was modeled by graphing hydrophilicity vs. amino acid number. This produced a visual representation of trends in hydrophilicity along the sequence. Many locations in the sequence showed antigenicity and five sites had antigenicity greater than 2. This information can be used in the design of affinity reagents to the protein.

Membrane spanning regions were predicted by graphing hydrophobicity vs. amino acid number. Thirteen regions were found to be somewhat hydrophobic. The algorithm TMpred predicted a model with 6 strong transmembrane helices located on the web at ch.embnet.org software TMPRED form.html.

NNPSL is a neural network algorithm developed by the Sanger Center. It uses amino acid composition and sequence to predict cellular location. For the peptide sequence submitted its first choice was mitochondrial 51.1 expected accuracy . Its second choice was cytoplasmic 91.4 expected accuracy .

The BLASTN program was used to compare the sequence to the UniGene and dbEST databases. No significant alignments were found in either. It was then searched against the nr database and only alignments to unnamed genomic DNA clones were found.

CAP2 was used to cluster a group of unknowns including clone 486E11. The sequence for 486E11 was found to overlap others. These formed a contig of 1 010 residues which is shown below 

The sequence of the CAP2 contig was used in a BLAST search of the human genome. 934 out of 1 010 residues aligned to a region of chromosome 21. A gap of 61 residues divided the aligned region into two smaller fragments. The sequence of this region plus 100 kilobases on each side of it was downloaded and analyzed using the Genscan site at MIT located on the web at genes.mit.edu GENSCAN.html with the following settings 

The fragment was found to fall within one of several predicted genes in the chromosome region. The bases corresponding to the predicted gene including its predicted introns were saved as a separate file and used to search GenBank again with BLASTN to find any ESTs or UniGene clusters identified by portions of the sequence not included in the original unknown fragment. The nr database contained no significant matches. At least 100 significant matches to various parts of the predicted gene were found in the dbEST database but all of them were to unnamed cDNA clones. Comparison to UniGene produced fewer significant matches but all matches were to unnamed cDNAs.

The peptide sequence predicted by Genscan was also saved. Multiple types of analyses were performed on it using the resources mentioned in Table 3. BLASTP and TBLASTN were used to search the TrEMBL protein database located on the web at expasy.ch sprot and the GenBank nr database located on the web at ncbi.nlm.hih.gov BLAST which includes data from the SwissProt PIR PRF and PDB databases. No significant matches were found in any of these so no gene identity or tertiary structure was discovered.

The peptide sequence was also searched for similarity to known domains and motifs using BLASTP with the Prosite Blocks Pfam and ProDom databases. The searches produced no significant alignments to known domains. BLASTP comparison to the PRINTS database produced an alignment to the P450 protein family but with a low probability of accuracy e 6.9 .

Two methods were used to predict secondary structure the Garnier Osguthorpe Robson model and the Chou Fasman model. The two methods differed somewhat in their results but both produced representations of the peptide sequence with helical and sheet regions and locations of turns.

Antigenicity was plotted as a graph with amino acid number in the sequence on the x axis and hydrophilicity on the y axis. Several areas of antigenicity were observed but only one with antigenicity greater than 2. Hydrophobicity was plotted in the same way. Only one region from approximately residue 135 to residue 150 had notable hydrophobicity. TMpred accessed through ExPASy was used to predict transmembrane helices. No regions of the peptide sequence were predicted with reasonable confidence to be membrane spanning helices.

NNPSL predicted that the putative protein would be found either in the nucleus expected prediction accuracy 51.1 or secreted from the cell expected prediction accuracy 91.4 .

Two tubes were drawn per patient. Blood was drawn from either a standard peripheral venous blood draw or directly from a large bore intra arterial or intravenous catheter inserted in the femoral artery femoral vein subclavian vein or internal jugular vein. Care was taken to avoid sample contamination with heparin from the intravascular catheters as heparin can interfere with subsequent RNA reactions.

For each tube 8 ml of whole blood was drawn into a tube CPT Becton Dickinson order 362753 containing the anticoagulant Citrate 25 C. density gradient solution e.g. Ficoll Percoll and a polyester gel barrier that upon centrifugation was permeable to RBCs and granulocytes but not to mononuclear cells. The tube was inverted several times to mix the blood with the anticoagulant. The tubes were centrifuged at 1750 g in a swing out rotor at room temperature for 20 minutes. The tubes were removed from the centrifuge and inverted 5 10 times to mix the plasma with the mononuclear cells while trapping the RBCs and the granulocytes beneath the gel barrier. The plasma mononuclear cell mix was decanted into a 15 ml tube and 5 ml of phosphate buffered saline PBS is added. The 15 ml tubes were spun for 5 minutes at 1750 g to pellet the cells. The supernatant was discarded and 1.8 ml of RLT lysis buffer is added to the mononuclear cell pellet. The buffer and cells were pipetted up and down to ensure complete lysis of the pellet. The cell lysate was frozen and stored until it is convenient to proceed with isolation of total RNA.

Total RNA was purified from the lysed mononuclear cells using the Qiagen Rneasy Miniprep kit as directed by the manufacturer 10 99 version for total RNA isolation including homogenization Qiashredder columns and on column DNase treatment. The purified RNA was eluted in 50 ul of water.

Two 8 ml blood samples were drawn from a peripheral vein into a tube CPT Becton Dickinson order 362753 containing anticoagulant Citrate 25 C. density gradient solution Ficoll and a polyester gel barrier that upon centrifugation is permeable to RBCs and granulocytes but not to mononuclear cells. The mononuclear cells and plasma remained above the barrier while the RBCs and granulocytes were trapped below. The tube was inverted several times to mix the blood with the anticoagulant and the tubes were subjected to centrifugation at 1750 g in a swing out rotor at room temperature for 20 min. The tubes were removed from the centrifuge and the clear plasma layer above the cloudy mononuclear cell layer was aspirated and discarded. The cloudy mononuclear cell layer was aspirated with care taken to rinse all of the mononuclear cells from the surface of the gel barrier with PBS phosphate buffered saline . Approximately 2 mls of mononuclear cell suspension was transferred to a 2 ml microcentrifuge tube and centrifuged for 3 min. at 16 000 rpm in a microcentrifuge to pellet the cells. The supernatant was discarded and 1.8 ml of RLT lysis buffer Qiagen were added to the mononuclear cell pellet which lysed the cells and inactivated Rnases. The cells and lysis buffer were pipetted up and down to ensure complete lysis of the pellet. Cell lysate was frozen and stored until it was convenient to proceed with isolation of total RNA.

RNA samples were isolated from 8 mL of whole blood. Yields ranged from 2 ug to 20 ug total RNA for 8 mL blood. A260 A280 spectrophotometric ratios were between 1.6 and 2.0 indicating purity of sample. 2 ul of each sample were run on an agarose gel in the presence of ethidium bromide. No degradation of the RNA sample and no DNA contamination were visible.

In some cases specific subsets of mononuclear cells were isolated from peripheral blood of human subjects. When this was done the StemSep cell separation kits manual version 6.0.0 were used from StemCell Technologies Vancouver Canada . This same protocol can be applied to the isolation of T cells CD4 T cells CD8 T cells B cells monocytes NK cells and other cells. Isolation of cell types using negative selection with antibodies may be desirable to avoid activation of target cells by antibodies.

Control RNA was prepared using total RNA from Buffy coats and or total RNA from enriched mononuclear cells isolated from Buffy coats both with and without stimulation with ionomycin and PMA. The following control RNAs were prepared 

Some samples were prepared using the following protocol Buffy coats from 38 individuals were obtained from Stanford Blood Center. Each buffy coat is derived from 350 mL whole blood from one individual. 10 ml buffy coat was removed from the bag and placed into a 50 ml tube. 40 ml of Buffer EL Qiagen was added the tube was mixed and placed on ice for 15 minutes then cells were pelleted by centrifugation at 2000 g for 10 minutes at 4 C. The supernatant was decanted and the cell pellet was re suspended in 10 ml of Qiagen Buffer EL. The tube was then centrifuged at 2000 g for 10 minutes at 4 C. The cell pellet was then re suspended in 20 ml TRIZOL GibcoBRL per Buffy coat sample the mixture was shredded using a rotary homogenizer and the lysate was then frozen at 80 C. prior to proceeding to RNA isolation.

Other control RNAs were prepared from enriched mononuclear cells prepared from Buffy coats. Buffy coats from Stanford Blood Center were obtained as described above. 10 ml buffy coat was added to a 50 ml polypropylene tube and 10 ml of phosphate buffer saline PBS was added to each tube. A polysucrose 5.7 g dL and sodium diatrizoate 9.0 g dL solution at a 1.077 0.0001 g ml density solution of equal volume to diluted sample was prepared Histopaque 1077 Sigma cat. no 1077 1 . This and all subsequent steps were performed at room temperature. 15 ml of diluted buffy coat PBS was layered on top of 15 ml of the histopaque solution in a 50 ml tube. The tube was centrifuged at 400 g for 30 minutes at room temperature. After centrifugation the upper layer of the solution to within 0.5 cm of the opaque interface containing the mononuclear cells was discarded. The opaque interface was transferred into a clean centrifuge tube. An equal volume of PBS was added to each tube and centrifuged at 350 g for 10 minutes at room temperature. The supernatant was discarded. 5 ml of Buffer EL Qiagen was used to resuspend the remaining cell pellet and the tube was centrifuged at 2000 g for 10 minutes at room temperature. The supernatant was discarded. The pellet was resuspended in 20 ml of TRIZOL GibcoBRL for each individual buffy coat that was processed. The sample was homogenized using a rotary homogenizer and frozen at 80 C until RNA was isolated.

RNA was isolated from frozen lysed Buffy coat samples as follows frozen samples were thawed and 4 ml of chloroform was added to each buffy coat sample. The sample was mixed by vortexing and centrifuged at 2000 g for 5 minutes. The aqueous layer was moved to new tube and then repurified by using the RNeasy Maxi RNA clean up kit according to the manufacturer s instruction Qiagen PN 75162 . The yield purity and integrity were assessed by spectrophotometer and gel electrophoresis.

Some samples were prepared by a different protocol as follows. The further use of RNA prepared using this protocol is described in Example 14.

50 whole blood samples were randomly selected from consented blood donors at the Stanford Medical School Blood Center. Each buffy coat sample was produced from 350 mL of an individual s donated blood. The whole blood sample was centrifuged at 4 400 g for 8 minutes at room temperature resulting in three distinct layers a top layer of plasma a second layer of buffy coat and a third layer of red blood cells. 25 ml of the buffy coat fraction was obtained and diluted with an equal volume of PBS phosphate buffered saline . 30 ml of diluted buffy coat was layered onto 15 ml of sodium diatrizoate solution adjusted to a density of 1.077 0.001 g ml Histopaque 1077 Sigma in a 50 mL plastic tube. The tube was spun at 800 g for 10 minutes at room temperature. The plasma layer was removed to the 30 ml mark on the tube and the mononuclear cell layer removed into a new tube and washed with an equal volume of PBS and collected by centrifugation at 2000 g for 10 minutes at room temperature. The cell pellet was resuspended in 10 ml of Buffer EL Qiagen by vortexing and incubated on ice for 10 minutes to remove any remaining erthythrocytes. The mononuclear cells were spun at 2000 g for 10 minutes at 4 degrees Celsius. The cell pellet was lysed in 25 ml of a phenol guanidinium thiocyanate solution TRIZOL Reagent Invitrogen . The sample was homogenized using a PowerGene 5 rotary homogenizer Fisher Scientific and Omini disposable generator probes Fisher Scientific . The Trizol lysate was frozen at 80 degrees C. until the next step.

The samples were thawed out and incubated at room temperature for 5 minutes. 5 ml chloroform was added to each sample mixed by vortexing and incubated at room temperature for 3 minutes. The aqueous layers were transferred to new 50 ml tubes. The aqueous layer containing total RNA was further purified using the Qiagen RNeasy Maxi kit PN 75162 per the manufacturer s protocol October 1999 . The columns were eluted twice with 1 ml Rnase free water with a minute incubation before each spin. Quantity and quality of RNA was assessed using standard methods. Generally RNA was isolated from batches of 10 buffy coats at a time with an average yield per buffy coat of 870 g and an estimated total yield of 43.5 mg total RNA with a 260 280 ratio of 1.56 and a 28S 18S ratio of 1.78.

Quality of the RNA was tested using the Agilent 2100 Bioanalyzer using RNA 6000 microfluidics chips. Analysis of the electrophorgrams from the Bioanalyzer for five different batches demonstrated the reproducibility in quality between the batches.

Total RNA from all five batches were combined and mixed in a 50 ml tube then aliquoted as follows 2 10 ml aliquots in 15 ml tubes and the rest in 100 l aliquots in 1.5 ml microcentrifuge tubes. The aliquots gave highly reproducible results with respect to RNA purity size and integrity. The RNA was stored at 80 C.

When compared with BC38 and Stimulated mononuclear reference samples the R50 performed as well if not better than the other reference samples as shown in .

In an analysis of hybridizations where the R50 targets were fluorescently labeled with Cy 5 using methods described herein and the amplified and labeled aRNA was hybridized as in example 14 to the olignoucleotide array described in example 13. The R50 detected 97.3 of probes with a Signal to Noise ratio S N of greater than three and 99.9 of probes with S N greater one.

Rheumatoid arthritis hereinafter RA is a chronic and debilitating inflammatory arthritis. The diagnosis of RA is made by clinical criteria and radiographs. A new class of medication TNF blockers are effective but the drugs are expensive have side effects and not all patients respond to treatment. In addition relief of disease symptoms does not always correlate with inhibition of joint destruction. For these reasons an alternative mechanism for the titration of therapy is needed.

An observational study was conducted in which a cohort of patients meeting American College of Rheumatology hereinafter ARC criteria for the diagnosis of RA was identified. Arnett et al. 1988 31 315 24. Patients gave informed consent and a peripheral blood mononuclear cell RNA sample was obtained by the methods as described herein. When available RNA samples were also obtained from surgical specimens of bone or synovium from effected joints and synovial fluid. Also T cells were isolated from the peripheral blood for some patients for expression analysis. This was done using the protocol given in Example 7.

From each patient the following clinical information was obtained if available Demographic information information relating to the ACR criteria for RA presence or absence of additional diagnoses of inflammatory and non inflammatory conditions data from laboratory test including complete blood counts with differentials CRP ESR ANA Serum IL6 Soluble CD40 ligand LDL HDL Anti DNA antibodies rheumatoid factor C3 C4 serum creatinine and any medication levels data from surgical procedures such as gross operative findings and pathological evaluation of resected tissues and biopsies information on pharmacological therapy and treatment changes clinical diagnoses of disease flare hospitalizations quantitative joint exams results from health assessment questionnaires HAQs other clinical measures of patient symptoms and disability physical examination results and radiographic data assessing joint involvement synovial thickening bone loss and erosion and joint space narrowing and deformity. In some cases data includes pathological evaluation of synovial memebranes and joint tissues from RA and control patients. Pathology scoring systems were used to determine disease category inflammation type of inflammatory infiltrate cellular and makeup of the synovial inflammation.

For some specimens of synovium mononuclear cells or subsets of mononuclear cells such as T cells can be isolated for expression profiling. The relative number of lyphocyte subsets for some specimens can be determined by fluorescence activated cell sorting. Examples are determination of the CD4 CD8 T cell ratio for a specimen. This information can be used as a variable to correlate to other outcomes or as an outcome for correlation analysis.

From these data measures of improvement in RA are derived as exemplified by the ACR 20 and 50 response improvement rates Felson et al. 1996 . Measures of disease activity over some period of time is derived from these data as are measures of disease progression. Serial radiography of effected joints is used for objective determination of progression e.g. joint space narrowing peri articular osteoporosis synovial thickening . Disease activity is determined from the clinical scores medical history physical exam lab studies surgical and pathological findings.

The collected clinical data disease criteria is used to define patient or sample groups for correlation of expression data. Patient groups are identified for comparison for example a patient group that possesses a useful or interesting clinical distinction verses a patient group that does not possess the distinction. Examples of useful and interesting patient distinctions that can be made on the basis of collected clinical data are listed here 

Samples from patients during a clinically diagnosed RA flare versus samples from these same or different patients while they are asymptomatic.

Samples from patients who subsequently have high measures of disease activity versus samples from those same or different patients who have low subsequent disease activity.

Samples from patients who subsequently have high measures of disease progression versus samples from those same or different patients who have low subsequent disease progression.

Samples from patients who subsequently respond to a given medication or treatment regimen versus samples from those same or different patients who subsequently do not respond to a given medication or treatment regimen for example TNF pathway blocking medications .

Samples from patients with tissue biopsy results showing a high degree of inflammation versus samples from patients with lesser degrees of histological evidence of inflammation on biopsy.

Expression profiles correlating with progression of RA are identified. Subsets of the candidate library or a previously identified diagnostic nucleotide set are identified according to the above procedures that have predictive value for the progression of RA.

Diagnostic nucleotide set s are identified which predict respond to TNF blockade. Patients are profiled before and during treatment with these medications. Patients are followed for relief of symptoms side effects and progression of joint destruction e.g. as measured by hand radiographs. Expression profiles correlating with response to TNF blockade are identified. Subsets of the candidate library or a previously identified diagnostic nucleotide set are identified according to the above procedures that have predictive value for response to TNF blockade.

SLE is a chronic systemic inflammatory disease characterized by dysregulation of the immune system. Clinical manifestations affect every organ system and include skin rash renal dysfunction CNS disorders arthralgias and hematologic abnormalities. SLE clinical manifestations tend to both recur intermittently or flare and progress over time leading to permanent end organ damage.

An observational study was conducted in which a cohort of patients meeting American College of Rheumatology hereinafter ACR criteria for the diagnosis of SLE were identified. See Tan et al. 1982 25 1271 7. Patients gave informed consent and a peripheral blood mononuclear cell RNA sample or a peripheral T cell sample was obtained by the methods as described in example 7.

From each patient the following clinical information was obtained if available Demographic information ACR criteria for SLE additional diagnoses of inflammatory and non inflammatory conditions data from laboratory testing including complete blood counts with differentials CRP ESR ANA Serum IL6 Soluble CD40 ligand LDL HDL Anti DNA antibodies rheumatoid factor C3 C4 serum creatinine and other measures of renal dysfunction medication levels data from surgical procedures such as gross operative findings and pathological evaluation of resected tissues and biopsies e.g. renal CNS information on pharmacological therapy and treatment changes clinical diagnoses of disease flare hospitalizations quantitative joint exams results from health assessment questionnaires HAQs SLEDAIs a clinical score for SLE activity that assess many clinical variables Bombadier C Gladman D D Urowitz M B Caron D Chang C H and the Committee on Prognosis Studies in SLE Derivation of the SLEDAI for Lupus Patients. 35 630 640 1992 other clinical measures of patient symptoms and disability physical examination results and carotid ultrasonography.

The collected clinical data disease criteria is used to define patient or sample groups for correlation of expression data. Patient groups are identified for comparison for example a patient group that possesses a useful or interesting clinical distinction verses a patient group that does not possess the distinction. Measures of disease activity in SLE are derived from the clinical data described above to divide patients and patient samples into groups with higher and lower disease activity over some period of time or at any one point in time. Such data are SLEDAI scores and other clinical scores levels of inflammatory markers or complement number of hospitalizations medication use and changes biopsy results and data measuring progression of end organ damage or end organ damage including progressive renal failure carotid atherosclerosis and CNS dysfunction.

Expression profiles correlating with progression of SLE are identified including expression profiles corresponding to end organ damage and progression of end organ damage. Expression profiles are identified predicting disease progression or disease flare response to treatment or likelihood of response to treatment predict likelihood of low or high disease measures optionally described using the SLEDAI score and presence or likelihood of developing premature carotid atherosclerosis. Subsets of the candidate library or a previously identified diagnostic nucleotide set are identified according to the above procedures that have predictive value for the progression of SLE.

Further examples of useful and interesting patient distinctions that can be made on the basis of collected clinical data are listed here. Samples can be grouped and groups are compared to discover diagnostic gene sets 

1. Samples from patients during a clinically diagnosed SLE flare versus samples from these same or different patients while they are asymptomatic or while they have a documented infection.

2. Samples from patients who subsequently have high measures of disease activity versus samples from those same or different patients who have low subsequent disease activity.

3. Samples from patients who subsequently have high measures of disease progression versus samples from those same or different patients who have low subsequent disease progression.

4. Samples from patients who subsequently respond to a given medication or treatment regimen versus samples from those same or different patients who subsequently do not respond to a given medication or treatment regimen.

5. Samples from patients with premature carotid atherosclerosis on ultrasonography versus patients with SLE without premature atherosclerosis.

Mononuclear RNA samples were collected from patients with SLE and patients with Rheumatoid or Osteoarthritis RA and OA or controls using the protocol described in example 7. The patient diagnoses were determined using standard diagnostic algorithms such as those that are employed by the American College of Rheumatology see example See Tan et al. 1982 25 1271 7 Arnett et al. 1988 31 315 24 .

32 samples were included in the anaysis. 15 samples were derived from patients with a clinical diagnosis of SLE and the remainder were derived from patients with RA 9 OA 4 and subjects without known disease 4 who served as controls. Samples from patients with SLE or RA were classified as Active or Controlled with respect to disease activity by the patient s physician based on objective and subjective criteria such as patient history physical exam and lab studies. An attempt was made to match SLE patients and controls with respect to important variables such as medication use sex age and secondary diagnoses.

After preparation of RNA example 7 amplification labeling hybridization scanning feature extraction and data processing were done as described in Example 14 using the oligonucleotide microarrays described in Example 13. The resulting log ratio of expression of Cy3 patient sample Cy5 R50 reference RNA was used for analysis.

Initially significance analysis for microarrays SAM Tusher 2001 Example 16 was used to discover that were differentially expressed between 7 of the Lupus samples and 17 control samples. 1 gene was identified that was expressed at a higher level in the lupus patients than in all controls. This gene had a 0.5 false detection rate using SAM. This means that there is statistically a 99.5 chance that the gene is truly differentially expressed between the Lupus and control samples. This gene was oligonucleotide and SEQ ID 4637. The oligonucleotide GCCTCTTGCTTGGCGTGATAACCCTGTCATCTTCCCAAAGCTCATTTATG detects a specific human gene sialyltransferase SIAT4A Unigene Hs.301698 Locus NM003033 GI 4506950. Expression ratios for the gene are given for each sample in . The average fold change in expression between SLE and controls was 1.48.

When a larger data set was used 15 SLE samples were compared to 17 controls. Using SAM genes were identified as significantly differentially expressed between Lupus and controls. These genes and their FDRs are given in Table 10A. Supervised harvesting classification X Mine Brisbane Calif. and CART Salford Systems San Diego Calif. were also used on the same data to determine which set of genes best distinguish SLE from control samples Example 16 .

CART was used to build a decision tree for classification of samples as lupus or not lupus using the gene expression data from the arrays. The analysis identitifies sets of genes that can be used together to accurately identify samples derived from lupus patients. The set of genes and the identified threshold expression levels for the decision tree are referred to as models . Multiple models for diagnosis of Lupus were derived by using different settings and parameters for the CART algorithm and using different sets of genes in the analysis. When using CART it may be desirable to limit the number of independent variables. In the case of the genes on the arrays a subset of 8000 can be selected for analysis in CART based on significant differential expression discovered by using SAM or some other algorithm.

Model I was based on a data set consisting of thirty two samples fifteen SLE and seventeen non SLE . These samples were used to derive the model and are referred to a the training set . Model I used the expression values for twenty nine genes which were found to be most significant in differentiating SLE and non SLE samples in the analysis using SAM described above. SLE samples were designated as Class 1 and non SLE samples were designated as Class 2. For this analysis the following settings were used in the MODEL SETUP CART Salford Systems San Diego Calif. . In the Model settings the tree type selected for the analysis was classification. In the Categorical settings the default values were used. In the Testing settings V fold cross validation was selected with a value of 10. In the Select Cases settings the default values were used. In the Best Tree settings the default values were used. In the Combine settings the default values were used. In the Method settings Symmetric Gini was selected as the type of classification tree and Linear combinations for splitting was also selected. The default values were used for the linear combinations. In the Advance Settings the default values were used. In the Costs settings the default values were used. In the Priors settings Equal was selected as the priors for Class. In the penalty settings the default values were used.

From this analysis CART built two models a two gene model and a three gene model . The sensitivity and specificity for the identification of lupus in the training set samples of the two genes model were 100 and 94 respectively. The sensitivity and specificity for the 10 fold cross validation set of the two gene model were 100 and 88 respectively with a relative cost of 0.118. The sensitivity and specificity for the training set of the three genes model were 100 and 100 respectively. The sensitivity and specificity for the 10 fold cross validation set of the three genes model were 93 and 94 respectively with a relative cost of 0.125.

Model II was based on a data set consisted of thirty two samples fifteen SLE and seventeen non SLE training set and six thousand forty four genes with expression values for at least 80 of the samples. The MODEL SETUP for the analysis of this data set was the same as for the analysis above except for the following correction. In the Method settings Linear combination for splitting was unchecked after the analysis yielded no classification tree. The change in the linear combination setting resulted in the following.

The sensitivity and specificity for the training set of the one gene model were 87 and 82 respectively. The sensitivity and specificity for the 10 fold cross validation set of the one gene model were 80 and 59 respectively with a relative cost of 0.612. The sensitivity and specificity for the training set of the three genes model were 100 and 88 respectively. The sensitivity and specificity for the 10 fold cross validation set of the three genes model were 67 and 65 respectively with a relative cost of 0.686. The sensitivity and specificity for the training set of the five genes model were 100 and 94 respectively. The sensitivity and specificity for the 10 fold cross validation set of the five genes model were 67 and 59 respectively with a relative cost of 0.745. Results and models are summarized in C and F.

These genes can be used alone or in association with other genes or variables to build a diagnostic gene set or a classification algorithm. These genes can be used in association with known gene markers for lupus such as those identified in the prior art to provide a diagnostic algorithm.

Primers for real time PCR validation were designed for each of the genes as described in Example 15 and are listed in Table 10B.

Surrogates for some of the most useful genes were identified and are listed in Table 10C. Surrogates can be used in addition to or in place of a diagnostic gene in a method of detecting lupus or in diagnostic gene set. For genes that were splitters in CART surrogates were identified and reported by the software. In these cases the best available surrogates are listed. For other genes hierarchical clustering of the data was performed with default settings x miner X mine Brisbane Calif. and members of gene expression clusters were noted. A cluster was selected that included the gene of interest and the members of that cluster were recorded in Table 10C.

This Example describes the compilation of almost 8 000 unique genes and ESTs using sequences identified from the sources described below. The sequences of these genes and ESTs were used to design probes as described in the following Example.

Tables 3A 3B and 3C list the sequences identified in the subtracted leukocyte expression libraries. All sequences that were identified as corresponding to a known RNA transcript were represented at least once and all unidentified sequences were represented twice once by the sequence on file and again by the complementary sequence to ensure that the sense or coding strand of the gene sequence was included.

Table 3A. Table 3A contained all those sequences in the subtracted libraries of example 1 that matched sequences in GenBank s nr EST Human and UniGene databases with an acceptable level of confidence. All the entries in the table representing the sense strand of their genes were grouped together and all those representing the antisense strand were grouped. A third group contained those entries whose strand could not be determined. Two complementary probes were designed for each member of this third group.

Table 3B and 3C. Table 3B and 3C contained all those sequences in the leukocyte expression subtracted libraries of example 1 that did not match sequences in GenBank s nr EST Human and UniGene databases with an acceptable level of confidence but which had a high probability of representing real mRNA sequences. Sequences in Table 3B did not match anything in the databases above but matched regions of the human genome draft and were spatially clustered along it suggesting that they were exons rather than genomic DNA included in the library by chance. Sequences in Table 3C also aligned well to regions of the human genome draft but the aligned regions were interrupted by genomic DNA meaning they were likely to be spliced transcripts of multiple exon genes.

Table 3B lists 510 clones and Table 3C lists 48 clones that originally had no similarity with any sequence in the public databases. Blastn searches conducted after the initial filing have identified sequences in the public database with high similarity E values less than 1e 40 to the sequences determined for these clones. Table 3B contained 272 clones and Table 3C contained 25 clones that were found to have high similarity to sequences in dbEST. The sequences of the similar dbEST clones were used to design probes. Sequences from clones that contained no similar regions to any sequence in the database were used to design a pair of complementary probes.

Probes were designed from database sequences that had the highest similarity to each of the sequenced clones in Tables 3A 3B and 3C. Based on BLASTn searches the most similar database sequence was identified by locus number and the locus number was submitted to GenBank using batch Entrez located at the website ncbi.nlm.nih.gov entrez batchentrez.cgi db Nucleotide to obtain the sequence for that locus. The GenBank entry sequence was used because in most cases it was more complete or was derived from multi pass sequencing and thus would likely have fewer errors than the single pass cDNA library sequences. When only UniGene cluster IDs were available for genes of interest the respective sequences were extracted from the UniGene unique database build 137 downloaded from NCBI ftp ncbi.nlm.nih.gov repository UniGene . This database contains one representative sequence for each cluster in UniGene.

Example 2 describes searches of literature databases. We also searched for research articles discussing genes expressed only in leukocytes or involved in inflammation and particular disease conditions including genes that were specifically expressed or down regulated in a disease state. Searches included but were not limited to the following terms and various combinations of theses terms inflammation atherosclerosis rheumatoid arthritis osteoarthritis lupus SLE allograft transplant rejection leukocyte monocyte lymphocyte mononuclear macrophage neutrophil eosinophil basophil platelet congestive heart failure expression profiling microarray inflammatory bowel disease asthma RNA expression gene expression granulocyte.

A UniGene cluster ID or GenBank accession number was found for each gene in the list. The strand of the corresponding sequence was determined if possible and the genes were divided into the three groups sense coding strand anti sense strand or strand unknown. The rest of the probe design process was carried out as described above for the sequences from the leukocyte subtracted expression library.

Database mining was performed as described in Example 2. In addition the Library Browser at the NCBI UniGene web site located on the web at ncbi.nlm.nih.gov UniGene lbrowse.cgi ORG Hs DISPLAY ALL was used to identify genes that are specifically expressed in leukocyte cell populations. All expression libraries available at the time were examined and those derived from leukocytes were viewed individually. Each library viewed through the Library Browser at the UniGene web site contains a section titled Shown below are UniGene clusters of special interest only that lists genes that are either highly represented or found only in that library. Only the genes in this section were downloaded from each library. Alternatively every sequence in each library is downloaded and then redundancy between libraries is reduced by discarding all UniGene cluster IDs that are represented more than once. A total of 439 libraries were downloaded containing 35 819 genes although many were found in more than one library. The most important libraries from the remaining set were separated and 3 914 genes remained. After eliminating all redundancy between these libraries and comparing the remaining genes to those listed in Tables 3A 3B and 3C the set was reduced to 2 573 genes in 35 libraries as shown in Table 4. From these all genes in first 30 libraries were used to design probes. A random subset of genes was used from Library Lib.376 Activated T cells XX . From the last four libraries a random subset of sequences listed as ESTs found only in this library was used.

215 sequences derived from an angiogenic endothelial cell subtracted cDNA library obtained from Stanford University were used for probe design. Briefly using well known subtractive hybridization procedures as described in e.g. U.S. Pat. Nos. 5 958 738 5 589 339 5 827 658 5 712 127 5 643 761 5 565 340 modified to normalize expression by suppressing over representation of abundant RNA species while increasing representation of rare RNA species a library was produced that is enriched for RNA species messages that are differentially expressed between test stimulated and control resting HUVEC populations. The subtraction suppression protocol was performed as described by the kit manufacturer Clontech PCR select cDNA Subtraction Kit .

Pooled primary HUVECs Clonetics were cultured in 15 FCS M199 GibcoBRL with standard concentrations of Heparin Penicillin Streptomycin Glutamine and Endothelial Cell Growth Supplement. The cells were cultured on 1 gelatin coated 10 cm dishes. Confluent HUVECs were photographed under phase contrast microscopy. The cells formed a monolayer of flat cells without gaps. Passage 2 5 cells were used for all experiments. Confluent HUVECs were treated with trypsin EDTA and seeded onto collagen gels. Collagen gels were made according to the protocol of the Collagen manufacturer Becton Dickinson Labware . Collagen gels were prepared with the following ingredients Rat tail collagen type I Collaborative Biomedical 1.5 mg mL mouse laminin Collaborative Biomedical 0.5 mg mL 10 10 media 199 Gibco BRL . 1N NaOH 10 PBS and sterile water were added in amounts recommended in the protocol. Cell density was measured by microscopy. 1.2 10cells were seeded onto gels in 6 well 35 mm dishes in 5 FCS M199 media. The cells were incubated for 2 hrs at 37 C with 5 CO2. The media was then changed to the same media with the addition of VEGF Sigma at 30 ng mL media. Cells were cultured for 36 hrs. At 12 24 and 36 hrs the cells were observed with phase contrast microscopy. At 36 hours the cells were observed elongating adhering to each other and forming lumen structures. At 12 and 24 hrs media was aspirated and refreshed. At 36 hrs the media was aspirated the cells were rinsed with PBS and then treated with Collagenase Sigma 2.5 mg mL PBS for 5 min with active agitation until the collagen gels were liquefied. The cells were then centrifuged at 4 C 2000 g for 10 min. The supernatant was removed and the cells were lysed with 1 mL Trizol Reagent Gibco per 5 10cells. Total RNA was prepared as specified in the Trizol instructions for use. mRNA was then isolated as described in the micro fast track mRNA isolation protocol from Invitrogen. This RNA was used as the tester RNA for the subtraction procedure.

Ten plates of resting confluent p4 HUVECs were cultured with 15 FCS in the M199 media described above. The media was aspirated and the cells were lysed with 1 mL Trizol and total RNA was prepared according to the Trizol protocol. mRNA was then isolated according to the micro fast track mRNA isolation protocol from Invitrogen. This RNA served as the control RNA for the subtraction procedure.

The entire subtraction cloning procedure was carried out as per the user manual for the Clontech PCR Select Subtraction Kit. The cDNAs prepared from the test population of HUVECs were divided into tester pools while cDNAs prepared from the control population of HUVECs were designated the driver pool. cDNA was synthesized from the tester and control RNA samples described above. Resulting cDNAs were digested with the restriction enzyme RsaI. Unique double stranded adapters were ligated to the tester cDNA. An initial hybridization was performed consisting of the tester pools of cDNA with its corresponding adapter and an excess of the driver cDNA. The initial hybridization results in a partial normalization of the cDNAs such that high and low abundance messages become more equally represented following hybridization due to a failure of driver tester hybrids to amplify.

A second hybridization involved pooling unhybridized sequences from the first hybridization together with the addition of supplemental driver cDNA. In this step the expressed sequences enriched in the two tester pools following the initial hybridization can hybridize. Hybrids resulting from the hybridization between members of each of the two tester pools are then recovered by amplification in a polymerase chain reaction PCR using primers specific for the unique adapters. Again sequences originating in a tester pool that form hybrids with components of the driver pool are not amplified. Hybrids resulting between members of the same tester pool are eliminated by the formation of panhandles between their common 5 and 3 ends. The subtraction was done in both directions producing two libraries one with clones that are upregulated in tube formation and one with clones that are down regulated in the process.

The resulting PCR products representing partial cDNAs of differentially expressed genes were then cloned i.e. ligated into an appropriate vector according to the manufacturer s protocol pGEM Teasy from Promega and transformed into competent bacteria for selection and screening. Colonies 2180 were picked and cultured in LB broth with 50 ug mL ampicillin at 37 C overnight. Stocks of saturated LB 50 ug mL ampicillin and 15 glycerol in 96 well plates were stored at 80 C. Plasmid was prepared from 1.4 mL saturated LB broth containing 50 ug mL ampicillin. This was done in a 96 well format using commercially available kits according to the manufacturer s recommendations Qiagen 96 turbo prep .

2 probes to represent 22 of these sequences required therefore a total of 237 probes were derived from this library.

Several viruses may play a role in a host of disease including inflammatory disorders atherosclerosis and transplant rejection. Table 12 lists the viral genes represented by oligonucleotide probes on the microarray. Low complexity regions in the sequences were masked using RepeatMasker before using them to design probes.

It was necessary to design sense oligonucleotide probes because the labeling and hybridization protocol to be used with the microarray results in fluorescently labeled antisense cRNA. All of the sequences we selected to design probes could be divided into three categories 

It was not known whether the sequences from the leukocyte subtracted expression library were from the sense or antisense strand. GenBank sequences are reported with sequence given 5 to 3 and the majority of the sequences we used to design probes came from accession numbers with descriptions that made it clear whether they represented sense or antisense sequence. For example all sequences containing mRNA in their descriptions were understood to be the sequences of the sense mRNA unless otherwise noted in the description and all IMAGE Consortium clones are directionally cloned and so the direction or sense of the reported sequence can be determined from the annotation in the GenBank record.

For accession numbers representing the sense strand the sequence was downloaded and masked and a probe was designed directly from the sequence. These probes were selected as close to the 3 end as possible. For accession numbers representing the antisense strand the sequence was downloaded and masked and a probe was designed complementary to this sequence. These probes were designed as close to the 5 end as possible i.e. complementary to the 3 end of the sense strand .

Multiple copies of certain genes or segments of genes were included in the sequences from each category described above either by accident or by design. Reducing redundancy within each of the gene sets was necessary to maximize the number of unique genes and ESTs that could be represented on the microarray.

Three methods were used to reduce redundancy of genes depending on what information was available. First in gene sets with multiple occurrences of one or more UniGene numbers only one occurrence of each UniGene number was kept. Next each gene set was searched by GenBank accession numbers and only one occurrence of each accession number was conserved. Finally the gene name description or gene symbol were searched for redundant genes with no UniGene number or different accession numbers. In reducing the redundancy of the gene sets every effort was made to conserve the most information about each gene.

We note however that the UniGene system for clustering submissions to GenBank is frequently updated and UniGene cluster IDs can change. Two or more clusters may be combined under a new cluster ID or a cluster may be split into several new clusters and the original cluster ID retired. Since the lists of genes in each of the gene sets discussed were assembled at different times the same sequence may appear in several different sets with a different UniGene ID in each.

Sequences from Table 3A were treated differently. In some cases two or more of the leukocyte subtracted expression library sequences aligned to different regions of the same GenBank entry indicating that these sequences were likely to be from different exons in the same gene transcript. In these cases one representative library sequence corresponding to each presumptive exon was individually listed in Table 3A.

After redundancy within a gene set was sufficiently reduced a table of approximately 8 000 unique genes and ESTs was compiled in the following manner. All of the entries in Table 3A were transferred to the new table. The list of genes produced by literature and database searches was added eliminating any genes already contained in Table 3A. Next each of the remaining sets of genes was compared to the table and any genes already contained in the table were deleted from the gene sets before appending them to the table.

By way of example this section describes the design of four oligonucleotide probes using Array Designer Ver 1.1 Premier Biosoft International Palo Alto Calif. . The major steps in the process are given first.

1 Obtain best possible sequence of mRNA from GenBank. If a full length sequence reference sequence is not available a partial sequence is used with preference for the 3 end over the 5 end. When the sequence is known to represent the antisense strand the reverse complement of the sequence is used for probe design. For sequences represented in the subtracted leukocyte expression library that have no significant match in GenBank at the time of probe design our sequence is used.

2 Mask low complexity regions and repetitive elements in the sequence using an algorithm such as RepeatMasker.

3 Use probe design software such as Array Designer version 1.1 to select a sequence of 50 residues with specified physical and chemical properties. The 50 residues nearest the 3 end constitute a search frame. The residues it contains are tested for suitability. If they don t meet the specified criteria the search frame is moved one residue closer to the 5 end and the 50 residues it now contains are tested. The process is repeated until a suitable 50 mer is found.

4 If no such 50 mer occurs in the sequence the physical and chemical criteria are adjusted until a suitable 50 mer is found.

5 Compare the probe to dbEST the UniGene cluster set and the assembled human genome using the BLASTn search tool at NCBI to obtain the pertinent identifying information and to verify that the probe does not have significant similarity to more than one known gene.

Clone 40H12 was sequenced and compared to the nr dbEST and UniGene databases at NCBI using the BLAST search tool. The sequence matched accession number NM002310 a curated RefSeq project sequence see Pruitt et al. 2000 16 44 47 encoding leukemia inhibitory factor receptor LIFR mRNA with a reported E value of zero. An E value of zero indicates there is for all practical purposes no chance that the similarity was random based on the length of the sequence and the composition and size of the database. This sequence cataloged by accession number NM002310 is much longer than the sequence of clone 40H12 and has a poly A tail. This indicated that the sequence cataloged by accession number NM002310 is the sense strand and a more complete representation of the mRNA than the sequence of clone 40H12 especially at the 3 end. Accession number NM002310 was included in a text file of accession numbers representing sense strand mRNAs and sequences for the sense strand mRNAs were obtained by uploading a text file containing desired accession numbers as an Entrez search query using the Batch Entrez web interface and saving the results locally as a FASTA file. The following sequence was obtained and the region of alignment of clone 401 112 is outlined 

The FASTA file including the sequence of NM002310 was masked using the RepeatMasker web interface Smit AFA Green P RepeatMasker at genome.washington.edu RM RepeatMasker.html Smit and Green . Specifically during masking the following types of sequences were replaced with N s SINE MIR LINE L2 LINE L1 LTR MaLR LTR Retroviral Alu and other low informational content sequences such as simple repeats. Below is the sequence following masking 

The length of this sequence was determined using batch automated computational methods and the sequence as sense strand its length and the desired location of the probe sequence near the 3 end of the mRNA was submitted to Array Designer Ver 1.1 Premier Biosoft International Palo Alto Calif. . Search quality was set at 100 number of best probes set at 1 length range set at 50 base pairs Target Tm set at 75 C. degrees plus or minus 5 degrees Hairpin max deltaG at 6.0 kcal mol. Self dimmer max deltaG at 6.0 kcal mol Run repeat dinucleotide max length set at 5 and Probe site minimum overlap set at 1. When none of the 49 possible probes met the criteria the probe site would be moved 50 base pairs closer to the 5 end of the sequence and resubmitted to Array Designer for analysis. When no possible probes met the criteria the variation on melting temperature was raised to plus and minus 8 degrees and the number of identical basepairs in a run increased to 6 so that a probe sequence was produced.

In the sequence above using the criteria noted above Array Designer Ver 1.1 designed a probe corresponding to oligonucleotide number 2280 in Table 8 and is indicated by underlining in the sequence above. It has a melting temperature of 68.4 degrees Celsius and a max run of 6 nucleotides and represents one of the cases where the criteria for probe design in Array Designer Ver 1.1 were relaxed in order to obtain an oligonucleotide near the 3 end of the mRNA Low melting temperature was allowed .

Clone 463D12 was sequenced and compared to the nr dbEST and UniGene databases at NCBI using the BLAST search tool. The sequence matched accession number AI184553 an EST sequence with the definition line qd60a05.x1 Soares testis NHT cDNA clone IMAGE 1733840 3 similar to gb M29550 PROTEIN PHOSPHATASE 2B CATALYTIC SUBUNIT 1 HUMAN mRNA sequence. The E value of the alignment was 1.00 10. The GenBank sequence begins with a poly T region suggesting that it is the antisense strand read 5 to 3 . The beginning of this sequence is complementary to the 3 end of the mRNA sense strand. The accession number for this sequence was included in a text file of accession numbers representing antisense sequences. Sequences for antisense strand mRNAs were obtained by uploading a text file containing desired accession numbers as an Entrez search query using the Batch Entrez web interface and saving the results locally as a FASTA file. The following sequence was obtained and the region of alignment of clone 463D12 is outlined 

The FASTA file including the sequence of AA184553 was then masked using the RepeatMasker web interface as shown below. The region of alignment of clone 463D12 is outlined.

The sequence was submitted to Array Designer as described above however the desired location of the probe was indicated at base pair 50 and if no probe met the criteria moved in the 3 direction. The complementary sequence from Array Designer was used because the original sequence was antisense. The oligonucleotide designed by Array Designer corresponds to oligonucleotide number 4342 in Table 8 and is complementary to the underlined sequence above. The probe has a melting temperature of 72.7 degrees centigrade and a max run of 4 nucleotides.

Clone 72D4 was sequenced and compared to the nr dbEST and UniGene databases at NCBI using the BLAST search tool. No significant matches were found in any of these databases. When compared to the human genome draft significant alignments were found to three consecutive regions of the reference sequence NT008060 as depicted below suggesting that the insert contains three spliced exons of an unidentified gene.

Because the reference sequence contains introns and may represent either the coding or noncoding strand for this gene BioCardia s own sequence file was used to design the oligonucleotide. Two complementary probes were designed to ensure that the sense strand was represented. The sequence of the insert in clone 72D4 is shown below with the three putative exons outlined.

The sequence was submitted to RepeatMasker but no repetitive sequences were found. The sequence shown above was used to design the two 50 mer probes using Array Designer as described above. The probes are shown in bold typeface in the sequence depicted below. The probe in the sequence is oligonucleotide number 6415 SEQ ID NO 6415 in Table 8 and the complementary probe is oligonucleotide number 6805 SEQ ID NO 6805 .

Following probe design each probe sequence was confirmed by comparing the sequence against dbEST the UniGene cluster set and the assembled human genome using BLASTn at NCBI. Alignments accession numbers gi numbers UniGene cluster numbers and names were examined and the most common sequence used for the probe. The final probe set was compiled into Table 8. In this table the sequence ID is given which corresponds to the sequence listing. The origin of the sequence for inclusion on the array is noted as coming from one of the cDNA libraries described in example 1 mining from databases as described in examples 2 and 11 or identification from the published literature. The unigene number genebank accession and GI number are also given for each sequence when known. The name of the gene associated with the accession number is noted. The strand is noted as 1 or 1 meaning that the probe was designed from the complement of the sequence 1 or directly from the sequence 1 . Finally the nucleotide sequence of each probe is also given.

We produced an array of 8000 spotted 50mer oligonucleotides. Examples 11 and 12 exemplify the design and selection of probes for this array.

Sigma Genosys The Woodlands Tex. synthesized un modified 50 mer oligonucleotides using standard phosphoramidite chemistry with a starting scale of synthesis of 0.05 mole see e.g. R. Meyers ed. 1995 . Briefly to begin synthesis a 3 hydroxyl nucleoside with a dimethoxytrityl DMT group at the 5 end was attached to a solid support. The DMT group was removed with trichloroacetic acid TCA in order to free the 5 hydroxyl for the coupling reaction. Next tetrazole and a phosphoramidite derivative of the next nucleotide were added. The tetrazole protonates the nitrogen of the phosphoramidite making it susceptible to nucleophilic attack. The DMT group at the 5 end of the hydroxyl group blocks further addition of nucleotides in excess. Next the inter nucleotide linkage was converted to a phosphotriester bond in an oxidation step using an oxidizing agent and water as the oxygen donor. Excess nucleotides were filtered out and the cycle for the next nucleotide was started by the removal of the DMT protecting group. Following the synthesis the oligo was cleaved from the solid support. The oligonucleotides were desalted resuspended in water at a concentration of 100 or 200 M and placed in 96 deep well format. The oligonucleotides were re arrayed into Whatman Uniplate 384 well polyproylene V bottom plates. The oligonucleotides were diluted to a final concentration 30 M in 1 Micro Spotting Solution Plus Telechem arrayit.com Sunnyvale Calif. in a total volume of 15 l. In total 8 031 oligonucleotides were arrayed into twenty one 384 well plates.

Arrays were produced on Telechem arrayit.com Super amine glass substrates Telechem arrayit.com which were manufactured in 0.1 mm filtered clean room with exact dimensions of 25 76 0.96 mm. The arrays were printed using the Virtek Chipwriter with a Telechem 48 pin Micro Spotting Printhead. The Printhead was loaded with 48 Stealth SMP3B TeleChem Micro Spotting Pins which were used to print oligonucleotides onto the slide with the spot size being 110 115 microns in diameter.

Samples consisting of at least 2 g of intact total RNA were further processed for array hybridization. Amplification and labeling of total RNA samples was performed in three successive enzymatic reactions. First a single stranded DNA copy of the RNA was made hereinafter ss cDNA . Second the ss cDNA was used as a template for the complementary DNA strand producing double stranded cDNA hereinafter ds cDNA or cDNA . Third linear amplification was performed by in vitro transcription from a bacterial Tpromoter. During this step fluorescent conjugated nucleotides were incorporated into the amplified RNA hereinafter aRNA .

The first strand cDNA was produced using the Invitrogen kit Superscript II . The first strand cDNA was produced in a reaction composed of 50 mM Tris HCl pH 8.3 75 mM KCl and 3 mM MgCl 1 First Strand Buffer Invitrogen 0.5 mM dGTP 0.5 mM dATP 0.5 mM dTTP 0.5 mM dCTP 10 mM DTT 10 U reverse transcriptase Superscript II Invitrogen 18064014 15 U RNase inhibitor RNAGuard Amersham Pharmacia 27 0815 01 5 M T7T24 primer 5 GGCCAGTGAATTGTAATACGACTCACTATAGGGAGGCGGTTTTTTTTTTTT TTTTTTTTTTTT 3 SEQ ID NO 8831 and 2 g of selected sample total RNA. Several purified recombinant control mRNAs from the plant were added to the reaction mixture 2 20 pg of the following genes CAB RCA LTP4 NAC1 RCP1 XCP2 RBCL LTP6 TIM and PRKase Stratagene 252201 252202 252204 252208 252207 252206 252203 252205 252209 252210 respectively . The control RNAs allow the estimate of copy numbers for individual mRNAs in the clinical sample because corresponding sense oligonucleotide probes for each of these plant genes are present on the microarray. The final reaction volume of 20 l was incubated at 42 C. for 60 min.

For synthesis of the second cDNA strand DNA polymerase and RNase were added to the previous reaction bringing the final volume to 150 l. The previous contents were diluted and new substrates were added to a final concentration of 20 mM Tris HCl pH 7.0 Fisher Scientific Pittsburgh Pa. BP1756 100 90 mMKCl Teknova Half Moon Bay Calif. 0313 500 4.6 mM MgCl Teknova Half Moon Bay Calif. 0304 500 10 mM NH SO Fisher Scientific A702 500 1 Second Strand buffer Invitrogen 0.266 mM dGTP 0.266 mM dATP 0.266 mM dTTP 0.266 mM dCTP 40 U DNA polymerase Invitrogen 18010 025 and 2 U RNaseH Invitrogen 18021 014 . The second strand synthesis took place at 16 C. for 120 minutes.

Following second strand synthesis the ds cDNA was purified from the enzymes dNTPs and buffers before proceeding to amplification using phenol chloroform extraction followed by ethanol precipitation of the cDNA in the presence of glycogen.

Alternatively a silica gel column is used to purify the cDNA e.g. Qiaquick PCR cleanup from Qiagen 28104 . The cDNA was collected by centrifugation at 10 000 g for 30 minutes the supernatant is aspirated and 150 l of 70 ethanol 30 water was added to wash the DNA pellet. Following centrifugation the supernatant was removed and residual ethanol was evaporated at room temperature.

Linear amplification of the cDNA was performed by in vitro transcription of the cDNA. The cDNA pellet from the step described above was resuspended in 7.4 l of water and in vitro transcription reaction buffer was added to a final volume of 20 l containing 7.5 mM GTP 7.5 mM ATP 7.5 mM TTP 2.25 mM CTP 1.025 mM Cy3 conjugated CTP Perkin Elmer Boston Mass. NEL 580 1 reaction buffer Ambion Megascript Kit Austin Tex. and 1334 and 1 Tpolymerase enzyme mix Ambion Megascript Kit Austin Tex. and 1334 . This reaction was incubated at 37 C. overnight. Following in vitro transcription the RNA was purified from the enzyme buffers and excess NTPs using the RNeasy kit from Qiagen Valencia Calif. 74106 as described in the vendor s protocol. A second elution step was performed and the two eluates were combined for a final volume of 60 l. RNA is quantified using an Agilent 2100 bioanalyzer with the RNA 6000 nano LabChip.

Reference RNA was prepared as described above except Cy5 CTP was incorporated instead of Cy3CTP. Reference RNA from five reactions each reaction started with 2 ug total RNA was pooled together and quantitated as described above.

RNA was prepared for hybridization as follows for an 18 mm 55 mm array 20 g of amplified RNA aRNA was combined with 20 g of reference aRNA. The combined sample and reference aRNA was concentrated by evaporating the water to 10 l in a vacuum evaporator. The sample was fragmented by heating the sample at 95 C. for 30 minutes to fragment the RNA into 50 200 bp pieces. Alternatively the combined sample and reference aRNA was concentrated by evaporating the water to 5 l in a vacuum evaporator. Five l of 20 mM zinc acetate was added to the aRNA and the mix incubated at 60 C. for 10 minutes. Following fragmentation 40 l of hybridization buffer was added to achieve final concentrations of 5 SSC and 0.20 SDS with 0.1 g ul of Cot 1 DNA Invitrogen as a competitor DNA. The final hybridization mix was heated to 98 C. and then reduced to 50 C. at 0.1 C. per second.

Alternatively formamide is included in the hybridization mixture to lower the hybridization temperature.

The hybridization mixture was applied to a pre heated 65 C. microarray surface covered with a glass coverslip Corning 2935 246 and placed on a pre heated 65 C. hybridization chamber Telechem AHC 10 . 15 ul of 5 SSC was placed in each of the reservoir in the hybridization chamber and the chamber was sealed and placed in a water bath at 62 C. for overnight 16 20 hrs . Following incubation the slides were washed in 2 SSC 0.1 SDS for five minutes at 30 C. then in 2 SSC for five minutes at 30 C. then in 2 SSC for another five minutes at 30 C. then in 0.2 SSC for two minutes at room temperature. The arrays were spun at 1000 g for 2 minutes to dry them. The dry microarrays are then scanned by methods described above.

The microarrays were imaged on the Agilent Palo Alto Calif. scanner G2565AA. The scan settings using the Agilent software were as follows for the PMT Sensitivity 100 Red and 100 Green Scan Resolution 10 microns red and green dye channels used the default scan region for all slides in the carousel using the largest scan region scan date for Instrument ID and barcode for Slide ID. The full image produced by the Agilent scanner was flipped rotated and split into two images one for each signal channel using TIFFSplitter Agilent Palo Alto Calif. . The two channels are the output at 532 nm Cy3 labeled sample and 633 nm Cy5 labeled R50 . The individual images were loaded into GenePix 3.0 Axon Instruments Union City Calif. for feature extraction each image was assigned an excitation wavelength corresponding the file opened Red equals 633 nm and Green equals 532 nm. The setting file gal was opened and the grid was laid onto the image so that each spot in the grid overlaped with 50 of the feature. Then the GenePix software was used to find the features without setting minimum threshold value for a feature. For features with low signal intensity GenePix reports not found . For all features the diameter setting was adjusted to include only the feature if necessary.

The GenePix software determined the median pixel intensity for each feature F and the median pixel intensity of the local background for each feature B in both channels. The standard deviation SDFSDB for each is also determined. Features for which GenePix could not discriminate the feature from the background were flagged as described below.

Following feature extraction into a .gpr file the header information of the .gpr file was changed to carry accurate information into the database. An Excel macro was written to change the headers. The steps in that macro were 

Following analysis of the image and extraction of the data the data from each hybridization was pre processed to extract data that was entered into the database and subsequently used for analysis. The complete GPR file produced by the feature extraction in GenePix was imported into an excel file pre processing template. The same excel template was used to process each GPR file. The template performs a series of calculations on the data to differentiate poor features from others and to combine triplicate feature data into a single data point for each probe.

Each GPR file contained 31 rows of header information followed by rows of data for 24093 features. The last of these rows was retained with the data. Rows 31 through the end of the file were imported into the excel template. Each row contained 43 columns of data. The only columns used in the pre processing were Oligo ID F633 Median median value from all the pixels in the feature for the Cy5 dye B633 Median the median value of all the pixels in the local background of the selected feature for Cy5 B633 SD the standard deviation of the values for the pixels in the local background of the selected feature for Cy5 F532 Median median value from all the pixels in the feature for the Cy3 dye B532 Median the median value of all the pixels in the local background of the selected feature for Cy3 B532 SD the standard deviation of the values for the pixels in the local background of the selected feature for Cy3 and Flags. The GenePix Flags column contains the flags set during feature extraction. 75 indicates there were no features printed on the array in that position 50 indicates that GenePix could not differentiate the feature signal from the local background and 100 indicates that the user marked the feature as bad.

Once imported the rows with 75 flags were deleted. Then the median of B633 SD and B532 SD were calculated over all features with a flag value of 0 . The minimum values of B633 Median and B532 Median were identified considering only those values associated with a flag value of 0 . For each feature the signal to noise ratio S N was calculated for both dyes by taking the fluorescence signal minus the local background BGSS and dividing it by the standard deviation of the local background 

If the S N was less than 3 then an adjusted background subtracted signal was calculated as the fluorescence minus the minimum local background on the slide. An adjusted S N was then calculated as the adjusted background subtracted signal divided by the median noise over all features for that channel. If the adjusted S N was greater than three and the original S N were less than three a flag of 25 was set for the Cy5 channel a flag of 23 was set for the Cy3 channel and if both met these criteria then a flag of 20 was set. If both the adjusted S N and the original S N were less than three then a flag of 65 was set for Cy5 63 set for Cy3 and 60 set if both dye channels had an adjusted S N less than three. All signal to noise calculations adjusted background subtracted signal and adjusted S N were calculated for each dye channel. If the BGSS value was greater than or equal to 64000 a flag was set to indicate saturation 55 for Cy5 53 for Cy3 50 for both.

The BGSS used for further calculations was the original BGSS if the original S N was greater than or equal to three. If the original S N ratio was less than three and the adjusted S N ratio was greater than or equal to three then the adjusted BGSS was used. If the adjusted S N ratio was less than three then the adjusted BGSS was used but with knowledge of the flag status.

To facilitate comparison among arrays the Cy3 and Cy5 data were scaled to have a median of 1. For each dye channel the median value of all features with flags 0 20 23 or 25 was calculated. The BGSS for each dye in each feature was then divided by this median value. The Cy3 Cy5 ratio was calculated for each feature using the scaled

The flag setting for each feature was used to determine the expression ratio for each probe a combination of three features. If all three features had flag settings in the same category categories negatives 0 to 25 50 55 and 60 65 then the average and CV of the three feature ratios was calculated. If the CV of all three features was less than 15 the average was used. If the CV was greater than 15 then the CV of each combination of two of the features was calculated and the two features with the lowest CV were averaged. If none of the combinations of two features had a CV less than 15 then the median ratio of the three features was used as the probe feature.

If the three features do not have flags in the same category then the features with the best quality flags were used 0 25 23 20 55 53 50 65 63 60 . Features with negative flags were never used. When the best flags were two features in the same category the average was used. If a single feature had a better flag category than the other two then that feature was used.

Once the probe expression ratio was calculated from the three features the log of the ratio was taken as described below and stored for use in analyzing the data. Whichever features were used to calculate the probe value the worst of the flags from those features was carried forward and stored as the flag value for that probe. 2 different data sets can be used for analysis. Flagged data uses all values including those with flags. Filtered data sets are created by removing flagged data from the set before analysis.

In example 10 leukocyte gene expression was used to discover expression markers and diagnostic gene sets for clinical outcomes. It is desirable to validate the gene expression results for each gene using a more sensitive and quantitative technology such as real time PCR. Further it is possible for the diagnostic nucleotide sets to be implemented as a diagnostic test as a real time PCR panel. Alternatively the quantitative information provided by real time PCR validation can be used to design a diagnostic test using any alternative quantitative or semi quantitative gene expression technology.

To validate the results of the microarray experiments we used real time or kinetic PCR. In this type of experiment the amplification product is measured during the PCR reaction. This enables the researcher to observe the amplification before any reagent becomes rate limiting for amplification. In kinetic PCR the measurement is of C threshold cycle or C crossing point . This measurement C C is the point at which an amplification curve crosses a threshold fluorescence value. The threshold is set to a point within the area where all of the reactions were in their linear phase of amplification. When measuring C a lower Cvalue is indicative of a higher amount of starting material since an earlier cycle number means the threshold was crossed more quickly.

Several fluorescence methodologies are available to measure amplification product in real time PCR. Taqman Applied BioSystems Foster City Calif. uses fluorescence resonance energy transfer FRET to inhibit signal from a probe until the probe is degraded by the sequence specific binding and Taq 3 exonuclease activity. Molecular Beacons Stratagene La Jolla Calif. also use FRET technology whereby the fluorescence is measured when a hairpin structure is relaxed by the specific probe binding to the amplified DNA. The third commonly used chemistry is Sybr Green a DNA binding dye Molecular Probes Eugene Oreg. . The more amplified product that is produced the higher the signal. The Sybr Green method is sensitive to non specific amplification products increasing the importance of primer design and selection. Other detection chemistries can also been used such as ethedium bromide or other DNA binding dyes and many modifications of the fluorescent dye quencher dye Taqman chemistry for example scorpions.

Initially samples are chosen for validation which have already been used for microarray based expression analysis. They are also chosen to represent important disease classes or disease criteria. For the first steps of this example primer design primer endpoint testing and primer efficiency testing we examined actin and GUS. These genes are considered housekeeping genes because they are required for maintenance in all cells. They are commonly used as a reference that is expected to not change with experimental treatment. We chose these two particular genes as references because they varied the least in expression across 5 mRNA samples examined by real time PCR.

The inputs for real time PCR reaction are gene specific primers cDNA from specific patient samples and the standard reagents. The cDNA was produced from mononuclear RNA prepared as in example 7 by reverse transcription using OligodT primers Invitrogen 18418 012 and random hexamers Invitrogen 48190 011 at a final concentration of 0.5 ng l and 3 ng l respectively. For the first strand reaction mix 1.45 g l of total RNA R50 universal leukocyte reference RNA as described in Example 8 and 1 l of the Oligo dT Random Hexamer Mix were added to water to a final volume of 11.5 l. The sample mix was then placed at 70 C. for 10 minutes. Following the 70 C. incubation the samples were chilled on ice spun down and 88.5 l of first strand buffer mix dispensed into the reaction tube. The final first strand buffer mix produced final concentrations of 1 first strand buffer Invitrogen Y00146 Carlsbad Calif. 0.01 mM DTT Invitrogen Y00147 0.1 mM dATP NEB N0440S Beverly Mass. 0.1 mM dGTP NEB N0442S 0.1 mM dTTP NEB N0443S 0.1 mM dCTP NEB N0441S 2U of reverse transcriptase Superscript II Invitrogen 18064 014 and 0.18U of RNase inhibitor RNAGaurd Amersham Pharmacia 27 0815 01 Piscataway N.J. . The reaction was incubated at 42 C. for 1 hour. After incubation the enzyme was heat inactivated at 70 C. for 15 minutes 1 l of RNAse H added to the reaction tube and incubated at 37 C. for 20 minutes.

Two methods were used to design primers. The first was to use the software Primer Express and recommendations for primer design that are provided with the GeneAmp 7700 Sequence Detection System supplied by Applied BioSystems Foster City Calif. . The second method used to design primers was the PRIMER3 ver 0.9 program that is available from the Whitehead Research Institute Cambridge Mass. The program can also be accessed on the World Wide Web at genome.wi.mit.edu cgi bin primer primer3 www.cgi. Primers and Taqman hybridization probes were designed as described below using both programs.

The Primer Express literature explains that primers should be designed with a melting temperature between 58 and 60 degrees C. while the Taqman probes should have a melting temperature of 68 to 70 under the salt conditions of the supplied reagents. The salt concentration is fixed in the software. Primers should be between 15 and 30 basepairs long. The primers should produce and amplicon in size between 50 and 150 base pairs have a C G content between 20 and 80 have no more than 4 identical base pairs next to one another and no more than 2 C s and G s in the last 5 bases of the 3 end. The probe cannot have a G on the 5 end and the strand with the fewest G s should be used for the probe.

Primer3 has a large number of parameters. The defaults were used for all except for melting temperature and the optimal size of the amplicon was set at 100 bases. One of the most critical is salt concentration as it affects the melting temperature of the probes and primers. In order to produce primers and probes with melting temperatures equivalent to Primer Express a number of primers and probes designed by Primer Express were examined using PRIMER3. Using a salt concentration of 50 mM these primers had an average melting temperature of 3.7 degrees higher than predicted by Primer Express. In order to design primers and probes with equivalent melting temperatures as Primer Express using PRIMER3 a melting temperature of 62.7 plus minus 1.0 degree was used in PRIMER3 for primers and 72.7 plus minus 1.0 degrees for probes with a salt concentration of 50 mM.

The C source code for Primer3 was downloaded and complied on a Sun Enterprise 250 server using the GCC complier. The program was then used from the command line using a input file that contained the sequence for which we wanted to design primers and probes along with the input parameters as described by help files that accompany the software. Using scripting it was possible to input a number of sequences and automatically generate a number of possible probes and primers.

Primers for Actin Beta Actin Genbank Locus NM001101 and GUS glucuronidase beta GUSB Genbank Locus NM000181 two reference genes were designed using both methods and are shown here as examples.

The first step was to mask out repetitive sequences found in the mRNA sequences using RepeatMasker program that can be accessed at the web site located at repeatmasker.genome.washington.edu cgi bin RepeatMasker Smit AFA Green P RepeatMasker at the web site located at ftp.genome.washington.edu RM RepeatMasker.html . The last 500 basepairs on the last 3 end of masked sequence was then submitted to PRIMER3 using the following exemplary input file PRIMER SEQUENCE ID ACTB Beta Actin PRIMER EXPLAIN FLAG 1 PRIMER MISPRIMING LIBRARY SEQUENCE TTGGCTTGACTCAGGATTTAAAAACTGGAACGGTGAAGGTGACAGCAGTCGGTTGGACGAGCA TCCCCCAAAGTTCACAATGTGGCCGAGGACTTTGATTGCACATTGTTGTTTTTTAATAGTCATTCCAAATAT GAGATGCATTGTTACAGGAAGTCCCTTGCCATCCTAAAAGCACCCCACTTCTCTCTAAGGAGAATGGCCCAG TCCTCTCCCAAGTCCACACAGGGGAGGGATAGCATTGCTTTCGTGTAAATTATGTAATGCAAAATTTTTTTA ATCTTCGCCTTAATCTTTTTTATTTTGTTTTATTTTGAATGATGAGCCTTCGTGCCCCCCCTTCCCCCTTTT TTCCCCCAACTTGAGATGTATGAAGGCTTTTGGTCTCCCTGGGAGTGGGTGGAGGCAGCCGGGCTTACCTGT ACACTGACTTGAGACCAGTTGAATAAAAGTGCACACCTTA SEQ ID NO 8833 PRIMER PRODUCT OPT SIZE 100 PRIMER NUM RETURN 100 PRIMER MAX END STABILITY 9.0 PRIMER MAX MISPRIMING 12.00 PRIMER PAIR MAX MISPRIMING 24.00 PRIMER MIN SIZE 18 PRIMER OPT SIZE 20 PRIMER MAX SIZE 32 PRIMER MIN TM 61.7 PRIMER OPT TM 62.7 PRIMER MAX TM 63.7 PRIMER MAX DIFF TM 100.0 PRIMER MIN GC 20.0 PRIMER MAX GC 80.0 PRIMER SELF ANY 8.00 PRIMER SELF END 3.00 PRIMER NUM NS ACCEPTED 0 PRIMER MAX POLY X 4 PRIMER OUTSIDE PENALTY 0 PRIMER GC CLAMP 0 PRIMER SALT CONC 50.0 PRIMER DNA CONC 50.0 PRIMER LIBERAL BASE 1 PRIMER MIN QUALITY 0 PRIMER MIN END QUALITY 0 PRIMER QUALITY RANGE MIN 0 PRIMER QUALITY RANGE MAX 100 PRIMER WT TM LT 1.0 PRIMER WT TM GT 1.0 PRIMER WT SIZE LT 1.0 PRIMER WT SIZE GT 1.0 PRIMER WT GC PERCENT LT 0.0 PRIMER WT GC PERCENT GT 0.0 PRIMER WT COMPL ANY 0.0 PRIMER WT COMPL END 0.0 PRIMER WT NUM NS 0.0 PRIMER WT REP SIM 0.0 PRIMER WT SEQ QUAL 0.0 PRIMER WT END QUAL 0.0 PRIMER WT POS PENALTY 0.0 PRIMER WT END STABILITY 0.0 PRIMER PAIR WT PRODUCT SIZE LT 0.05 PRIMER PAIR WT PRODUCT SIZE GT 0.05 PRIMER PAIR WT PRODUCT TM LT 6.0 PRIMER PAIR WT PRODUCT TM GT 0.0 PRIMER PAIR WT DIFF TM 0.0 PRIMER PAIR WT COMPL ANY 0.0 PRIMER PAIR WT COMPL END 0.0 PRIMER PAIR WT REP SIM 0.0 PRIMER PAIR WT PR PENALTY 1.0 PRIMER PAIR WT IO PENALTY 0.0 PRIMER INTERNAL OLIGO MIN SIZE 18 PRIMER INTERNAL OLIGO OPT SIZE 20 PRIMER INTERNAL OLIGO MAX SIZE 35 PRIMER INTERNAL OLIGO MIN TM 71.7 PRIMER INTERNAL OLIGO OPT TM 72.7 PRIMER INTERNAL OLIGO MAX TM 73.7 PRIMER INTERNAL OLIGO MIN GC 20.0 PRIMER INTERNAL OLIGO MAX GC 80.0 PRIMER INTERNAL OLIGO SELF ANY 12.00 PRIMER INTERNAL OLIGO SELF END 12.00 PRIMER INTERNAL OLIGO NUM NS 0 PRIMER INTERNAL OLIGO MAX POLY X 5 PRIMER INTERNAL OLIGO MISHYB LIBRARY PRIMER INTERNAL OLIGO MAX MISHYB 12.00 PRIMER INTERNAL OLIGO MIN QUALITY 0 PRIMER INTERNAL OLIGO SALT CONC 50.0 PRIMER INTERNAL OLIGO DNA CONC 50.0 PRIMER IO WT TM LT 1.0 PRIMER IO WT TM GT 1.0 PRIMER IO WT SIZE LT 1.0 PRIMER IO WT SIZE GT 1.0 PRIMER IO WT GC PERCENT LT 0.0 PRIMER IO WT GC PERCENT GT 0.0 PRIMER IO WT COMPL ANY 0.0 PRIMER IO WT NUM NS 0.0 PRIMER IO WT REP SIM 0.0 PRIMER IO WT SEQ QUAL 0.0 PRIMER TASK pick per primers and hyb probe PRIMER PRODUCT SIZE RANGE 50 150 PRIMER FIRST BASE INDEX 1 PRIMER PICK ANYWAY 1 PRIMER SEQUENCE ID GUSB PRIMER EXPLAIN FLAG 1 PRIMER MISPRIMING LIBRARY SEQUENCE GAAGAGTACAGAAAAGTCTGCTAGAGCAGTACCATCTGGGTCTGGATCAAAAACGCAGAAAA TATGTGGTTGGAGAGCTCATTTGGAATTTTGCCGATTTCATGACTGAACAGTCACCGACGAGAGTGCTGGGG AATAAAAAGGGGATCTTCACTCGGCAGAGACAACCAAAAAGTGCAGCGTTCCTTTTGCGAGAGAGATACTGG AAGATTGCCAATGAAACCAGGTATCCCCACTCAGTAGCCAAGTCACAATGTTTGGAAAACAGCCCGTTTACT TGAGCAAGACTGATACCACCTGCGTGTCCCTTCCTCCCCGAGTCAGGGCGACTTCCACAGCAGCAGAACAAG TGCCTCCTGGACTGTTCACGGCAGACCAGAACGTTTCTGGCCTGGGTTTTGTGGTCATCTATTCTAGCAGGG AACACTAAAGGTGGAAATAAAAGATTTTCTATTATGGAAATAAAGAGTTGGCATGAAAGTCGCTACTG SEQ ID NO 8834 PRIMER PRODUCT OPT SIZE 100 PRIMER NUM RETURN 100 PRIMER MAX END STABILITY 9.0 PRIMER MAX MISPRIMING 12.00 PRIMER PAIR MAX MISPRIMING 24.00 PRIMER MIN SIZE 18 PRIMER OPT SIZE 20 PRIMER MAX SIZE 32 PRIMER MIN TM 61.7 PRIMER OPT TM 62.7 PRIMER MAX TM 63.7 PRIMER MAX DIFF TM 100.0 PRIMER MIN GC 20.0 PRIMER MAX GC 80.0 PRIMER SELF ANY 8.00 PRIMER SELF END 3.00 PRIMER NUM NS ACCEPTED 0 PRIMER MAX POLY X 4 PRIMER OUTSIDE PENALTY 0 PRIMER GC CLAMP 0 PRIMER SALT CONC 50.0 PRIMER DNA CONC 50.0 PRIMER LIBERAL BASE 1 PRIMER MIN QUALITY 0 PRIMER MIN END QUALITY 0 PRIMER QUALITY RANGE MIN 0 PRIMER QUALITY RANGE MAX 100 PRIMER WT TM LT 1.0 PRIMER WT TM GT 1.0 PRIMER WT SIZE LT 1.0 PRIMER WT SIZE GT 1.0 PRIMER WT GC PERCENT LT 0.0 PRIMER WT GC PERCENT GT 0.0 PRIMER WT COMPL ANY 0.0 PRIMER WT COMPL END 0.0 PRIMER WT NUM NS 0.0 PRIMER WT REP SIM 0.0 PRIMER WT SEQ QUAL 0.0 PRIMER WT END QUAL 0.0 PRIMER WT POS PENALTY 0.0 PRIMER WT END STABILITY 0.0 PRIMER PAIR WT PRODUCT SIZE LT 0.05 PRIMER PAIR WT PRODUCT SIZE GT 0.05 PRIMER PAIR WT PRODUCT TM LT 0.0 PRIMER PAIR WT PRODUCT TM GT 0.0 PRIMER PAIR WT DIFF TM 0.0 PRIMER PAIR WT COMPL ANY 0.0 PRIMER PAIR WT COMPL END 0.0 PRIMER PAIR WT REP SIM 0.0 PRIMER PAIR WT PR PENALTY 1.0 PRIMER PAIR WT IO PENALTY 0.0 PRIMER INTERNAL OLIGO MIN SIZE 18 PRIMER INTERNAL OLIGO OPT SIZE 20 PRIMER INTERNAL OLIGO MAX SIZE 35 PRIMER INTERNAL OLIGO MIN TM 71.7 PRIMER INTERNAL OLIGO OPT TM 72.7 PRIMER INTERNAL OLIGO MAX TM 73.7 PRIMER INTERNAL OLIGO MIN GC 20.0 PRIMER INTERNAL OLIGO MAX GC 80.0 PRIMER INTERNAL OLIGO SELF ANY 12.00 PRIMER INTERNAL OLIGO SELF END 12.00 PRIMER INTERNAL OLIGO NUM NS 0 PRIMER INTERNAL OLIGO MAX POLY X 5 PRIMER INTERNAL OLIGO MISHYB LIBRARY PRIMER INTERNAL OLIGO MAX MISHYB 12.00 PRIMER INTERNAL OLIGO MIN QUALITY 0 PRIMER INTERNAL OLIGO SALT CONC 50.0 PRIMER INTERNAL OLIGO DNA CONC 50.0 PRIMER IO WT TM LT 1.0 PRIMER IO WT TM GT 1.0 PRIMER IO WT SIZE LT 1.0 PRIMER IO WT SIZE GT 1.0 PRIMER IO WT GC PERCENT LT 0.0 PRIMER IO WT GC PERCENT GT 0.0 PRIMER IO WT COMPL ANY 0.0 PRIMER IO WT NUM NS 0.0 PRIMER IO WT REP SIM 0.0 PRIMER IO WT SEQ QUAL 0.0 PRIMER TASK pick per primers and hyb probe PRIMER PRODUCT SIZE RANGE 50 150 PRIMER FIRST BASE INDEX 1 PRIMER PICK ANYWAY 1

After running PRIMER3 100 sets of primers and probes were generated for ACTB and GUSB. From this set nested primers were chosen based on whether both left primers could be paired with both right primers and a single Taqman probe could be used on an insert of the correct size. With more experience we have decided not use the mix and match approach to primer selection and just use several of the top pairs of predicted primers.

In addition to the primer pairs above Primer Express predicted the following primers for GUSB Forward 178 TACCACCTGCGTGTCCCTTC SEQ ID NO 8843 and Reverse 242 GAGGCACTTGTTCTGCTGCTG SEQ ID NO 8844 . This pair of primers worked to amplify the GUSB mRNA.

Using this approach multiple primers were designed for genes that were shown to have expression patterns that correlated with clinical data in example 10. These primer pairs are shown in Table 10B and are added to the sequence listing. Primers can be designed from any region of a target gene using this approach.

Granzyme B is an important marker of CMV infection and transplant rejection. For Granzyme B the following sequence NM004131 was used as input for Primer3 

GGGGACTCTGGAGGCCCTCTTGTGTGTAACAAGGTGGCCCAGGGCATTGT CTCCTATGGACGAAACAATGGCATGCCTCCACGAGCCTGCACCAAAGTCT CAAGCTTTGTACACTGGATAAAGAAAACCATGAAACGCTACTAACTACAG GAAGCAAACTAAGCCCCCGCTGTAATGAAACACCTTCTCTGGAGCCAAGT CCAGATTTACACTGGGAGAGGTGCCAGCAACTGAATAAATACCTCTCCCA GTGTAAATCTGGAGCCAAGTCCAGATTTACACTGGGAGAGGTGCCAGCAA CTGAATAAATACCTCTTAGCTGAGTGG SEQ ID No 9086 

Primers were first tested to examine whether they would produce the correct size product without non specific amplification. The standard real time PCR protocol was used with out the Rox and Sybr green dyes. Each primer pair was tested on cDNA made from universal mononuclear leukocyte reference RNA that was produced from 50 individuals as described in Example 8 R50 .

The PCR reaction consisted of 1 RealTime PCR Buffer Ambion Austin Tex. 3 mM MgCl2 Applied BioSystems B02953 0.2 mM dATP NEB 0.2 mM dTTP NEB 0.2 mM dCTP NEB 0.2 mM dGTP NEB 1.25U AmpliTaq Gold Applied BioSystems Foster City Calif. 0.3 M of each primer to be used Sigma Genosys The Woodlands Tex. 5 l of the R50 reverse transcription reaction and water to a final volume of 19 l.

Following 40 cycles of PCR one microliter of the product was examined by agarose gel electrophoresis and on an Agilent Bioanalyzer DNA1000 chip Palo Alto Calif. . Results for 2 genes are shown in . From the primer design and the sequence of the target gene one can calculate the expected size of the amplified DNA product. Only primer pairs with amplification of the desired product and minimal amplification of contaminants were used for real time PCR. Primers that produced multiple products of different sizes are likely not specific for the gene of interest and may amplify multiple genes or chromosomal loci.

Once primers passed the end point PCR the primers were tested to determine the efficiency of the reaction in a real time PCR reaction. cDNA was synthesized from starting total RNA as described above. A set of 5 serial dilutions of the R50 reverse transcribed cDNA as described above were made in water 1 10 1 20 1 40 1 80 and 1 160.

The Sybr Green real time PCR reaction was performed using the Taqman PCR Reagent kit Applied BioSystems Foster City Calif. N808 0228 . A master mix was made that consisted of all reagents except the primes and template. The final concentration of all ingredients in the reaction was 1 Taqman Buffer A Applied BioSystems 2 mM MgCl2 Applied BioSystems 200 M dATP Applied BioSystems 200 M dCTP Applied BioSystems 200 M dGTP Applied BioSystems 400 M dUTP Applied BioSystems 1 400 000 diluted Sybr Green dye Molecular Probes 1.25U AmpliTaq Gold Applied BioSystems . The master mix for 92 reactions was made to a final volume of 2112 l. 1012 l of PCR master mix was dispensed into two light tight tubes. Each Actin primer F75 and R178 Genosys was added to one tube of PCR master mix and Each GUS primer F178 and R242 Genosys was added to the other tube of PCR master mix to a final primer concentration of 300 nM and a final volume of 1035 l per reaction tube. 45 l of the Actin master mix was dispensed into 23 wells in a 96 well plate Applied BioSystems . 45 l of the GUS master mix was dispensed into 23 wells in a 96well plate Applied BioSystems . 5 l of the template dilution series was dispensed into triplicate wells for each primer. The reaction was run on an ABI 7700 Sequence Detector Applied BioSystems .

The Sequence Detector v1.7 software was used to analyze the fluorescent signal from each well. A threshold value was selected that allowed most of the amplification curves to cross the threshold during the linear phase of amplification. The cycle number at which each amplification curve crossed the threshold C was recorded and the file transferred to MS Excel for further analysis. The Cvalues for triplicate wells were averaged. The data were plotted as a function of the logof the calculated starting concentration of RNA. The starting RNA concentration for each cDNA dilution was determined based on the original amount of RNA used in the RT reaction the dilution of the RT reaction and the amount used 5 l in the real time PCR reaction. For each gene a linear regression line was plotted through all of the dilutions series points. The slope of the line was used to calculate the efficiency of the reaction for each primer set using the equation 

Using this equation Pfaffl 2001 the efficiency for these actin primers is 2.28 and the efficiency for these GUS primers is 2.14 . This efficiency was used when comparing the expression levels among multiple genes and multiple samples. This same method was used to calculate reaction efficiency for primer pairs for each gene we studied.

Once primers were designed and tested and efficiency analysis was completed primers were used examine expression of a single gene among many clinical samples. The basic design was to examine expression of both the experimental gene and a reference gene in each sample and at the same time in a control sample. The control sample we used was the universal mononuclear leukocyte reference RNA described in example 8 R50 .

In this example three patient samples from patients with known CMV infection were compared to three patient samples from patients with no diagnosis of CMV infection based on standard diagnostic algorithms for active CMV infection including viral PCR assays serologies culture and other tests . cDNA was made from all six RNA samples and the R50 control as described above. The cDNA was diluted 1 10 in water and 5 l of this dilution was used in the 50 l PCR reaction. Each 96 well plate consisted of 32 reactions each done in triplicate. There were 17 templates and 3 primer sets. The three primer sets were GUS Actin and Granzyme B AS described above. Each of the three primer sets was used to measure template levels in 8 templates the six experimental samples R50 and water no template control . The GUS primers were also used to measure template levels a set of 8 templates identical except for the absence of the reverse transcriptase enzyme in the cDNA synthesis reaction RT . The real time PCR reactions were performed as described above in primer optimization efficiency .

The GUS amplification with RT and RT cDNA synthesis reaction templates were compared to measure the amount of genomic DNA contamination of the patient RNA sample . The only source of amplifiable material in the RT cDNA synthesis reaction is contaminating genomic DNA. Separation by at least four Cbetween the RT and RT for each sample was required to consider the sample useful for analysis of RNA levels. Since a Cdecrease of one is a two fold increase in template a difference of four Cwould indicate that genomic DNA contamination level in the RT samples was 6.25 of the total signal. Since we used these reactions to measure 30 or greater differences a 6 contamination would not change the result.

For samples with sufficiently low genomic DNA contamination the data were used to identify differences in gene expression by measuring RNA levels. Cvalues from the triplicate wells for each reaction were averaged and the coefficient of variation CV determined. Samples with high CV 2 were examined and outlier reaction wells were discarded from further analysis. The average of the wells for each sample was taken as the Cvalue for each sample. For each gene the Cwas the R50 control Cminus the sample C. The equation below was then used to identify an expression ratio compared to a reference gene Actin and control sample R50 for Granzyme B expression in each experimental sample Pfaffl M. W. 2001 . E is the amplification efficiency determined above.

The complete experiment was performed in duplicate and the average of the two ratios taken for each gene. When Actin was used as the reference gene the data show that Granzyme B is expressed at 25 fold higher levels in mononuclear cell RNA from patients with CMV than from patients without CMV . In this graph each circle represents a patient sample and the black bars are the average of the three samples in each category.

After generation and processing of expression data sets from microarrays as described in Example 14 a log ratio value is used for most subsequent analysis. This is the logarithm of the expression ratio for each gene between sample and universal reference. The processing algorithm assigns a number of flags to data that are of low signal to noise or are in some other way of uncertain quality. Correlation analysis can proceed with all the data including the flagged data or can be done on filtered data sets where the flagged data is removed from the set. Filtered data should have less variability and may result in more significant results. Flagged data contains all information available and may allow discovery of genes that are missed with the filtered data set.

In addition to expression data clinical data are included in the analysis. Continuous variables such as the ejection fraction of the heart measured by echocardiography or the white blood cell count can be used for correlation analysis. In some cases it may be desirable to take the logarithm of the values before analysis. These variables can be included in an analysis along with gene expression values in which case they are treated as another gene . Sets of markers can be discovered that work to diagnose a patient condition and these can include both genes and clinical parameters. Categorical variables such as male or female can also be used as variables for correlation analysis. For example the sex of a patient may be an important splitter for a classification tree.

Clinical data are used as supervising vectors for the significance or classification analysis. In this case clinical data associated with the samples are used to divide samples in to clinically meaningful diagnostic categories for correlation or classification analsysis. For example pathologic specimens from kidney biopsies can be used to divide lupus patients into groups with and without kidney disease. A third or more categories can also be included for example unknown or not reported . After generation of expression data and definition of using supervising vectors correlation significance and classification analysis is used to determine which set of genes are most appropriate for diagnosis and classification of patients and patient samples.

Significance analysis for microarrays SAM Tusher 2001 is a method through which genes with a correlation between their expression values and the response vector are statistically discovered and assigned a statistical significance. The ratio of false significant to significant genes is the False Discovery Rate FDR . This means that for each threshold there are a set of genes which are called significant and the FDR gives a confidence level for this claim. If a gene is called differentially expressed between 2 classes by SAM with a FDR of 5 there is a 95 chance that the gene is actually differentially expressed between the classes. SAM takes into account the variability and large number of variables of microarrays. SAM will identify genes that are most globally differentially expressed between the classes. Thus important genes for identifying and classifying outlier samples or patients may not be identified by SAM.

After generation of data from patient samples and definition of categories using clinical data as supervising vectors SAM is used to detect genes that are likely to be differentially expressed between the groupings. Those genes with the highest significance can be validated by real time PCR Example 15 or can be used to build a classification algorithm as described here.

Supervised harvesting of expression trees Hastie 2001 identifies genes or clusters that best distinguish one class from all the others on the data set. The method is used to identify the genes clusters that can best separate one class versus all the others for datasets that include two or more classes from each other. This algorithm can be used to identify genes that are used to create a diagnostic algorithm. Genes that are identified can be used to build a classification tree with algorithms such as CART.

CART is a decision tree classification algorithm Breiman 1984 . From gene expression and or other data CART can develop a decision tree for the classification of samples. Each node on the decision tree involves a query about the expression level of one or more genes or variables. Samples that are above the threshold go down one branch of the decision tree and samples that are not go down the other branch. Genes from expression data sets can be selected for classification building using CART by significant differential expression in SAM analysis or other significance test identification by supervised tree harvesting analysis high fold change between sample groups or known relevance to classification of the target diseases. In addition clinical data can also be used as variables for CART that are of know importance to the clinical question or are found to be significant predictors by multivariate analysis or some other technique. CART identifies surrogates for each splitter genes that are the next best substitute for a useful gene in classification . Analysis is performed in CART by weighting misclassification costs to optimize desired performance of the assay. For example it may be most important the sensitivity of a test for a given diagnosis be near 100 while specificity is less important.

Once a set of genes and expression criteria for those genes have been established for classification cross validation is done. There are many approaches including a 10 fold cross validation analysis in which 10 of the training samples are left out of the analysis and the classification algorithm is built with the remaining 90 . The 10 are then used as a test set for the algorithm. The process is repeated 10 times with 10 of the samples being left out as a test set each time. Through this analysis one can derive a cross validation error which helps estimate the robustness of the algorithm for use on prospective test samples. When a gene set is established for a diagnosis with a low cross validation error this set of genes is tested using samples that were not included in the initial analysis test samples . These samples may be taken from archives generated during the clinical study. Alternatively a new prospective clinical study can be initiated where samples are obtained and the gene set is used to predict patient diagnoses.

