---

title: Geometric patterns and lipid bilayers for DNA molecule organization and uses thereof
abstract: The invention is related to nucleic acid arrays and methods of using nucleic acid arrays.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09255289&OS=09255289&RS=09255289
owner: THE TRUSTEES OF COLUMBIA UNIVERSITY IN THE CITY OF NEW YORK
number: 09255289
owner_city: New York
owner_country: US
publication_date: 20101025
---
This application is a continuation in part of International Application No. PCT US2009 41434 filed on Apr. 22 2009 which claims the benefit of priority of U.S. Ser. No. 61 047 657 filed on Apr. 24 2008 and U.S. Ser. No. 61 116 815 filed on Nov. 21 2008 the contents of each of which are hereby incorporated by reference in their entireties.

All patents patent applications and publications cited herein are hereby incorporated by reference in their entirety. The disclosures of these publications in their entireties are hereby incorporated by reference into this application in order to more fully describe the state of the art as known to those skilled therein as of the date of the invention described and claimed herein.

This patent disclosure contains material that is subject to copyright protection. The copyright owner has no objection to the facsimile reproduction by anyone of the patent document or the patent disclosure as it appears in the U.S. Patent and Trademark Office patent file or records but otherwise reserves any and all copyright rights.

The work described herein was supported in whole or in part by National Institute of Health Grant Nos. PA 03 058 GM074739 GM082848. Thus the United States Government has certain rights to the invention.

The instant application contains a Sequence Listing which has been submitted in ASCII format via EFS Web and is hereby incorporated by reference in its entirety. Said ASCII copy created on Nov. 23 2010 is named 19240524.txt and is 10 165 bytes in size.

Recent years have witnessed a dramatic increase in the use of technologies that allow the detailed interrogation of individual biological macromolecules in aqueous environments under near native conditions. This increase can be attributed to the development and availability of highly sensitive experimental tools such as atomic force microscopy AFM laser and magnetic tweezers and fluorescence based optical detection all of which have all been used to study biological phenomena such as protein folding and unfolding DNA dynamics and protein nucleic acid interaction.

The invention is based in part on the discovery that nucleic acid molecules can be disposed on a substrate and positionally aligned to allow analysis of individual nucleic acid molecules. Accordingly in one aspect the invention features an array that includes a substrate and nucleic acid molecules attached to the substrate. The nucleic acid molecules can be attached to the substrate by means of a linkage e.g. a linkage between cognate binding proteins e.g. neutravidin and biotin or an antibody and antigen e.g. anti digoxigenin antibody and digoxigenin or a crosslinking linkage e.g. disulfide linkage or coupling between primary amines using gluteraldehyde. In some embodiments the nucleic acid molecules are attached at one end. In some embodiments the nucleic acid molecules are attached at both ends. In some embodiment the nucleic acid molecule is reversibly attached to the lipid bilayer along its contour.

The array further includes a coating material e.g. lipids e.g. a lipid layer e.g. a lipid bilayer deposited onto the substrate. In one embodiment the lipids are zwitterionic lipids. In one embodiment polyethylene glycol PEG is added to the lipid bilayer. For example 1 2 3 4 5 6 7 8 9 10 12 w w or more of PEG can be included in the lipid bilayer.

The substrate can be e.g. glass fused silica SiO quartz borosilicate glass polydimethylsiloxane polymerized Langmuir Blodgett film functionalized glass Si Ge GaAs GaP SiO SiN modified silicon or a polymer e.g. poly tetrafluoroethylene poly vinylidenedifluoride polystyrene or polycarbonate . In one embodiment the substrate is fused silica. The substrate can be e.g. a disc square rectangle sphere or circle. The substrate can be a suitable to be used in the methods described herein. In one embodiment the substrate is a slide used for fluorescent microscopy.

The nucleic acid molecules can be e.g. single stranded DNA double stranded DNA or RNA. The nucleic acid molecules can be about 10 20 30 40 50 100 150 200 500 1000 2000 5000 10000 50000 100000 200000 or more nucleotides in length. The number of nucleic acid molecules that can be attached to the substrate can be determined by the size of the substrate and by the design of the array. In some embodiments about 50 100 250 500 1000 2000 5000 or more nucleic acid molecules are attached to the substrate.

The nucleic acid molecules can be coupled to a label e.g. a fluorescent label e.g. YOYO1 or other fluorescent label described herein or to a quantum dot.

In another aspect the invention features an array that includes a substrate a lipid bilayer disposed on the substrate and nucleic acid molecules attached to the lipid bilayer by a linkage. In one embodiment a polypeptide e.g. neutravidin is linked to the lipid head groups and a cognate polypeptide e.g. biotin is linked to the nucleic acid molecules. The nucleic acid molecules are attached to the lipid bilayer by a linkage between the neutravidin and the biotin. In some embodiments the nucleic acid molecules are attached at one end. In some embodiments the nucleic acid molecules are attached at both ends.

In one embodiment the substrate further includes a diffusion barrier e.g. a non linear mechanical chemical or protein barrier that prevents lipid diffusion. The diffusion barrier comprises a non linear geometric diffusion barrier. For example the non linear diffusion barrier can comprise a repetitive triangular wave producing a sawtooth pattern that repeats at nanometer scale intervals or a repetitive series of angles of varying degrees. The non linear diffusion barrier can also comprise turns bends curves or interruptions. In one embodiment the barrier is non linear over its entire length. In another embodiment the barrier comprises at least one angle. In a further embodiment the barrier comprises at least one angle less than about 180 . In another embodiment the barrier comprises at least one angle of varying degrees. In some embodiments the barrier comprises multiple angles of varying degrees. In another embodiment the diffusion barrier is a non linear nanoscale barrier. A non linear mechanical barrier can be e.g. a rough scratch or etch on the substrate for example wherein the scratch or etch is not linear. Protein barriers include e.g. fibronectin. Protein barriers can be deposited onto a substrate e.g. a substrate described herein in well defined patterns. Protein barriers can have a thickness of e.g. 2 3 4 5 6 7 8 9 10 15 20 or more m thick. In one embodiment the barrier materials comprising a chemical barrier can comprise metals such as chromium aluminum gold titanium platinum osmium or nickel. In another embodiment the barrier materials can comprise metal oxides such as aluminum oxide or titanium oxide. In a further embodiment the solid support comprises a geometric barrier wherein the barrier is non linear. In another embodiment the barrier comprises a repetitive triangular wave. In other embodiments the barrier forms a nanowell by connecting vertices of the non linear geometric barrier while in some embodiments the nanowell comprises a nanopore for entry of nucleic acids. In some embodiments the non linear geometric barrier controls the lateral displacement of the nucleic acid molecule s . The non linear geometric barrier can be generated by nanofabrication for example comprising electron beam lithography or photolithographic techniques. For example nanofabrication techniques that can lay down thin 100 nm barriers of a metal such as chromium can be used.

In another aspect the invention features an array that includes a substrate a diffusion barrier described herein a lipid bilayer disposed on the substrate and nucleic acid molecules attached to the diffusion barrier by a linkage. In one embodiment the diffusion barrier is coupled to a protein e.g. biotin. A cognate protein e.g. neutravidin is then bound directly to the biotinylated diffusion barriers and biotinylated nucleic acid molecules are attached to the diffusion barriers by binding to the cognate protein e.g. neutravidin. In some embodiments the nucleic acid molecules are attached at one end. In some embodiments the nucleic acid molecules are attached at both ends.

In another aspect the invention features a cell e.g. a flowcell e.g. a microfluidic flowcell that includes an array described herein. The flowcell can be configured to allow a fluid to interact with the lipid bilayer e.g. to flow over the lipid bilayer. In some embodiments a substrate described herein further includes two openings e.g. an inlet port and an outlet port. The cell e.g. flowcell includes the substrate and a cover e.g. a glass cover e.g. a glass coverslip adhesively attached at its perimeter to the substrate creating a chamber between the substrate and the cover. The inlet port and the outlet port open into the chamber allowing the application of a hydrodynamic force into the chamber and over the lipid bilayer deposited on the substrate. For example a buffer can be forced through the inlet port into the chamber such that the buffer flows over the lipid bilayer and exits the chamber through the outlet port. In one embodiment the flowcell further comprises a staging area a bifurcated nanochannel at least one pair of parallel channels at least one pore or a combination thereof. In another embodiment the nucleic acid molecule of the invention is aligned along the non linear geometric barrier through application of a hydrodynamic force an electrophoretic force or a combination of the two forces.

In one embodiment the nucleic acid molecules of the array are positioned into a desired orientation by application of a hydrodynamic force an electrophoretic force or combination of either to the flowcell. For example upon application of a hydrodynamic force to the flowcell e.g. introduction of a buffer as described herein the nucleic acid molecules are aligned in the direction of the hydrodynamic force. In embodiments in which the nucleic acid molecules are attached at one end the hydrodynamic force results in the extension of the nonattached ends of the nucleic acid molecules in the direction of the flow of the hydrodynamic force. In embodiments in which the nucleic acid molecules are attached to the lipid heads of the lipid bilayer the nucleic acid molecules will flow in the direction of the hydrodynamic force until the lipid head encounters a non linear geometric diffusion barrier resulting in the extension of the nucleic acid molecule at a desired position in a desired orientation.

In one aspect the invention provides a method for isolating a length specific nucleic acid from a plurality of nucleic acids. The method comprises a providing the microfluidic cell of the invention wherein the attached nucleic acid molecule is a DNA molecule coupled to a fluorescent label that permits visualization of the DNA molecule b applying a first hydrodynamic or electrophoretic force perpendicular to the surface of the support to localize the attached DNA molecules to the nanowell c applying a second hydrodynamic or electrophoretic force tangential to the surface of the support to align the attached DNA molecules in a desired orientation within the pair of parallel channels and d visualizing the DNA molecule. In one embodiment the method can optionally comprise applying continuously the second hydrodynamic or electrophoretic force tangential to the surface of the support.

In another aspect the invention features a method for visualizing individual nucleic acid molecules. The method includes attaching nucleic acid molecules coupled to a fluorescent label to a substrate to a lipid bilayer or to a non linear geometric diffusion barrier as described herein to form an array. The array is then included in a flowcell and the nucleic acid molecules are aligned in a desired orientation as described herein. The arrays are then excited with a light source e.g. a laser at the excitation wavelength of the particular fluorescent label and the resulting fluorescence at the emission wavelength is detected. Detection of the fluorescence signal utilizes a microscope e.g. a fluorescent microscope. In another embodiment excitation and detection is mediated by Total Internal Reflection Fluorescence Microscopy TIRFM as described herein.

In another aspect the invention features methods for analyzing the interactions between a nucleic acid and a polypeptide. The method includes e.g. providing an array within a flowcell as described herein. The nucleic acid molecules can be aligned in a desired orientation by application of a hydrodynamic force and the nucleic acid molecules can be visualized as described herein. A target polypeptide is then added to the flowcell e.g. by being added to the buffer that mediates the hydrodynamic force across the array. In one embodiment the target polypeptide is coupled to a fluorescent label that is different than the fluorescent label coupled to the nucleic acid molecule. The localization of the target polypeptide to the nucleic acid molecule can be visualized and such localization is indicative of interaction between the target polypeptide and the nucleic acid molecule.

In one embodiment the signals from the array are collected serially over time allowing the movement of the target polypeptides on the nucleic acid molecules to be determined.

In one embodiment the length of the nucleic acid molecules is determined before and after the addition of the polypeptide wherein if the polypeptide causes the nucleic acid molecule to change length e.g. shorten or lengthen this indicates that the polypeptide causes a structural change in the nucleic acid molecule.

In another aspect the invention features methods for identifying a nucleic acid sequence e.g. a mutation in a nucleic acid sequence that disrupts an interaction between a nucleic acid molecule and a polypeptide. The method includes providing a first array within a first flowcell as described herein. The first array contains a first population of identical nucleic acid molecules that are coupled to a first fluorescent label. The method also includes providing a second array within a second flowcell as described herein. The second array contains a second population of identical nucleic acid molecules that are coupled to a first fluorescent label. In another embodiment the nucleotide sequence of the second population of nucleic acid molecules differs from the nucleotide sequence of the first population of nucleic acid molecules by at least one nucleotide. A polypeptide is then added to the flowcells e.g. by being added to the buffer that mediates the hydrodynamic force across the arrays. In an embodiment of the invention the polypeptide is coupled to a second fluorescent label e.g. one that is different from the fluorescent label coupled to the nucleic acid molecules. The localization of the polypeptide to the nucleic acid molecules on the arrays can be visualized and the localization of the polypeptide to the nucleic acid molecules of the first array but not of the second array is indicative that the nucleic acid molecules of the second array contain a nucleic acid sequence e.g. a mutation that disrupts the interaction between the nucleic acid molecules of the first array and the polypeptide.

In another aspect the invention features methods for identifying an agent that disrupts the interaction of a polypeptide and a nucleic acid. The method includes e.g. providing an array within a flowcell as described herein. The nucleic acid molecules coupled to a first fluorescent label can be aligned in a desired orientation by application of a hydrodynamic force and the nucleic acid molecules can be visualized as described herein. A polypeptide is then added to the flowcell e.g. by being added to the buffer that mediates the hydrodynamic force across the array. In another embodiment the polypeptide is coupled to a fluorescent label that is different than the fluorescent label coupled to the nucleic acid molecule. In another embodiment the polypeptide is a polypeptide that is known to bind to the nucleic acid molecules. The localization of the polypeptide to the nucleic acid molecule can be visualized. A candidate agent e.g. a compound or drug is then added to the flowcell e.g. by being added to the buffer and whether the localization of the polypeptide can be visualized. An agent that causes loss of localization of the polypeptide anywhere along the length of the nucleic acid molecule is indicative of an agent that disrupts the interaction between the nucleic acid molecule and the polypeptide.

In another aspect the invention features methods for sequencing a nucleic acid molecule. The method includes e.g. providing a single stranded nucleic acid molecule e.g. a single stranded DNA molecule. The single stranded nucleic acid molecule is mixed with DNA polymerase and a mix of fluorescently labeled nucleotide analogs e.g. fluorescently labeled dNTPs. In another embodiment each dNTP e.g. dATP dCTP dGTP and dTTP is coupled to a different fluorescent label. The mixture is reacted under conditions that allow the addition of the nucleotide analogs to the single stranded nucleotide molecules. The reacted nucleic acid molecules are then added to an array as described herein. The nucleic acid molecules can be aligned in a desired orientation by application of a hydrodynamic force and the nucleic acid molecules can be visualized as described herein.

In one embodiment the nucleic acid molecules are identical and the sequence can be determined by parallel lines of color representing particular nucleotides across the array. In one embodiment the nucleic acid molecules are different.

In another aspect the invention features methods for high throughput physical mapping of single DNA molecules for example using restriction enzymes hybridization with fluorescent proteins or fluorescence in situ hybridization. In one embodiment the method for mapping a nucleic acid molecule comprises a providing an array of the invention described herein wherein the array comprises a plurality of heterogeneous nucleic acid molecules and wherein the nucleic acid molecules are DNA molecules coupled to a first fluorescent label that permits visualization of the DNA molecules b contacting a plurality of DNA probes to the DNA molecules wherein each DNA probe is coupled to a fluorescent label that is not the first fluorescent label c applying a hydrodynamic force or an electrophoretic force tangential to the surface of the support to align the DNA molecule in a desired orientation and d visualizing the locations of binding of the DNA probes. In some embodiments the plurality of nucleic acid molecules comprises heterogeneous nucleic acid molecules. In other embodiments the plurality of nucleic acid molecules comprises identical e.g. homogeneous nucleic acid molecules. In further embodiments of the invention the method further comprises e contacting a restriction enzyme to the DNA molecule f collecting the digested DNA molecule of interest and g determining the changes in the length of the DNA molecule following the contacting step. The method can optionally comprise reversibly attaching the nucleic acid molecule along its contour to the lipid bilayer for example by exposing the DNA molecules to a flowcell buffer contain an effective concentration of calcium. The DNA molecule can unattach itself from the lipid bilayer upon washing away the calcium present in the buffer. In one embodiment the calcium concentration is at least about 0.5 mM at least about 1 mM at least about 1.5 mM at least about 2 mM at least about 2.5 mM at least about 3 mM at least about 3.5 mM at least about 4 mM at least about 4.5 mM at least about 5 mM at least about 5.5 mM at least about 6 mM at least about 6.5 mM at least about 7 mM at least about 7.5 mM at least about 8 mM at least about 8.5 mM at least about 9 mM at least about 9.5 mM at least about 10 mM or at least about 10.5 mM.

In another aspect the invention features a plurality of microfluidic flowcells described herein arranged in parallel. The plurality of flowcells can be used in parallel in any method described herein.

In another aspect the invention features a diagnostic method that uses the arrays described herein for detecting a mutation in a nucleic acid. Detection can be achieved in a variety of ways including but not limited either through sequencing of the nucleic acids or hybridization methods.

One aspect of the invention features methods for reversibly attaching a nucleic acid molecule along its contour to a lipid bilayer. The method comprises a providing the array of claim wherein the nucleic acid is a DNA molecule coupled to a first fluorescent label that permits visualization of the DNA molecule b applying a hydrodynamic force or an electrophoretic force tangential to the surface of the support to align the DNA molecule in a desired orientation c adding an effective concentration of Ca to the buffer flow and d optionally washing away the Ca from the buffer. In one embodiment the calcium concentration is at least about 0.5 mM at least about 1 mM at least about 1.5 mM at least about 2 mM at least about 2.5 mM at least about 3 mM at least about 3.5 mM at least about 4 mM at least about 4.5 mM at least about 5 mM at least about 5.5 mM at least about 6 mM at least about 6.5 mM at least about 7 mM at least about 7.5 mM at least about 8 mM at least about 8.5 mM at least about 9 mM at least about 9.5 mM at least about 10 mM or at least about 10.5 mM.

Unless otherwise defined all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention suitable methods and materials are described below. All publications patent applications patents and other references mentioned herein are incorporated by reference in their entirety. In case of conflict the present specification including definitions will control. In addition the materials methods and examples are illustrative only and not intended to be limiting. Other features and advantages of the invention will be apparent from the following detailed description and from the claims.

The present invention is based in part on the discovery that nucleic acid molecules can be disposed on a substrate and positionally aligned along a non linear geometric diffusion barrier e.g. a non linear geometric nanobarrier to allow analysis of individual nucleic acid molecules. The methods and compositions described herein include a substrate coating material e.g. a lipid bilayer and nucleic acid molecules attached directly to the substrate attached to the substrate via a linkage or attached to the lipid layer via a linkage. The nucleic acids are capable of interacting with their specific targets while attached to the substrate and by appropriate labeling of the nucleic acid molecules and the targets the sites of the interactions between the targets and the nucleic acid molecules can be derived. Because the nucleic acid molecules are positionally defined the sites of the interactions will define the specificity of each interaction. As a result a map of the patterns of interactions with nucleic acid molecules on the substrate is convertible into information on specific interactions between nucleic acid molecules and targets.

The invention is based in part on technologies useful for high throughput single molecule research which can be applied to a variety of systems involving protein DNA interactions. For example fluorescent nucleosomes have been reconstituted and it has been shown in the Examples below that DNA curtains can be used to define intrinsic energy landscapes. The invention is also based in part on assays that allow for directly visualization of 1 dimensional diffusion of proteins on DNA. For example proteins involved in mismatch repair such as Msh2 Msh6 and Mlh1 Pms1 both diffuse on DNA but have different characteristics suggesting that they travel along DNA via distinct mechanisms see Examples discussed herein .

As used herein geometric diffusion barriers geometric barriers and geometric barrier patterns are non linear non smooth diffusion barriers. For example the non linear geometric diffusion barrier can comprise a repetitive series of angles of varying degrees. The non linear geometric diffusion barrier can also comprise turns bends curves or interruptions. In one embodiment the non linear geometric diffusion barrier is non linear over its entire length. In another embodiment the non linear geometric diffusion barrier comprises at least one angle. In a further embodiment the barrier comprises at least one angle less than 180 . In another embodiment the non linear geometric diffusion barrier comprises at least one angle of varying degrees. In some embodiments the non linear geometric diffusion barrier comprises multiple angles of varying degrees. The non linear geometric barriers can also comprise a repetitive non linear geometric shape such as a repetitive triangular wave producing a sawtooth pattern that repeats at nanometer scale intervals e.g. the interval is at least about 5 nm at least about 10 nm at least about 20 nm at least about 25 nm at least about 30 nm at least about 40 nm 50 nm at least about 100 nm at least about 150 nm at least about 200 nm at least about 250 nm at least about 300 nm at least about 350 nm at least about 400 nm at least about 450 at least about 500 nm at least about 600 nm at least about 700 nm at least about 750 nm at least about 800 nm at least about 900 nm at least about 1000 nm at least about 1500 nm or at least about 2000 nm intervals . The vertex of each adjacent triangle of the repetitive triangular wave can form a nanoscale well within the non linear geometric barrier. This well can be referred to as a geometric nanowell. The vertex of each adjacent triangle of the repetitive triangular wave can have peak to peak distances of at least about 50 nm at least about 100 nm at least about 150 nm at least about 200 nm at least about 250 nm at least about 300 nm at least about 350 nm at least about 400 nm at least about 450 at least about 500 nm at least about 600 nm at least about 700 nm at least about 750 nm at least about 800 nm at least about 900 nm at least about 1000 nm at least about 1500 nm or at least about 2000 nm.

Essentially any conceivable substrate can be employed in the compositions and methods described herein. The substrate can be biological nonbiological organic inorganic or a combination of any of these existing e.g. as particles strands precipitates gels sheets tubing spheres containers capillaries pads slices films plates or slides. The substrate can have any convenient shape such as e.g. a disc square sphere or circle. The substrate and its surface can form a rigid support on which to carry out the reactions described herein. The substrate can be e.g. a polymerized Langmuir Blodgett film functionalized glass Si Ge GaAs GaP SiO SiN modified silicon or any one of a wide variety of gels or polymers such as poly tetrafluoroethylene poly vinylidenedifluoride polystyrene polycarbonate or combinations thereof. Other substrate materials will be readily apparent to those of skill in is the art upon review of this disclosure. In some embodiments the substrate is a made of SiOand is flat.

In some embodiments the substrate is coated with a linker to which the nucleic acid molecules attach. Such linkers can be e.g. chemical or protein linkers. For example the substrate can be coated with a protein such as neutravidin or an antibody.

In some embodiments the substrate includes a diffusion barrier e.g. a mechanical chemical barrier or protein barrier. Diffusion barriers can be prepared by applying barrier materials onto the substrate prior to deposition of the lipid bilayer the bilayer then forms around the barriers. A mechanical barrier can be e.g. a scratch or etch on the substrate which physically prevents lipid diffusion.

In the case of a chemical barrier the chemical nature of the barrier and not its surface topography is the primary factor in preventing lipid diffusion Powell et al. 2002 10 1262 1263 . For example a chemical barrier can comprise a metal a metal oxide or a combination thereof. In one embodiment the metal comprises chromium aluminum gold or titanium. In another embodiment the metal oxide comprises chromium oxide aluminum oxide or titanium oxide. Barrier materials can be made that are similar to the thickness of the bilayer itself e.g. 6 8 nm or thinner than the bilayer.

Protein barriers can be deposited onto substrates e.g. SiOsubstrates by a variety of methods. For example protein barriers can be deposited in well defined patterns by a process called microcontact printing Kato et al. 2000 . 45 133 137 Davies et al. 2001 7 273 282 . Microcontact printing uses a PDMS poly dimethylsiloxane template as a stamp for generating specific patterns on substrates. PDMS stamps can transfer proteins to a SiOsubstrate in patterns with features as small as 1 m and thicknesses on the order of 5 10 nm Kato et al. 2000 . 45 133 137 Davies et al. 2001 7 273 282 . The PDMS stamps used for microcontact printing can be made e.g. by soft lithography as described in Davies et al. 2001 7 273 282. Once made the PDMS can be incubated with a solution of protein dried and then placed into contact with the substrate e.g. SiO resulting in transfer of the protein ink from the PDMS stamp to the substrate and yielding a pattern defined by the stamp design. For example protein barriers can be made from fibronectin.

In one embodiment electron beam lithography can be used to engineer chromium barriers to lipid diffusion with nanometer nm scale features in order to make molecular curtains of DNA suspended above an inert bilayer. The shape of the barriers and the fluidity of the bilayer are used to direct the organization of the DNA into well defined patterns in which all of the molecules are arranged in the same orientation and aligned with respect to one another. These barriers are simple and robust they do not interfere with optical imaging of the fluorescent DNA molecules and they can be precisely constructed at predefined locations on the surface of a microfluidic sample chamber. In another embodiment the barriers can be generated as depicted in .

To the substrate is then attached a layer of a material. In one embodiment the material is one that renders the substrate inert. For example the material can be lipids forming e.g. a lipid bilayer. In another embodiment the layer is made of zwitterionic lipids. A lipid bilayer can be deposited onto the substrate by applying liposomes to the substrate. Liposomes can be produced by known methods from e.g. 1 2 dioleoyl sn glycero 3 phosphocholine DOPC or 0.5 biotin phosphatidylethanolamine biotin PE plus 99.5 DOPC Avanti Polar Lipids Alabaster Ala. . In some embodiments the lipid bilayer can include polyethylene glycol PEG . For example in embodiments where quantum dots are used to label nucleic acid molecules and or polypeptides PEG can be included in the lipid bilayer. PEG can also be included to make the surface of the bilayer inert to reagents added to the array.

As described herein the nucleic acid molecules can be attached to the substrate to the lipid bilayer or to the non linear geometric diffusion barrier to form an array. The nucleic acid molecules can be attached by a linkage either at one end of the nucleic acid molecule or at both ends. For example when a protein is coated on the substrate prior to the deposition of the lipid bilayer the nucleic acid molecule can be linked to a cognate protein that binds to the protein coated on the substrate. In one embodiment the substrate is coated with neutravidin and the nucleic acid molecule linker is biotin. Linkers can be added to the nucleic acid molecules using standard molecular biology techniques known to those of ordinary skill in the art.

Alternatively the nucleic acid molecule can be linked to the lipid bilayer. In one embodiment the lipid bilayer is deposited onto the substrate and a protein e.g. neutravidin is linked to the lipid head groups. Biotinylated nucleic acid molecules are then introduced linking the nucleic acid molecules to the lipid bilayer.

In other embodiments the nucleic acid molecules can be linked to the non linear geometric diffusion barriers. In one embodiment the diffusion barrier is a protein e.g. biotinylated bovine serum albumin BSA deposited on the substrate. Neutravidin is then bound directly to the biotinylated BSA protein barriers and biotinylated nucleic acid molecules are linked to the biotinylated BSA protein barriers. Other known protein cognate protein pairs can be used in the methods described herein. For example antibodies e.g. anti digoxigenin antibodies can be used as protein barriers and the cognate antigen e.g. digoxigenin linked to the nucleic acid molecule. In another embodiment one end of the nucleic acid molecule is attached by a linkage for example to the substrate or to a non linear geometric diffusion barrier. In a further embodiment both ends of the nucleic acid molecule are attached by linkages for example to the substrate to a non linear geometric diffusion barrier or to a combination of the two surfaces. Double tethered DNA substrates can be used for visualizing 1D diffusion For example DNA molecules can be biotinylated at both ends. While a constant moderate hydrodynamic flow force is applied DNA is suspended above an inert lipid bilayer. The only interaction between the DNA and the surface is through the biotinylated ends of the molecule. For example 80 extension of the DNA molecule corresponds to 0.5 pN of force e.g. where the DNA is not distorted .

In some embodiments attaching both ends of a nucleic acid molecule to the barriers inert substrate or a combination thereof can generate a rack for example a DNA rack as shown in and . In some embodiments the rack can be generated by reversibly anchoring the entire contour length of the nucleic acid molecule e.g. a DNA molecule to the lipid bilayer of an array described herein by exposing the nucleic acid molecules to an effective calcium concentration for example see . In one embodiment the calcium concentration is at least about 0.5 mM at least about 1 mM at least about 1.5 mM at least about 2 mM at least about 2.5 mM at least about 3 mM at least about 3.5 mM at least about 4 mM at least about 4.5 mM at least about 5 mM at least about 5.5 mM at least about 6 mM at least about 6.5 mM at least about 7 mM at least about 7.5 mM at least about 8 mM at least about 8.5 mM at least about 9 mM at least about 9.5 mM at least about 10 mM or at least about 10.5 mM.

In another embodiment the attached nucleic acid molecules and or the interacting nucleic acid molecules or polypeptides are visualized by detecting one or more labels attached to the nucleic acid molecules or polypeptides. The labels can be incorporated by any of a number of means well known to those of skill in the art. The nucleic acid molecules on the array can be coupled to a nonspecific label e.g. a dye e.g. a fluorescent dye e.g. YOYO1 Molecular Probe Eugene Oreg. TOTO1 TO PRO acridine orange DAPI and ethidium bromide that labels the entire length of the nucleic acid molecule. The nucleic acid molecules can also be labeled with Quantum dots as described herein.

In another embodiment the nucleic acid molecules e.g. the nucleic acid molecules on the array or target nucleic acid molecules can be coupled to a label at defined locations using known methods. The label can be incorporated during an amplification step in the preparation of the sample nucleic acids. For example polymerase chain reaction PCR with labeled primers or labeled nucleotides will provide a labeled amplification product. The nucleic acid molecule is amplified in the presence of labeled deoxynucleotide triphosphates dNTPs .

Alternatively a label can be added directly to the nucleic acid molecule or to an amplification product after an amplification is completed. Means of attaching labels to nucleic acids include for example nick translation or end labeling e.g. with a labeled RNA by kinasing of the nucleic acid and subsequent attachment ligation of a nucleic acid linker joining the sample nucleic acid to a label e.g. a fluorophore .

Detectable labels suitable for use in the methods and compositions described herein include any composition detectable by spectroscopic photochemical biochemical immunochemical electrical optical or chemical means. Useful labels include biotin for staining with labeled streptavidin conjugate magnetic beads e.g. Dynabeads fluorescent dyes e.g. fluorescein Texas red rhodamine green fluorescent protein and the like see e.g. Molecular Probes Eugene Oreg. radiolabels e.g. H I S C or P enzymes e.g. horse radish peroxidase alkaline phosphatase and others commonly used in an ELISA and colorimetric labels such as colloidal gold e.g. gold particles in the 40 80 nm diameter size range scatter green light with high efficiency or colored glass or plastic e.g. polystyrene polypropylene latex beads. Patents teaching the use of such labels include U.S. Pat. Nos. 3 817 837 3 850 752 3 939 350 3 996 345 4 277 437 4 275 149 and 4 366 241.

In some embodiments fluorescent labels are used. The nucleic acid molecules can all be labeled with a single label e.g. a single fluorescent label. Alternatively different nucleic acid molecules have different labels. For example one nucleic acid molecule can have a green fluorescent label and a second nucleic acid molecule can have a red fluorescent label. Suitable chromogens which can be employed include those molecules and compounds that absorb light in a distinctive range of wavelengths so that a color can be observed or alternatively which emit light when irradiated with radiation of a particular wave length or wave length range e.g. fluorescers.

A wide variety of suitable dyes are available being primary chosen to provide an intense color with minimal absorption by their surroundings. Illustrative dye types include quinoline dyes triarylmethane dyes acridine dyes alizarine dyes phthaleins insect dyes azo dyes anthraquinoid dyes cyanine dyes phenazathionium dyes and phenazoxonium dyes.

A wide variety of fluorescers can be employed either by alone or alternatively in conjunction with quencher molecules. Fluorescers of interest fall into a variety of categories having certain primary functionalities. These primary functionalities include 1 and 2 aminonaphthalene p p diaminostilbenes pyrenes quaternary phenanthridine salts 9 aminoacridines p p diaminobenzophenone imines anthracenes. oxacarbocyanine marocyanine 3 aminoequilenin perylene bisbenzoxazole bis p oxazolyl benzene 1 2 benzophenazin retinol bis 3 aminopyridinium salts hellebrigenin tetracycline sterophenol benzimidzaolylphenylamine 2 oxo 3 chromen indole xanthen 7 hydroxycoumarin phenoxazine salicylate strophanthidin porphyrins triarylmethanes and flavin. Individual fluorescent compounds that have functionalities for linking or that can be modified to incorporate such functionalities include e.g. dansyl chloride fluoresceins such as 3 6 dihydroxy 9 phenylxanthhydrol rhodamineisothiocyanate N phenyl 1 amino 8 sulfon atonaphthalene N phenyl 2 amino 6 sulfonatonaphthalene 4 acetamido 4 isothiocyanato stilbene 2 2 disulfonic acid pyrene 3 sulfonic acid 2 toluidinonaphthalene 6 sulfonate N phenyl N methyl 2 aminoaphthalene 6 sulfonate ethidium bromide stebrine auromine 0 2 9 anthroyl palmitate dansyl phosphatidylethanolamine N N dioctadecyl oxacarbocyanine N N dihexyl oxacarbocyanine merocyanine 4 3 pyrenyl butyrate d 3 aminodesoxy equilenin 12 9 anthroyl stearate 2 methylanthracene 9 vinylanthracene 2 2 vinylene p phenylene bisbenzoxazole p bis 2 4 methyl 5 phenyl oxazolyl benzene 6 dimethylamino 1 2 benzophenazin retinol bis 3 aminopyridinium 1 10 decandiyl diiodide sulfona phthylhydrazone of hellibrienin chlorotetracycline N 7 dimethylamino 4 methyl 2 oxo 3 chromenyl maleimide N p 2 benzimidazolyl phenyl maleimide N 4 fluoranthyl maleimide bis homovanillic acid resazarin 4 chloro 7 nitro 2 1 3benzooxadiazole merocyanine 540 resorufin rose bengal and 2 4 diphenyl 3 2H furanone.

The label can be a direct label i.e. a detectable label that is directly attached to or incorporated into the nucleic acid molecule. Alternatively the label can be an indirect label i.e. a label joined to the nucleic acid molecule after attachment to the substrate. The indirect label can be attached to a binding moiety that has been attached to the nucleic acid molecule prior to attachment to the substrate. For a detailed review of methods of labeling nucleic acids and detecting labeled hybridized nucleic acids see M. Anderson s Springer Verlag N.Y. 1999 and Sambrook and Russell Cold Spring Harbor Laboratory Press NY 3edition 2001 .

Polypeptides can be visualized by coupling them to e.g. fluorescent labels described herein using known methods. Alternatively other labels such as Quantum dots Invitrogen can be used as described herein.

As discussed above the use of a fluorescent label is an embodiment of the invention. Standard procedures are used to determine the positions of the nucleic acid molecules and or a target e.g. a second nucleic acid molecule or a polypeptide. For example the position of a nucleic acid molecule on an array described herein can be detected by the signal emitted by the label. In other examples when a nucleic acid molecule on the array and a target nucleic acid molecule or polypeptide are labeled the locations of both the nucleic acid molecules on the array and the target will exhibit significant signal. In addition to using a label other methods can be used to scan the matrix to determine where an interaction e.g. between a nucleic acid molecule on an array described herein and a target takes place. The spectrum of interactions can of course be determined in a temporal manner by repeated scans of interactions that occur at each of a multiplicity of conditions. However instead of testing each individual interaction separately a multiplicity of interactions can be simultaneously determined on an array e.g. an array described herein.

In certain embodiments the array is excited with a light source at the excitation wavelength of the particular fluorescent label and the resulting fluorescence at the emission wavelength is detected. In certain embodiments the excitation light source is a laser appropriate for the excitation of the fluorescent label.

Detection of the fluorescence signal can utilize a microscope e.g. a fluorescent microscope. The microscope can be equipped with a phototransducer e.g. a photomultiplier a solid state array or a ccd camera attached to an automated data acquisition system to automatically record the fluorescence signal produced by the nucleic acid molecules and or targets on the array. Such automated systems are known in the art. Use of laser illumination in conjunction with automated confocal microscopy for signal detection permits detection at a resolution of better than about 100 m better than about 50 m and better than about 25 m.

The detection method can also incorporate some signal processing to determine whether the signal at a particular position on the array is a true positive or can be a spurious signal. For example a signal from a region that has actual positive signal can tend to spread over and provide a positive signal in an adjacent region that actually should not have one. This can occur e.g. where the scanning system is not properly discriminating with sufficiently high resolution in its pixel density to separate the two regions. Thus the signal over the spatial region can be evaluated pixel by pixel to determine the locations and the actual extent of positive signal. A true positive signal should in theory show a uniform signal at each pixel location. Thus processing by plotting number of pixels with actual signal intensity should have a clearly uniform signal intensity. Regions where the signal intensities show a fairly wide dispersion can be suspect and the scanning system can be programmed to more carefully scan those positions.

Total internal reflection fluorescence microscopy TIRFM is used to detect the nucleic acid molecules and polypeptides described herein. For TIRFM a laser beam is directed through a microscope slide and reflected off the interface between the slide and a buffer containing the fluorescent sample. If the angle of incidence is greater than the critical angle sin n n where nand nare the refractive indexes of the slide and aqueous samples respectively then all of the incident light is reflected away from the interface. However an illuminated area is present on the sample side of the slide. This is called the evanescent wave and its intensity decays exponentially away from the surface Axelrod D. 1989 30 245 70 Forkey et al. 2000 74 1 35 . For most applications the evanescent wave penetrates approximately 100 nm into the aqueous medium. This geometry reduces the background signal by several orders of magnitude compared to conventional fluorescence microscopy and readily allows the detection of single fluorescent molecules because contaminants and bulk molecules in solution are not illuminated and do not contribute to the detected signal. Forkey et al. 2000 74 1 35 . By using TIRFM to visualize the arrays described herein it is possible to simultaneously monitor hundreds of aligned DNA molecules within a single field of view.

The methods described herein use microfluidic flowcells composed of substrates that are rendered inert by deposition of a lipid bilayer as described herein. For example DNA molecules can be suspended above an inert lipid bilayer. By applying a hydrodynamic force to the arrays described herein the attached nucleic acid molecules are aligned in a desired orientation that is optimal for detection by e.g. TIRFM. In one embodiment a hydrodynamic force can be used to organize DNA molecules along the leading edge of a nanoscale diffusion barrier for example a non linear geometric diffusion barrier. In another embodiment an electrophoretic force can be used to organize DNA molecules along the edge of a barrier. Thousands of DNA molecules can be visualized in a single experiment using an array described herein. All molecules are physically aligned with one another.

A microfluidic flowcell that can be used in the methods described herein is depicted in . Generally a substrate described herein is overlaid with a coverslip e.g. a glass coverslip to form a sample chamber and the substrate contains an inlet port and an outlet port through which a hydrodynamic force is applied. The hydrodynamic force can be mediated by e.g. a buffer solution that flows over the lipid bilayer described herein. An exemplary microfluidic flowcell can be constructed from 76.2 25.4 1 mm L W H fused silica slides ESCO Products Oak Ridge N.J. . Inlet and outlet holes can be drilled through the slides using e.g. a diamond coated bit 1.4 mm O.D. Eurotool Grandview Mo. . A sample chamber can be prepared from a borosilicate glass coverslip Fisher Scientific USA and e.g. double sided tape 25 m thick 3M USA or a polyethylene gasket. Inlet and outlet ports can be attached using preformed adhesive rings Upchurch Scientific Oak Harbor Wash. and cured at 120 C. under vacuum for 2 hours. The dimensions of the exemplary sample chamber are 3.5 0.45 0.0025 cm L W H . The total volume of the exemplary flowcell is 4 l. A syringe pump Kd Scientific Holliston Mass. is used to control buffer delivery to the sample chamber. This exemplary apparatus is not meant to be limiting and one of skill in the art would appreciate modifications that could be made.

A total internal reflection fluorescence microscope is depicted in . An exemplary microscope is a modified Nikon TE2000U inverted microscope. Groves et al. 1998 14 3347 3350 A 488 nm laser Coherent Inc. Santa Clara Calif. and a 532 nm laser CrystaLaser Reno Nev. were focused through a pinhole 10 m using an achromatic objective lens 25 Melles Griot Marlow Heights Md. then collimated with another achromatic lens f 200 mm . The beam was directed to a focusing lens f 500 mm and passed through a custom made fused silica prism J.R. Cumberland Inc placed on top of the flowcell. Fluorescence images were collected through an objective lens 100 Plan Apo NA 1.4 Nikon passed through a notch filter Semrock Rochester N.Y. and captured with a back thinned EMCCD Cascade 512B Photometrics Tucson Ariz. . Image acquisition and data analysis were performed with Metamorph software Universal Imaging Corp. Downington Pa. . All DNA length measurements were performed by calculating the difference in y coordinates from the beginning to the end of the fluorescent molecules. Diffusion estimates for the lipid tethered DNA substrates were performed by manually tracking the tethered ends of four different molecules and diffusion coefficients were calculated using D MSD 4t where MSD the mean square displacement is the square of the average step size measured over time interval t 0.124 sec Kelman Z. 1997 14 629 640. .

The arrays described herein can be used to detect individual nucleic acid molecules e.g. nucleic acid molecules coupled to a label. For example an array can be constructed as part of a microfluidic flowcell described herein. The nucleic acid molecules e.g. labeled nucleic acid molecules can be attached to a substrate to a lipid bilayer or to a non linear geometric diffusion barrier as described herein. Upon the application of hydrodynamic force e.g. introduction of a buffer as described herein the nucleic acid molecules are aligned in direction of the hydrodynamic force with the nonattached ends of the nucleic acid molecules extending in the direction of the flow of the hydrodynamic force. Individual nucleic acid molecules on the array can be visualized before and or after the application of the hydrodynamic force using e.g. TIRFM as described herein.

In some embodiments the interactions of nucleic acid molecules on the arrays with target polypeptides are determined. The nucleic acid molecules can be visualized before and or after the application of a hydrodynamic force as described herein. To visualize the interactions with target polypeptides the polypeptides can be coupled to a label and introduced into the array e.g. a microfluidic cell including the array as a component of the buffer that mediates the hydrodynamic force. Individual nucleic acid molecules and individual target polypeptides can be visualized e.g. by TIRFM as described herein and interactions can be determined by colocalization of the signals from the nucleic acid molecules and the polypeptides. Such interactions can be further analyzed by collecting signals over a period of time. Such methods can be used to visualize e.g. the movement of polypeptides along the length of individual nucleic acid molecules as described herein.

The methods and compositions described herein can be used to screen for compounds e.g. drug compounds that affect e.g. disrupt the interactions between nucleic acid molecules and polypeptides. For example an array can be constructed as part of a microfluidic flowcell described herein. The nucleic acid molecules e.g. labeled nucleic acid molecules can be attached to a substrate to a lipid bilayer or to a non linear geometric diffusion barrier as described herein. To visualize the interactions with target polypeptides the polypeptides can be coupled to a label and introduced into the array e.g. a microfluidic cell including the array as a component of the buffer that mediates the hydrodynamic force. In some embodiments the polypeptides are known to interact with the nucleic acid molecules and the interactions are visualized as described herein. For example the polypeptides can be proteins involved in DNA replication recombination and or repair. Candidate compounds can then be added to the array e.g. as a component of the buffer that mediates the hydrodynamic force and the effect of the compound on the interactions between individual nucleic acid molecules and the polypeptides can be visualized. Compounds that disrupt the interactions can be visually identified. Such methods can be automated.

For example the methods described herein can be used to screen for therapeutic compounds to treat cancer e.g. cancer of the breast prostate lung bronchus colon rectum urinary bladder kidney pancreas oral cavity pharynx ovary skin thyroid stomach brain esophagus liver cervix larynx soft tissue testis small intestine anus anal canal anorectum vulva gallbladder bones joints hypopharynx eye nose nasal cavity ureter gastrointestinal tract non Hodgkin lymphoma Multiple Myeloma Acute Myeloid Leukemia Chronic Lymphocytic Leukemia Hodgkin Lymphoma Chronic Myeloid Leukemia and Acute Lymphocytic Leukemia.

The methods and compositions described herein can be used to sequence nucleic acid molecules. The arrays described herein can be constructed with identical nucleic acid molecules e.g. single stranded DNA molecules or with different nucleic acid molecules e.g. single stranded DNA molecules. Before attaching the DNA molecules to the substrate an oligonucleotide primer is annealed to the DNA molecules. Polymerase is then added along with the fluorescent dNTP mix. Such methods are known in the art. Fluorescent nucleotide analogs that do not terminate extension of the DNA strand are used. The DNA molecules are then attached to the substrate and the array is visualized as described herein. The color of the nucleotide incorporated into the growing chain reveals the sequence of the DNA molecules. If all of the DNA molecules within the array are identical then the incorporation of the first nucleotide during polymerization will yield a fluorescent line extending horizontally across the array. Subsequent nucleotide addition will also yield horizontal lines and the color of each line will correspond the DNA sequence. When sequencing different DNA molecules the differences in DNA sequences are revealed as the incorporation of different fluorescent nucleotides across the array rather than the lines of identical color seen when sequencing identical DNA molecules. In some embodiments these methods are automated.

Examples are provided below to facilitate a more complete understanding of the present invention. The following examples illustrate the exemplary modes of making and practicing the present invention. However the scope of the invention is not limited to specific embodiments disclosed in these Examples which are for purposes of illustration only since alternative methods can be utilized to obtain similar results.

Single molecule visualization of protein DNA complexes can reveal details of reaction mechanisms and macromolecular dynamics inaccessible to traditional biochemical assays. However these techniques are often limited by the inherent difficulty of collecting statistically relevant information from experiments explicitly designed to look at single events. New approaches that increase throughput capacity of single molecule methods have the potential for making these techniques more readily applicable to a variety of biological questions involving different types of DNA transactions. Here we present a method for organizing DNA molecules into curtains along the leading edges of nanofabricated chromium barriers which are located at strategic positions on a fused silica slide otherwise coated with a supported lipid bilayer. The individual molecules that make up the DNA curtains are visualized by total internal reflection fluorescence TIRFM allowing simultaneous imaging of thousands of perfectly aligned molecules. These DNA curtains present a robust and powerful experimental platform portending massively parallel data acquisition of individual protein DNA interactions in real time.

Single molecule techniques have revealed many insights into previously inaccessible aspects of biology and this field is now poised to profoundly impact the way that all biological macromolecules can be studied. However many single molecule methods suffer from disadvantages that limit their broader application and meeting the oncoming challenges will require the development of more robust user friendly high throughput experimental platforms that can be readily applied to a broad range of biochemical systems.

For example one common problem of many single molecule techniques is the requirement for the macromolecules under investigation be anchored to a solid support surface which is unlike anything encountered within a cellular environment. It is essential to minimize nonspecific interactions with the surface that can perturb their biological properties. Traditional approaches for passivating surfaces have included nonspecific blocking agents e.g. BSA or casein or covalent modification with polyethylene glycol PEG A1 A2 . Nonspecific blocking proteins often do not work well enough to prevent surface adsorption of other molecules A3 . PEGylated surfaces are efficient at preventing nonspecific interactions between proteins or nucleic acids and the underlying surface but PEG alone can not be sufficient in all cases. More recently vesicle encapsulated reactions have been used in single molecule analysis A4 A5 . Vesicle encapsulation is a very promising approach that makes use of the native environment provided by lipid membranes but has limited potential for biochemical experiments involving macromolecules that can not be confined within vesicles or those requiring successive addition of high molecular weight components.

Single molecule techniques are also impeded by the difficulty of collecting statistically relevant information using procedures designed to image just one or at best a few molecules at any given time. This can be problematic when the reactions under investigation require the use of long DNA substrates especially when the reactions themselves are inefficient and or involve rare intermediates. Procedures for anchoring numerous long DNA molecules to surfaces are present in the literature and each has great potential for specific situations but they also suffer from specific drawbacks with respect to biochemical applications. For example Bensimon et al. developed DNA combing A6 which has evolved into a powerful tool reviewed in A7 . Combed DNA is anchored to a hydrophobic glass slide and aligned with a receding air water meniscus yielding molecules adhered to the glass by multiple contact points and stretched 150 percent beyond the length of normal B DNA. The hydrophobic surfaces required for combing and the resulting distortion of the DNA can not be compatible with many proteins. In addition while the combed DNA molecules are aligned along a common direction their ends are not aligned relative to one another nor is the orientation of the DNA defined with respect to its sequence. In another elegant approach Kabata and colleagues reported that belts of DNA could be stretched between two aluminum electrodes by dielectrophoresis which they used to visualize the motion of RNAP and EcoRI by fluorescence microscopy A8 A9 . However the molecules in these belts are not oriented in the same direction with respect to their sequence it remains unclear how the DNA links to the aluminum and broader use of this technique has not been realized A10 . Recently Guan and Lee have demonstrated that highly ordered arrays of DNA molecules can be stamped onto PDMS polydimethyl siloxane with an intriguing method based on molecular combing A11 . This technology is promising but protein adsorption to unmodified PDMS can present a limitation for biochemical applications. Prentiss and colleagues have used an approach in which magnetic beads were linked to the free ends of DNA molecules anchored to a glass surface A12 . Kim et al. reported a similar approach in which they anchored molecules of DNA to a PEGylated surface and stretched the DNA with buffer flow A13 . In each of these examples they concurrently detect 100 200 molecules but required 10 magnification to expand the field of view thus the overall density of the anchored DNA remained quite low A13 . Finally Schwartz and co workers have pioneered single DNA molecule optical mapping techniques A14 A15 but these approaches may not be applicable for real time biochemical analysis of protein DNA interactions.

To address these challenges we have developed DNA curtains which allow simultaneous imaging of on the order of one hundred individual DNA molecules within a single field of view A16 . Curtains are assembled by anchoring one end of a biotinylated DNA molecule to a lipid bilayer which provides an inert environment compatible with a wide range of biological molecules A17 . The bilayer also permits long range two dimensional motion of the lipid tethered DNA molecules. We have taken advantage of this mobility by using hydrodynamic force to organize the DNA molecules at microscale diffusion barriers which are manually etched into the surface of the flowcell and oriented perpendicular to the direction of buffer flow. Lipids within the bilayer can not traverse the etched barrier A18 therefore the lipid tethered DNA molecules accumulate along the leading edges of these barriers A16 . A drawback of this approach is that manual etching greatly limits user control over the dimensions and locations of the microscale diffusion barriers. The etched barriers also compromise the quality of the optical surface leading to problems such as light scattering uneven alignment of DNA nonspecific protein adsorption inefficient coverage of the viewing area and a high failure rate. Together these problems can undermine the use of DNA curtains for single molecule biological research.

In this work we use electron beam lithography to engineer chromium barriers to lipid diffusion with nanometer nm scale features in order to make molecular curtains of DNA suspended above an inert bilayer. The shape of the barriers and the fluidity of the bilayer are used to direct the organization of the DNA into well defined patterns in which all of the molecules are arranged in the same orientation and aligned with respect to one another. These barriers are simple and robust they do not interfere with optical imaging of the fluorescent DNA molecules and they can be precisely constructed at predefined locations on the surface of a microfluidic sample chamber. Using these nanoscale barriers we can concurrently image several hundred and even several thousand aligned DNA molecules in a single field of view. These uniform patterns of DNA provide a unique and powerful experimental platform enabling massively parallel data acquisition from thousands of individual molecules and offer a myriad of potential applications.

The use of mechanical or chemical barriers to corral lipids within supported bilayers has been pioneered by S.G. Boxer and colleagues A20 . Inspired by these studies we demonstrated that mechanical barriers to lipid diffusion can also be used to organize DNA molecules into curtains at defined locations on a fused silica surface A16 . We have shown that these curtains serve as a highly effective experimental platform for the study of protein DNA interactions at the single molecule level A21 23 . The general principles behind this approach are outlined in . To make the curtains DNA is first anchored by one end to a supported lipid bilayer coating the surface of the sample chamber . In the absence of a hydrodynamic force the molecules are randomly distributed on the surface but lie outside of the detection volume defined by the penetration depth of the evanescent field 150 200 nm A24 . Application of flow pushes the DNA through the sample chamber with one end anchored to the bilayer. The barriers are oriented perpendicular to the direction of flow at strategic locations in the path of the DNA this halts the movement of the molecules causing them to accumulate at the edges of the barriers where they are extended parallel to the surface enabling visualization along their full contour length by TIRFM A16 .

In the past we used micrometer scale diffusion barriers prepared by manually scoring the surface with a diamond tipped scribe A16 A19 A21 23 . Manual etching is simple yet inherently problematic because it is difficult to control. As an alternative we sought to apply lithographic techniques for generating precisely nanoscale patterned barriers that could be used to organize DNA molecules into curtains making the most efficient use of available surface area. shows a cartoon representation of a desired surface pattern comprised of an interlocking series of bracket shaped barriers and the important features of the design are indicated. Guide channels oriented parallel to the direction of flow ensure efficient capture of approaching DNA molecules tethered to the bilayer. Perpendicular barriers form the curtain rods against which the DNA molecules are aligned. The parallel barriers of the guide channels also prevent the molecules from sliding off the edges of the perpendicular barriers when buffer flow is transiently paused see below . Collectively these features can organize the tethered DNA molecules into curtains wherein all of the constituent molecules are aligned in the same orientation.

An optical image of a chromium barrier pattern with the afore mentioned design elements prepared by direct write electron beam E beam lithography is shown in . Fluorescence images of the same type of barrier collected at 60 magnification after deposition of a supported bilayer containing fluorescent lipids 0.5 rhodamine DHPE confirm that the lipids coat the fused silica but that they do not cover the chromium barriers A20 . The image in shows a section of fused silica surface with a 2 3 series of chromium barrier sets. shows an atomic force microscopy AFM image illustrating a representative single barrier that is 31 nm tall and we have also made functional patterns with barriers ranging from 20.5 nm up to 173 nm. The height of the barriers is dictated by the amount of chromium evaporated onto the surface and can be accurately controlled as required for specific experimental needs. shows a scanning electron microscopy SEM image of a parallel chromium barrier revealing a width 100 9 nm. As a point of comparison show AFM and SEM images of manually etched barriers respectively. In contrast to the highly uniform chromium barriers the width of the etched barriers can be on the order of 5 10 m and they also have highly irregular topology as previously reported A18 .

To assemble DNA curtains biotinylated DNA is tethered to the bilayer through tetravalent neutravidin that is in turn attached to a subset of lipids that have biotinylated head groups. The DNA molecules are pushed in the direction of the diffusion barriers through the application of a flow force. The initial application of buffer flow pushes the DNA into the barrier patterns where they accumulate at the ends of the guide channels. Once all of the molecules have accumulated within the barriers flow is briefly terminated allowing the molecules to diffuse freely within the bilayer. This step permits the DNA molecules to diffuse laterally within the bilayer so that they become evenly distributed along each of the barriers. The DNA molecules themselves are retained within the barrier set because flow is not stopped long enough to allow them to diffuse out of the guide channel openings. Flow can then be resumed to assess the distribution of the DNA and if necessary this process is repeated at short intervals to achieve even disbursement of the DNA along the barrier edges see Materials and Methods .

The design of the curtains should yield DNA molecules all aligned with the same sequence orientation based upon the location of the biotin tag. If some fraction of the DNA is bound to the bilayer via its unlabeled end then this second population of DNA will have reversed sequence orientation with respect to those molecules anchored via the biotin tag. Verifying the alignment is critical for future experiments meant to probe the sequence dependent behavior of DNA bound proteins. DNA has five EcoRI restriction sites located 21 226 bp 26 106 bp 31 747 bp 39 168 bp and 44 972 bp from the left end of the molecules. If the molecules are in a given orientation then complete EcoRI digestion of DNA anchored by its left end will yield a tethered fragment of approximately 21 kb and all of the downstream fragments will be washed from the sample chamber. Similarly an EcoRI digestion of a curtain comprised of DNA biotinylated at the right end should yield much smaller fragments corresponding to a final length of 3.5 kb. confirm these predictions proving that all of the DNA molecules making up the curtain are tethered in the same orientation.

Optical restriction mapping has evolved into a powerful technique for the physical analysis of large DNA molecules A14 A15 A25 and because the DNA curtains are organized with all of the molecules in a defined orientation they provide a very simple platform for mapping the locations of specific restriction sites. As shown in different combinations of single restriction sites can be easily mapped within the DNA curtain by successive introduction of the desired enzymes into the flowcell. In this example the curtain was sequentially cut with NheI XhoI EcoRI NcoI PvuI and SphI and the observed lengths m of the resulting DNA fragments were measured and plotted as a histogram to illustrate the location of the cleavage sites. As shown here complete restriction digests leave behind tethered DNA fragments whose lengths correspond to the cleavage site closest to the biotinylated ends of the DNA and any other downstream fragments are washed away. Complete restriction digests can only reveal single cleavage sites and can not be used to map multiple identical restriction sites throughout the DNA molecules. In contrast a partial digest with one or more restriction enzymes should yield a population of discrete fragments whose lengths correspond to each of the restriction sites present in the DNA molecules. To verify this prediction we preformed a partial EcoRI digest of curtains made with DNA molecules that were tethered by either the right or the left ends. The apparent lengths of the resulting DNA fragments were then measured and their distributions plotted as histograms . This partial digest strategy was sufficient to identify all five EcoRI sites within the phage genome. Together these experiments demonstrate that the locations of restriction sites within large molecules can be rapidly identified via optical mapping of the DNA curtains.

The DNA fragment lengths reported above are indicated in microns and represent an apparent observed value rather than a direct measure of the actual contour length. To estimate the actual size of any DNA fragment in either microns or base pairs the observed contour length of the DNA in microns must be corrected for the fact that the molecules are not fully extended. The mean extension x L of the DNA molecules examined in this study was approximately 0.80 corresponding to 0.6 pN of tension. A plot of all the different measured DNA fragment contour lengths in m versus the known length of fully extended DNA fragments based on their size in either microns or base pairs can be used as a calibration curve to estimate the actual size of the DNA fragments. Although sufficient for estimating the number of base pairs in relatively large tethered DNA fragments we note that this empirical relationship breaks down with shorter DNA molecules 9 kb because the tension experienced by the DNA and therefore the mean extended length decreases as the molecules get shorter. For example the SphI 2.2 kb fragments described above were too short to measure. In addition smaller DNA fragments tend to diffuse laterally along the barrier edges making it difficult count them directly. As consequence of these two effects the observed lengths for the shorter fragments are just an approximation and the total number observed was based on the initial number of uncut DNA molecules. While measurements of these smaller fragment lengths is beyond the scope of this study it should nevertheless be possible by including a rigorous analysis of signal intensity data and or accommodating for effects of shear flow on extended polymers and it can also be possible to restrict their lateral motion with alternative barrier designs.

Here we apply nanolithography to engineer arrays of diffusion barriers which are used to organize curtains of DNA on a surface coated with a supported lipid bilayer. With these tools we can visualize thousands of individual perfectly aligned DNA molecules all arranged in the exact same orientation in real time using TIRFM. These nanofabricated DNA curtains offer numerous advantages that overcome some current limitations of single molecule DNA imaging. The method is simple and robust the flowcells are reusable the barriers themselves are highly uniform and they do not compromise the optical quality of the fused silica or interfere with signal detection. In addition the bilayer provides an inert environment closely resembling a cell membrane and is compatible with many biological macromolecules ensuring that the DNA curtains can be used for imaging a range of biochemical systems A17 A20 .

Direct write electron beam lithography for nanofabricating barrier patterns offers tremendous reproducibility accuracy design flexibility and is advantageous for prototyping devices. The key elements of the barrier design barrier height barrier width barrier shape barrier material separation distance between adjacent barriers guide channel shape or width etc. can all be adjusted to accommodate any desired substrate and or experimental need with virtually no limitations on the overall pattern other than those spatial constraints imposed by the use of lithographic techniques. The shapes and dimensions of the barriers presented here were specifically constructed for visualizing DNA molecules. For example the parallel barriers within these sets are separated from one another by a distance of 16 m to allow maximal surface coverage. Moreover the design flexibility conferred by the use of nanolithography beckons the development of much more complex barrier elements to accommodate and or manipulate any desired substrate.

Our primary intent was to generate new tools that facilitate massively parallel data collection for single molecule analysis of protein DNA interactions yet it is also apparent that the DNA curtains offer a myriad of other potential applications. For example they enable rapid generation of physical maps of long DNA molecules and we have demonstrated this with a series of optical mapping assays based on restriction endonuclease cleavage. Because these reactions are performed within a microfluidic sample chamber and DNA is only anchored by one end collection of the liberated fragments in sufficient quantities for cloning and further analysis should prove straightforward. Similarly these curtains can also be used to generate maps of binding sites for any DNA binding protein of interest as long as it can be tagged with a fluorescent label. Finally the perfect alignment of the DNA molecules within the curtains greatly facilitates data evaluation and offers the future potential for automated image analysis.

Fused silica slides were cleaned in NanoStrip solution CyanTek Corp. for 20 minutes then rinsed with acetone and isopropanol and dried with N. The slides were spin coated with a bilayer of polymethylmethacrylate PMMA molecular weight 25K plus 3 anisole and 495K plus 1.5 in anisole MicroChem followed by a layer of Aquasave conducting polymer Mitsubishi Rayon . Each layer was spun at 4000 rpm for 45 seconds using a ramp rate of 300 rpm s. Patterns were written by E beam lithography using an FEI Sirion scanning electron microscope equipped with a pattern generator and lithography control system J. C. Nabity Inc. . After the pattern was written the Aquasave was washed off with deionized water and the sample dried with N. Resist was developed using a 3 1 solution of isopropanol to methyl isobutyl ketone MIBK for 1 minute with ultrasonic agitation at 5 C. The substrate was then rinsed in isopropanol and dried with N. A thin layer of chromium was deposited using a Semicore electron beam evaporator. To effect lift off the coated substrate was submerged in a 65 C. acetone bath for 30 minutes and then gently sonicated. Following lift off samples were rinsed with acetone to remove stray chromium flakes and dried with N. Barriers were imaged using a Hitachi 4700 scanning electron microscope and a PSIA XE 100 Scanning Probe Microscope in noncontact mode. Optical images of the barriers were taken with a Nikon Eclipse ME600 at either 10 or 20 magnification as indicated .

Flowcells and DNA curtains were constructed as previously described A16 . All lipids were purchased from Avanti Polar Lipids and liposomes were prepared as previously described. In brief a mixture of DOPC 1 2 dioleoyl sn glycero phosphocholine 0.5 biotinylated DPPE 1 2 dipalmitoyl sn glycero 3 phosphoethanolamine N cap biotinyl and 8 mPEG 550 DOPE 1 2 dioleoyl sn glycero 3 phosphoethanolamine N methoxy polyethylene glycol 550 . Liposomes were applied to the sample chamber for 30 minutes. Excess liposomes were flushed away with buffer containing 10 mM Tris HCl pH 7.8 and 100 mM NaCl. The flowcell was then rinsed with buffer A 40 mM Tris HCl pH 7.8 1 mM DTT 1 mM MgCl and 0.2 mg ml BSA and incubated for 15 minutes. Neutravidin 660 nM in buffer A was then injected into the sample chamber and incubated for 10 minutes. After rinsing thoroughly with additional buffer A biotinylated DNA 10 pM pre stained with YOYO1 1 dye per 600 base pairs was injected into the sample chamber incubated for 10 minutes and unbound DNA was removed by flushing with buffer at 0.1 ml min. For imaging the buffers also contained 100 pM YOYO1 along with an oxygen scavenging system comprised of 1 w v glucose 60 mM mercaptoethanol glucose oxidase 100 units ml and catalase 1 560 units ml . Application of buffer flow caused the lipid tethered DNA molecules to align along the leading edges of the diffusion barriers. The flow was stopped for 5 minutes allowing the DNA to diffuse towards the center of the barriers. The flow was started at 0.1 ml min for 30 seconds and the flow on off cycle was repeated 3 5 times until DNA curtains of even density formed along the diffusion barriers.

The basic design of the microscope used in this study has been previously described A19 . The beam intensity at the face of the prism was 10 15 mW. Images were detected with a back illuminated EMCCD detector Photometrics Cascade 512B . TIRFM images were collected using a 60 water immersion objective lens Nikon 1.2 NA Plan Apo or a 10 objective Nikon 0.45 NA Plan Apo as indicated.

For complete digests 700 l of the desired restriction enzyme in reaction buffer A 40 mM Tris HCl pH 7.8 1 mM MgCl 1 mM DTT and 0.2 mg ml BSA plus 50 mM NaCl and 10 mM MgClwas injected at 0.2 ml min. All restriction enzymes were purchased from NEB and the amounts of enzymes used were as follows NheI 100 units ml XhoI 100 units ml EcoRI 100 units ml NcoI 50 units ml PvuI 50 units ml and SphI 50 units ml . Images of the DNA molecules were collected before the restriction enzyme injection and after all of the enzyme solution had flown through. For partial digests the amount of EcoRI was reduced to 20 units ml and 700 l was injected at 0.4 ml min.

E beam lithography was used to create chromium nanoscale barriers in a defined pattern on a fused silica slide. DNA stained with YOYO1 was then assembled into curtains along the edges of the nanoscale barriers. Images were collected at 60 magnification at an acquisition rate of 10 frames per second and a time stamp is shown in the lower right corner. The movie begins with buffer flow on and the DNA extended. The flow is stopped showing that all of the DNA molecules diffuse away from the surface and the barrier edges. In the absence of flow the full length of the DNA can not be seen because it resides outside the excitation volume defined by the penetration depth of the evanescent field. Finally flow is resumed to re extend the DNA.

Images were captured at 10 magnification and the full field of view shows a 2 3 array of barrier sets from the surface of a slide that had a 4 6 array of barriers 24 identical barrier patterns in total . The movie was collected at 10 frames per second and a time stamp is indicated in the lower right hand corner. The movie begins with buffer flow on and the DNA extended then flow is stopped showing that the DNA diffuses away from the surface and the barrier edges and finally flow is resumed to re extend the DNA.

The analysis of individual molecules is evolving into an important tool for biological research and presents conceptually new ways of approaching experimental design strategies. However more robust methods are required if these technologies are to be made broadly available within the biological research community. To help achieve this goal we have combined nanofabrication techniques with single molecule optical microscopy for assembling and visualizing curtains comprised of thousands of individual DNA molecules organized at engineered diffusion barriers on lipid bilayer coated surface. Here we present an important extension of this technology that implements non linear geometric barrier patterns comprised of thousands of nanoscale wells that can be loaded with single molecules of DNA. We show that these geometric nanowells can be used to precisely control the lateral distribution of the individual DNA molecules within curtains assembled along the edges of the engineered barrier patterns. The individual molecules making up the DNA curtain can be separated from one another by a user defined distance dictated by the dimensions of the nanowells. We demonstrate the broader utility of these patterned DNA curtains in a real time restriction assay that we refer to as dynamic optical restriction mapping which can be used to rapidly identify entire sets of cleavage sites within a large DNA molecule.

Advances in biology are often made possible only through the concurrent establishment of new technologies that bring together different scientific disciplines to achieve a common goal. This maxim has proven especially true for single molecule research where aspects of physics chemistry and biology have all provided invaluable contributions which have led to the development of highly sensitive detection optics robust fluorescent tags and chemistries for anchoring individual molecules to surfaces. Taken together these technological achievements have enabled new approaches for isolating viewing and manipulating individual macromolecules while at the same time addressing important biological questions and have opened new avenues of research which have already revealed a wealth of new information. Nowhere is this more evident that in the study of nucleic acids and protein nucleic acid interactions B1 5 . Yet even as these approaches are revealing more and more details of specific reaction mechanisms broader implementation of the techniques remains hindered by their low throughput capacity and challenging technical nature as a consequence most studies have been restricted to relatively simple model systems conducted in a handful of specialized laboratories.

In an effort to help make these techniques more broadly accessible we have integrated nanoscale engineering microfluidics and inert lipid bilayer coated surfaces with optical microscopy to develop high throughput methods for making curtains of DNA molecules that can be used for massively parallel data acquisition from thousands of individual protein DNA complexes in real time using a robust experimental platform that is amenable to a wide variety of biological applications B6 B7 . These DNA curtains permit concurrent visualization of thousands of individual DNA molecules that are perfectly aligned with respect to one another and also enable real time detection of fluorescent proteins bound to the DNA B8 11 . However a remaining limitation of this approach is the lack of control over the lateral separation between each of the molecules making up the DNA curtains. This can potentially present a problem especially in instances where either implementation of the desired experiments or the subsequent data analysis is confounded by the presence of overlapping molecules within the curtain. In addition while relatively large DNA 48 502 bp substrates remain at fixed lateral locations along the smooth chromium barriers as long as constant buffer flow is maintained smaller DNA fragments exhibit significant lateral movement along the barriers. The lipid bilayers remain fluid at the edges of the chromium barriers B12 and the observed mobility of the shorter DNA arises from slippage of the lipid anchors along the smooth barrier edges. This is especially problematic for single molecule data analysis because molecules targeted for investigation will inevitably slip along the barrier edge and collide with and or bypass one another making it extremely difficult to distinguish similar looking molecules from one another over periods longer than just a few seconds. Taken together these complications can place significant design constraints on the experiments that can be conducted with DNA curtains especially when using DNA substrates shorter in length than the phage genome.

In an effort to solve these problems we sought to develop new barrier designs that could control the positions of the molecules within the DNA curtain. Here we take advantage of the fact that lipid tethered DNA molecules can slide along the barrier edges by demonstrating that simple non linear geometric barrier patterns made by nanolithography can be used to construct aligned DNA curtains with precisely controlled lateral displacement between the DNA molecules. These barriers consist of a repetitive triangular wave sawtooth pattern with nanometer scale features where the vertex of each adjacent triangle forms a tiny well which we will refer to as geometric nanowells. We show that single molecules of DNA can be loaded into the nanowells and retained indefinitely for continued observation and analysis so long as continuous buffer flow is maintained. The minimal separation distance between each of the DNA molecules within the curtain is defined by the distance between the adjacent nanowells within the non linear geometric barrier pattern and the total number of DNA molecules within in each well is controlled by the amount of DNA applied to the surface. Although designed explicitly for single molecule biochemical studies of protein DNA interactions DNA curtains made with these non linear geometric barrier patterns have a multitude of other potential applications. As an example of their broader utility we use the DNA curtains made with geometric nanowells in a dynamic optical restriction mapping assay which uses real time data collection to identify an entire set of specific cleavage sites within the phage genome in a reaction that takes under 2 minutes.

Fused silica slides were cleaned in NanoStrip solution CyanTek Corp Fremont Calif. for 20 minutes then rinsed with acetone and isopropanol and dried with N. The slides were spin coated with a bilayer of polymethylmethacrylate PMMA molecular weight 25K and 495K 3 in anisole MicroChem Newton Mass. . Each layer was spun at 4 000 rpm for 45 seconds using a ramp rate of 300 rpm s. Patterns were written by E beam lithography using an FEI Sirion scanning electron microscope equipped with a pattern generator and lithography control system J. C. Nabity Inc. Bozeman Mont. . Resist was developed using a 3 1 solution of isopropanol to methyl isobutyl ketone MIBK for 2 minutes with ultrasonic agitation at 5 C. The substrate was then rinsed in isopropanol and dried with N. A thin layer of chromium was deposited using a Semicore electron beam evaporator. To effect lift off the coated substrate was submerged in a 75 C. acetone bath for 30 minutes and then gently sonicated. Following lift off samples were rinsed with acetone to remove stray chromium flakes and dried with N. Barriers were imaged using a Hitachi 4700 scanning electron microscope and a PSIA XE 100 Scanning Probe Microscope in noncontact mode. Optical images of the barriers were taken with a Nikon Eclipse ME600 at either 10 or 20 magnification as indicated .

The flowcells were assembled from fused silica slides G. Finkenbeiner Inc. with chromium nanoscale diffusion barriers. Inlet and outlet ports were made by boring through the slide with a high speed precision drill press equipped with a diamond tipped bit 1.4 mm O.D. Kassoy . The slides were cleaned by successive immersion in 2 v v Hellmanex 1 M NaOH and 100 MeOH. The slides were rinsed with filtered sterile water between each wash and stored in 100 MeOH until use. Prior to assembly the slides were dried under a stream of nitrogen and baked in a vacuum oven for at least 1 hour. A sample chamber was prepared from a borosilicate glass coverslip Fisher Scientific and double sided tape 25 m thick 3M . Inlet and outlet ports Upchurch Scientific were attached with hot melt adhesive SureBonder glue sticks FPC Corporation . The total volume of the sample chambers was 4 l. A syringe pump Kd Scientific and actuated injection valves Upchurch Scientific were used to control sample delivery buffer selection and flow rate. The flowcell and prism were mounted in a custom built heater with computer controlled feedback regulation to control the temperature of the sample from between 25 37 C. 0.1 C. as necessary.

DNA curtains were constructed as described B7 . All lipids were purchased from Avanti Polar Lipids and liposomes were prepared as previously described. In brief a mixture of DOPC 1 2 dioleoyl sn glycero phosphocholine 0.5 biotinylated DPPE 1 2 dipalmitoyl sn glycero 3 phosphoethanolamine N cap biotinyl and 8 mPEG 550 DOPE 1 2 dioleoyl sn glycero 3 phosphoethanolamine N methoxy polyethylene glycol 550 . Liposomes were applied to the sample chamber for 30 minutes. Excess liposomes were flushed away with buffer containing 10 mM Tris HCl pH 7.8 and 100 mM NaCl. The flowcell was then rinsed with buffer A 40 mM Tris HCl pH 7.8 1 mM DTT 1 mM MgCl and 0.2 mg ml BSA and incubated for 15 minutes. Neutravidin 660 nM in buffer A was then injected into the sample chamber and incubated for 10 minutes. After rinsing thoroughly with additional buffer A biotinylated DNA 10 pM pre stained with 1 2 nM YOYO1 was injected into the sample chamber incubated for 10 minutes and unbound DNA was removed by flushing with buffer at 0.1 ml min. Application of buffer flow caused the lipid tethered DNA molecules to align along the leading edges of the diffusion barriers. The flow was stopped for 5 minutes allowing the DNA to diffuse towards the center of the barriers. The flow was started at 0.1 ml min for 30 seconds and the flow on off cycle was repeated 3 5 times until DNA curtains of even density formed along the diffusion barriers.

The basic design of the microscope used in this study has been previously described B11 . Briefly the system is built around a Nikon TE2000U inverted microscope with a custom made illumination system. A 488 nm 200 mW diode pumped solid state laser Coherent Sapphire CDHR was used as the excitation source. The laser was attenuated as necessary with a neutral density filter and centered over the DNA curtain by means of a remotely operated mirror New Focus . The beam intensity at the face of the prism was 10 15 mW. Images were detected with a back illuminated EMCCD detector Photometrics Cascade 512B . TIRFM images were collected using a 60 water immersion objective lens Nikon 1.2 NA Plan Apo unless otherwise indicated.

We have shown that mechanical barriers to lipid diffusion can be used to organize DNA molecules into curtains that serve as a unique and effective experimental platform for the study protein DNA interactions at the single molecule level B6 11 . The overall strategy for making DNA curtains is shown in . DNA molecules are first anchored by one end to a supported lipid bilayer coating the surface of the sample chamber and application of hydrodynamic force is used to push the lipid tethered DNA molecules towards lipid diffusion barriers. Lipids within supported bilayers cannot traverse these mechanical barriers B13 15 . As a consequence the barriers curtain rods also halt the forward movement of the DNA molecules causing them to accumulate at the barrier edges where they extend into the evanescent field. As we have previously shown molecules of DNA 48.5 kb can be assembled into curtains along these barrier edges and the molecules remain at fixed lateral locations B6 .

In contrast to the full length phage genome smaller DNA fragments diffuse laterally along the smooth barrier edges see below . This is because the lipid bilayers remain fluid even at the barrier edges B 12 and the tension exerted on the shorter DNAs by the hydrodynamic force is insufficient to overcome the tendency of the molecules to slide along the barriers. In addition these chromium barriers are designed to be oriented exactly perpendicular to the flow force and if they are misaligned by just a few degrees relative to the direction flow then the smaller DNA molecules slip rapidly along the barrier edges. In light of these potential design flaws when used for smaller DNA substrates we sought a new barrier design that could eliminate slippage of the molecules along the barrier edges. We reasoned that the locations of the DNA molecules along the barrier edges could be passively controlled by altering the shape of the barriers themselves. For example because the DNA molecules can move along the smooth edges then non linear geometric barrier patterns comprised of a simple repetitive triangular wave would limit the mobility and . This is because the hydrodynamic force exerted by the flowing buffer should push the DNA molecules into the vertex of each triangle with sufficient force to overcome any tendency of the molecules to move laterally within the bilayer . We refer to these triangular features as geometric nanowells conceptually similar to the wells of a sequencing gel because the DNA molecules loaded within each well would not be free to move laterally along the barrier as long as a continuous flow force is maintained. In addition to eliminating lateral slippage the peak to peak distance of the adjacent triangles within the patterns should dictate the minimal lateral separation of the DNA molecules that make up the curtain. For example non linear geometric patterns that repeat at 500 nm intervals should yield DNA molecules separated from one another by 500 nm. The separation will be greater than 500 nm if some of the wells are not occupied yet even in this case the distances should occur in defined increments divisible by 500 nm i.e. 500 1000 1500 2000 nm etc. . If this were true then these non linear geometric barrier patterns would confer precise control over the positioning of each individual DNA making up the curtain.

Examples of the non linear geometric barrier patterns tested here are shown in . We used electron beam E beam lithography to achieve these desired design features by engineering chromium barriers that were 100 nm wide and 30 nm tall arranged in a sawtooth pattern on a fused silica slide. shows an optical image of a pair of barrier sets and once assembled into a flowcell the chromium barriers will be oriented perpendicular to the hydrodynamic force with the apex of each triangle pointing in the same direction that the buffer is flowing as indicated in . shows an AFM image of a non linear geometric barrier pattern with 1 m peak to peak spacing and reveals a height of just 20 nm. Examples of SEM images of non linear geometric barriers with peak to peak distances of 2000 1000 750 500 350 and 200 nm are shown in . These images demonstrate that we can use standard lithographic techniques to construct chromium barriers with precise non linear geometric patterns having nanometer scale features of our desired designs.

If the DNA molecules within the curtains were in fact retained within the geometric nanowells as we predicted then the distance between the adjacent vertices of triangles within the pattern design should be reflected in the separation distance between the individual DNA molecules. shows examples of DNA curtains made with non linear geometric barrier patterns with 2000 1000 750 500 350 and 200 nm separation between adjacent nanowells left panels . As indicated above these curtains are comprised of the 20 kb DNA that is fluorescently stained with the intercalating dye YOYO1. To verify that this design pattern yielded DNA molecules with the desired spacing we plotted signal intensity of the stained DNA versus its lateral location along the barrier edge and these plots revealed peak to peak frequencies consistent with values from designs of the non linear geometric barrier patterns middle panels . The 2000 1000 750 500 350 and 200 nm barriers yielded mean separation distances corresponding to the pattern designs. This data demonstrates that the non linear geometric barrier patterns prevent slippage of the DNA molecules along the non linear geometric diffusion barriers and also confirms that the distance between adjacent nanowells dictates the extent of lateral separation between individual molecules that make up the DNA curtain.

The number of DNA molecules that make up the curtains can be varied by modulating several different parameters the simplest of which is the amount of DNA injected into the sample chamber B6 B7 . With these non linear geometric diffusion barrier patterns if too much DNA is present then multiple molecules of DNA will accumulate within each geometric nanowell. To avoid this problem these experiments were conducted with a relatively small amount of DNA such that less than one DNA molecule should be found per nanowell. In other words some of the nanowells will remain unoccupied many of the wells will have a single DNA molecule and some of the wells will have two or more molecules of DNA. This can be confirmed by measuring the fluorescence intensity of the DNA in each well and those nanowells harboring two molecules will be twice as bright as those harboring just one therefore allowing easy discrimination. shows a histogram of total integrated intensity per DNA molecule revealing a prominent peak and a less prominent peak. In addition a tally of the total number of geometric nanowells and the total number of DNA molecules within a barrier set. These values are also reflected in the fluorescence cross sections shown in which provides further evidence that the number of DNA molecules loaded per nanowell can be controlled and that any nanowells having more than one DNA can be readily identified based on signal intensity and discarded from further analysis if necessary.

As indicated above DNA molecules aligned along smooth linear chromium barriers can slip laterally along the barrier edges and this effect can be very pronounced with shorter molecules of DNA. To illustrate this problem a 20 kb PCR DNA fragment derived from the DNA was labeled with YOYO1 B16 and see Materials and Methods . These molecules were aligned along the edge of a smooth barrier and viewed over the course of 13.5 minutes. A kymogram illustrating the lateral locations of the DNA molecules over time was then generated from a cross section of the curtain. As shown in the labeled ends of the molecules exhibited large movements consistent with lateral slippage of the DNA along the barrier edge even over periods spanning just a few seconds. DNA curtains made with non linear geometric barrier patterns are shown in along with the corresponding kymograms. In contrast to the movement observed with the smooth linear barriers the DNA molecules aligned at the non linear geometric patterns displayed absolutely no evidence of lateral slippage. Similar experiments with DNA molecules of sequentially shorter lengths revealed that the lateral movement of 34 21 11 and 5 kb DNA fragments was also eliminated with the non linear geometric barrier patterns . This data provided evidence indicating that the non linear geometric barrier patterns were highly effective at restricting movement of the DNA molecules along the barrier edges further indicating that the molecules themselves were loaded into the geometric nanowells.

Using these non linear geometric barrier patterns permits studies of even relatively small DNA fragments by eliminating their lateral mobility along the barrier edges when maintained by a constant flow force. As we have previously shown the design of the curtains also yields DNA molecules that are all aligned with the same sequence orientation based upon the location of the biotin tag at a specific end of the DNA B6 . As a consequence of this perfect alignment restriction digest of the DNA molecules within the curtain yields identical tethered fragments whose lengths correspond to the furthest upstream cleavage site B6 . Complete digests can not be used to identify multiple sites cleaved by the same restriction enzyme because the downstream DNA is flushed from the sample chamber as soon as the DNA is cleaved. However partial restriction digests can be used to generate mixed populations of DNA molecules whose lengths reflect the distribution of cleavage sites throughout the DNA.

In principle a complete restriction digest of the DNA can also reveal all of the fragments corresponding to each of these intermediate lengths but only if the digest is viewed in real time and the cleavage rate is slower than the data acquisition frequency. In this case following each individual cleavage event the downstream fragment will be immediately flushed from the sample chamber by buffer flow leaving behind the intact biotinylated fragment whose shortened length would correspond to the location of the cleavage site. This can happen repeatedly on the same DNA until that particular molecule is cleaved at the furthest upstream site relative to the anchored end of the DNA. The large number of DNA molecules that can be viewed with these DNA curtains makes it possible to detect all of the potential sites within the DNA and the geometric nanowells prevent the DNA molecules from slipping back and forth after they are cleaved. Eventually all of the DNA molecules will be cut to a final length corresponding to the furthest upstream restriction site. We refer to this real time digest as dynamic optical restriction mapping and an example of such an assay is shown in . Here the DNA was digested with EcoRI and the digest was observed in real time. shows the DNA before the digest and shows the DNA after the digest was completed which demonstrates that all of the DNA were eventually trimmed corresponding to the 21 kb fragment and confirming that all of the molecules were in the identical orientation. shows examples of kymograms made from individual DNA molecules during the digestion reaction. These kymograms were selected because they provide examples where each of the five EcoRI sites can be identified. The total time required for this digest was just 30 seconds and we anticipate that this time can be reduced significantly with further optimization. The length of the resulting fragments observed at the intermediate time points was measured and their distribution plotted as a histogram in which reveals all five of the EcoRI cuts sites in the phage genome. This assay is not possible with smooth linear barrier designs because the DNA molecules start to rapidly slip back and forth as the fragment lengths become shorter and shorter thus interfering with measurements of the same DNA molecule over time. Thus this real time dynamic optical restriction mapping assay is greatly facilitated when used in combination with the non linear geometric barrier patterns because the nanowells prevent the DNA molecules from moving as they get shorter and shorter during the digest.

High throughput approaches to single molecule biochemistry have the potential to reveal numerous new details of reaction mechanisms and macromolecular dynamics which were previously inaccessible to traditional ensemble approaches. In an effort to make these techniques more broadly applicable we have used nanoscale engineering to establish a high throughput approach that we call DNA curtains which facilitates massively parallel data acquisition from thousands of individual molecules viewed in real time. Here we have shown that non linear geometric barrier patterns can also be used to align curtains of DNA molecules tethered to a fluid lipid bilayer and that the geometry of the barriers dictates the lateral distance between the individual molecules that make up the curtains. We demonstrate that single molecules of DNA can be loaded into the geometric nanowells where they can be retained for further analysis and experimentation. This work brings the power of nanofabrication to bear on single molecule biology by providing a very simple and robust means of controlling the lateral dispersion of DNA molecules assembled into a molecular curtain.

In our previous studies we demonstrated that DNA curtains are advantageous for analysis of protein DNA interactions in particular with systems that involve lateral movement of proteins along DNA B8 B9 . One complication of these experiments is that it is not always possible to irrefutably rule out the possibility that a protein under observation is interacting with two or more DNA molecules that are closely juxtaposed. Although the density of the DNA curtains can be decreased to help mitigate this issue it still is not always possible to demonstrate that a specific protein under observation is bound to a particular DNA molecule. Moreover while our DNA curtain assays have a demonstrated capability to enable massively parallel data acquisition evaluation of the resulting data remains a challenge because automated image analysis programs can not readily distinguish two or more identical objects if they overlap to any significant extent. With the advent of the non linear geometric barrier patterns these problems are eliminated because the DNA molecules are maintained at fixed distances from one another and we can verify that only one molecule is loaded in a particular well based on the fluorescence intensity of the labeled DNA. Moreover the non linear geometric patterns used here are extremely simple yet highly effective at directing DNA molecules to defined locations on a surface. The ability to load and retain individual DNAs into geometric nanowells portends the future possibility of individually handling or isolating these single molecules and we envision that the flexibility afforded by nanolithography will enable construction of intricate barrier patterns that can be used for even more complex manipulations of lipid anchored DNA molecules. While the primary motivation for this work remains development of robust high throughput tools for single molecule optical detection of protein DNA interactions we envision that the DNA curtains and geometric nanowells have numerous potential applications pertaining to physical analysis of large DNA molecules. For example a major aspect of genome research remains characterization of large DNA fragments and numerous approaches have been developed for manipulating analyzing and isolating these molecules B17 21 . We have shown that DNA curtains made up of these non linear geometric patterns can be used to generate optical restriction maps of the tethered DNA molecules and because these assays are performed in the context of a microfluidic system the cleaved DNA fragments can easily be isolated for further analysis. This type of real time restriction digest imaging can be used for rapidly mapping the DNA molecules and the large numbers of molecules aligned in the curtains ensures acquisition of statistically relevant data from just a single run we refer to this as dynamic optical restriction mapping. Finally these perfectly ordered DNA substrates now offer the future potential for applying rigorous machine vision techniques for fully automated data analysis with little or no user intervention.

We have previously demonstrated that we can use nanofabricated barriers to control lipid diffusion to align molecular curtains of DNA which can be visualized by TIRF microscopy and used for physical analysis of the DNA molecules themselves or can serve as a tool for probing protein DNA interactions. The advantages of this technology is that it permits these studies to be conducted at the single molecule level but in a high throughput format that permits massively parallel data acquisition. However one drawback of these prior barrier patterns is that they require the continuous application of a hydrodynamic or electrophoretic force to confine the mobility of the aligned DNA molecules. In the absence of this force the DNA molecules diffuse away from the barrier edge and making it difficult to probe biochemical interactions that involve the formation of DNA loops e.g. the interactions between RNA polymerase bound to a promoter and a transcription factor bound to a distal enhancer element . To overcome this problem here we have developed a technology for aligning curtains of DNA wherein the constituent molecules are trapped at defined locations. This is accomplished using barriers patterns that contain long narrow entry pores which are used to guide single DNA molecules into individual geometric nanowells. In the absence of buffer flow the DNA molecules diffuse away from the lipid coated surface but they can not escape the geometric nanowells over time scales ranging up to tens of minutes. These barrier designs are ideal for studying any protein DNA interaction that involves the association of distal components bound along the same DNA chain.

Uses Trapping DNA for continuous analysis involving steps that necessitate the absence of buffer flow Physical analysis of DNA looping Analysis of protein protein interactions that require DNA looping and Will enable selective manipulation and isolation of single DNA molecules from within a more complex mixed population.

Here we present a combination of different non linear geometric barrier patterns that enable complex manipulation of DNA molecules that are anchored to a fluid lipid bilayer on the surface of a microfluidic sample chamber see . The non linear geometric barrier elements include a nanopore gated staging area which is used to irreversibly capture linear DNA molecules that are anchored by both ends to the bilayer and pushed into the staging area using hydrodynamic force. Once captured the DNA molecules are driven in the perpendicular direction using tangentially applied force generated either using buffer flow or an electric field. This pushed the DNA molecules to one side of the staging area where they encounter a bifurcated nanochannel that directs each end of the same DNA molecules down a pair of parallel channels the DNA rack which are separated by a user defined distance designed to stretch the double tethered DNA molecules to a desired length. The exit ports at the ends of each channel are designed to retain any DNA using a tension based selection while at the same time filtering out any broken molecules. Once the molecules are aligned within the rack they are maintained in a stretched configuration and can be visualized by TIRFM in the absence of either a flow or electrophoretic force. These aligned arrays of double tethered DNA curtains offer an ideal high throughput format for analyzing DNA molecules or protein DNA interactions in the absence of a perturbing hydrodynamic force.

Uses Any biochemical biophysical studies of DNA or protein DNA interactions that must be conducted in the absence of a perturbing hydrodynamic or electrophoretic force for example promoter enhance interactions during transcription Studies of one dimensional diffusion and target site binding by proteins i.e. visualizing where and how transcription factors or DNA repair proteins bind DNA and Size selective fractionation of single DNA molecules the rack elements can be designed to accommodate any desired DNA or even a range of different sized DNA molecules .

Here we develop new strategies to construct curtains of DNA in which the substrates themselves are aligned with respect to one another and maintained in an extended configuration with both ends anchored to the surface of a microfluidic sample chamber that is otherwise coated with an inert lipid bilayer. This is accomplished through the use of nanofabricated racks of metallic barrier patterns with different functional elements which together enable alignment and anchoring of hundreds of individual DNA molecules. Linear barriers to lipid diffusion are used to align the DNA and antibody coated pentagons provide solid anchor points for the free ends of the molecules. These double tethered curtains of anchored DNA can be visualized using total internal reflection fluorescent microscopy under conditions that do not require the use of continuous buffer flow to stretch the DNA. This offers the potential for data acquisition without perturbing the behavior of the proteins under investigation through the application of an externally applied hydrodynamic force. We provide a proof of principle demonstration that these DNA racks can be used in a 1D diffusion assay that monitors the motion of the mismatch repair proteins along DNA.

Linear barriers to lipid diffusion are used to align the DNA and antibody coated pentagons provide solid anchor points for the free ends of the molecules. These double tethered curtains of anchored DNA can be visualized using total internal reflection fluorescent microscopy under conditions that do not require the use of continuous buffer flow to stretch the DNA. This offers the potential for data acquisition without perturbing the behavior of the proteins under investigation through the application of an externally applied hydrodynamic force. We provide a proof of principle demonstration that these DNA racks can be used in a 1D diffusion assay that monitors the motion of the mismatch repair protein complex Msh2 Msh6 along DNA.

Dynamic interactions between proteins and DNA underlie many biological processes and as such are the subject of intense investigation. Many laboratories are now tackling these problems using new experimental methods that enable the visualization of protein DNA complexes at the single molecule level in real time and the information garnered from these experiments is being used to build detailed mechanistic models of many different types of reactions. However one major drawback of many single molecule techniques is that they are inherently designed to probe individual reactions and as a consequence it can be challenging to gather statistically relevant data. This difficulty is often compounded by the fact that these experiments are often technically demanding. Therefore it is highly advantageous to establish new approaches that can increase throughput capacity of single molecule methods and make these approaches both easier and more readily applicable to biological reactions involving different types of DNA transactions.

In an effort to help make these techniques more accessible we have integrated nanoscale engineering microfluidics and lipid bilayer coated surfaces with single molecule optical microscopy to develop high throughput methods for making molecular curtains comprised of thousands of individual DNA molecules C1 3 . A key aspect of these experimental platforms is that they take advantage of the fluid nature of lipid bilayers to control the organization of DNA molecules tethered to individual lipids C4 C5 . This is accomplished by first coating a sample chamber surface with a bilayer containing a small fraction of biotinylated head groups and DNA molecules are then anchored by one end to the bilayer via a biotin neutravidin linkage. These lipid tethered DNA molecules are assembled into molecular curtains by pushing them to the leading edge of either micro or nanoscale lipid diffusion barriers for example linear or non linear geometric barriers through the application of a hydrodynamic force. In our initial studies we demonstrated that microscale barriers to lipid diffusion made by manually etching a fused silica surface with a diamond tipped scribe could be used to align hundreds of lipid tethered DNA molecules C3 . More recently we have established methods for fabricating barriers with nanoscale dimensions that allow for much more precise control over both the location and lateral distribution of the DNA molecules C1 C2 . These nanofabricated DNA curtains permit simultaneous visualization of thousands of individual DNA molecules that are perfectly aligned with respect to one another and can be used for massively parallel data acquisition from thousands of individual protein DNA complexes in real time using a robust experimental platform that is amenable to a wide variety of biological applications.

We have also shown that these DNA curtains are highly advantageous for studying protein DNA interactions at the single molecule level and we have applied these tools to biological systems such as chromatin remodeling homologous DNA recombination and post replicative mismatch repair C6 9 . This experimental platform is especially applicable to the real time detection of fluorescent proteins bound to the DNA. However these previous DNA curtains require the continuous application of a hydrodynamic force during data collection because just one end of the DNA is anchored to the lipid bilayer. If buffer flow is terminated the DNA does not remain stretched and as a consequence it can not be visualized along its full contour length because it drifts outside of the detection volume defined by the penetration depth of the evanescent field. The need for buffer flow is not problematic for many types of measurements however it can potentially impact the behavior of bound proteins or protein complexes and the magnitude of this impact can scale in proportion to the hydrodynamic radius of the molecules under observation. The influence of buffer flow is especially apparent during measurements involving proteins that slide on DNA by one dimensional diffusion because an applied flow force can strongly bias the direction that the sliding proteins travel along the DNA.

The need for buffer flow is not problematic for many types of measurements however it can potentially impact the behavior of bound proteins or protein complexes and the magnitude of this impact can scale in proportion to the hydrodynamic radius of the molecules under observation. The influence of buffer flow is especially apparent during measurements involving proteins that slide on DNA by one dimensional diffusion because an applied flow force can strongly bias the direction that the sliding proteins travel along the DNA.

Here we sought to develop new procedures for making curtains of aligned DNA molecules that were held in an extended configuration anchored by both ends to the surface of a microfluidic sample chamber and suspended above an inert bilayer. To accomplish this we have developed new patterns of metallic barriers with distinctive functional elements. These metallic patterns incorporate both linear barriers to lipid diffusion and non linear geometric barriers comprising arrayed pentagons and together these elements function to align and tether long molecules of DNA. The DNA molecules are first anchored by one end to a fluid lipid bilayer via a biotin neutravidin interaction and initially aligned with one another by using hydrodynamic force to push them into the linear barriers. The arrayed pentagons serve as solid anchor points positioned at a defined distance downstream from the linear barriers and are coated with antibodies directed against either digoxigenin or bromodeoxyuridine which are covalently attached to the free ends of the DNA molecules. Importantly once linked to the surface the DNA molecules are maintained in an extended state and confined within the detection volume defined by the penetration depth of the evanescent field allowing for continuous observation along their full contour length even in the absence of an applied hydrodynamic force.

Fused silica slides were cleaned in NanoStrip solution CyanTek Corp Fremont Calif. for 20 minutes then rinsed with acetone and isopropanol and dried with N. The slides were spin coated with a bilayer of polymethylmethacrylate PMMA molecular weight 25K 3 in anisole and 495K 1.5 in anisole MicroChem Newton Mass. followed by a layer of Aquasave conducting polymer Mitsubishi Rayon . Each layer was spun at 4 000 rpm for 45 seconds using a ramp rate of 300 rpm s. Polygon patterns for non linear geometric barriers and linear barriers were written by Ebeam lithography using an FEI Sirion scanning electron microscope equipped with a pattern generator and lithography control system J. C. Nabity Inc. Bozeman Mont. . After patterning Aquasave was rinsed off with deionized water. Resist was developed using a 3 1 solution of isopropanol to methyl isobutyl ketone MIBK for 1 minute with ultrasonic agitation at 5 C. The substrate was then rinsed in isopropanol and dried with N. A 15 20 nm layer of gold atop a 3 5 nm adhesion layer of either chromium or titanium was deposited using a Semicore electron beam evaporator. Liftoff was effected at 80 C. in a 9 1 ratio of methylene chloride to acetone. Alternatively barriers were made out of just a 15 20 nm layer of chromium as previously described. Following liftoff samples were rinsed with acetone to remove stray chromium flakes and dried with N. Barriers were imaged using a Hitachi 4700 scanning electron microscope and a PSIA XE 100 Scanning Probe Microscope in noncontact mode. Optical images of the barriers were taken with a Nikon Eclipse ME600 at either 50 or 100 magnification as indicated .

The flowcells were assembled from fused silica slides G. Finkenbeiner Inc. with chromium nanoscale diffusion barriers. Inlet and outlet ports were made by boring through the slide with a high speed precision drill press equipped with a diamond tipped bit 1.4 mm O.D. Kassoy . The slides were cleaned by successive immersion in 2 v v Hellmanex 1 M NaOH and 100 MeOH. The slides were rinsed with filtered sterile water between each wash and stored in 100 MeOH until use. Prior to assembly the slides were dried under a stream of nitrogen and baked in a vacuum oven for at least 1 hour. A sample chamber was prepared from a borosilicate glass coverslip Fisher Scientific and double sided tape 25 m thick 3M . Inlet and outlet ports Upchurch Scientific were attached with hot melt adhesive SureBonder glue sticks FPC Corporation . The total volume of the sample chambers was 4 l. A syringe pump Kd Scientific and actuated injection valves Upchurch Scientific were used to control sample delivery buffer selection and flow rate. The flowcell and prism were mounted in a custom built heater with computer controlled feedback regulation to control the temperature of the sample from between 25 37 C. 0.1 C. as necessary. After each use the slides were soaked in MeOH to remove the ports and tape rinsed with water washed briefly 15 20 minutes with Nanostrip and finally rinsed with water. This procedure was sufficient to clean the slide surfaces for reuse and each slide could be used multiple times degrading the quality of the optical surface or the metallic patterns.

DNA curtains were constructed as described. All lipids were purchased from Avanti Polar Lipids and liposomes were prepared as previously described. In brief a mixture of DOPC 1 2 dioleoyl sn glycero phosphocholine 0.5 biotinylated DPPE 1 2 dipalmitoyl sn glycero 3 phosphoethanolamine N cap biotinyl and 8 10 mPEG 550 PE 1 2 dioleoyl sn glycero 3 phosphoethanolamine N methoxy polyethylene glycol 550 . The mPEG is does not affect bilayer formation or assembly of the DNA curtains but rather serves to further passivated the surface against nonspecific adsorption of quantum dots which we use in our studies of protein DNA interactions . Liposomes were applied to the sample chamber in three injections of 200 l followed by 5 minute incubations. Excess liposomes were flushed away with 1 ml of buffer A which contained 10 mM Tris HCl pH 7.8 plus 100 mM NaCl and the bilayer was incubated for an addition 30 minutes. Buffer A plus 25 g ml anti DIG Fab Roche or anti BrdU IgG Sigma was then injected into the sample chamber and incubated for 30 minutes. The sample chamber was then flushed with 1 ml of buffer B which contained 40 mM Tris HCl pH 7.8 1 mM DTT 1 mM MgCl and 0.2 g ml BSA and incubated for an additional 5 minutes. 1 ml of buffer A containing Neutravidin 330 nM was then injected into the sample chamber and incubated for 20 minutes. The flowcell was then rinsed with 3 ml of buffer B to remove any unbound Neutravidin. 1 ml of DNA 20 pM labeled at one end with biotin and at the other end with digoxigenin see below and pre stained with 1 2 nM YOYO1 was injected into the sample chamber in five 200 l aliquots with a 2 3 minute incubation period following each injection. The DNA was then aligned at the linear barriers using a flow rate of 0.02 ml min and this rate was then increased to 2 3 ml min to anchor the second end of the DNA molecules.

DNA substrates. The DNA substrates were made by ligating oligonucleotides to the 12 nucleotide overhangs at the end of the phage genome 48.5 kb . Ligation mixes 1 ml total volume contained 4 nM DNA Invitrogen 1 M biotinylated oligonucleotiode 5 pAGG TCG CCG CCC BioTEG 3 SEQ ID NO 1 1 M BrdU labeled oligonucleotide or DIG labeled oligonucleotide 5 pGGG CGG CGA CCT BrdU 3 SEQ ID NO 2 or 5 pGGG CGG CGA CCT DIG 3 SEQ ID NO 3 and 1 ligase buffer NEB . The reaction mix was warmed to 65 C. for 10 minutes and then cooled slowly to room temperature. After cooling ligase was added T4 DNA ligase 400 U l or Taq ligase 40 U l NEB and the mixture was incubated overnight at 42 C. Reactions performed with T4 ligase were then heat inactivated at 65 C. for 10 minutes and the ligated DNA products were purified over a Sephacryl S200HR column GE Healthcare run in 10 mM Tris HCl pH 7.8 1 mM EDTA plus 150 mM NaCl. The purified DNA was stored at 20 C.

For inserting a DIG labeled oligonucleotide complementary to position 26 151 26 166 on 500 ul of DNA 200 pM labeled at the ends with biotin and BrdU as described above was incubated for 2 h with the nicking enzymes Nb.BsmI and Nt.BstNBI 50 U each NEB at 55 C. in buffer containing 10 mM Tris HCl pH 7.8 10 mM MgCl 150 mM NaCl. 1 uM of oligonucleotide 5 pCAT TCT TGA GTC CAA TTT TT DIG 3 SEQ ID NO 4 was then added to the solution along with 10 mM EDTA and the mixture was incubated at 55 C. for 20 minutes and then allowed to slowly cool to room temperature over the course of an hour. ATP was added to a final concentration of 1 mM along with 2 000 units of T4 ligase and an additional 10 mM MgCl. The reaction was then incubated at room temp. for 90 min. Additional EDTA C 20 mM was added to the solution and the nicking enzymes and ligase were heat denatured at 80 C. for 20 minutes. This procedure was also used for concurrently inserting oligonucleotides at 26 151 26 166 and 33 779 33 791 in the same DNA.

For inserting an oligonucleotide at only position 33 779 33 791 on 500 ul DNA 200 pM labeled at the ends with biotin and BrdU was incubated for 2 h with the nicking enzyme Nt.BstNBI 50 U NEB at 55 C. in buffer containing 10 mM Tris HCl pH 7.8 10 mM MgCl 150 mM NaCl. The enzyme was heat denatured by incubation at 80 C. for 20 min. 1 M of oligonucleotide 5 pTTC AGA GTC TGA CTT TT DIG 3 SEQ ID NO 5 was added to the solution and the mixture was incubated at 55 C. for 20 min. then cooled slowly to room temp. over the course of 1 h. ATP was then added to a final concentration of 1 mM along with 2 000 units of T4 ligase and the reaction was incubated at room temp. for 90 min. The ligase was then heat denatured by incubation at 65 C. for 20 min.

The microscope used in this study has been previously described C3 . In brief the system is built around a Nikon TE2000U inverted microscope with a custom made illumination system. A 488 nm 200 mW diode pumped solid state laser Coherent Sapphire CDHR was used as the excitation source. The laser was attenuated as necessary with a neutral density filter and centered over the DNA curtain by means of a remotely operated mirror New Focus . The beam intensity at the face of the prism was 10 15 mW. Images were detected with a back illuminated EMCCD detector Photometrics Cascade 512B . TIRFM images were collected using a 60 water immersion objective lens Nikon 1.2 NA Plan Apo unless otherwise indicated.

We have previously demonstrated that mechanical barriers that disrupt the continuity of lipid bilayers can be used to construct molecular curtains of aligned DNA molecules C1 3 . We have also shown that these DNA curtains provide a unique and advantageous tool for studying protein DNA interactions at the single molecule level when visualized by TIRFM. Here we expand on our previous work and demonstrate the development of new barrier elements which we call DNA racks that can be used to make DNA curtains in which both ends of the molecules are anchored to the flowcell surface. An overview of the general design is presented in and utilizes a combination of two distinct pattern elements as described below. To make the curtains one end of the DNA is first anchored via a biotin neutravidin interaction to a supported lipid bilayer coating the surface of the sample chamber and . In the absence of a hydrodynamic force the molecules are randomly distributed on the surface but lie primarily outside of the detection volume defined by the penetration depth of the evanescent field 150 200 nm . Application of flow pushes the DNA through the sample chamber with one end remaining anchored to the bilayer. The first pattern elements are linear barriers to lipid diffusion which are oriented perpendicular to the direction of flow at strategic locations in the path of the DNA and these linear barriers are designed to halt the movement of the lipid tethered DNA molecules causing them to accumulate at leading edge of the barriers where they then extend parallel to the surface.

The second elements of the pattern are a series of arrayed pentagons for example a non linear geometric barrier positioned behind the linear barriers and separated from one another by small nanochannels. The non linear geometric shape of the pentagons is intended to act as a funnel and direct any lipid tethered DNA molecules through the nanochannels so that they do not accumulate at the leading edge of the pentagons but rather are pushed along the surface until they encounter the next successive linear barrier in the pattern. The distance between the linear barriers and the pentagons is optimized for the length of the DNA to be used for the experiments. The pentagons themselves present a large surface that can be coated with antibodies directed against digoxigenin DIG which is covalently linked to the ends of the DNA opposite the ends bearing the biotin tag. When the DIG coupled DNA end encounters the antibody coated pentagons they should become immobilized and the DNA should remain stretched parallel to the surface even when no buffer is being pushed through the sample chamber. The distance between each of the successive linear barriers should influence the density of aligned DNA because the total number of molecules anchored to the bilayer is directly proportional to the area encompassed by the bilayer upstream from each barrier. Varying the total distance between adjacent barrier sets on the flowcell surface should yield double tethered DNA curtains of varying density enabling the user to scan the surface for regions with an appropriate number of molecules for an experiment. Therefore the patterns were designed with separation distances of either 0.1 0.2 0.3 0.4 or 0.5 m between the adjacent sets of barriers .

The overall design relies upon the selective but nonspecific adsorption of antibodies to the exposed surface of the metallic pentagons but the antibodies should not interact with the inert lipid bilayer. To assemble the DNA curtains the surface of the flowcell was first coated with a lipid bilayer as previously described with the exception that BSA was omitted from all buffers used prior to deposition of the antibody. Once the bilayer was assembled on the surface anti DIG Fab fragments or anti BrdU IgG was injected into the sample chamber where they were allowed to adhere nonspecifically to the exposed metal barriers. The antibodies could potentially bind both the linear barriers as well as the pentagons but the very small surface area of the linear barriers should ensure selective adsorption of more antibody to the much larger pentagons. Following a brief incubation the free Fab was rinsed from the flowcell and replaced with buffer containing 0.2 mg ml BSA which served as a nonspecific blocking agent to passivate any remaining exposed surfaces. DNA stained with the fluorescent intercalating dye YOYO1 was then injected into the sample chamber incubated briefly without buffer flow and then buffer flow was applied to push the anchored molecules into the linear barriers. Buffer flow was then terminated and the anchored DNA molecules imaged by TIRFM.

As shown in this procedure yielded curtains of DNA molecules that remained fully extended and visible even though no buffer was flowing through the sample chamber. Any DNA molecules that failed to become anchored by the DIG labeled end remained near the linear barriers but only the ends of the molecules could be observed rather than their full contour lengths. These single tethered DNA molecules gradually diffused away from the barrier edges and could also be pushed away from the barrier edge simply by reversing the direction of buffer flow but the double tethered DNA molecules remained anchored to their original locations providing additional conformation that the molecules making up the curtain are anchored in the desired configuration. The number of DNA molecules at each of the different barrier locations was proportional to the distance between the adjacent linear barriers. Similar results were obtained with DNA that was end labeled with a single bromodeoxyuridine BrdU and pentagons coated with anti BrdU antibodies see below although the BrdU anchored DNA ends had a greater tendency to dissociate from the pentagons. Finally control experiments verified that the presence of both the DIG labeled and the Fab fragments were necessary for efficiently anchoring both ends of the DNA.

If the DNA anchored between the barriers was nonspecifically adhered to the sample chamber surface along its contour length rather than suspended above the inert bilayer then introduction of a double stranded break should not cause the molecules to retract from the surface. In contrast if the DNA were only anchored by the biotin and DIG tagged ends as designed and suspended above the bilayer then a double strand break in the strands should cause the molecules to retract away from the surface. When DNA molecules randomly break they can immediately retract from the surface as they diffuse out of the evanescent field and disappear from view. DNA molecules can only interact with the sample chamber surface through their anchored ends and cannot interact nonspecifically with the lipid bilayer.

The differential chemistries used to anchor the two ends of the DNA was predicted to define the orientation of the DNA molecules within the curtains. Specifically the biotinylated end should be anchored at the edge of the linear barrier and the other end should be at the pentagons and based on this design all of the molecules should be in the exact same orientation. If the DNA molecules were aligned as expected then a fluorophore located at a single specific site within the DNA and offset from its center should appear as a fluorescent line spanning the DNA curtain. The line should be oriented perpendicular to the long axis of the extended DNA molecules and should coincide precisely with the know location of the fluorescent tag. In contrast if the DNA molecules were randomly oriented then the fluorescent tags should appear as two lines demarking each of the two possible orientations of the DNA. To confirm that the DNA was oriented correctly the molecules were labeled at the free ends with BrdU and labeled at a specific internal position with DIG. The internal labels were made using an oligonucleotide replacement strategy wherein see Materials and Methods . Experiments with DNA curtains anchored by a single end confirmed that the fluorescent tags were present at a single location within the DNA molecules as dictated by the sequences of the oligonucleotides. The double tethered curtains were then assembled using pentagons coated with anti BrdU antibodies and after assembly the DIG tags with labeled with anti DIG coated Qdots. As shown in the anti DIG Qdots were all aligned with one another confirming the orientation of the DNA molecules.

The primary motivation for development of these double tethered DNA curtains was for use in experiments designed to visualize motion of proteins along DNA. We have previously demonstrated that the protein complex Msh2 Msh6 can diffuse in one dimension along duplex DNA C6 see also and . Msh2 Msh6 is an essential component of the post replicative mismatch repair machinery and is responsible for locating and initiating repair of biosynthetic DNA replication errors. Our initial studies with Msh2 Msh6 relied upon DNA molecules that were tethered by either end to neutravidin nonspecifically absorbed to a fused silica surface that was otherwise coated with a lipid bilayer.

With this approach we would obtain only 10 30 DNA molecules per flowcell that were suitable for making diffusion measurements. These double tethered DNA molecules were randomly distributed and had to be manually located before use by visually scanning the entire surface making these 1D diffusion measurements technically demanding. As shown above using the engineered surfaces we can now visualize thousands of molecules per flowcell. Here we demonstrate that these anchored curtains of DNA are suitable for visualizing protein DNA interactions.

For these experiments we used the mismatch repair protein complex Mlh1 Pms1 and it was labeled with a single quantum dot via a FLAG epitope tag that was engineered into the Mlh1 subunit. The labeled proteins were injected into a flowcell containing a curtain of double tethered DNA molecules and videos were collected over a 60 second period. As shown in we could readily detect binding of Mlh1 PMs1 to the DNA molecules that were aligned and anchored on the surface. There were approximately 100 DNA molecules and 250 DNA bound proteins observed in this single field of view illustrating the improvement of this approach compared to our previous technique for making double tethered DNA molecules. The proteins bound to the DNA rapidly diffused along the DNA molecules and this motion was revealed in kymograms made from representative examples of molecules within the DNA curtains . As with the naked DNA molecules the protein bound DNA molecules also retracted from view when the DNA strands were broken verifying that there were no nonspecific interactions with the surface.

See also and for additional studies pertaining to double tethered curtains used to visualize Mlh1 diffusion as well as the diffusion of the Mlh1 Pms1 complex.

Here we used direct write Ebeam lithography to nanofabricate arrays of diffusion barriers followed by pentagonal anchor points which together utilize different functional features to anchor curtains of DNA by both ends on the surface of a microfluidic sample chamber. We refer to these patterned surfaces as DNA racks because the DNA is stretched out and anchored to the flowcell surface where they can be viewed by TIRFM. An important aspect of these devices is that the fused silica surface is coated with a supported lipid bilayer and the DNA molecules are suspended above this bilayer ensuring that they are maintained within an inert microenvironment compatible with a range of biological molecules. We have demonstrated that these tools can be used along with wide field TIRF microscopy to visualize hundreds of individual perfectly aligned DNA molecules all of which are arranged in the same orientation and anchored by both ends to the sample chamber surface. As with our previous nanofabricated devices this new approach is simple and robust the flowcells are reusable the barriers themselves are uniform and they do not compromise the optical quality of the fused silica or interfere with signal detection.

As a first conceptual demonstration of the utility of these DNA racks we show here that they can be used to image 1D diffusion of DNA binding proteins. Our previous approach to these experiments relied upon DNA that was randomly anchored to a surface via biotin neutravidin interactions which yielded no more than 1 3 molecules of DNA per field of view and there were just 10 30 double tethered DNA molecules present over the entire surface of the flowcell. Moreover in our prior studies the orientation of the DNA could not be defined because the molecules were anchored with a biotin located at either end of the DNA. Therefore the molecules were randomly oriented on the flowcell surface making it difficult to locate and compare different DNA molecules to one another in order to dissect sequence specific events. While this original approached proved useful for our initial studies it was tedious and remained challenging to collect sufficient data for thorough analysis. As demonstrated here we are now able to visualize on the order of one hundred DNA molecules per field of view thousands of molecules are present on the surface of a flowcell and all of these DNA molecules are aligned in the exact same orientation which will make 1D diffusion measurements and molecule to molecule comparisons straightforward for future work.

During postreplicative mismatch repair MMR proteins must communicate information over long distances along the DNA helix. Without being bound by theory evidence from bulk biochemical studies deminstrates this communication occurs predominantly through mechanisms involving motion of proteins along DNA. Consistent with this we have used single molecule optical microscopy to demonstrate that the MMR proteins Msh2 Msh6 and Mlh1 Pms1 can both travel along DNA via a mechanism most consistent with sliding. Neither the bulk biochemical studies nor our single molecule observations could completely rule out other mechanisms such as hopping jumping and or intersegmental transfer. Here we have devised an assay that enables us to determine whether Msh2 Msh6 and or Mlh1 Pms1 can travel along DNA while employing one or more of these alternative mechanisms. This new assay relies upon nanofabricated metallic structures for arranging individual DNA molecules into crisscrossed patterns suspended above the surface of a microfluidic sample chamber otherwise coated with a lipid bilayer. We used these substrates to determine whether or not Msh2 Msh6 and or Mlh1 Pms1 could transfer directly from one strand of DNA to another as the proteins slid past the junctions between intersecting DNA molecules.

MMR is conserved from bacteria to man. MMR involves various proteins such as MutS which comprises Msh2 Msh6 MutL which comprises Mlh1 Pms1 ExoI RPA PCNA pol of which MutS and MutL will be discussed herein. Defects in this process result in 100 1000 increase in mutation frequency. For example defects in MMR is associated with a cancer prone syndrome hereditary nonpolyposis colorectal cancer HNPCC . Up to 90 of HNPCC cases are due to mutations in MSH2 or MLH1.

Biochemical reactions involving genomic transactions often require proteins to communicate regulatory information over long distances along a DNA strand. Postreplicative mismatch repair MMR is an evolutionarily biochemical pathway requiring protein mediated signaling over long distances of DNA during the repair of replication errors see for example . MMR has been most studied in where the proteins MutS MutL and MutH promote the initiation of strand specific repair by taking advantage of the transiently unmethylated state of the newly synthesized daughter DNA. MutS binds mismatched DNA and together with MutL activates the MutH nicking endonuclease. This leads to cleavage of the daughter strand at the nearest hemi methylated dGATC which serves as a strand discrimination signal enabling distinction between the parental DNA and the error containing daughter strand. The nick generated by MutH provides an entry point for other proteins that complete downstream steps of the pathway. The stand discrimination signals can be located up to 2 kb away from the mismatch and still elicit repair in reconstituted in vitro systems. Moreover once removal of the daughter strand ensues the excision machinery stops only after it has bypassed the original location of the mispaired base. These findings highlight that communication along the DNA helix is necessary to coordinate mismatch binding with events at the distal strand discrimination signal and the proteins involved must keep track of which strand is which throughout the entire process.

In eukaryotes MMR involves up to eleven different proteins including Msh2 Msh6 MutS and Mlh1 Pms1 MutL both of which are necessary for the initial stages of the repair reaction. Cancer associated mutations can be mapped to the MSH and MLH genes indicating the importance of these proteins in genome maintenance. Msh2 Msh6 in a heterodimer comprised of two MutS homologs and is necessary for initial mismatch recognition and binding. Mlh1 Pms1 interacts with Msh2 Msh6 after mismatch recognition and is thought to coordinate downstream steps in the repair pathway. ATP binding and hydrolysis can provoke conformational changes in Mlh1 Pms1 necessary to regulate these protein protein interactions. Homologs of MutH are not found in eukaryotes but Mlh1 Pms1 itself is a latent endonuclease that is activated in the presence of Msh2 Msh6 PCNA RFC mismatched DNA and ATP. This endonuclease activity is necessary for processing the daughter DNA strand during repair. How the daughter stand is identified remains unclear because the strand discrimination signals in eukaryotes are unknown. Pre existing nicks between adjacent Okazaki fragments in the lagging strand or the 3 hydroxyl present on the leading strand of replication restart intermediates can participate in strand discrimination again highlighting the need for long distance communication along the DNA helix. Structures of MutL family members can be found in Guarn et al. 2004 EMBO J. 23 4134 4135 and in Kosinski et al. 2005 Journal of Molecular Biology. 351 895 909 .

It is necessary for the MMR proteins to communicate information between the lesion and the distal strand discrimination signal. The mechanism by which this communication occurs has been the subject of intensive investigation. Accumulating evidence supports the idea that 1D diffusion is an integral aspect of MMR protein function see . In support of this we have visualized Msh2 Msh6 at the single molecule level using total internal reflection fluorescence microscopy TIRFM and demonstrated that the protein can rapidly diffuse in one dimension along DNA. We have also demonstrated that Mlh1 Pms1 can also diffuse on DNA and it travels an order of magnitude faster that Msh2 Msh6. Lateral movement of these proteins can contribute to lesion recognition post recognition steps necessary for locating strand discrimination signals and or processing of the daughter DNA strand. From a mechanistic perspective the diffusion appears to occur via a sliding mechanism wherein the proteins travel while remaining engaged with bound DNA strand. However we could not unambiguously rule out jumping hopping and or intersegmental transfer all of which potential mechanisms enabling a protein to passively travel along DNA without need for a high energy cofactor. A common aspect of these mechanisms is that they each must invoke an intermediate state in which the proteins are either partially or fully disengaged from the DNA substrate and therefore capable of establishing interactions with a second DNA site. Consequently hopping jumping and or intersegmental transfer can allow direct transfer of a protein from one DNA strand another. In contrast a protein traveling along DNA purely via a sliding mechanism would be incapable of directly transferring from one DNA to the next because the protein must by definition remain engaged with the substrate as it is moving.

To determine whether Msh2 Msh6 and Mlh1 Pms1 traveled along DNA via sliding or whether hopping jumping and or intersegmental transfer contributed to the observed motion we sought to directly visualize proteins on two intersecting DNA molecules to ask whether the proteins could transfer directly from one strand to the next. This was accomplished using nanofabricated patterns of crisscrossed DNA molecules that were suspended above an inert lipid bilayer and anchored to the surface of a microfluidic sample chamber. We then used single molecule optical microscopy to visualize the movements of individual fluorescent proteins as they traversed the intersecting DNA molecules. We reasoned that any events involving direct transfer of proteins between DNA molecules would be apparent as abrupt 90 turns in the protein motion coinciding with the intersecting junctions of the DNA. The existence of such events would provide an unambiguous readout arguing that mechanism of protein movement involved a 3D component wherein the originally bound DNA was released. In contrast the absence of protein transfer between crossed DNA strands at the junction points would argue strongly that the observed lateral motion was a pure sliding mechanism wherein the proteins remained in continuous uninterrupted contact with the DNA.

Experiments using catenated plasmids have been used to probe protein mediated communication between distal DNA sites and elegant examples of these studies have included promoter enhancer interactions restriction endonucleases and site specific recombination. We sought to devise an experiment using intersecting molecules of DNA in a format compatible with single molecule imaging. The design strategy for engineering crisscrossed patterns of DNA suspended above an inert lipid bilayer is illustrated in . The approach for making arrays of intersecting DNA molecules relied upon a combination of hydrodynamic force and diffusion barriers such as non linear geometric nanoscale diffusion barriers to control the organization of DNA molecules anchored by one end to a lipid bilayer via a biotin neutravidin connection. A key feature of the current design is that the different pattern elements are organized into squares wherein two adjacent sides are comprised of linear barriers and the other two sides are non linear geometric barriers made up of arrayed pentagons. The patterns are placed at the center of a four channel flowcell such that each of the linear barriers is oriented perpendicular to one of the two buffer inlet channels. The linear elements serve as diffusion barriers which cannot be traversed by the lipids within the bilayer. Arrays of pentagons are positioned downstream from the linear barriers and serve as solid attachment points for the other end of the DNA. The pentagons are nonspecifically coated with antibodies directed against digoxigenin which is covalently attached to the free ends of the DNA. When the lipid tethered DNA molecules are pushed into the linear diffusion barriers they extend parallel to the surface and the DIG labeled free end of the DNA then binds the antibodies adsorbed to the exposed pentagon surfaces. The triangular faces of the pentagons are designed to funnel DNA into the channels between each adjacent pentagon to minimize accumulation of DNA molecules within the interior of the square patterns.

Assembly of the crisscrossed DNA curtains involved the sequential addition of different components to achieve the desired substrate configuration. First the surfaces of the flowcell were first coated with a lipid bilayer containing a small fraction 0.5 of biotinylated lipid. The inert bilayer covers the exposed fused silica but can not form over the gold surface patterns leaving the metallic surfaces exposed to solution. Anti DIG Fab fragments were then injected into the sample chamber and incubated for a brief period allowing them to adhere to any surfaces not protected by the lipid bilayer. The large surface area of the pentagons relative to the much smaller linear barriers ensures that the pentagons are selectively coated with more antibodies. The sample chamber surface was then blocked with buffer containing BSA and neutravidin was coupled to the bilayer to provide a mobile attachment point for the DNA. Finally DNA labeled at one end with biotin and at the other end with digoxigenin was injected into the sample chamber and pushed into the linear barrier elements. Once the molecules encounter the linear barriers they stretch out and the free ends adhered to the pentagons. shows examples of YOYO1 stained DNA molecules that were aligned and anchored to the metallic racks. One end of the DNA was linked to the lipid bilayer through a biotin neutravidin connection and the other end of the DNA was labeled with digoxigenin DIG and coupled to the pentagons via nonspecifically adsorbed anti DIG Fab fragments. No buffer was flowing during image acquisition confirming that the DNA molecules were anchored by both the biotinylated and digoxigenin labeled ends. As shown here the anchored DNA molecules are oriented at 90 relative to one another ensuring that they are in close physically proximity only at the intersecting points.

We have previously shown that Msh2 Msh6 can travel along DNA by 1D diffusion and that the observed motion is consistent with a mechanism involving a rotational component wherein the proteins continually track the phosphate backbone. In addition we have demonstrated that this motion appears to involve continuous contact between Msh2 Msh6 and the phosphate backbone with no obvious evidence that the linear motion included any hops or jumps along the DNA.

Our previous work has shown that Mlh1 Pms1 can also slide along DNA but travels roughly an order of magnitude more rapidly than Msh2 Msh6 suggesting that these two protein complexes move via fundamentally distinct mechanisms. The structures of these two protein complexes is also strikingly different. Msh2 Msh6 is a compact heterodimeric complex that engages DNA through close contact with the phosphate backbone. Mlh1 Pms1 has N and C terminal domains separated by long linker arms that are thought to encircle DNA. Interestingly bulk biochemical experiments have revealed that Mlh1 Pms1 binds tightly and nonspecifically to DNA with a strong preference for circular substrates and does not dissociate over time scales spanning several minutes. However despite the apparent tight binding affinity Mlh1 Pms1 can transfer to a competitor DNA in solution suggesting that the addition of excess competitor DNA provokes dissociation from the initial bound substrate.

HA tagged Msh2 Msh6 and FLAG tagged Mlh1 Pms1 were purified from as previously described. The protein complexes were labeled with quantum dots Invitrogen conjugated to either anti HA or anti FLAG as appropriate. The epitope tagged proteins are fully functional for MMR in vivo and the labeled proteins retained wild type DNA binding activities in gel shift and filter binding assays as previously described.

Fused silica slides were cleaned in NanoStrip solution CyanTek Corp Fremont Calif. for 20 minutes then rinsed with acetone and isopropanol and dried with N. The slides were spin coated with a bilayer of polymethylmethacrylate PMMA molecular weight 25K 3 in anisole and 495K 1.5 in anisole MicroChem Newton Mass. followed by a layer of Aquasave conducting polymer Mitsubishi Rayon . Each layer was spun at 4 000 rpm for 45 seconds using a ramp rate of 300 rpm s. Polygon patterns and linear elements forming the non linear V non linear geometric barriers were written by Ebeam lithography using an FEI Sirion scanning electron microscope equipped with a pattern generator and lithography control system J. C. Nabity Inc. Bozeman Mont. . After patterning Aquasave was rinsed off with deionized water. Resist was developed using a 3 1 solution of isopropanol to methyl isobutyl ketone MIBK for 1 minute with ultrasonic agitation at 5 C. The substrate was then rinsed in isopropanol and dried with N. A 15 20 nm layer of gold atop a 3 5 nm adhesion layer of either chromium or titanium was deposited using a Semicore electron beam evaporator. Liftoff was effected at 80 C. in a 9 1 ratio of methylene chloride to acetone. Following liftoff slides were rinsed with acetone and dried with N. Barriers were imaged using a Hitachi 4700 scanning electron microscope and a PSIA XE 100 Scanning Probe Microscope in noncontact mode. Optical images of the barriers were collected with a Nikon Eclipse ME600.

The flowcells were assembled from fused silica slides G. Finkenbeiner Inc. with chromium nanoscale diffusion barriers. Inlet and outlet ports were made by boring through the slide with a high speed precision drill press equipped with a diamond tipped bit 1.4 mm O.D. Kassoy . The slides were cleaned by successive immersion in 2 v v Hellmanex 1 M NaOH and 100 MeOH. The slides were rinsed with filtered sterile water between each wash and stored in 100 MeOH until use. Prior to assembly the slides were dried under a stream of nitrogen and baked in a vacuum oven for at least 1 hour. A sample chamber was prepared from a borosilicate glass coverslip Fisher Scientific and double sided tape 25 m thick 3M . Inlet and outlet ports Upchurch Scientific were attached with hot melt adhesive SureBonder glue sticks FPC Corporation . The total volume of the sample chambers was 4 l. A syringe pump Kd Scientific and actuated injection valves Upchurch Scientific were used to control sample delivery buffer selection and flow rate. The flowcell and prism were mounted in a custom built heater with computer controlled feedback regulation to control the temperature of the sample from between 25 37 C. 0.1 C. as necessary. After each use the slides were soaked in MeOH to remove the ports and tape rinsed with water washed briefly 15 20 minutes with Nanostrip and finally rinsed with water. This procedure was sufficient to clean the slide surfaces for reuse and each slide could be used multiple times degrading the quality of the optical surface or the metallic patterns.

DNA curtains were constructed as described. All lipids were purchased from Avanti Polar Lipids and liposomes were prepared as previously described. In brief a mixture of DOPC 1 2 dioleoyl sn glycero phosphocholine 0.5 biotinylated DPPE 1 2 dipalmitoyl sn glycero 3 phosphoethanolamine N cap biotinyl and 8 mPEG 550 DOPE 1 2 dioleoyl sn glycero 3 phosphoethanolamine N methoxy polyethylene glycol 550 . The mPEG is does not affect bilayer formation or assembly of the DNA curtains but rather serves to further passivated the surface against nonspecific adsorption of quantum dots which we use in our studies of protein DNA interactions . Liposomes were applied to the sample chamber for 30 minutes. Excess liposomes were flushed away with buffer containing 10 mM Tris HCl pH 7.8 and 100 mM NaCl. The flowcell was then rinsed with buffer A 40 mM Tris HCl pH 7.8 1 mM DTT and 1 mM MgCl and incubated for 15 minutes. Buffer A plus 30 g ml anti DIG Fab Roche was then injected into the sample chamber and incubated for 20 minutes. The sample chamber was then flushed with buffer A plus and 0.2 mg ml BSA for 5 minutes. Neutravidin 330 nM in buffer A was then injected into the sample chamber and incubated for 20 minutes. After rinsing thoroughly with additional buffer A plus 0.2 mg ml BSA DNA 15 20 pM labeled at one end with biotin and at the other end with digoxigenin and pre stained with 1 2 nM YOYO1 was injected into the sample chamber incubated for 10 minutes and unbound DNA was removed by flushing with buffer at 0.1 ml min. Application of buffer flow caused the lipid tethered DNA molecules to align along the leading edges of the diffusion barriers.

The basic design of the microscope used in this study has been previously described. In brief the system is built around a Nikon TE2000U inverted microscope with a custom made illumination system. For this study a 488 nm 200 mW diode pumped solid state laser Coherent Sapphire CDHR was used as the excitation source. The laser was attenuated as necessary and centered over the DNA curtain by means of a remotely operated mirror New Focus . The beam intensity at the face of the prism was 10 15 mW. Images were detected with a back illuminated EMCCD detector Photometrics Cascade 512B . TIRFM images were collected using a 60 water immersion objective lens Nikon 1.2 NA Plan Apo unless otherwise indicated.

For complete digests 700 l of the desired restriction enzyme in reaction buffer A 40 mM Tris HCl pH 7.8 1 mM MgCl2 1 mM DTT and 0.2 mg ml BSA plus 50 mM NaCl and 10 mM MgCl2 was injected at 0.2 mL min. All restriction enzymes were purchased from NEB and the amounts of enzymes used were as follows NheI 100 units ml XhoI 100 units ml EcoRI 100 units ml NcoI 50 units ml PvuI 50 units ml and SphI 50 units ml . Images of the DNA molecules were collected before the restriction enzyme injection and after all of the enzyme solution had flown through. For partial digests the amount of EcoRI was reduced to 20 units ml and 700 l was injected at 0.4 mL min.

Msh2 Msh3 was purified as described in Surtees J. A. Alani E. J. Mol. Biol. 2006 360 523 536 and TIRFM experiments with Msh2 Msh3 were performed essentially as described for our previously published experiments with Msh2 Msh6 Gorman J. et al. Mol. Cell. 2007 28 359 370 . In brief HA tagged Msh2 Msh3 was incubated with anti HA antibody conjugated quantum dots at 1 2 protein quantum dot ratio for 15 min in Buffer A plus 50 mM NaCl. 50 L of 1.5 nM quantum dot tagged Msh2 Msh3 was then injected at 0.1 mL min to allow efficient binding. The flow rate was increased to 0.4 mL min for data collection and the binding distributions were quantitated as described Prasad T. K. Robertson R. B. Visnapuu M. L. Chi P. Sung P. Greene E. C. J. Mol. Biol. 2007 369 940 953 .

These DNA curtains are useful for analysis of protein DNA interactions at the level of individual molecules and are capable of providing statistically relevant information in a single experimental run. To demonstrate this utility we examined the binding distribution of Msh2 Msh3 a protein complex that is involved in postreplicative mismatch repair of small insertion deletion loops and DNA processing during genetic recombination Surtees J. A. Alani E. J. Mol. Biol. 2006 360 523 536 Langston L. D. Symington L. S. EMBO J. 2005 24 2214 2223 For this experiment recombinant Msh2 Msh3 bearing an HA epitope tag on Msh2 was labeled with anti HA tagged quantum dots Gorman J. Chowdhury A. Surtees J. A. Shimada J. Reichman D. R. Alani E. Greene E. C. Mol. Cell 2007 28 359 370 . The tagged proteins were then injected into a sample chamber containing DNA curtains and the unbound proteins were removed by buffer flow. The DNA molecules and remaining bound proteins were then viewed in the presence and absence of buffer flow and . This transient pause in buffer flow is used as a control to verify that the observed proteins are bound only to the DNA and are not bound to the flowcell surface compare panels A and B . As shown in and Msh2 Msh3 bound to the curtains of DNA but did not display any favored regions or sites for undamaged homoduplex DNA substrates. There are 226 individual DNA molecules and 548 complexes of Msh2 Msh3 in this single field of view collected at 60 magnification highlighting the statistical power of this approach for viewing single protein DNA complexes.

As indicated above DNA molecules aligned along smooth linear chromium barriers can slip laterally along the barrier edges and this effect becomes more pronounced with shorter molecules of DNA. To illustrate this problem either 48.5 kb DNA or a 23 kb PCR DNA fragment derived from the human globin locus was labeled with YOYO1 see Materials and Methods in Example 2 and these molecules were aligned along the edge of a smooth barrier. Kymograms illustrating the lateral locations of the DNA molecules over a period of 13.5 s were then generated from a cross section of the curtain. As shown in the fluorescently labeled molecules exhibited movement consistent with lateral slippage of the DNA along the barrier edge even over periods spanning just a few seconds. In addition quantification of the lateral motion position variance divided by time of the DNA molecules aligned along the smooth barrier edges yielded a broad range of values with a mean of 1.6 10 2.1 10 m s. DNA curtains made using non linear geometric barrier patterns with 500 nm spacing are shown in along with the corresponding kymograms. In contrast to the movement observed with the smooth barriers the DNA molecules aligned at the non linear geometric patterns displayed absolutely no evidence of lateral slippage over the same 13.5 s period. Similar experiments with DNA molecules of sequentially shorter lengths revealed that the lateral movement of 34 21 11 and 5 kb DNA fragments was also eliminated with the non linear geometric barrier patterns . Quantification of the lateral motion of these DNA molecules yielded values of just 2.00 10 6 1.73 10 6 m2 s demonstrating that the DNA was effectively immobilized. These DNA molecules remained stationary even over much longer periods spanning tens of minutes and this was true for each of the different barrier dimensions. This data provides evidence indicating that the non linear geometric barrier patterns effectively restricted lateral movement of the DNA molecules along the barrier edges.

To further confirm the assignment of the lowest relative signal intensity as arising from just one DNA the DNA molecules were partially digested in real time with very low concentrations of the restriction endonuclease AVaI and see below . If just one DNA molecule is present in a nanowell then stochastic cleavage events will coincide with the complete and near instantaneous loss of any DNA downstream from the corresponding cleavage site because the YOYO1 stained DNA fragment liberated by the AVaI cut will be washed away by buffer flow. In contrast if multiple DNA molecule are present in a single nanowell then a single stochastic cleavage event by AVaI will not yield instantaneous loss of all downstream signal but rather will result in a decease in signal intensity downstream of the cleavage site corresponding to loss of just one DNA molecule. As shown in when a single DNA is loaded in the nanowell cleavage by the restriction enzyme leads to immediate loss of the corresponding downstream DNA fragment. In contrast when multiple DNA molecules are present in a nanowell in this example there are two molecules in the nanowell cleavage by the restriction enzyme results in an incremental decrease in the downstream signal because just one of the DNA molecules is cut over the course of the digest. The results from these partial restriction digests confirmed that the lowest YOYO1 fluorescence signal corresponds to just one molecule of DNA.

As shown above these non linear geometric barrier patterns permit studies of even relatively small DNA fragments by eliminating their lateral mobility along the barrier edges. We have also demonstrated that we can load the geometric nanowells with single DNA molecules. In previous work we have shown that the design of the curtains also yields DNA molecules that are all aligned with the same sequence orientation based upon the location of the biotin tag at a specific end of the DNA. As a consequence of this consistent alignment complete restriction digest of the DNA molecules within the curtain yields identical tethered fragments whose lengths correspond to the furthest upstream cleavage site. Complete digests at high concentrations of restriction enzyme cannot be used to identify multiple sites cleaved by the same restriction enzyme because the downstream DNA is flushed from the sample chamber as soon as the DNA is cleaved. As shown above restriction digests done at low concentrations of enzyme can reveal stochastic cleavage events which could be used to reveal all of the fragments corresponding to each of these intermediate lengths. However this is only true if the digest is viewed in real time and sufficient numbers of molecules are observed to detect all sites and the cleavage rate is slower than the data acquisition frequency. If these criteria are met then following each individual cleavage event the downstream fragment will be immediately flushed from the sample chamber by buffer flow leaving behind the intact biotinylated fragment whose shortened length would correspond to the location of the cleavage site. This can happen repeatedly on the same DNA until that particular molecule is cleaved at the furthest upstream site relative to the anchored end of the DNA. The large number of DNA molecules that can be viewed with these DNA curtains makes it possible to detect all of the potential sites within the DNA and the geometric nanowells prevent the DNA molecules from slipping back and forth after they are cleaved. Eventually all of the DNA molecules will be cut to a final length corresponding to the furthest upstream restriction site. We refer to this real time digest as dynamic optical restriction mapping and an example of such an assay is shown in using the restriction enzymes EcoRI and BamHI respectively. These kymograms were selected because they provide examples where each of the five EcoRI sites or five BamHI sites can be identified. The total time required for this digest was just under 120 s. This assay is not possible with smooth barrier designs because the DNA molecules start to rapidly slip back and forth as the fragment lengths become shorter and shorter thus interfering with measurements of the same DNA molecule over time. Thus this real time dynamic optical restriction mapping assay is greatly facilitated when used in combination with barrier patterns comprised of the geometric nanowells.

Nucleosomes are the fundamental unit of organization in eukaryotic chromatin and contribute to all aspects of DNA metabolism. The importance of chromatin structure has driven efforts to establish computational methods for predicting genome wide nucleosome distributions based on intrinsic sequence preferences. Here we use high throughput single molecule imaging of nanofabricated DNA curtains to directly determine intrinsic energy landscapes for nucleosome deposition on model substrates. We then ask whether in silico predictions accurately reflect course grain features of the experimentally derived intrinsic energy landscapes. Our results show that nucleosome distribution patterns were anticorrelated with predictions from the widely cited model of Segal et al but were strongly correlated with the more recent model of Field et al. Without being bound by theory these findings reinforce that DNA contains intrinsic positioning information necessary for dictating distributions of canonical nucleosomes and indicate that exclusionary sequences play a dominant role in this process. We also demonstrate that intrinsic landscapes are largely independent of constituent histone identity and nucleosomes containing variant histones H2AZ or CenH3 are well correlated with patterns observed for canonical nucleosomes indicating they are subject to similar thermodynamic principles despite disparate in vivo distributions. However the non histone protein Scm3 drastically alters the intrinsic landscape enabling centromeric nucleosomes to overcome exclusionary effects of poly dA dT rich sequences. Finally we show that cis regulatory sequences in human DNA coincide with peaks in the intrinsic landscape whereas valleys correspond to non regulatory regions and we present evidence arguing that patterns of nucleosome deposition in vertebrate genomes are influenced by both extrinsic factors and possible intrinsic sequence elements not yet accounted for by current theory.

The distribution of nucleosomes throughout the genome has profound consequences for DNA transcription repair and chromosome segregation. Nucleosomes consist of 147 base pairs bp of DNA wrapped in 1.7 turns around an octamer containing two of each histone H2A H2B H3 and H4. There has been tremendous interest in developing in silico models of genome wide nucleosome positions the first of which came from Segal et al. and calculated probabilistic nucleosome distributions based upon the universal preference of nucleosomes for bendable DNA containing AA TT AT dinucleotides with 10 bp periodicities in counter phase with GC dinucleotides see also . Without being bound by theory this supported the idea of a second genetic code which asserts that genomes intrinsically encode information dictating nucleosome distributions and posits that extrinsic factors such as chromatin remodeling proteins play a limited role in establishing steady state positions. However without being bound by theory accumulating evidence demonstrates exclusion signals such as poly dA dT also influence nucleosome distribution yet these sequences were unaccounted for in the original Segal predictions. A more recent model from Segal and colleagues which shall be referred to as the Field Kaplan Fondufe Mittendorf FKFM model used 2000 fold more sequence information from isolated nucleosomes to train the pattern matching algorithm and also included exclusion effects from motifs enriched in linker DNA see also .

The relative performance of these models as general tools for predicting intrinsic landscapes has yet to be thoroughly explored prompting us to ask whether we could verify the predictions by high throughput single molecule imaging with total internal reflection fluorescence microscopy. For this assay DNA is anchored to a lipid bilayer on the surface of a microfluidic chamber and hydrodynamic force is used to align thousands of molecules into curtains along nanofabricated barriers where they are viewed in real time and . To make curtains with nucleosomes recombinant octamers containing FLAG H2B were deposited onto biotinylated DNA by salt dialysis which recapitulates thermodynamically favorable distributions and nucleosomes were labeled in situ with anti FLAG quantum dots QDs . Transient termination of hydrodynamic force provoked entropic collapse of the curtains causing the DNA and nucleosomes to drift outside the detection volume defined by the evanescent field verifying the molecules were anchored only via the biotin tag . For distribution analysis conditions were selected to yield 5 nucleosomes per DNA equivalent to 1.5 coverage of the 48.5 kb substrate and time courses confirmed the nucleosomes did not move or dissociate during the experiments .

 DNA is not subject to evolutionary pressure to position nucleosomes but analysis of the phage genome revealed an advantage for its use as a model substrate the Segal and FKFM models yielded strikingly dissimilar predictions for the nucleosome distribution patterns and . Pearson correlation analysis revealed these predictions were actually anticorrelated r 0.76 p

Information within eukaryotic genomes is enriched through targeted deposition of histone variants. Histone H2AZ is a conserved lineage of H2A that influences gene regulation. CenH3 Cse4 in yeast is an H3 variant that serves as a centromere specific epigenetic marker. Cse4 can replace CenH3 in human centromeres indicating a high degree of functional conservation. In vivo deposition of H2AZ and CenH3 requires extrinsic factors but it remains unclear whether variants are subject to the same energetic landscapes that dictate favorable binding by canonical nucleosomes. Nucleosomes bearing H2AZ displayed the same distribution trends as canonical nucleosomes demonstrating that H2AZ does not alter thermodynamic sequence preferences and . The intrinsic landscape is intriguing for Cse4 which binds poly dA dT rich CEN DNA in vivo and forms an unusual hexasome wherein H2A H2B is replaced with the non histone protein Scm3. If centromeric nucleosomes obey the same principles as canonical nucleosomes with respect to site preferences then the in vitro distributions should be similar.

Alternatively if centromeric nucleosomes do not obey the same thermodynamic principles then they should exhibit distinct distribution patterns. As shown in Cse4 did cause 7 of the total nucleosomes to redistribution away from the left 10 kb of the DNA reflecting a subtle perturbation of the intrinsic landscape. However the overall distribution of Cse4 nucleosomes N 2 033 was still correlated with that of the canonical nucleosomes r 0.58 p

All of the work described above utilized phage DNA as a simple model substrate. To determine the intrinsic landscape of a more complex eukaryotic substrate we tested a 23 kb fragment from the human globin locus . Comparison of the two theoretical distributions for this substrate revealed they were again anticorrelated r 0.74 p

In contrast valleys within the intrinsic landscape corresponded to non transcribed and non regulatory DNA. This data reveals that cis regulatory regions within the human globin locus are poised with thermodynamically favored nucleosome binding sites. Without being bound by theory this evolutionarily driven architecture has arisen because precise nucleosome positioning within regulatory sequences can be much more critical compared to other areas of the genome. It remains to be determined whether this organizational relationship is a general trend defining the intrinsic landscape of eukaryotic DNA. Interestingly the locations of nucleosomes across the globin locus have been mapped in CD4 T cells but there was no obvious correlation between these positions and the theoretical FKFM prediction . The in vivo nucleosome distribution also did not fully coincide with the course grain features of the experimentally observed intrinsic landscape although the promoter proximal regions of the and globin genes were occupied by positioned nucleosomes based on the intrinsic landscape . However the in vivo data were paradoxically dissimilar with the experimentally defined intrinsic landscape in the region encompassing the A globin gene illustrating that thermodynamically favored sequences alone can not predict nucleosome occupancy within the human genome. Therefore extrinsic factors must play a more prevalent role for establishing in vivo nucleosome distribution patterns in the human genome.

Histones were expressed in purified from inclusion bodies and reconstituted as described . H2AZ was cloned into pET 100 D TOPO and purified using the same procedure . Cse4 was purified and reconstituted as described . Scm3 remained soluble after cell lysis and was ammonium sulfate precipitated 45 saturation resuspended in unfolding buffer 7 M guanidinium HCl 1 M NaCl 50 mM Tris HCl pH 7.8 1 mM EDTA 1 mM DTT and dialyzed against urea buffer 7 M urea 1 M NaCl 10 mM Tris HCl pH 7.8 1 mM EDTA 5 mM mercaptoethanol . Scm3 was purified on Ni NTA agarose in urea buffer eluted with 200 mM imidazole and dialysed against 10 mM Tris HCl pH 7.8 plus 5 mM mercaptoethanol followed by 10 mM Tris HCl pH 7.8 then lyophilized and stored at 20 C. Nucleosomes were deposited onto DNA by salt dialysis and the DNA was assembled into curtains as described. Nucleosomes were labeled with 2 nM QDs Invitrogen 705 nm emission conjugated to anti FLAG antibodies Sigma . Illumination was by a 488 nm laser Coherent Sapphire CDHR images were collected using a water immersion objective Nikon 1.2 NA Plan Apo 60 . YOYO1 and QD emissions were separated using a dichroic mirror 630 nm DCXR Chroma Technologies and recorded with an EMCCD Photometrics Cascade 512B . Images for position analysis were acquired at a flow rate of 0.4 ml min in 40 mM Tris HCl pH 7.8 1 mM DTT and 0.2 mg ml BSA. Flow was transiently paused to exclude non specifically bound QDs from analysis. Only full length DNA molecules well resolved from adjacent neighboring molecules were used for distribution measurements. Nucleosome position measurements and statistical analysis were performed using Igor Pro. All correlation trends were cross validated on a random subset comprised of half the experimental data.

Electron beam lithography was used to nanofabricate chromium diffusion barriers on the fused silica surface of a microfluidic sample chamber. DNA molecules stained with YOYO1 green were then anchored via a biotin neutravidin linkage to a fluid lipid bilayer deposited on the surface of the slide. Application of buffer flow aligns the DNA molecules along the leading edges of the nanofabricated barriers and extends the DNA parallel to the sample chamber surface confining the molecules within the excitation volume defined by the penetration depth of the evanescent field. When buffer flow is transiently paused the DNA molecules collapse due to entropic forces and move away from the surface of the sample chamber demonstrating that the molecules do not interact nonspecifically with the lipid bilayer.

A YOYO1 stained DNA curtain green was assembled as described with the exception that the DNA was pre bound by nucleosomes deposited through salt dialysis. The nucleosomes were labeled in situ using fluorescent quantum dots magenta . When buffer flow is transiently paused the nucleosome bound DNA molecules collapse due to entropic forces and move away from the surface of the sample chamber demonstrating that the molecules do not interact nonspecifically with the lipid bilayer and verifying that the nucleosomes are bound to DNA.

Single molecule studies of biological macromolecules can benefit from new approaches that facilitate experimental design and data acquisition. Here we develop new strategies to construct molecular curtains of DNA in which the substrates are aligned with respect to one another and maintained in an extended configuration. Importantly both ends of the DNA are anchored to the surface of a microfluidic sample chamber that is otherwise coated with an inert lipid bilayer. This is accomplished through the use of nanofabricated racks of metallic barrier patterns with two distinct functional elements. Linear barriers to lipid diffusion are first used to align the DNA. Pentagons coated with antibodies then provide solid anchor points for the free ends of the aligned DNA molecules. These devices enable the alignment and anchoring of hundreds of individual DNA molecules which can then be visualized using total internal reflection fluorescent microscopy under conditions that do not require continuous application of buffer flow to stretch the DNA. This offers the potential for imaging and data acquisition without perturbing the behavior of the proteins under investigation by eliminating the need for an externally applied hydrodynamic force. We provide a demonstration that these DNA racks can be used in a 1D diffusion assay that monitors the motion of quantum dot tagged proteins along DNA.

Dynamic interactions between proteins and DNA underlie many biological processes and as such are the subject of intense investigation. Many laboratories are now tackling these problems using new experimental methods that enable the visualization of DNA molecules or protein DNA complexes at the single molecule level in real time and the information garnered from these experiments is being used to build detailed mechanistic models of many different types of reactions. Similarly micro and nanoscale devices are becoming increasingly powerful tools for the manipulation and analysis of individual DNA molecules. However one major drawback of many single molecule techniques is that they are inherently designed to probe individual reactions and as a consequence it can be challenging to gather statistically relevant data. This difficulty is often compounded by the fact that these experiments are technically demanding. Therefore it is advantageous to establish new approaches that can increase throughput capacity of single molecule methods while at the same time making these approaches both easier and more readily applicable to biological reactions involving different types of DNA transactions.

In an effort to help make single molecule imaging techniques more accessible we have integrated nanoscale engineering microfluidics and lipid bilayer coated surfaces with single molecule optical microscopy to develop high throughput methods for making molecular curtains comprised of thousands of individual DNA molecules. A key aspect of these experimental platforms is that they utilize inert lipid bilayers to passivate the fused silica surface of a microfluidic sample chamber. Artificial lipid membranes deposited on solid supports have proven useful for many types of biochemical studies. An important aspect of artificial membranes is that the chemical characteristics of the bilayer can be readily controlled through careful selection of the constituent lipids. In addition artificial membranes is that they can be partitioned with chemical or mechanical barriers and the distributions of molecules anchored to lipids can also be manipulated using photochemical modulation electrical fields or hydrodynamic force. Taken together the aforementioned features suggested the potential for controlling the organization of molecules anchored to lipid membranes and accordingly we have demonstrated that mechanical barriers that disrupt the continuity of lipid bilayers can be used to control the two dimensional organization of DNA molecules anchored to a surface. This controlled organization is accomplished by first coating a sample chamber surface with a bilayer containing a small fraction of biotinylated head groups and DNA molecules are anchored by one end to the bilayer via a biotin neutravidin linkage. The lipid tethered DNA molecules are assembled into molecular curtains by pushing them to the leading edge of either micro or nano scale lipid diffusion barriers through the application of a hydrodynamic force. We have demonstrated that microscale barriers to lipid diffusion made by manually etching a fused silica surface with a diamond tipped scribe could be used to align hundreds of lipid tethered DNA molecules and we refer to these as DNA curtains. More recently we have we have employed electron beam ebeam lithography to fabricate barriers with nanoscale dimensions which allows for much more precise control over both the location and lateral distribution of the DNA molecules within the curtains. These nanofabricated DNA curtains permit simultaneous visualization of thousands of individual DNA molecules that are perfectly aligned with respect to one another and can be used for massively parallel data acquisition from thousands of individual protein DNA complexes in real time using a robust experimental platform that is amenable to a wide variety of biological applications. We have also shown that these DNA curtains are advantageous for studying protein DNA interactions at the single molecule level and we have applied these tools to biological systems such as nucleosomes and chromatin remodeling homologous DNA recombination and post replicative mismatch repair.

Our previous DNA curtain designs required continuous application of a hydrodynamic force during data collection because just one end of the DNA is anchored to the lipid bilayer. If buffer flow is terminated the DNA does not remain stretched and as a consequence it can not be visualized along its full contour length because it drifts outside of the detection volume defined by the penetration depth of the evanescent field. The need for buffer flow is not problematic for many types of measurements however it can potentially impact the behavior of bound proteins or protein complexes and the magnitude of this impact can scale in proportion to the hydrodynamic radius of the molecules under observation. The influence of buffer flow is especially apparent during measurements involving proteins that slide on DNA by one dimensional diffusion because an applied flow force can strongly bias the direction that the sliding proteins travel along the DNA.

Here we sought to develop new procedures for making curtains of aligned DNA molecules that were held in an extended configuration anchored by both ends to the surface of a microfluidic sample chamber and suspended above an inert bilayer. These double tethered curtains of DNA are specifically intended for use in experiments designed to look at passive protein motion along DNA molecules. To achieve this design this we have developed new patterns of metallic barriers with made using either electron beam or nanoimprint lithography. These metallic patterns incorporate both linear barriers to lipid diffusion and arrayed pentagons and together these distinct functional elements serve to align and tether long molecules of DNA. The DNA molecules are first anchored by one end to a fluid lipid bilayer via a biotin neutravidin interaction and initially aligned with one another by using hydrodynamic force to push them into the linear barriers. The arrayed pentagons serve as solid anchor points positioned at a defined distance downstream from the linear barriers and are coated with antibodies directed against either digoxigenin or bromodeoxyuridine which are covalently attached to the free ends of the DNA molecules. Importantly once linked to the surface the DNA molecules are maintained in an extended state suspended above an inert lipid bilayer and remain confined within the detection volume defined by the penetration depth of the evanescent field allowing for continuous observation along their full contour length even in the absence of an applied hydrodynamic force.

Fused silica slides were cleaned in NanoStrip solution CyanTek Corp Fremont Calif. for 20 minutes then rinsed with acetone and isopropanol and dried with N. The slides were spin coated with a bilayer of polymethylmethacrylate PMMA molecular weight 25K 3 in anisole and 495K 1.5 in anisole MicroChem Newton Mass. followed by a layer of Aquasave conducting polymer Mitsubishi Rayon . Each layer was spun at 4 000 rpm for 45 seconds using a ramp rate of 300 rpm s. Polygon patterns and linear barriers were written by Ebeam lithography using an FEI Sirion scanning electron microscope equipped with a pattern generator and lithography control system J. C. Nabity Inc. Bozeman Mont. . After patterning Aquasave was rinsed off with deionized water. Resist was developed using a 3 1 solution of isopropanol to methyl isobutyl ketone MIBK for 1 minute with ultrasonic agitation at 5 C. The substrate was then rinsed in isopropanol and dried with N. A 15 20 nm layer of gold atop a 3 5 nm adhesion layer of either chromium or titanium was deposited using a Semicore electron beam evaporator. Liftoff was effected at 80 C. in a 9 1 ratio of methylene chloride to acetone. Alternatively barriers were made out of just a 15 20 nm layer of chromium as previously described. Following liftoff samples were rinsed with acetone to remove stray chromium flakes and dried with N. Barriers were imaged using a Hitachi 4700 scanning electron microscope and a PSIA XE 100 Scanning Probe Microscope in noncontact mode. Optical images of the barriers were taken with a Nikon Eclipse ME600.

Nanoimprint masters were fabricated using electron beam lithography liftoff and inductively coupled plasma etching. Briefly a bilayer of poly methyl methacrylate PMMA 25K and 495K was spun onto a silicon wafer with a thin coating of silicon dioxide. Patterns were written by an FEI Sirion SEM outfitted with a Nabity Nanopattern Generation System then developed in a mixture of isopropanol methyl isobutyl ketone 3 1 at 5 C. in a bath sonicator. Samples were rinsed with isopropanol and dried with N. A Semicore Ebeam evaporator was used to evaporate 20 nm Cr onto the masters. Liftoff was performed in acetone at 65 C. The patterned masters were plasma etched to a depth of 100 nm in a mixture of CF O 9 1 for 90 seconds at a power of 300 W using an Oxford ICP etch tool. Nanoimprint masters were coated with a fluorinated self assembled monolayer Nanonex Princeton N.J. to prevent adhesion between the master and resist.

To make nanoimprinted barriers PMMA 35K Microresist Technologies Germany was spin coated on a fused silica microscope slide and baked on a hotplate for 5 min. at 180 C. Nanoimprint was performed in two stages a 2 min. pre imprint phase with a pressure of 120 psi and pre temp. of 120 C. followed by a 5 min. imprint phase with a pressure of 480 psi and temp. of 190 C. This heated PMMA well above its glass transition temp. and allowed it to conform to the mold. After imprinting a de scum process was done to remove 10 nm of residual PMMA. De scum was done in an inductively coupled plasma under CHF O 1 1 and a power of 200 W for 40 seconds total two iterations of 20 seconds . After descum 15 20 nm of Cr was evaporated on the samples and liftoff was performed in acetone at 65 C. for several hours followed by bath sonication to remove stray metal flakes. Nanopatterned slides were then rinsed in acetone and dried with N.

The flowcells were assembled from fused silica slides G. Finkenbeiner Inc. with chromium nanoscale diffusion barriers. Inlet and outlet ports were made by boring through the slide with a high speed precision drill press equipped with a diamond tipped bit 1.4 mm O.D. Kassoy . The slides were cleaned by successive immersion in 2 v v Hellmanex 1 M NaOH and 100 MeOH. The slides were rinsed with filtered sterile water between each wash and stored in 100 MeOH until use. Prior to assembly the slides were dried under a stream of nitrogen and baked in a vacuum oven for at least 1 hour. A sample chamber was prepared from a borosilicate glass coverslip Fisher Scientific and double sided tape 25 m thick 3M . Inlet and outlet ports Upchurch Scientific were attached with hot melt adhesive SureBonder glue sticks FPC Corporation . The total volume of the sample chambers was 4 l. A syringe pump Kd Scientific and actuated injection valves Upchurch Scientific were used to control sample delivery buffer selection and flow rate. The flowcell and prism were mounted in a custom built heater with computer controlled feedback regulation to control the temperature of the sample from between 25 37 C. 0.1 C. as necessary. After each use the slides were soaked in MeOH to remove the ports and tape rinsed with water washed briefly 15 20 minutes with Nanostrip and rinsed with water. This procedure was sufficient to clean the slide surfaces for reuse and each slide could be used multiple times without degrading the quality of the optical surface or the metallic patterns.

DNA curtains were constructed as described with the exception of additional steps necessary for anchoring the second end of the DNA. All lipids were purchased from Avanti Polar Lipids and liposomes were prepared as previously described. In brief a mixture of DOPC 1 2 dioleoyl sn glycero phosphocholine 0.5 biotinylated DPPE 1 2 dipalmitoyl sn glycero 3 phosphoethanolamine N cap biotinyl and 8 10 mPEG 550 PE 1 2 dioleoyl sn glycero 3 phosphoethanolamine N methoxy polyethylene glycol 550 . The mPEG is does not affect bilayer formation or assembly of the DNA curtains but rather serves to further passivated the surface against nonspecific adsorption of quantum dots which we use in our studies of protein DNA interactions . Liposomes were applied to the sample chamber in three injections of 200 l followed by 5 minute incubations. Excess liposomes were flushed away with 1 ml of buffer A which contained 10 mM Tris HCl pH 7.8 plus 100 mM NaCl and the bilayer was incubated for an addition 30 minutes. Buffer A plus 25 g ml anti DIG Fab Roche anti fluorescein Sigma or anti BrdU IgG Sigma was then injected into the sample chamber and incubated for 30 minutes. The sample chamber was then flushed with 1 ml of buffer B which contained 40 mM Tris HCl pH 7.8 1 mM DTT 1 mM MgCl and 0.2 mg ml BSA and incubated for an additional 5 minutes. 1 ml of buffer A containing Neutravidin 330 nM was then injected into the sample chamber and incubated for 20 minutes. The flowcell was then rinsed with 3 ml of buffer B to remove any unbound Neutravidin. 1 ml of DNA 20 pM labeled at one end with biotin and at the other end with either digoxigenin fluorescein or bromodeoxyuridine as indicated and pre stained with 1 2 nM YOYO1 was injected into the sample chamber in five 200 l aliquots with a 2 3 minute incubation period following each injection. The DNA was then aligned at the linear barriers using a flow rate of 0.02 ml min and this rate was then increased to 2 3 ml min to anchor the second end of the DNA molecules.

The DNA substrates were made by ligating oligonucleotides to the 12 nucleotide overhangs at the end of the phage genome 48.5 kb . Ligation mixes 1 ml total volume contained 4 nM DNA Invitrogen 1 M biotinylated oligonucleotiode 5 pAGG TCG CCG CCC BioTEG 3 SEQ ID NO 1 1 M fluorescein labeled oligonucleotide or DIG labeled oligonucleotide 5 pGGG CGG CGA CCT fluor 3 SEQ ID NO 6 or 5 pGGG CGG CGA CCT DIG 3 SEQ ID NO 3 and 1 ligase buffer NEB . The reaction mix was warmed to 65 C. for 10 minutes and then cooled slowly to room temperature. After cooling ligase was added T4 DNA ligase 400 U l or Taq ligase 40 U l NEB and the mixture was incubated overnight at 42 C. Reactions performed with T4 ligase were then heat inactivated at 65 C. for 10 minutes and the ligated DNA products were purified over a Sephacryl S200HR column GE Healthcare run in 10 mM Tris HCl pH 7.8 1 mM EDTA plus 150 mM NaCl. The purified DNA was stored at 20 C.

For inserting a DIG labeled oligonucleotide complementary to position 26 151 26 166 on 500 ul of DNA 200 pM labeled at the ends with biotin and fluorescein as described above was incubated for 2 hours with the nicking enzymes Nb.BsmI and Nt.BstNBI 50 units each NEB at 55 C. in buffer containing 10 mM Tris HCl pH 7.8 10 mM MgCl 150 mM NaCl. Subsequently 1 uM of oligonucleotide 5 pCAT TCT TGA GTC CAA TTT TT DIG 3 SEQ ID NO 4 was added to the solution along with 10 mM EDTA and the mixture was incubated at 55 C. for 20 minutes and then allowed to slowly cool to room temperature over the course of an hour. ATP was added to a final concentration of 1 mM along with 2 000 units of T4 ligase and an additional 10 mM MgCl. The reaction was then incubated at room temp. for 90 min. Additional EDTA C 20 mM was added to the solution and the nicking enzymes and ligase were heat denatured at 80 C. for 20 min.

For inserting an oligonucleotide at position 33 779 33 791 on 500 l DNA 200 pM labeled at the ends with biotin and fuorescein as described above was incubated for 2 hours with the nicking enzyme Nt.BstNBI 50 units NEB at 55 C. in buffer containing 10 mM Tris HCl pH 7.8 10 mM MgCl 150 mM NaCl. The enzyme was heat denatured by incubation at 80 C. for 20 minutes. 1 M of oligonucleotide 5 pTTC AGA GTC TGA CTT TT DIG 3 SEQ ID NO 5 was added to the solution and the mixture was incubated at 55 C. for 20 minutes then cooled slowly to room temperature over the course of 1 hr. ATP was then added to a final concentration of 1 mM along with 2 000 units of T4 ligase and the reaction was incubated at room temperature for 90 min. The ligase was then heat denatured by incubation at 65 C. for 20 minutes.

The basic design of the microscope used in this study has been previously described. In brief the system is built around a Nikon TE2000U inverted microscope with a custom made illumination system. For this study a 488 nm 200 mW diode pumped solid state laser Coherent Sapphire CDHR was used as the excitation source. The laser was attenuated as necessary with a neutral density filter and centered over the DNA curtain by means of a remotely operated mirror New Focus . The beam intensity at the face of the prism was 10 15 mW. Images were detected with a back illuminated EMCCD detector Photometrics Cascade 512B . TIRFM images were collected using a 60 water immersion objective lens Nikon 1.2 NA Plan Apo unless otherwise indicated.

Here we expand on our previous work and demonstrate the development of new barrier patterns which we call DNA racks that can be used to make DNA curtains in which both ends of the molecules are anchored to the flowcell surface. An overview of the general design is presented in and utilizes a combination of two distinct pattern elements as described below. In principle one end of the DNA is first anchored via a biotin neutravidin interaction to a supported lipid bilayer coating the surface of the sample chamber . In the absence of a hydrodynamic force the molecules are randomly distributed on the surface but lie primarily outside of the detection volume defined by the penetration depth of the evanescent field 150 200 nm . Application of flow pushes the DNA through the sample chamber with one end remaining anchored to the bilayer. The first pattern elements are linear barriers to lipid diffusion which are oriented perpendicular to the direction of flow at strategic locations in the path of the DNA these linear barriers are designed to halt the movement of the lipid tethered DNA molecules causing them to accumulate at leading edge of the barriers where they then extend parallel to the surface. The second elements of the pattern are a series of arrayed pentagons positioned behind the linear barriers and separated from one another by small channels. The non linear geometric shape of the pentagons is intended to act as a funnel and direct any lipid tethered DNA molecules through the channels so that they do not accumulate at the leading edge of the pentagons but rather are pushed along the surface until they encounter the next successive linear barrier in the pattern. The distance between the linear barriers and the pentagons is optimized for the length of the DNA to be used for the experiments. The pentagons themselves present a large surface that can be coated with antibodies directed against digoxigenin DIG which is covalently linked to the ends of the DNA opposite the ends bearing the biotin tag. When the DIG coupled DNA end encounters the antibody coated pentagons they should become immobilized and the DNA should remain stretched parallel to the surface even when no buffer is being pushed through the sample chamber.

The barrier patterns necessary to achieve the desired design features were made by either Ebeam lithography as previously described or by nanoimprint lithography. shows the characterization of a DNA rack made by Ebeam lithography and highlights specific features of the pattern geometry deemed critical for its function. shows an optical image of the overall pattern design including the dimensions of critical pattern parameters. shows measurements of these parameters using atomic force microscopy AFM . shows characterization of the patterns with scanning electron microscopy SEM . As demonstrated from these images the height of the patterns elements was on the order of 20 nm the width of the channels was 500 nm and the optimal distance between the leading edge of the linear barriers and the back of the pentagon array was 13 m see below . This distance was specifically selected for use with DNA which is 48 502 base pairs with a fully extended contour length of approximately 16.5 m thus 11 13 m would correspond to a mean extended length of 65 79 that of the full contour length respectively.

The overall design of the DNA rack relies upon the selective but nonspecific adsorption of antibodies to the relatively large exposed surface of the metallic pentagons and the antibodies should not interact with the inert lipid bilayer. All nanofabricated elements of the pattern are made of the same material therefore the antibodies could potentially bind both the linear barriers as well as the pentagons. However each pentagon has a surface area of 1.55 mand for each 260 m long linear barrier there are 167 pentagons. This corresponds to a total surface area of 372 mfor the pentagons compared to just 36 mfor the entire linear barrier. The smaller total surface area of the linear barriers should ensure adsorption of more antibody to the much larger area encompassed by the pentagons.

To assemble the DNA curtains the surface of the flowcell was first coated with a lipid bilayer as previously described with the exception that BSA was omitted from all buffers used prior to deposition of the antibody. Once the bilayer was assembled on the surface anti DIG Fab fragments or anti fluorescein IgG was injected into the sample chamber where they were allowed to adhere nonspecifically to the exposed metal barriers. Following a brief incubation the free antibody was rinsed from the flowcell and replaced with buffer containing 0.2 mg ml BSA which served as a nonspecific blocking agent to passivate any remaining exposed surfaces. DNA stained with the fluorescent intercalating dye YOYO1 was then injected into the sample chamber incubated briefly without buffer flow and then buffer flow was applied to push the anchored molecules into the linear barriers. Buffer flow was then terminated and the anchored DNA molecules imaged by TIRFM. As shown in this procedure yielded curtains of DNA molecules that remained fully extended and visible even though no buffer was flowing through the sample chamber. Any DNA molecules that failed to become anchored by the DIG labeled end remained near the linear barriers but only the ends of the molecules could be observed rather than their full contour lengths. These single tethered DNA molecules gradually diffused away from the barrier edges and could also be pushed away from the barrier edge by reversing the direction of buffer flow but the double tethered DNA molecules remained anchored to their original locations providing additional conformation that the molecules making up the curtain are anchored in the desired configuration.

The number of DNA molecules at each of the different barrier locations was proportional to the distance between the adjacent linear barriers. The number of double tethered DNA molecules decreased slowly over time with a half life of 1 hour for the DIG labeled molecules. Finally control experiments verified that the presence of both the DIG labeled and the Fab fragments were necessary for efficiently anchoring the two ends of the DNA. Approximately 70 of the anchored DNA could be attributed to specific antibody hapten interactions with the remaining 30 due to nonspecific anchoring of the DNA ends to the antibody coated pentagons. Similar double tethered DNA curtains could be made using substrates that were end labeled with either a single bromodeoxyuridine BrdU or fluorescein moiety and pentagons coated with anti BrdU or anti fluorescein antibodies see below respectively. The half life of the fluorescein anchored DNA was comparable to that of the DIG anchored DNA. However the BrdU anchored DNA molecules had a greater tendency to dissociate from the pentagons displaying a half life on the order of just a few minutes.

The differential chemistries used to anchor the two ends of the DNA were predicted to define the orientation of the molecules within the curtains. Specifically the biotinylated end should be anchored at the edge of the linear barrier and the other end should be at the pentagons and based on this design all of the molecules should be in the exact same orientation. If the DNA molecules were aligned as expected then a fluorophore located at a single specific site within the DNA and offset from its center should appear as a fluorescent line spanning the DNA curtain. The line should be oriented perpendicular to the long axis of the extended DNA molecules and should coincide precisely with the know location of the fluorescent tag. In contrast if the DNA molecules were randomly oriented then the fluorescent tags should appear as two lines demarking each of the two possible orientations of the DNA. To confirm that the DNA was oriented correctly the molecules were labeled at the free ends with fluorescein and labeled at a specific internal position with DIG. The internal labels were made using an oligonucleotide replacement strategy wherein the DNA was nicked at specific sites with a nicking endonuclease and short ssDNA fragments flanked by the resulting nicks were replaced with a DIG tagged oligonucleotide see Materials and Methods . Experiments with DNA curtains anchored by a single end confirmed that the fluorescent tags were present at a single location within the DNA molecules as dictated by the sequences of the oligonucleotides . The double tethered curtains were then assembled using pentagons coated with anti BrdU antibodies and after assembly the DIG tags with labeled with anti DIG coated Qdots. As shown in the anti DIG Qdots were all aligned with one another confirming the orientation of the DNA molecules. These results demonstrate that the DNA molecules anchored to the surface via the rack elements were all in the same orientation as defined by the distinct function groups at the opposing ends of the DNA.

Many biological reactions are dependent upon protein motion along DNA and these processes often involve passive diffusion. The primary motivation for development of these double tethered DNA curtains was for use in experiments designed to visualize motion of proteins along DNA. We have previously demonstrated that the protein complex Msh2 Msh6 can diffuse in one dimension along duplex DNA. Msh2 Msh6 is an essential component of the post replicative mismatch repair machinery and is responsible for locating and initiating repair of biosynthetic DNA replication errors. Our initial studies with Msh2 Msh6 relied upon DNA molecules that were tethered by either end to neutravidin nonspecifically absorbed to a fused silica surface that was otherwise coated with a lipid bilayer. With this approach we would obtain only 10 30 DNA molecules per flowcell that were suitable for making diffusion measurements. These double tethered DNA molecules were randomly distributed and had to be manually located before use by visually scanning the entire surface making these 1D diffusion measurements technically demanding and time consuming. As shown above using the engineered surfaces we can now visualize thousands of molecules per flowcell. Here we demonstrate that these anchored curtains of DNA are suitable for visualizing protein DNA interactions.

To determine whether the DNA racks could be used to study 1D diffusion of proteins along DNA we used the mismatch repair protein Mlh1 and it was labeled with a single quantum dot via a FLAG epitope tag complete analysis of quantum dot tagged Mlh1 will be presented elsewhere. The labeled proteins were injected into a flowcell containing a curtain of double tethered DNA molecules and videos were collected over a 60 second period. As shown in we could readily detect binding of Mlh1 to the DNA molecules that were aligned and anchored on the surface. There were 79 DNA molecules and 235 DNA bound proteins observed in this single field of view illustrating the dramatic improvement of this approach compared to our previous technique for making double tethered DNA molecules. The proteins bound to the DNA rapidly diffused along the DNA molecules and this motion was revealed in kymograms made from representative examples of molecules within the DNA curtains .

If protein bound molecules were nonspecifically linked to the sample chamber surface along its contour length rather than suspended above the inert bilayer then they should remain in view even if the DNA breaks or otherwise detaches from one of its anchor points. In contrast if the proteins only interacted with the DNA then detachment of the DNA should cause the molecules to retract away from the surface and disappear from view. As shown in when the DNA molecules randomly detached from the surface the proteins immediately disappeared from view as they drifted outside the detection volume define by the penetration depth of the evanescent field. This data demonstrates that the fluorescently tagged proteins were bound to the DNA the DNA molecules only interacted with the sample chamber surface through their anchored ends and neither the quantum dot tagged protein nor the DNA interacted nonspecifically with the lipid bilayer.

Single molecule studies offer many unique opportunities to probe biological reactions in ways not possible through conventional biochemical or biophysical approaches. We are developing new techniques for controlling the spatial organization of surface anchored DNA molecules within the confines of a microfluidic sample chamber. Procedures for anchoring numerous long DNA molecules to surfaces are present in the literature. However none of these methods offers the ability to patterns hundreds of DNA molecules all aligned in the same orientation using an experimental platform that is both compatible with protein biochemistry and single molecule fluorescence imaging. Our unique approaches are based upon surface engineering techniques that enable us to control the distribution of DNA substrates with micro and nanoscale precision. Here we build upon our previous work by using direct write Ebeam or nanoimprint lithography to fabricate arrays of diffusion barriers followed by pentagonal anchor points which together utilize different functional features to anchor curtains of DNA by both ends on the surface of a microfluidic sample chamber. We refer to these patterned surfaces as DNA racks because the DNA is stretched out and anchored to the flowcell surface where they can be viewed by TIRFM. An important aspect of these devices is that the fused silica surface is coated with a supported lipid bilayer and the DNA molecules are suspended above this bilayer ensuring that they are maintained within an inert microenvironment compatible with a range of biological molecules. We have demonstrated that these tools can be used along with wide field TIRF microscopy to visualize hundreds of individual perfectly aligned DNA molecules all of which are arranged in the same orientation and anchored by both ends to the sample chamber surface. As with our previous nanofabricated devices this new approach is simple and robust the flowcells are reusable the barriers themselves are uniform and they do not compromise the optical quality of the fused silica or interfere with signal detection.

As a first conceptual demonstration we show here that the curtains of double tethered DNA can be used to image 1D diffusion of DNA binding proteins. Our previous approach to these experiments relied upon DNA that was randomly anchored to a surface via biotin neutravidin interactions which yielded no more than 1 3 molecules of DNA per field of view and there were just 10 30 double tethered DNA molecules present over the entire surface of the flowcell. Moreover the prior studies showed that the orientation of DNA could not be defined because the molecules were anchored with biotin located at either end of the DNA. Therefore the molecules were randomly oriented on the flowcell surface making it difficult to locate and compare different DNA molecules to one another in order to dissect sequence specific events. While this original approached proved useful for our initial studies it was tedious and remained challenging to collect sufficient data for thorough analysis. As demonstrated here we are now able to visualize on the order of one hundred DNA molecules per field of view thousands of molecules are present on the surface of a flowcell and all the DNA molecules are aligned in the exact same orientation which will make 1D diffusion measurements and molecule to molecule comparisons straightforward for future work.

DNA is attached to lipid bilayer as shown in . DNA is then aligned at the diffusion barrier edge for example a linear diffusion barrier or a non linear geometric diffusion barrier using either a hydrodynamic force or and electric field . A calcium solution of about 1 mM is then injected into the flow cell. The DNA becomes reversibly anchored to the lipid bilayer throughout the DNA s entire contour length . The following calcium concentration can also be used for example a calcium concentration of at least about 0.5 mM at least about 1 mM at least about 1.5 mM at least about 2 mM at least about 2.5 mM at least about 3 mM at least about 3.5 mM at least about 4 mM at least about 4.5 mM at least about 5 mM at least about 5.5 mM at least about 6 mM at least about 6.5 mM at least about 7 mM at least about 7.5 mM at least about 8 mM at least about 8.5 mM at least about 9 mM at least about 9.5 mM at least about 10 mM or at least about 10.5 mM.

Some advantages of using this system are as follows the need for a second barrier surface for anchoring DNA molecule is eliminated the DNA remains anchored when flow is reduced or eliminated Anchoring of DNA is completely reversible upon washing the flowcell with buffer lacking calcium and the system would enable manipulation and or collections of fragment after cleavage with a restriction enzyme.

A sample of homogeneous DNA molecules is attached to lipid bilayer as shown in . Fluorescently tagged oligonucleotides or PNAs protein nucleic acids directed against specific target sequences can be hybridized to DNA molecules aligned at the edge of a diffusion barrier for example a linear diffusion barrier or a non linear geometric diffusion barrier using either a hydrodynamic force or and electric field . An advantage of using this system includes course mapping of DNA for genomics applications which can be carried out without the requirement for cutting the DNA. This type of mapping technology is being used by others but not in the context of a DNA curtain. The DNA curtain design top view offers two potential benefits over what is being practiced 1 easier data analysis because DNA molecules are aligned and 2 the DNA can be collected after analysis by cleaving it with a restriction enzyme. For example one could design a single specific cut site right at the point of attachment. No other single molecule DNA mapping technique allows the user to harvest the DNA after analysis.

A sample of heterogeneous DNA molecules for example a heterogeneous preparation of genomic DNA can be attached to lipid bilayer as shown in . Fluorescently tagged oligonucleotides or PNAs protein nucleic acids directed against specific target sequences can be hybridized to DNA molecules aligned at the edge of a diffusion barrier for example a linear diffusion barrier or a non linear geometric diffusion barrier using either a hydrodynamic force or and electric field . Individual DNA molecules can then be designated or numbered and from the organization scheme of these DNA molecules a DNA map can be generated based on overlapping signals top view . An example of a DNA map is seen in . An advantage of using this system includes course mapping of DNA for genomics applications which can be carried out without the requirement for cutting the DNA. Specifically this system allows the mapping of more complex mixtures of genomic DNA. One would need to prepare clean samples of genomic DNA comprised of fragments that could range in size from 10 300 kb. One would also have to ensure that the DNA is labeled at ONE end not both ends with a tag such as biotin or another tag used in the art . This would allow for the DNA molecule to be anchored to the surface and is not trivial for heterogeneous DNA samples have different ends sequences.

In physiological settings nucleic acid translocases must act on substrates occupied by other proteins and an increasingly appreciated role of translocases is to catalyze protein displacement from RNA and DNA. However little is known regarding the inevitable collisions that must occur and the fate of protein obstacles and the mechanisms by which they are evicted from DNA remain unexplored. Here we sought to establish the mechanistic basis for protein displacement from DNA using RecBCD as a model system. Using nanofabricated curtains of DNA and multi color single molecule microscopy we visualized collisions between a model translocase and different DNA bound proteins in real time. We show that the DNA translocase RecBCD can disrupt core RNA polymerase RNAP holoenzyme stalled elongation complexes and transcribing RNAP in either head to head or head to tail orientations as well as EcoRI lac repressor and even nucleosomes. RecBCD did not pause during collisions and often pushed proteins thousands of base pairs before evicting them from DNA. We conclude that RecBCD overwhelms obstacles through direct transduction of chemomechanical force with no need for specific protein protein interactions and that proteins can be removed from DNA through active disruption mechanisms that act on a transition state intermediate as they are pushed from one nonspecific site to the next.

RecBCD is a heterotrimeric translocase involved in initiating homologous recombination and processing stalled replication forks. RecB is a 3 5 SF1A helicase and contains a nuclease domain for DNA processing RecD is a 5 3 SF1B helicase RecC holds the complex together and coordinates the response to cis acting Chi sequences 5 dGCTGGTGG 3 SEQ ID NO 7 . RecD is the lead motor before Chi RecB is the lead motor after Chi and Chi recognition is accompanied by a reduced rate of translocation corresponding to the slower velocity of RecB. Chi prompts RecBCD to process DNA yielding 3 ssDNA overhangs onto which RecA is loaded.

We monitored RecBCD activity using total internal reflection fluorescence microscopy TIRFM and a DNA curtain assay that allows us to visualize hundreds of aligned molecules . When assayed on DNA curtains RecBCD displayed rapid translocation 1 484 167 bp sec 37 C. 1 mM ATP N 100 high processivity 36 000 12 500 bp and also decreased velocity in response to Chi 549 155 bp sec 37 C. 1 mM ATP N 100 in agreement with previous studies.

We used quantum dots QDs to fluorescently label RNAP See Methods . The binding distribution of QD RNAP holoenzyme overlapped with known promoters promoter targeting was dependent and promoter bound holoenzymes were highly stable t 23.2 1.42 min N 58 . Core QD RNAP dissociated when challenged with heparin t 3.4 0.03 sec N 150 whereas promoter bound holoenzyme was heparin resistant t 6.7 min N 58 confirming open complex formation . Bulk assays verified QD RNAP produced transcripts and single molecule assays revealed a transcription velocity of 15.7 8.6 bp sec N 20 25 C. 250 M of each rNTP .

When RecBCD collided with RNAP the polymerase was rapidly ejected from DNA t 2.4 0.13 sec . Remarkably RNAP could be pushed long distances 10 460 7 690 bp N 44 and RecBCD could disrupt core holoenzyme stalled elongation complexes ECs and active ECs . Out of 47 collisions with QD RNAP holoenzyme 15 7 47 immediately stalled RecBCD 8.5 4 47 resulted in dissociation of RNAP with no sliding 76.5 36 47 of RNAP was pushed and 71 of pushed molecules were eventually ejected . The population of RNAP molecules that was directly ejected from the DNA increased 5 fold for stalled and active elongation complexes . RecBCD also pushed and evicted RNAP labeled with 40 nm fluorescent beads or Alexa Fluor 488 IgG arguing against nonspecific interactions between RecBCD and the QDs . RecBCD did not slow or pause upon colliding with RNAP nor was there any reduction in processivity compared to naked DNA 29 000 15 500 bp . Similar outcomes were observed before and after Chi indicating that RecBCD could dislodge RNAP regardless of whether RecB or RecD was the lead motor. We could unambiguously assign the orientation of RNAP at P and RecBCD dislodged RNAP bound at Pduring collisions in either direction . RecBCD also pushed and ejected RNAP bound at all other locations regardless of DNA orientation . RecBCD even dislodged RNAP at lower velocities 446 192 bp sec 122 128 bp sec and 78 27 bp sec at 100 M 25 M and 15 M ATP respectively indicating proteins could be dislodged even under sub optimal translocation conditions. We conclude that RecBCD disrupts RNAP regardless of orientation transcriptional status or translocation velocity.

We next asked whether RecBCD could dislodge other proteins. EcoRIis a catalytically inactive version of EcoRI which has high affinity K 2.5 fM for cognate sites and even binds tightly to nonspecific DNA K 4.8 pM . EcoRIcan halt RNA polymerase T7 and SP6 RNA polymerases SV40 large T antigen UvrD DnaB and Dda helicases SV40 replication forks and replication forks. EcoRI withstands up to 20 40 pN and EcoRIbinds cognate sites 3000 fold stronger than wild type EcoRI K 6.7 pM thus we infer the catalytic mutant can resist at least as much force as the wild type protein. Lac repressor LacI is representative of a large family of bacterial transcription factors that has served as a paradigm for transcriptional regulation and protein DNA interactions. LacI binds tightly to specific sites K 10 fM for a 21 bp symmetric operator but binds weakly to nonspecific DNA K 1 nM and slides rapidly along nonspecific DNA rather than remaining at fixed locations. LacI blocks RNAP and replication forks both in vitro and in vivo highlighting this protein as a potent and physiologically relevant barrier to translocase progression.

EcoRIand LacI were labeled with QDs See Methods . QD EcoRIand QD LacI were targeted to the correct locations on the DNA substrates and QD LacI was rapidly released from DNA with IPTG as expected . When RecBCD collided with EcoRIit pushed the proteins 13 000 9 100 bp N 70 before ejecting them from DNA . In contrast LacI was immediately ejected and was not pushed within our resolution limits . There was no change in velocity or processivity upon colliding with either protein D . Out of 70 collisions with QD EcoRI 11.2 5 70 stalled the translocase 11.4 8 70 resulted in immediate dissociation of EcoRIwith no detectable sliding 81.4 57 70 of EcoRIwas pushed along DNA and 92 of pushed molecules were eventually ejected . Out of 30 collisions with LacI 3.3 1 30 stalled the translocase 93.3 28 30 resulted in immediate dissociation of LacI with no detectable sliding and 3.3 1 30 exhibited sliding before dissociation . A greater fraction of LacI might slide but if so the sliding events fall below our resolution limits. Control experiments confirmed RecBCD disrupted EcoRIlabeled with fluorescent beads or Alexa Fluor 488 . As with RNAP RecBCD could strip EcoRIafter Chi and RecBCD also disrupted EcoRIand LacI during low velocity collisions. These findings confirm that RecBCD readily displaces tightly bound proteins from DNA.

In eukaryotes nucleosomes are the most frequently encountered DNA bound obstacles. Replisomes transcription machinery and ATP dependent chromatin remodelers all act through mechanisms requiring force generation and the response of nucleosomes to these forces remains a long standing question in chromatin biology. Heterologous systems have revealed fundamental principles underlying these processes experiments with SP6 RNAP provided a theoretical framework for nucleosome repositioning and studies with phage T4 proteins were among the first to address the fate of nucleosomes during replication. Eukaryotic translocases exert forces in the same net direction as RecBCD and RecBCD can unwind nucleosome bound DNA arguing that it can serve as a good protein based force probe for studying the fate of nucleosomes when rammed by a translocase.

Recombinant nucleosomes were deposited on DNA curtains by salt dialysis as deseribed. Remarkably RecBCD could push nucleosomes 7 311 5 373 bp N 75 and similar results were obtained with fluorescently labeled H2A H2B or H3 H4 . Control experiments demonstrated that RecBCD could also push nucleosomes labeled with either fluorescent beads or Alexa Fluor 488 . Out of 357 collisions with QD nucleosomes 24 84 357 immediately stalled RecBCD 11 40 357 resulted in direct nucleosome ejection 65 233 357 led to sliding and 50 of these were eventually ejected t 3.93 0.21 sec . Nucleosomes reduced the processivity of RecBCD to 14 000 7 000 bp as anticipated and a larger fraction of these collisions caused the translocase to stall 24 compared to RNAP 15 stall EcoRI 7 stall and LacI 3.3 stall . Fewer of the pushed nucleosomes 50 were subsequently ejected from the DNA compared to the other roadblock proteins and there was a 10 reduction t test p 0.0005 in velocity while pushing nucleosomes . These results demonstrate that intact nucleosomes can be pushed along DNA as theoretically predicted but indicated that RecBCD had more difficulty pushing and evicting nucleosomes compared to the other protein roadblocks. The finding that RecBCD pushes and evicts nucleosomes also rules out mechanisms requiring species specific protein protein interactions.

Protein disruption mechanisms can be described by at least four models which differ in the nature of mobile intermediates and the stage of the chemomechanical cycle during which the proteins dissociate . For the first model passive release the proteins S are dislodged from a high affinity specific site and then pushed from one sequential nonspecific site to the next. Subsequent dissociation occurs spontaneously simply because the proteins are bound to lower affinity nonspecific DNA N . This model assumes the proteins have similar low affinities for all nonspecific sites sampled and predicts that the observed rates of RecBCD induced dissociation k would be similar to spontaneous dissociation from nonspecific DNA in the absence of RecBCD k k . This model also predicts that the distance d over which proteins are pushed will be dictated by their affinity for nonspecific DNA and will be proportional to velocity V such that faster translocation will lead to longer distances and slower translocation will yield shorter distances.

The second model preferred site release accounts for a scenario where proteins encounter rare sequences of exceptionally low affinity N such that they preferentially dissociate from these sites k k .

In the third model structural disruption translocase collisions alter the conformation of the proteins e.g. by permanently rupturing a subset of protein DNA contacts such that they persist as structurally perturbed complexes X after displacement from the high affinity site. In this scenario the mobile intermediates have a characteristic lifetime dictated by their weakened affinity for DNA and this lifetime should be insensitive to translocation velocity. Therefore the distance d over which proteins are pushed will be proportional to velocity V and faster translocation will lead to longer distances whereas slower translocation will yield shorter distances. The most important feature of this model which distinguishes it from all of the other models is that the structurally disrupted proteins are more weakly bound to DNA specifically as a consequence of the collision such that the observed rate of RecBCD induced dissociation k would be greater than the rate of spontaneous dissociation from nonspecific DNA k k k .

The fourth model transition state ejection is characterized by a series of tightly bound nonspecific complexes N that must pass through a weakly bound transition state T as they are pushed from one position to the next. This model predicts that dissociation occurs predominantly during the transition state k k . The time required to pass through the transition state during one round of the chemomechanical cycle is equivalent to the time required for the translocase to take a single step k which is a fixed intrinsic value independent of ATP concentration. This relationship can be rationalized by considering that the velocity of RecBCD can be controlled by modulating ATP concentration see below with slower velocities manifesting from longer dwell times between steps while awaiting new ATP rather than from changes in k. Therefore the time it takes the roadblock to pass through the transition state during a single step will be independent of ATP concentration whereas the cumulative time spent in the transition state will increase linearly with step number n irrespective of the overall observed translocation velocity. The probability of dissociation will then increase with step number the observed lifetimes will be inversely proportional to velocity and the total distance the proteins are pushed before dissociation will be independent of velocity i.e. the roadblocks will be pushed similar distances regardless of how fast the translocase moves .

Each aforementioned model makes distinct predictions that can be experimentally evaluated. This evaluation is made easier for RNAP EcoRI and nucleosomes because these proteins were pushed long distances LacI is considered separately below . We first measured dissociation of these proteins from specific and nonspecific sites in the absence of RecBCD Supplementary Information and compared these results to RecBCD induced rates of dissociation . RNAP EcoRI and nucleosomes all bind tightly to nonspecific DNA and RecBCD induced dissociation was 200 fold faster than spontaneous dissociation from nonspecific sites which is inconsistent with passive release. We next analyzed pushing trajectories to determine if there was any evidence supporting preferred site release. Comparison of these trajectories revealed that RecBCD induced dissociation of all three roadblock proteins occurred at random locations arguing against preferred site release. To distinguish between structural disruption and transition state eviction we compared protein lifetimes and pushing distances at four different translocation velocities . Remarkably a 3.3 fold decrease in RecBCD velocity 446 192 bp secat 100 M ATP led to a 1.5 7.0 and 3.4 fold increase in the post collision half life of EcoRI RNAP and nucleosomes respectively while the distribution of distances over which the proteins were pushed remained largely unaltered Table 1 .

Half lives are reported in seconds sec and pushing distances are reported in kilobases kb at each of the four different ATP concentrations tested. See for a graphical representation of the data.

This effect was even more obvious at 15 M ATP where a 19 fold decrease in RecBCD velocity 78 27 bp sec led to a 36 93 and 24 fold increase in post collision half life of EcoRI RNAP and nucleosomes respectively while pushing distances were either unaltered or increased compared to the faster velocities. These results indicated that dissociation was dictated by the number of steps the proteins were forced to take rather than the cumulative time it took to be pushed a given distance which is most consistent with transition state ejection. While our experiments did not reveal any evidence for a structural disruption mechanism of eviction this does not rule out the possibility that EcoRI RNAP and nucleosomes are structurally altered when acted upon by RecBCD. However if they are structurally perturbed this alone does not result in their eventual dissociation from DNA.

LacI differs from the other roadblocks in that it was immediately evicted from DNA and the RecBCD induced dissociation rate was comparable to the rate of spontaneous dissociation from nonspecific sites which would seem consistent with a passive release model. However with current resolution limits we cannot completely rule out other mechanisms and future studies will be necessary to fully address this issue. Importantly RNAP EcoRI and nucleosomes all bind tightly to nonspecific DNA whereas LacI binds much more weakly to nonspecific sequences suggesting that LacI is released more rapidly from DNA after the collisions due to its weaker affinity for nonspecific sites. This result demonstrates that the roadblock proteins and the nature of their interactions with nonspecific DNA are critical contributing factors to the outcome of the collisions.

This leaves the question of how much force RecBCD exerts and how much is sufficient to disrupt obstacles. While our experiments do not yield a direct read out of force we can safely conclude that the force exerted by RecBCD is sufficient to overcome RNAP EcoRI LacI and nucleosomes. Our work has revealed unprecedented details of protein collisions on DNA and provides new insights into how translocases can disrupt nucleoprotein complexes. Given the flexibility of our experimental platform we anticipate these studies can be extended to other translocases and roadblock proteins and it will be important to determine whether the mechanistic concepts developed here apply to different types of collisions between proteins on DNA

We conducted TIRFM experiments on a home built microscope using nanofabricated DNA curtains as previously described. For all initial experiments and for all kymograms shown in the manuscript we used YOYO1 to stain the DNA. YOYO1 does not affect the translocation rate or processivity of RecBCD nor did it affect the binding distributions of RNAP EcoRI or nucleosomes. In the presence of YOYO1 the roadblocks showed the same general response to collisions with RecBCD with identical distributions of ejection stalling and pushing and pushing velocities seen YOYO1. However YOYO1 reduced the distance obstacles were pushed by 20 30 compared to minus YOYO1 reactions. Therefore all sliding distances and half lives reported in the manuscript correspond to values measured in the absence of YOYO1. Sliding distances are only reported for roadblocks that did not encounter any other QD tagged proteins as they were pushed along the DNA. This ensures that each analyzed collision dissociation event only involved a single QD tagged protein. Many reactions were observed in which multiple QD tagged roadblocks were pushed into one another but in these cases we could not determine the order in which each QD protein was displaced from the DNA and therefore could not measure sliding distances. For categorizing the event type distributions we defined sliding as any QD tagged roadblock that moved more than 0.53 m 1 950 bp and anything less than this was scored as a direct dissociation event.

RecBCD was purified from JM109 co transformed with plasmids pPB800 and pPB520 which were a generous gift from Dr. Jeff Gelles Brandeis University Cells were grown in 2YT in the presence of 34 g mlchloramphenicol and 50 g mlcarbenicillin. The cells were grown to an OD 0.6 induced with IPTG collected 4 hours after induction resuspended in buffer R 50 mM Tris HCl pH 7.5 0.1 mM PMSF 10 sucrose and lysed by freezing and sonication. The lysate was clarified by high speed centrifugation and fractionated with 0.282 g mlammonium sulfate. The precipitated protein was recovered by centrifugation at 20 000 g for 20 mM. The pellet was resuspended in buffer A 20 mM Tris HCl pH 7.5 0.1 mM DTT 0.1 mM EDTA and loaded onto a 5 ml HiTrap Q FF column GE Healthcare . The protein was eluted with a gradient to 100 buffer B 20 mM Tris HCl pH 7.5 0.1 mM DTT 0.1 mM EDTA 1 M NaCl . The protein containing fraction were diluted with buffer A to a final NaCl concentration of less than 0.1 M and loaded onto a HiPrep Heparin 16 10 FF column GE Healthcare . The protein was eluted with a gradient to 50 buffer B loaded directly onto a Mono Q 5 50 GL column and eluted with a gradient to 100 buffer B. The pooled protein fractions were loaded onto a HiPrep 16 60 Sephacryl S 300 HR column GE Healthcare . Following purification the protein was dialyzed into storage buffer 50 mM Tris HCl pH 7.5 0.1 M NaCl 0.1 mM EDTA 1 mM DTT 50 glycerol and stored at 20 C. Care was taken to minimize exposure of the protein to high NaCl concentrations to avoid dissociation of the RecD subunit. Protein concentration was determined using 4 10Mcm. An 8 SDS PAGE gel stained with SafeStain Invitrogen indicated that the preparation was 95 pure with a 1 1 1 ratio for the three protein components. Exonuclease and DNA dependent ATPase activities were assayed for the wild type protein see below .

Wild type EcoRI was purchased from New England Biolabs. Constructs encoding the hydrolytically defective EcoRIwere also used. For overexpression and purification the gene for EcoRIwas sub cloned into the pTXB3 vector New England Biolabs generating vector pTXBERI. To prepare EcoRIwith a triple FLAG epitope tag repeat the EcoRIgene was PCR amplified with oligos IF01 and IF02 see Table 2 .

The PCR amplicon was digested with SapI and NcoI New England Biolabs and ligated into pTXB3 to generate plasmid pTXBERI 3FL. The plasmid was transformed into HMS174 DE3 pLysS cells. The cells were grown to an OD 0.6 in the presence of 34 g mlchloramphenicol and 50 g mlcarbenicillin induced with IPTG collected 4 hours after induction resuspended in buffer R and lysed by freezing and sonication. The lysate was clarified by high speed centrifugation and loaded onto a chitin binding domain column according to the manufacturer suggested protocol New England Biolabs . The column was washed with 20 column volumes of washing buffer W 20 mM Tris HCl pH 8.5 and 0.5 M NaCl . The intein CBD tag was cleaved by flushing the column with buffer W supplemented with 50 mM DTT and incubating overnight at 4 C. Protein containing fractions were pooled dialyzed into storage buffer 40 mM Tris HCl pH 7.5 300 mM NaCl 10 mM 2 Mercaptoethanol 0.1 mM EDTA 50 Glycerol 0.15 Triton X 100 and stored at 20 C.

Wild type RNA polymerase holo and core enzymes were purchased from Epicentre Biotechnologies which were used in bulk experiments as controls. The single molecule experiments used either FLAG tagged RNAP or biotinylated RNAP. RNA polymerase containing an N terminal 6 His SEQ ID NO 12 and C terminal triple FLAG tagged a subunit were prepared from constructs described in. Plasmid p706awas digested with BsiWI and XhoI and ligated with oligos 1F03 and 1F04 see Table 2 . The resulting plasmid p706a 3FL was transformed into HMS174 DE3 pLysS cells. The 6His tagged 6His disclosed as SEQ ID NO 12 RNA polymerase was purified as described previously. Prior to separation of the holo and core enzymes on a MonoQ column an additional HisTrap column was used to separate the FLAG tagged from wild type proteins. Experiments employing a biotinylated RNA polymerase were purified from a construct described in. Cells harboring a chromosomal copy of the RNA polymerase with an in vivo biotynilation peptide on the C terminus of the subunit were purified as describedand the core and holoenzyme fractions were separated on a MonoQ column. Protein activity for all RNAP constructs was assayed by in vitro transcription runoff assays see below .

Nucleosomes where prepared as described. Histones H2A H2B H3 H4 and 3 FLAG H3 were expressed in purified from inclusion bodies and reconstituted as described. In brief inclusion bodies were resuspended in unfolding buffer 7 M guanidinium HCl 1 M NaCl 50 mM Tris HCl pH 7.8 1 mM EDTA 1 mM DTT dialyzed against urea buffer 7 M urea 10 mM Tris HCl pH 7.8 1 mM EDTA 5 mM mercaptoethanol and 100 mM NaCl for H2A H2B 3 FLAG H3 or 200 mM NaCl for H3 H4 then loaded onto tandem HiTrap Q and SP columns GE Healthcare . 3 FLAG H2B was expressed in precipitated from cell lysate in 50 NHSO dialyzed against urea buffer and 100 mM NaCl and loaded onto HiTrap SP column GE Healthcare . Histones were eluted from the SP column with a 100 400 mM NaCl gradient for H2A H2B and 3 FLAG H2B and 3 FLAG H3 and a 200 500 mM NaCl gradient for H3 and H4. Purified histones were dialyzed against 10 mM Tris HCl pH 7.8 plus 5 mM mercaptoethanol followed by 10 mM Tris HCl pH 7.8 then lyophilized and stored at 20 C. Lyophilized histones were unfolded in 7 M guanidinium HCl 50 mM Tris HCl pH 7.8 plus 10 mM DTT combined at equimolar ratios and dialyzed into 2 M NaCl 20 mM Tris HCl pH 7.8 1 mM EDTA 5 mM mercaptoethanol with several buffer changes over 48 hours. Reconstituted octamers were purified by gel filtration and deposited onto DNA by salt dialysis.

Wild type LacI was PCR amplified for plasmid pTYB21 New England Bioloabs using oligos IF11 and IF12 and sub cloned into pTXB3 generating pTXLACI. A FLAGepitope tag was introduced at the C terminus by ligating oligos encoding FLAG repeats at the C terminal ClaI restriction site to generated pTXLACI 6FL. Plasmids encoding either wt or FLAG labeled LacI were transformed into BL21 DE3 cells grown to OD 0.6 in the presence of 50 mlcarbenicillin and induced with 2.5 mM IPTG. After a three hour induction the cells were harvested and purified using the same protocol as for EcoRI see above .

RecBCD activity was verified using ATPase reactions under the same buffer conditions as the single molecule flowcell assay. Reaction mixtures containing 1.5 nM in molecules 3 nM in DNA ends DNA New England Biolabs 0.1 M P ATP 1 mM ATP 40 mM Tris HCl pH 8 2 mM MgCl 0.2 g lBSA and 1 mM DTT. For the time courses reactions contained 0.1 nM RecBCD and were terminated with the addition of EDTA to a final concentration of 80 mM after incubations for the indicated periods . Reaction products were resolved by thin layer chromatography on PEI cellulose Sigma Aldrich run in 0.5 M formic acid with 0.5 M LiCl. Products were detected with a PhosphorImager and analysis was done with ImageQuant 5.2 software.

Exonuclease assays were carried out essentially as described previouslyand confirmed the proteins were fully active. Under reaction conditions were ATP 

Reactions containing 50 mM Tris HCl pH 8.5 10 mM MgCl 40 M ATP 40 M dsDNA in nucleotides 0.67 mM DTT and 0.2 g lBSA were pre incubated at 37 C. for several minutes. Reactions were initiated by addition of RecBCD in reaction buffer minus DNA and ATP to a final concentration of 0.1 nM. At the indicated time points 20 L aliquots were withdrawn and quenched by adding 100 L of 10 ice cold TCA and 5 L of 0.5 mg mlsheared salmon sperm DNA Ambion . The quenched aliquots were kept on ice for at least ten minutes and undigested DNA collected as an invisible pellet by centrifugation at 14 000 rcf for 10 minutes at 4 C. Following centrifugation 110 L of the supernatant was added to 5 mL of scintillation counting fluid Fisher Scientific and quantified by liquid scintillation counting. All measurements were performed at least three times. Control experiments containing all components except ATP indicated no nuclease activity.

RNA polymerase run off transcription assays were performed to ensure that the introduction of an epitope tag and QD labeling did not adversely affect enzymatic activity . A 467 bp segment of DNA containing the Ppromoter was amplified by PCR using cloned Pfu DNA polymerase Stratagene and primers IF09 and IF10. The amplicon was purified with a QIAGEN PCR purification kit. Transcription assays were carried out in 25 l reactions containing transcription buffer 40 mM Tris HCl pH 8.0 150 mM KCl 10 mM MgCl 250 M each rNTP supplemented with 0.5 Ci P ATP 40 U SUPERase In RNAse inhibitor Ambion 40 nM dsDNA template and 20 nM holo RNA polymerase. When indicated QDs were added to a final concentration of 40 nM. The reaction was incubated for 1 hr at 37 C. Reactions were centrifuged through a NucAway desalting column Ambion and mixed with 2 RNA loading dye solution Fermentas . Transcription products were resolved on 6 polyacrylamide urea gels Invitrogen . Gels were run at room temperature for 1 hr constant voltage 175V in 1 TBE running buffer. The expected transcription productswere detected with a Typhoon Phosphorlmager GE and analysis was done with ImageQuant 5.2 software.

Single turnover transcription experiments were performed on stalled RNAP elongation complexes to determine the percent of stalled polymerases that can re initiate transcription. Stalled complexes were prepared by incubating 20 nM holo RNAP with 40 nM dsDNA template in transcription buffer supplemented with 40 U SUPERase In RNAse inhibitor 150 M ApU RiboMed 0.5 Ci P ATP and 25 M each of rATP rCTP rGTP. The reaction was incubated for ten minutes at 37 C. Half of the reaction was quenched with EDTA to 25 mM and kept on ice. The second half reaction was rapidly supplemented with 0.1 mg ml heparin and 250 M each of rATP rCTP rGTP rUTP. After a five minute incubation at 37 C. transcription products were resolved on an 18 polyacrylamide urea gel Invitrogen . Gels were run at room temperature for 1 45 hr constant voltage 175V in 1 TBE running buffer and analyzed as above. All transcription reactions were repeated at least three times.

EcoRI digestion protection assays were performed essentially as described. DNA 1.5 nM five EcoRI sites was incubated with 30 nM EcoRI concentration in dimers in binding buffer 25 mM Tris Cl pH 8.0 150 mM NaCl 1 mM DTT 0.2 g lBSA 10 mM MgCl for 30 minutes on ice followed by 2 minutes at 37 C. Where indicated QDots were included as a two fold excess over EcoRIdimers. DNA EcoRIcomplexes were challenged with 10 U lwtEcoRI New England Biolabs for 5 60 minutes at 37 C. Reactions were stopped with 20 mM EDTA 0.5 SDS and deproteinized with 1 mg mlProteinase K for 30 minutes at 50 C. The products were run at 80V on a 0.6 room temperature agarose gel stained with ethidium bromide and imaged on a UV trans illuminator SynGene . Time courses were in agreement with previously published results. All assays were repeated at least three times.

Most experiments used bacteriophage 48 502 bp lacking a Chi sequence. To determine whether RecBCD could strip obstacles when RecB was the lead motor a triple Chi sequencewas cloned into bacteriophage . In vitro RecBCD recognizes each Chi sequence with a 30 probabilityand a triple Chi sequence repeat was previously shown to be recognized by 90 of translocating RecBCD molecules. Phage DNA harboring the cI857 and S7 mutations New England Biolabs was digested with XhoI and NheI and ligated with oligos IF05 and IF06 Table S2 which abolish both cut sites. The ligated product was digested again with XhoI and NheI prior to phage DNA packaging according to the manufacturer supplied protocol EpiCentre . Individual phage plaques were screened for the insert by PCR and confirmed by DNA sequencing. Phage particles were isolated from heat inducible lysogens and the DNA containing the Chi locus was purified with a commercial phage DNA extraction kit Qiagen . The final DNA construct containing Chi was 47 321 bp.

For experiments with LacI RecBCD collisions a single high affinity ideal LacO sequencewas introduced into phage by ligating the complimentary oligonucleotides IF13 and IF14 between XhoI and NheI restriction sites as described above. The final DNA construct containing LacO was 47 331 bp.

Phage DNA with a biotinylated cos end was prepared according to previously published protocols. Briefly the cos end was annealed with oligos IF07 or IF08 Table 2 ligated and filtered over an Sephacryl S 200 HR column GE Healthcare to remove excess oligonucleotide and ligation reaction components.

Streptavidin quantum dots QDs were purchased from Invitrogen. To prepare antibody labeled QDs amine functionalized QDs were labeled with affinity purified reduced anti FLAG antibodies Sigma Aldrich using SMCC succinimidyl 4 N maleimidomethyl cyclohexane 1 carboxylate . The resulting QD antibody conjugates were then purified over a Superdex 200 10 300 GL gel filtration column GE Healthcare and were stored in PBS pH 7.4 at 4 C. Although the manufacturer reported antibody QD ratio is 4 1 the percentage of QDs with an active antibody may be substantially lower. There may be dark proteins that are not coupled to QDs or QDs that are not fluorescent. The presence of dark QDs or unlabeled proteins would increase the total number of RecBCD roadblock collisions but those involving dark roadblocks would not be observed. The possible presence of dark roadblocks does not affect any of our conclusions.

A complete description of the DNA curtains made with nanofabricated diffusion barriers containing geometric nanowells can be found in Visnapuu et al. 2008. In brief fused silica slides G. Finkenbeiner Inc. were cleaned in NanoStrip solution CyanTek Corp Fremont Calif. for 20 minutes rinsed with acetone and isopropanol and dried with N. Slides were spin coated with a bilayer of polymethylmethacrylate PMMA 25K and 495K MicroChem Newton Mass. followed by a layer of Aquasave Mitsubishi Rayon . Patterns were written with a FEI Sirion scanning electron microscope J. C. Nabity Inc. Bozeman Mont. . Aquasave was removed with deionized water and resist was developed using isopropanol methyl isobutyl ketone 3 1 for 1 minute with ultrasonic agitation at 5 C. The substrate was rinsed in isopropanol and dried with N. Barriers were made with a 15 20 nm layer of chromium Cr and following liftoff samples were rinsed with acetone and dried with N as described.

Inlet and outlet ports were made by boring through the slide with a precision drill press equipped with a diamond tipped bit 1.4 mm O.D. Kassoy . The slides were cleaned by successive immersion in 2 v v Hellmanex 1 M NaOH and 100 MeOH. Slides were rinsed with MilliQ between each wash and stored in 100 MeOH until use. Prior to assembly slides were dried under a stream of nitrogen and baked in a vacuum oven for at least 1 hour. A sample chamber was prepared from a borosilicate glass coverslip Fisher Scientific and double sided tape 100 m thick 3M . Ports Upchurch Scientific were attached with hot melt adhesive SureBonder glue sticks FPC Corp. . The total volume of the sample chambers was 14 l. A syringe pump Kd Scientific and actuated injection valves Upchurch Scientific were used to control sample delivery. The flowcell and prism were mounted in a custom built heater with computer controlled feedback regulation.

Lipids were purchased from Avanti Polar Lipids and liposomes were prepared as previously described. In brief a mixture of DOPC 1 2 dioleoyl sn glycero phosphocholine 0.5 biotinylated DPPE 1 2 dipalmitoyl sn glycero 3 phosphoethanolamine N cap biotinyl and 8 mPEG 2000 DOPE 1 2 dioleoyl sn glycero 3 phosphoethanolamine N methoxy polyethylene glycol 2000 . The mPEG prevented nonspecific adsorption of QDs. Liposomes were applied to the sample chamber for 30 minutes. Excess liposomes were removed with buffer containing 10 mM Tris HCl pH 7.8 and 100 mM NaCl. The sample chamber was then flushed with buffer A 40 mM Tris HCl pH 7.8 1 mM DTT and 1 mM MgCl plus 0.2 mg mlBSA for 5 minutes. Streptavidin 0.02 mg ml in buffer A was injected into the sample chamber and incubated for 20 minutes. After rinsing with additional buffer A plus 0.2 mg mlBSA DNA 15 20 pM labeled at one end with biotin and pre stained with 0.5 nM YOYO1 was injected into the chamber incubated for 10 minutes and unbound DNA was removed by flushing with buffer at 0.1 ml min. Application of flow aligned the DNA molecules along the diffusion barriers pushed them into the nanowells and stretched the molecules parallel to the sample chamber surface. We could readily distinguish nanowells harboring 1 DNA from those containing 2 or more molecules based on YOYO1 signal intensity and RecBCD translocation data were only collected from nanowells containing single full length DNA molecules see below .

DNA curtains without any roadblock proteins were assembled on nanofabricated silica slides as described herein. See also Fazio et al. Langmuir. 2008 Sep. 16 24 18 10524 31 . RecBCD was diluted to 20 nM in imaging IM buffer 40 mM Tris HCl pH 8.0 2 mM MgCl 1 mM DTT 0.5 nM YOYO1 Invitrogen 50 mM mercaptoethanol 1.4 mM glucose glucose oxidase and catalase and injected slowly 70 l min into the flowcell over the course of several minutes. After excess RecBCD was washed out the flow rate was increased to 400 l min and data acquisition was initiated. At this flow rate the DNA was approximately 80 extended relative to its full contour length see below . A hundred frames were acquired prior to switching to digestion buffer IM buffer supplemented with 1 mM ATP unless otherwise indicated . Upon ATP injection RecBCD begins to unwind and nucleolytically degrade dsDNA leading to a reduction in DNA length and concomitant ejection of YOYO1 and the time dependent DNA length is used as a readout of RecBCD translocation see below . All experiments were conducted at 37 C.

DNA curtains with stationary holo or core RNA polymerase enzymes were prepared using biotinylated RNAP. The biotinylated polymerase was incubated with a five fold molar excess of Streptavidin conjugated QDs on ice for 15 minutes. After labeling the reaction was diluted to 1 nM RNA polymerase 5 nM QDs with RNAP Buffer 40 mM Tris pH 8.0 100 mM KCl 2 mM MgCl 1 mM DTT 0.2 mg mlBSA supplemented with 10 M free biotin to block remaining streptavidin sites on the quantum dots. Prior to QD RNAP injection remaining biotin binding sites on the flowcell surface immobilized streptavidin were blocked by incubating the flowcell in imaging buffer containing 10 M free biotin. Control experiments indicated that all free flowcell streptavidin sites were blocked by biotin. QD RNA polymerase was injected into the flowcell at 70 l minin RNAP buffer over the course of several minutes. Inclusion of free biotin in the flow buffer further prevented potential interactions between the biotinylated enzyme and lipid bound streptavidin. After all free RNA polymerase and biotin were flushed out of the sample chamber flow was switched to IM buffer for further RecBCD experiments. Reactions with FLAG tagged RNAP were carried out by pre incubating 20 l of 150 pM DNA with 1 l of 100 nM FLAG RNAP in RNAP buffer for 10 minutes at 37 C. The reaction was diluted to a total volume of 1 ml with RNAP buffer injected into the flowcell and incubated 5 minutes at room temperature allowing the biotinylated ends to attach to the streptavidin on the lipid bilayer surface. Excess DNA and RNAP were flushed out and the FLAG tagged proteins were labeled in situ by flushing 700 l of 1 nM anti FLAG QDs through the flowcell in IM buffer. The use of FLAG tagged RNAP eliminated the need to block the sample chamber surface with free biotin.

Experiments involving transcribing RNA polymerase molecules were carried out using the FLAG tagged construct. Stalled elongation complexes were prepared on DNA as described. Briefly holo RNA polymerase was diluted to 15 nM in 20 l of Buffer T 10 mM Tris pH 8.0 100 mM KCl 10 mM MgCl 1 mM DTT supplemented with 150 pM DNA 150 M ApU 25 M rATP 25 M rGTP and 25 M rCTP for 10 minutes. After incubation the reaction was diluted to a total volume of 1 ml with RNAP buffer supplemented with 0.2 mg mlheparin injected into the flowcell and incubated 5 minutes at room temperature. Heparin effectively competes for DNA binding dissociating all RNAP complexes that are bound at nonspecific DNA sites see . Free proteins and rNTPs were washed out and the DNA bound RNA polymerases were labeled in situ with anti FLAG QDs as described above. Active elongation complexes were prepared from stalled ECs by injecting buffer T supplemented with 250 M CTP GTP UTP and 1 mM ATP to maintain consistent RecBCD velocity .

For DNA curtain assays using EcoRI 20 l of 150 pM DNA 750 pM EcoRI sites was incubated with 1 l of 100 nM EcoRI concentration in dimers in EcoRI Buffer 10 mM Tris HCl pH 8.0 150 mM NaCl 10 mM MgCl 0.2 mg mlBSA for 30 minutes on ice followed by 5 minutes at 37 C. The reaction was diluted to a total volume of 1 ml with EcoRI buffer injected into the flowcell and incubated 5 minutes at room temperature allowing the biotinylated DNA ends to attach to the streptavidin on the lipid bilayer surface. Excess DNA and EcoRIwere flushed out and the FLAG tagged proteins were labeled in situ as described above.

DNA curtains with specifically bound LacI were constructed as for EcoRIabove. DNA containing LacO 20 l of 150 pM DNA was incubated with 1 l of 50 nM LacI concentration in dimers in LacI Buffer 10 mM Tris HCl pH 8.0 150 mM NaCl 1 mM EDTA 0.2 mg mlBSA for 5 minutes at 37 C. The reaction was diluted to a total volume of 1 ml with LacI buffer. After the DNA curtain was constructed and excess protein and DNA flushed out of the flowcell LacI was labeled with antiFLAG QDs in situ.

Control experiments with LacI incubated with wild type DNA lacking a LacO sequence indicated that the enzyme bound the non specific DNA transiently and rapidly dissociated from the DNA in the presence of buffer flow. Lifetimes of non specific LacI DNA complexes were estimated by incubating FLAG LacI with a five fold excess of antiFlag QDs and injecting the enzyme into flow cells with pre formed wild type DNA curtains. The reaction buffer flow rate and illumination conditions were identical to those used for RecBCD digest experiments. Under these conditions we observed a number of transient and DNA specific LacI binding events N 445 that lasted 0.2 5 seconds. A histogram of 445 LacI DNA association events data not shown was fit with an exponential lifetime of 0.24 seconds but since the majority of observed events lasted one frame 0.2 seconds we report an upper bound of 300 msec for the non specific half life.

Nucleosome bound DNA curtains were constructed as described previouslyand the nucleosomes were labeled in situ by flowing 700 l of 2 nM antiFLAG QDs through the flowcell in IM buffer. Nucleosome collision experiments were conducted as described above for RNAP EcoRIand LacI.

The custom built total internal reflection microscope used in this study has been described previouslyStreams of 1000 3000 images were acquired at 5.0 Hz using a 200 millisecond integration time. For RecBCD experiments at lower ATP concentrations and roadblock dissociation measurements the laser beam was shuttered for up to two seconds between individual 200 ms exposures. All data were collected using NIS Elements software Nikon and saved as uncompressed 16 bit TIFF files. A Dual View image splitting device Optical Insights with a dichroic mirror 630 DCXR Chroma Technologies was used for two color detection. This set up allowed us to simultaneously image the 705 nm QD protein complexes and YOYO stained DNA 509 nm on separate halves of the same CCD chip. Alignment of red and green channels was performed during post processing ImageJ software with Align RGB Planes plug in using dark signal from the nanofabricated barriers as a reference. Alignment was confirmed by observing fluorescent signals from surface imperfections that were occasionally observed in both channels. Aligned images were pseudo colored and digitally recombined in ImageJ.

DNA length tracking for experiments involving RecBCD collisions with RNAP or EcoRIwere performed using an automated algorithm MATLAB that iteratively fit the intensity profiles of individual YOYO stained DNA molecules according to equation 1 

In experiments studying RecBCD collisions with nucleosomes DNA length tracking was performed using a custom algorithm IgorPro that iteratively fit the images of individual YOYO stained DNA molecules according to the equation 2 

For both cases the conversion factor from pixels to base pairs for each flowcell was obtained by averaging the observed length of at least five full length undigested DNA molecules and dividing by the known number of base pairs. This conversion factor was computed for every flowcell prior to injection of RecBCD under the same imaging conditions. We restricted data analysis to individual DNA molecules that are full length at the start of the experiment and are clearly separated from neighboring DNA. The resolution of the DNA tracking algorithm was estimated by tracking several hundred frames of full length DNA molecules under the same conditions as used for single molecule RecBCD digestion experiments. The fluctuations in the measured DNA length set a resolution of 1 000 by for full length DNA.

QD tagged proteins were tracked as described previously using an automated algorithm MATLAB or IgorPro . Briefly the x and y positions of the point spread function were fit to a 2D Gaussian function in conjunction with a region of interest mask. Both the x coordinates i.e. perpendicular to the long axis of the DNA and y coordinates i.e. parallel to the long axis of the DNA molecules were recorded for each tracked particle. All RecBCD velocity histograms were fit to a Gaussian function using Origin Pro OriginLabs Inc. and the reported values for the means and standard deviations were determined directly from the Gaussian fits to the data. Roadblock dissociation histograms were fit to a single exponential decay in Origin Pro and the reported values are for the half life and standard fit error.

For all initial experiments and for all kymograms shown in the manuscript YOYO1 was used to stain the DNA and locate the curtains. YOYO1 did not affect the translocation rate or processivity of RecBCD and did not affect the binding distributions of RNAP EcoRI or nucleosomes. In the presence of YOYO1 the roadblocks showed the same general response to collisions with RecBCD as seen in the absence of YOYO1. However when YOYO1 was present the overall distance that the obstacles were pushed was reduced compared to minus YOYO1 reactions yielding values of 6720 4800 6670 4390 and 6265 3555 base pairs for RNAP EcoRI and nucleosomes respectively. The reduction in sliding distance and the similarity of the three values suggests that the YOYO1 was inducing DNA damage possible nicks or otherwise perturbing the nucleoprotein complexes and that the distances over which the obstacles were pushed were limited by the presence of the YOYO1 dye. Therefore all sliding distances and half lives reported in the main manuscript correspond to values that were measured in the absence of any YOYO1.

Sliding distances were only reported for QD tagged roadblocks that were pushed along the DNA and did not encounter any other QD tagged proteins as they moved along the DNA. This ensures that each analyzed collision dissociation event only involved a single QD tagged protein. Many reactions were observed in which multiple QD tagged roadblocks were pushed into one another but in these cases we could not definitely determine the order in which each different QD protein was eventually displaced from the DNA and therefore could not measure sliding distances. Finally for categorizing the event type distributions we defined sliding as any QD tagged roadblock that moved more than 0.53 m approximately 1 950 bp and anything less than this was scored as a direct dissociation event.

Streptavidin labeled 40 nm far red fluorescent TransFluoSpheres 488 645 were purchased from Invitrogen Cat. No. T 10711 . To prepare antibody labeled microspheres 30 L of 66 nM biotinylated monoclonal anti FLAG antibodies Sigma Aldrich in conjugation buffer 40 mM Tris HCl pH 8.0 1 mM MgCl 1 mM DTT 0.2 mg ml BSA were incubated with 30 L of TransFluoSpheres at stock concentration 0.5 solids for 20 minutes on ice. The microsphere antibody complexes were isolated from free antibodies by centrifugation at 16 100 RCF for 20 minutes at 4 C. The flowcell surface was blocked with 0.1 saturated biotin solution C 0.1 mM biotin in conjugation buffer. The pelleted microsphere anti FLAG conjugates were diluted to 100 l of 1M buffer and injected into flowcells that contained DNA curtains with pre bound roadblock proteins as described for the QD labeling experiments. For experiments with biotinylated RNA polymerase the bead enzyme complexes was prepared as described above and injected into flow cells with pre formed DNA at a slow rate 0.05 ml min to facilitate binding of RNAP to the DNA curtain.

The Alexa 488 dye labeled anti FLAG antibodies were prepared according to the manufacturer s protocol Invitrogen Cat. No. A 20181 . In brief amine reactive Alexa Fluor 488 was reacted with monoclonal anti FLAG antibody Sigma Aldrich and purified on a size exclusion spin column. Labeling efficiency was calculated according to the manufacture s recommendation Moles dye per mole protein A494 71 000 M where 71 000 cmMis the molar extinction coefficient of the Alexa Fluor 488 dye at 494 nm and M A280 A494 0.11 203 000 where 203 000 cmMis the molar extinction coefficient of the antibody at 280 nm and 0.11 is a correction factor that accounts for the fluorophore s contribution to the absorbance at 280 nm. This calculation yielded a value of 5 Alexa Fluor 488 dyes per antibody. The fluorescent antibodies were diluted to 5 nM in 1 ml of IM buffer and injected into the flowcell that contained DNA curtains with pre bound roadblock proteins as described for the QD experiments. Single molecule RecBCD collision experiments were conducted with the bead and Alexa labeled roadblock proteins as described above for the QD labeled roadblock proteins.

DNA binding proteins survey genomes for targets using facilitated diffusion which typically includes a one dimensional 1D scanning component for sampling local regions. Eukaryotic proteins must accomplish this task while navigating through chromatin. Yet it is unknown whether nucleosomes disrupt 1D scanning or whether eukaryotic DNA binding factors can circumnavigate nucleosomes without falling off DNA. Here we use single molecule microscopy in conjunction with nanofabricated curtains of DNA to show that the post replicative mismatch repair MMR protein complex Mlh1 Pms1 diffuses in 1D along DNA via a hopping stepping mechanism and readily bypasses nucleosomes. This is the first experimental demonstration that a passively diffusing protein can traverse stationary obstacles. In contrast Msh2 Msh6 an MMR protein complex that slides while maintaining continuous contact with DNA experiences a boundary upon encountering nucleosomes. These differences reveal important mechanistic constraints impacting intranuclear trafficking of DNA binding proteins.

Virtually all DNA binding proteins must use some form of facilitated diffusion e.g. hopping jumping sliding and or intersegmental transfer to scan the genome and locate targets. The advent of single molecule imaging has led to a resurgence of interest in facilitated diffusion and an emerging consensus agrees that many proteins can scan DNA via one dimensional 1D diffusion where the proteins undergo a random walk while moving laterally along the helix. However all of these studies have been limited to naked DNA substrates which do not resemble the crowded environments that would be encountered in vivo leaving the role of 1D diffusion in question under physiologically relevant settings. In eukaryotes these processes must occur within the context of chromatin which has the potential to hinder protein mobility. Motor proteins such as RNA polymerase and other DNA translocases solve this problem by using the chemomechanical energy derived from nucleotide hydrolysis to push their way through nucleosome obstacles. However most DNA binding proteins such as transcription factors or DNA repair proteins cannot mechanically disrupt nucleosomes therefore other mechanisms must come into play if these proteins are to scan chromatin. Whether or not proteins can circumnavigate nucleosomes without dissociating from DNA remains an unresolved issue with direct bearing on how all eukaryotic DNA binding proteins are trafficked throughout the nucleus. This problem led us to ask whether eukaryotic proteins that diffuse in 1D along DNA could circumnavigate individual nucleosomes and travel along nucleosomal arrays and if so what mechanistic principles affect mobility along chromatin.

We have chosen the post replicative mismatch repair MMR protein complexes Msh2 Msh6 and Mlh1 Pms1 as model systems for studying the physical basis of facilitated diffusion. MMR is a ubiquitous repair pathway that corrects errors mismatches and small insertion deletion loops left behind by the replication machine. Defects in MMR lead to elevated mutation rates are linked to hereditary nonpolyposis colon cancer HNPCC and are associated with many sporadic tumors. Msh2 Msh6 and Mlh1 Pms1 are DNA binding proteins required for MMR. During MMR Msh2 Msh6 must locate lesions and also helps identify nearby signals differentiating parental and nascent DNA strands whereas Mlh1 Pms1 must locate lesion bound Msh2 Msh6 and then coordinates downstream steps in the reaction. Although Msh2 Msh6 and Mlh1 Pms1 are both ATPases neither uses ATP for generating chemomechanical force rather nucleotide binding and hydrolysis are thought to serve as signaling mechanisms for coordinating the various stages of repair by regulating protein protein interactions in the case of Mlh1 Pms1 or protein DNA interactions with Msh2 Msh6. These or closely related protein complexes are also involved in mitotic and meiotic recombination triplet repeat expansion class switch recombination somatic hypermutation and DNA damage signaling checkpoints. All known functions of Msh2 Msh6 and Mlh1 Pms1 require targeting to specific structures within the genome and the later stages of the MMR reaction involved in strand discrimination are also thought to involve 1D movement along DNA making these protein complexes good candidates as model systems for single molecule studies of facilitated diffusion.

Using total internal reflection fluorescent microscopy TIRFM we have previously demonstrated that Msh2 Msh6 moves on DNA via a sliding mechanism consistent with a model where it tracks the phosphate backbone. To determine whether Mlh1 Pms1 also moves on DNA the proteins were engineered with epitope tags FLAG and or HA and labeled with antibody coupled quantum dots QDs . Gel shift and nitrocellulose filter binding assays confirmed labeling specificity and demonstrated that labeling did not disrupt DNA binding activity . For TIRFM we used microfluidic devices with hybrid surfaces comprised of fluid lipid bilayers and nanofabricated metallic barrier patterns made by electron beam lithography. The DNA substrates DNA 48 502 bp were anchored by one end to the bilayer through a biotin streptavidin linkage and hydrodynamic force was then used to push the DNA and align it along the leading edges of nanofabricated barriers to lipid diffusion . The second end of the DNA was then anchored to antibody coated pentagons positioned downstream from the linear barriers . This strategy yields double tethered curtains of DNA in which the individual DNA molecules are suspended above a lipid bilayer and anchored by both ends such that they can be viewed across their entire contour length by TIRFM in the absence of a perturbing hydrodynamic flow .

When imaged by TIRFM Mlh1 Pms1 co localized with DNA and 95 of the DNA bound proteins moved rapidly back and forth along the DNA molecules Table 3 .

Two color labeling experiments revealed that most 98.4 of the complexes were single heterodimers under the conditions used for these experiments. Mlh1 Pms1 often remained bound to the DNA for several minutes without dissociating consistent with bulk biochemical studies. Analysis of the motion revealed linear MSD plots as expected for 1D diffusion yielding a mean diffusion coefficient of D 0.143 0.29 msec N 25 at 150 mM NaCl and 1 mM MgCl . 1D diffusion was observed ADP ATP and ATP S and the differences in the diffusion coefficients measured under the different nucleotide conditions were statistically insignificant student t test p 01 . These results indicate that nucleotide binding and hydrolysis were unnecessary for movement consistent with the notion that nucleotide binding is primarily involved in promoting protein protein interactions or structural rearrangements with little impact on DNA binding.

The Mlh1 Pms1 heterodimer is maintained through protein protein interactions between the CTDs and the NTDs and CTDs are separated by very long linker arms . It has been previously hypothesized that this structural organization allows Mlh1 Pms1 and related proteins to adopt a ring like architecture which would enable them to encircle DNA . This type of topological binding mechanism makes several specific experimentally testable predictions as follows i dissociation from DNA should occur preferentially from the free ends of linear DNA molecules ii protein dissociation should be prevented if the DNA ends are sterically occluded iii dissociation should be less prevalent from internal positions and iv an intact heterodimer would be necessary for stable DNA binding activity and for end dependent dissociation. We first asked whether Mlh1 Pms1 preferentially dissociated from DNA ends . When hydrodynamic force 100 fN was used to push Mlh1 Pms1 most complexes 95 did not dissociate upon encountering anchored i.e. sterically blocked DNA ends N N 1 23 dissociated total pushed to DNA ends nor did Mlh1 Pms1 dissociate from the apex of looped DNA N N 0 4 . In contrast Mlh1 Pms1 immediately dissociated from free ends of single tethered DNA N N 880 1000 and from free ends of photo chemically induced double stranded breaks DSB N N 14 14 . Mlh1 alone can exist as monomers or dimers Kof 3.14 0.19 M but the Mlh1 NTDs do not self associate. Mlh1 alone could bind DNA exhibiting a diffusion coefficient 6.9 fold greater p80 dissociated from the anchored DNA ends N N 285 350 this finding was strikingly different from results with the intact heterodimer indicating that the presence of Pms1 was necessary to observe end dependent dissociation. Finally we engineered TEV cleavage sites into the linker of Mlh1 and Pms1 and proteolytic cleavage of one or both linker arms abolished detectable DNA binding activity highlighting the importance of the linker arms for DNA binding. We conclude that formation of an intact Mlh1 Pms1 heterodimer stabilizes the DNA bound complex and that the heterodimer preferentially dissociates from DNA ends. These experimental findings are all consistent with predictions for the previously proposed mechanism where Mlh1 Pms1 can adopt a ring like architecture that wraps around DNA although we are careful note that we do not yet know structural details of the wrapped complex.

We have previously shown that Msh2 Msh6 complexes traveling on the same molecule cannot pass one another arguing that the proteins maintain continuous close contact with the DNA which is consistent with a sliding mechanism. In contrast two color labeling experiments revealed that Mlh1 Pms1 complexes could bypass one another as they traveled along the same DNA molecule which is only consistent with a hopping stepping mechanism wherein the individual hops or steps span distances comparable to or greater than the dimensions of the QD tagged proteins. Closed ring like architecture is difficult to reconcile with the observed protein bypass and would require two Mlh1 Pms1 complexes to thread through one another as they moved along the DNA. A threading mechanism specifically predicts that Mlh1 Pms1 would be unable to bypass obstacles larger than the internal diameter of the large central pore formed the protein complex. Alternatively bypass could also be accomplished through transient ring opening whereupon the proteins could simple step past one another in an open configuration. This type of open stepping mechanism predicts that Mlh1 Pms1 would be capable of bypassing obstacles larger than the internal diameter of the protein ring. Given the combined length of the Mlh1 Pms1 linker arms 51.7 14.6 nm the corresponding maximal diameter of the central pore would be 16.5 4.6 nm in diameter which is too small to accommodate passage of a QD 20 nm dia. ruling out a threading mechanism for obstacle bypass. We conclude that Mlh1 Pms1 most likely bypasses obstacles by stepping over them in an open ring configuration implying that the protein is capable of transitioning back and forth between an open and closed conformation.

The finding that Mlh1 Pms1 complexes could bypass one another suggested that these proteins might be able to undergo 1D diffusion on crowded DNA substrates similar to what would be found in an in vivo environment. Therefore we next asked whether MMR proteins could traverse nucleosomes which are anticipated to be the most abundant obstacles encountered in eukaryotes. For these experiments unlabeled recombinant nucleosomes were deposited onto the DNA substrates by salt dialysis at a ratio of either 5 10 or 80 100 nucleosomes per DNA molecule. The Mlh1 Pms1 diffusion measurements were performed as described above and the nucleosomes were then located by labeling with QDs after the diffusion measurements were completed Supplementary Information . As shown in Mlh1 Pms1 still diffused on nucleosome bound DNA D 0.027 0.021 msec N 26 and repeatedly bypassed unlabeled nucleosomes 10 nm dia. exhibiting no evident boundary effects upon colliding with single nucleosomes upper panel N 1 000 Mlh1 Pms1 complexes each giving rise to multiple bypass events . Mlh1 Pms1 also moved freely along DNA bound by up to 80 100 unlabeled nucleosomes middle panel D 0.034 0.018 msec N 25 providing an unequivocal demonstration that nucleosomes do not prevent 1D diffusion of Mlh1 Pms1. We conclude Mlh1 Pms1 can travel along a simple chromatin lattice by 1D diffusion while bypassing protein obstacles at it travels along the DNA . As indicated above for all of these experiments the nucleosomes were labeled only after making the diffusion measurements to ensure that the large QDs would not interfere with Mlh1 Pms1 movement. However Mlh1 Pms1 could also bypass QD labeled nucleosomes N 63 Mlh1 Pms1 complexes each yielding multiple bypass events although in this case Mlh1 Pms1 exhibited characteristics of bounded diffusion upon colliding with the QD nucleosomes with the large QD nucleosomes acting as semi penetrable barriers lower panel . Given the large diameter of the QD nucleosome 30 nm compared to the size of Mlh1 Pms1 we conclude that nucleosome bypass must occur via a stepping mechanism where the protein transiently adopts an open ring configuration. These results provide the first experimental demonstration that a protein undergoing 1D diffusion can circumnavigate protein obstacles that lie in its path.

In striking contrast to Mlh1 Pms1 the movement of Msh2 Msh6 past unlabeled nucleosomes was highly restricted exhibiting characteristics of bounded diffusion with nucleosomes acting as semi penetrable barriers and Msh2 Msh6 typically became trapped between nucleosomes upper and middle panels . On higher density nucleosome arrays 80 100 nucleosomes per 48.5 kb DNA substrate most molecules of Msh2 Msh6 were immobile or oscillated within tightly confined regions N 964 1000 and displayed little evidence of free 1D diffusion within our detection limits D 1 10 msec . A small subpopulation of Msh2 Msh6 remained mobile on the high density arrays N 36 1000 3.6 suggesting they were bound in an alternate conformation. In further contrast to Mlh1 Pms1 Msh2 Msh6 never bypassed QD tagged nucleosomes lower panel indicating that large obstacles 30 nm dia. present insurmountable barriers which is fully consistent with expectations based on the structure of Msh2 Msh6 which wraps around DNA making intimate contacts with the phosphate backbone over nearly 1.5 turns of helix and is also consistent with a continuous sliding mechanism that does not involve extensive hoppine. Rare nucleosome bypass by Msh2 Msh6 might occur through occasional hopping events or through limited excursions into 2D sliding where Msh2 Msh6 maintains contact with the DNA without tracking the helical pitch of the phosphate backbone H6 H8. In either case the mechanism does not permit efficient mobility of Msh2 Msh6 along the higher density nucleosome arrays this conclusion agrees with bulk biochemical studies showing that nucleosomes or other stationary obstacles can trap Msh2 Msh6 or its homologs on DNA.

Intranuclear trafficking of all DNA binding proteins is governed by facilitated diffusion. Theoretical descriptions and bulk measurements of facilitated diffusion have long been reported in the literature beginning with the classical studies of lac repressor and more recently with NMR experiments of transcription factors but direct measurements of diffusion have only recently become possible through development of new single molecule techniques. Together these studies support an emerging consensus that many DNA binding proteins can travel long distances along DNA by 1D diffusion in vitro. However the validity of this conclusion with respect to physiological settings remains unclear despite years of experimental and theoretical efforts specifically because it remains unknown whether or how 1D diffusion can occur in the presence of nucleosomes and other nucleoprotein structures. Here we sought to resolve this issue by using single molecule imaging nanofabricated curtains of double tethered DNA molecules and MMR proteins as model systems for facilitated diffusion.

We have demonstrated that both Msh2 Msh6 and Mlh1 Pms1 can diffuse in 1D along DNA but do so using very different mechanisms. Mlh1 Pms1 hops or steps along the DNA but Msh2 Msh6 moves predominantly by sliding along the DNA while remaining in continuous close contact with the phosphate backbone. The functional consequences of these mechanistic differences are that Mlh1 Pms1 can readily traverse nucleosomes and travel along chromatin whereas Msh2 Msh6 cannot. These results provide an unambiguous demonstration that 1D diffusion can occur on crowded DNA substrates in the presence of protein obstacles and that the ability to bypass obstacles is dependent upon the how the protein in question diffuses along DNA. We anticipate that these behaviors displayed by Mlh1 Pms1 and Msh2 Msh6 in response to collisions with nucleosomes will reflect general mechanistic attributes of their respective modes of 1D diffusion which in principle will apply to any proteins that diffuse on DNA e.g. DNA repair proteins transcription factors etc. proteins that track the phosphate backbone while sliding along DNA will experience a barrier upon encountering obstacles and must either disengage the DNA and enter a 2D or 3D mode of diffusion to continue searching for targets or the DNA must be cleared of obstacles before hand to allow unhindered access to the DNA see below in contrast proteins that do not track the backbone can traverse obstacles without experiencing significant boundary effects.

The different modes of diffusion found for Msh2 Msh6 and Mlh1 Pms1 also impose specific constraints on the mechanisms of MMR. Msh2 Msh6 is the first to arrive at lesions and helps identify nearby signals differentiating the parental and nascent DNA strands. Many models for strand discrimination invoke 1D movement of Msh2 Msh6 along DNA and even transient loss of contact with the DNA during this second phase of the reaction could compromise repair. Nucleosomes or other DNA binding proteins have the potential to thwart Msh2 Msh6 and a single nucleosome deposited near a lesion could render it irreparable suggesting regions in need of repair must be kept free from obstacles. Replication forks disrupt nucleosomes leaving stretches of naked DNA in their wake. While speculative if Msh2 Msh6 were restricted to the region behind the fork possibly through direct association with PCNA then it would be free to scan newly replicated naked DNA. Mlh1 Pms1 is thought to arrive later than Msh2 Msh6 implying it must survey the entire genome for lesion bound Msh2 Msh6 without the benefit of confined searches in regions already cleared by a replication fork. The ability of Mlh1 Pms1 to hop or step along DNA and freely traverse nucleosomes ensures it could efficiently bypass stationary obstacles while searching the genome for its binding targets.

Mlh1 Pms1 was expressed and purified using pMH1 GAL1 MLH1 VMA CBD 2 TRP1 and pMH8 GAL10 PMS1 2 LEU2 vectors transformed into the strain BJ2168. Mlh1 Pms1 complexes containing FLAG HA and or TEV tags were purified from BJ2168 containing the relevant pMH1 and pMH8 derivatives described in the Supplemental Information. All Mlh1 Pms1 constructs were functional for mismatch repair in vivo.

The TIRFM system particle tracking and data analysis have been described. Unless otherwise indicated data were collected using DNA molecules anchored with nanofabricated patterns made by electron beam lithography. QDs were prepared using a protocol that yields 0.076 0.014 epitope binding sites per QD i.e. approximately 1 in 13 QDs has of a functional IgG . Assuming the conjugation reaction is a Poisson process the probability of a QD having two or three epitope binding sites is P 0.0027 and P 7.02 10 respectively. FLAG tagged Mlh1 Pms1 30 250 nM was mixed with 2 fold molar excess anti FLAG QDs in buffer containing 40 mM Tris HCl pH 7.7 150 mM NaCl 1 mM MgCl 1 mM DTT and 0.4 mg mlBSA and incubated for 15 20 minutes on ice. Reactions were diluted 10 fold 3 25 nM Mlh1 Pms1 prior to injection. Crosslinking assays verified Mlh1 Pms1 did not dissociate to monomers under dilute conditions not shown . TIRFM experiments were done using 40 mM Tris HCl pH 7.7 150 mM NaCl unless otherwise indicated 1 mM MgCl 1 mM DTT 0.4 mg mlBSA 140 mM mercaptoethanol 1 mM nucleotide ADP ATP or ATP S as indicated . Labeled proteins were injected into the sample chamber unbound proteins were quickly flushed away and flow was terminated prior to data acquisition. YOYO1 0.5 nM Invitrogen and an Oscavenging system glucose oxidase 34 units ml catalase 520 units ml and 1 glucose w v was included in reactions requiring fluorescent DNA. Bulk experiments verified that YOYO1 and the Oscavenging system did not affect Mlh1 Pms1 DNA binding activity. Recombinant histones were purified from reconstituted into octamers and deposited on the DNA as deseribed.

Proteins that hop cycle rapidly between a free and a bound state and increasing ionic strength increases the lifetime of unbound intermediate while decreasing the lifetime of the bound intermediate hence increasing the overall observed rate of travel Berg et al. Biochemistry 1981 Blainey et al. PNAS 2006 Kochaniak et al. J. Biol. Chem. 2008 Komazin Meredith et al. PNAS 2008 . Stepping can be considered virtually identical to hopping with the exception that at least two separate parts of the protein must cycle between free and bound states i.e. hop . The effect of salt on a stepping process would be similar to that which is observed for a simpler hopping mechanism and would lead to an apparent overall increase in the rate of travel along the DNA. It is important to note that different forms of facilitated diffusion are not mutually exclusive and for example the bound states that exist during a hopping stepping mechanism may in fact slide on DNA Givaty Levy J. Mol. Biol. 2009 . Nevertheless our data indicate that Msh2 Msh6 moves predominantly by sliding and Mlh1 Pms1 moves predominantly by hopping stepping. The functional consequence of this difference is that Mlh1 Pms1 can readily traverse nucleosomes and chromatin whereas Msh2 Msh6 cannot.

Jumping is distinct from hopping stepping in that it is an uncorrelated search involving a free 3D diffusion component enabling the protein to move long distances between each independent jumping event von Hippel Berg J. Biol. Chem. 1989 . Similar to hopping stepping jumping frequency would also increase at higher ionic strengths which in the case of jumping would lead to increased dissociation of the protein from the DNA and single molecule experiments done in the presence of buffer flow induce dissociation of jumping proteins as the free state is readily pushed away from the DNA and irretrievably lost to solution. We do not completely rule out the possibility that occasional jumping is a potential mechanism that could contribute to Mlh1 Pms1 movement. However in a single molecule assay jumping would appear as the sudden disappearance of a protein followed by its near immediate reappearance at a distant location or even on a different DNA molecule Bonnet et al. Nucleic Acids Res. 2008 . The vast majority of the diffusion trajectories observed for Mlh1 Pms1 involved continuous 1D motion along the DNA which is only consistent with a correlated scanning mechanism i.e. sliding and or hopping stepping but inconsistent with extensive jumping. Moreover extensive jumping is inconsistent with the end dependent DNA dissociation observed for Mlh1 Pms1 and the wrapped DNA binding topology that we propose as an explanation for the end dependent dissociation.

Plasmids containing tagged MLH1 under the native MLH1 promoter were created by overlap extension PCR as derivatives of pEAA213 Heck et al. 2006 pEAA373 contains a FLAG tag underlined flanked on either side by three alanines AAAAAA SEQ ID NO 25 and inserted immediately after amino acid 448T of MLH1 pEAA516 contains a TEV site underlined flanked on either side by three alanines AAAAAA SEQ ID NO 26 inserted immediately after amino acid 448T and the FLAG tag inserted immediately after amino acid 499Y of MLH1. For expression the tagged MLH1 constructs were sub cloned into pMH1 and purified as described Hall and Kunkel 2001 . Sub cloning from pEAA373 and pEAA516 into pMH1 created pEAE267 and pEAE295 respectively. For tagged PMS1 pEAE296 was created as a derivative of pMH8 by overlap extension PCR and included a HA tag underlined flanked by three alanine residues AAAAAA SEQ ID NO 27 inserted after amino acid D565 of PMS1. For complementation the epitope tag from pEAE296 was sub cloned into pEAA238 Heck et al. 2006 which has PMS1 under its native promoter and then cloned into pEAA248 which is the same as pEAA238 but contains the URA3 selectable marker to create pEAA517. All clones were sequenced Cornell BioResource Center and additional details on vector construction will be provided upon request.

MLH1 constructs were tested in vivo for the mlh1 mutator phenotype in ARS CEN LEU2 vectors containing MLH1 expressed from its native promoter. The HA tagged PMS1 was tested in vivo for the pms1 mutator phenotype Heck et al. 2006 . To test complementation the semiquantitative canavanine resistance assay was used to measure mutation rates in the S288c strains EAY874 MAT leu2 3 112 trp1 289 ura3 argD cyhS mlh1 KanMX4 and EAY1087 MAT ura3 52 leu2 1 trp1 63 his3 200 lys2 202 pms1 KanMX4 . EAY874 strains containing pRS415 mutant control pEAA213 wild type control or pEAA373 pEAA515 and pEAA516 epitope tagged MLH1 were streaked on leucine dropout plates to obtain single colonies. EAY310 strains containing pRS416 mutant control pEAA248 wild type control and pEAA517 HA PMS1 were streaked onto uracil dropout plates to obtain single colonies. 35 independent colonies from each strain were patched onto appropriate dropout plates containing L canavanine 60 mg L and incubated for 3 days at 30 C. The number of canavanine resistant papillations in each patch was counted and the median number for each strain was used for comparison. All tested derivatives of MLH1 and PMS1 conferred mutation frequencies indistinguishable from wild type in contrast to the 10 fold higher frequency found in the corresponding null strains carrying an empty vector.

Protein concentrations were determined by Bradford assay using BSA as a standard. Gel mobility shift assays with oligonucleotide substrates were performed as described Kijas et al. 2003 . Binding reactions were performed at room temperature RT for 5 minutes in 15 l reactions containing 60 nM 5 P end labeled 40 bp substrate 120 nM Mlh1 Pms1 25 mM Hepes pH 7.6 40 mlBSA 1 mM DTT 50 mM NaCl and 8 w v sucrose. In super shift experiments either 0.65 g of FLAG antibody Sigma Cat. No. F3165 or 3.25 g of HA antibody Sigma Cat. No. H3663 were pre incubated with Mlh1 Pms1 on ice for 30 min prior to the addition of DNA substrates. Samples were loaded on 4 w v non denaturing polyacrylamide gels containing 0.5 TBE and electrophoresed at 130 V for 1 hour at RT. Gels were dried on 3mM Whatman paper and visualized by Phosphorlmaging. Analysis was done using ImageJ. The 40 bp substrate was created by annealing S1 5 dACC GAA TTC TGA CTT GCT AGG ACA TCT TTG CCC ACG TTG A SEQ ID NO 28 and S2 5 dTCA ACG TGG GCA AAG ATG TCC TAG CAA GTC AGA ATT CGG T SEQ ID NO 29 Integrated DNA Technologies Surtees and Alani 2006 .

Nitrocellulose filter binding assays were performed as described Chi and Kolodner 1994 . Briefly nitrocellulose filters BA85 0.45 m 25 mm Whatman Schleicher Schuell were pre soaked in 0.5 M KOH for 20 minutes rinsed thoroughly with sterile deionized distilled water washed once with reaction buffer 25 mM Hepes pH 7.6 0.01 mM EDTA and stored in same buffer prior to use. The 3 kb plasmid pEAO242 was linearized with NcoI followed by treatment with calf intestinal phosphatase CIP New England Biolabs . CIP treated DNA was purified QIAquick Qiagen and end labeled with T4 polynucleotide kinase and P ATP followed by heat inactivation and removal of unincorporated nucleotides as described by the manufacturer New England Biolabs . Binding reactions 30 l were performed in buffer supplemented with 1 mM DTT and 40 g mlBSA without EDTA and included 100 pM of 5 P labeled 3 kb linear plasmid. Mlh1 Pms1 was incubated with 40 nM IgG or IgG coupled QDs as indicated on ice for 30 minutes. Binding reactions were incubated at RT for 10 minutes and filtration was performed on a Hoefer filter manifold FH225V Hoefer Scientific . Binding reactions were added to 2.5 ml of ice cold reaction buffer overlaying nitrocellulose filters and passed through with a flow rate of 2.5 ml min. Dried filters were placed in 5 ml of Ecoscint scintillant National Diagnostics and the bound radioactivity measured in a Beckman LS 5000 scintillation counter. Background was determined in reactions without protein and was typically around 1 5 of total radioactivity. Percent DNA binding was determined by dividing the background subtracted count for each filter by the total radioactivity per reaction.

3 g of Mlh1 Pms1 was incubated with 0.03 g of TEV protease in 15 l reactions containing 25 mM Hepes pH 7.6 1 mM DTT and 40 g mlBSA. TEV protease was a generous gift from Dr. Ailong Ke Cornell . TEV cleaved proteins were assayed for DNA binding activity using gel shift assays as described above. To confirm TEV cleavage samples were incubated at 30 C. for 30 minutes after which 7.5 l of 3 SDS loading buffer 0.195 M Tris pH 6.8 30 glycerol 3 mercaptoethanol 6 SDS was added to each and samples were boiled for 3 minutes. Samples were analyzed by 10 SDS PAGE and stained with Coomassie blue see .

DNA substrates were made by ligating oligonucleotides to the 12 nucleotide overhangs at the end of the DNA. Oligonucleotides were purchased from Operon Technologies and gel purified prior to use. Ligation mixes 1 ml total volume contained 4 nM DNA Invitrogen 1 M biotinylated oligonucleotiode 5 pAGGT CGCCGCCC BioTEG 3 SEQ ID NO 30 1 M DIG digoxigenin or FITC fluorescein isothiocyanate labeled oligonucleotide 5 pGGG CGG CGA CCT DIG 3 SEQ ID NO 31 or 5 pGGG CGG CGA CCT FITC 3 SEQ ID NO 32 and lx ligase buffer New England Biolabs . The reaction mix was warmed to 65 C. for 10 minutes and then cooled slowly to RT. After cooling ligase was added T4 DNA ligase 400 units l or Taq ligase 40 units l New England Biolabs and the mixture was incubated overnight at 42 C. Reactions performed with T4 ligase were heat inactivated at 65 C. for 10 minutes and ligated DNA products were purified over a Sephacryl S200HR column GE Healthcare run in 10 mM Tris HCl pH 7.8 1 mM EDTA plus 150 mM NaCl. Purified DNA was stored at 20 C.

QDs were prepared by following the manufacturer s recommended protocol as described in Gorman et al. 2007 and Pathak et al. 2007 Qdot 585 Antibody Conjugation Kit Cat. No. Q22011MP and Qdot 705 Antibody Conjugation Kit Cat. No. Q22061MP Invitrogen . In brief QDs 4 M were activated with 1 mM SMCC succinimidyl 4 N maleimidomethyl cyclohexane 1 carboxylate for 1 hour at RT yielding maleimide functional groups on the QD surface. Antibodies 1 mg mlin PBS Sigma Cat. No. H3663 and F3165 for anti HA and anti FLAG respectively were reduced with 20 mM DTT for 30 minutes at RT which cleaves disulfide bonds between heavy chains generating free thiols that can be coupled the maleimide QDs. Reduced IgG was purified on a NAP 5 column and then mixed with the maleimide activated QDs for 1 hour at RT. Reactions were quenched with 100 M mercaptoethanol for 30 minutes at RT and the resulting antibody QD conjugates were purified over a Superdex 200 10 300 GL gel filtration column GE Healthcare run in PBS to remove unreacted IgG. Purified conjugates were stored in PBS pH 7.4 plus 0.1 mg mlacetylated BSA at 4 C.

Individual QDs blink and this well known phenomenon enables one to distinguish single vs. multiple QDs M. Dahan et al. Science 2003 J. Yao et al. PNAS 2005 Q. Zhang et al. Science 2009 . In our experiments a non blinking QD signal could arise from either QD aggregation or protein aggregation therefore any QDs that did not blink were discarded from analysis ensuring all reported data arose from single fluorescent molecules. Apparent differences in signal intensities within the kymograms can arise from several sources including normal variations in QD fluorescence variations in QDs blinking frequency stationary QDs are brighter because their signal is confined to a fixed location during frame acquisition nonblinking signals are brighter because they arise from multiple QDs and some faint signals in the kymograms arise from bleed through of signal from QD proteins bound to an adjacent DNA in the curtain. These variations are expected and we have confined our analysis and calculations to blinking QDs to ensure we are monitoring single fluorescent molecules.

Two color QD labeling experiments as described in S. L. Reck Peterson et al. Cell 2006 were used to assess whether the Mlh1 Pms1 complexes under investigation were consistent with single heterodimers or higher order oligomers. In brief we premixed equimolar amounts of anti FLAG green QDs Qdot 585 and anti FLAG magenta QDs Qdot 705 and then used this two color mixture to label Mlh1 FLAG Pms1. Heterodimeric Mlh1 FLAG Pms1 contains just one FLAG epitope and therefore could only be labeled with one QD either green OR magenta but not both . Whereas higher order oligomers would contain multiple FLAG tags the exact number of FLAG tags would scale in proportion to the number of Mlh1 Pms1 subunits within the oligomer and therefore could be labeled with two or more QDs leading to colocalization of both green and magenta QDs. These two color colocalization experiments revealed that 94.8 N 1 254 1 323 of the proteins were either only green or only magenta arguing that most Mlh1 Pms1 complexes contained only a single FLAG epitope. This result is consistent with a heterodimeric Mlh1 Pms1 complex but inconsistent with the formation of larger Mlh1 Pms1 oligomers at the low protein concentrations used for our diffusion experiments.

A complete description of the double tethered DNA curtains can be found in Gorman et al. 2010. Fused silica slides G. Finkenbeiner Inc. were cleaned in NanoStrip solution CyanTek Corp Fremont Calif. for 20 minutes rinsed with acetone and isopropanol and dried with N. Slides were spin coated with a bilayer of polymethylmethacrylate PMMA 25K and 495K MicroChem Newton Mass. followed by a layer of Aquasave Mitsubishi Rayon . Patterns were written with a FEI Sirion scanning electron microscope J. C. Nabity Inc. Bozeman Mont. . Aquasave was removed with deionized water and resist was developed using isopropanol methyl isobutyl ketone 3 1 for 1 minute with ultrasonic agitation at 5 C. The substrate was rinsed in isopropanol and dried with N. Barriers were made with a 15 20 nm layer of chromium Cr and following liftoff samples were rinsed with acetone and dried with N as described Gorman et al. 2010 .

Inlet and outlet ports were made by boring through the slide with a precision drill press equipped with a diamond tipped bit 1.4 mm O.D. Kassoy . The slides were cleaned by successive immersion in 2 v v Hellmanex 1 M NaOH and 100 MeOH. Slides were rinsed with MilliQ between each wash and stored in 100 MeOH until use. Prior to assembly slides were dried under a stream of nitrogen and baked in a vacuum oven for at least 1 hour. A sample chamber was prepared from a borosilicate glass coverslip Fisher Scientific and double sided tape 100 m thick 3M . Ports Upchurch Scientific were attached with hot melt adhesive SureBonder glue sticks FPC Corp. . The total volume of the sample chambers was 13 l. A syringe pump Kd Scientific and actuated injection valves Upchurch Scientific were used to control sample delivery. The flowcell and prism were mounted in a custom built heater with computer controlled feedback regulation.

Lipids were purchased from Avanti Polar Lipids and liposomes were prepared as previously described Gorman et al. 2010 . In brief a mixture of DOPC 1 2 dioleoyl sn glycero phosphocholine 0.5 biotinylated DPPE 1 2 dipalmitoyl sn glycero 3 phosphoethanolamine N cap biotinyl and 8 mPEG 550 DOPE 1 2 dioleoyl sn glycero 3 phosphoethanolamine N methoxy polyethylene glycol 550 . The mPEG prevented nonspecific adsorption of QDs. Liposomes were applied to the sample chamber for 15 minutes. Excess liposomes were removed with buffer containing 10 mM Tris HCl pH 7.8 and 100 mM NaCl. The flowcell was then rinsed with the same buffer and incubated for 30 minutes. 30 g mlanti DIG Fab Roche Cat. No. 1214667001 or anti FITC Invitrogen Cat. No. 71 1900 was injected into the chamber and incubated for 20 minutes. The sample chamber was then flushed with buffer A 40 mM Tris HCl pH 7.8 1 mM DTT and 1 mM MgCl plus and 0.2 mg mlBSA for 5 minutes. Streptavidin 0.02 mg ml in buffer A was injected into the sample chamber and incubated for 20 minutes. After rinsing with additional buffer A plus 0.2 mg mlBSA DNA 15 20 pM labeled at one end with biotin and at the other end with DIG or FITC and pre stained with 0.5 nM YOYO1 was injected into the chamber incubated for 10 minutes and unbound DNA was removed by flushing with buffer at 0.1 ml min. Application of flow aligned the DNA molecules along the diffusion barriers and stretched the molecules so the free ends could attach to the pentagons.

All experiments where proteins were pushed along DNA by hydrodynamic force were done at a flow rate of 0.5 ml min. When considering flow through a channel in which the height h is much less than the width w one can use the Navier Stokes equations to determine the flow profile y obtained when a pressure difference is applied between the two ends of the channel. In this case the top and bottom surfaces of the flowcell chamber create a drag on the buffer that results in a parabolic flow where the velocity of the buffer at a distance y from the surface can be described by the equation 

We approximated the distance y from the surface to be 100 nm and the radius of the protein QD complex to be 13 nm. The height of a typical channel was measured to be 100 m and the width of the channel was measured to be 4 500 m. The resulting calculated force applied to a protein QD complex at a buffer flow of 0.5 ml minis approximately 25 fN. At 200 nm from the surface this theoretical value increases to approximately 50 fN.

We then determined the force acting on a complex experimentally and compared these experimental results to the above theoretical calculations. The drift velocity of a particle is equal to the force exerted on that molecule F divided by the drag coefficient of the molecule f Berg 1993 

We were therefore able to calculate the force applied to a single Mlh1 QD complex on DNA by first determining its diffusion coefficient and subsequently pushing the molecule with buffer flow and tracking the movement to determine its drift velocity. Using these values we determined the actual force on the molecule to be 100 fN which is in close agreement to the theoretical values calculated above. Importantly this experimental method implicitly includes the drag components from the hydrodynamic radius of the complex as well as the protein DNA interaction without needing to determine these components directly and also makes no assumptions regarding the distance of the protein from the surface.

Nucleosomes were prepared as described Visnapuu Greene 2009 . Histones H2A FLAG H2B H3 and H4 were expressed in purified from inclusion bodies and reconstituted as described Wittmeyer et al. 2004 . In brief inclusion bodies were resuspended in unfolding buffer 7 M guanidinium HCl 1 M NaCl 50 mM Tris HCl pH 7.8 1 mM EDTA 1 mM DTT dialyzed against urea buffer 7 M urea 1 M NaCl 10 mM Tris HCl pH 7.8 1 mM EDTA 5 mM mercaptoethanol then loaded onto tandem HiTrap Q and SP columns GE Healthcare . Histones were eluted from the SP column with a 100 400 mM NaCl gradient for H2A and FLAG H2B and a 200 500 mM NaCl gradient for H3 and H4. Purified histones were dialysed against 10 mM Tris HCl pH 7.8 plus 5 mM mercaptoethanol followed by 10 mM Tris HCl pH 7.8 then lyophilized and stored at 20 C. Lyophilized histones were unfolded in 7M guanidinium HCl 1 M NaCl 50 mM Tris HCl pH 7.8 plus 10 mM DTT combined at equimolar ratios and dialyzed into 2 M NaCl 20 mM Tris HCl pH 7.8 1 mM EDTA 5 mM mercaptoethanol with several buffer changes over 48 hours. Reconstituted octamers were purified by gel filtration and deposited onto DNA by salt dialysis Luger et al. 1999 Th str m et al. 2004 .

The FLAG tagged nucleosomes were labeled with 0.5 nM QDs QD 585 Invitrogen conjugated to anti FLAG antibodies Sigma . Nucleosome labeling was done in situ as described Visnapuu Greene 2009 either before or after the injection of Mlh1 Pms1 as indicated see . For experiments with unlabeled nucleosomes we first conducted the diffusion experiment using QD Mlh1 Pms1 QD 705 Invitrogen in buffer containing 40 mM Tris HCl pH 7.7 150 mM NaCl 1 mM ATP O mM Mg 1 mM DTT and 0.4 mg mlBSA. Mg induces condensation of the chromatin substrates by promoting nucleosome nucleosome interactions and for this reason was omitted from the diffusion experiments. Mlh1 Pms1 was then flushed from the chamber with 300 mM NaCl which does not disrupt the nucleosomes M L.V and E.C.G unpublished Burton et al. 1978 and Park et al. 2004 and the nucleosomes were then located by labeling with QDs QD 705 Invitrogen . The QD signal from the labeled nucleosomes was then pseudocolored and superimposed on the kymograms of Mlh1 Pms1 diffusion see upper and middle panels . For reactions with labeled nucleosomes the nucleosomes were labeled in situ with anti FLAG QDs QD 585 Invitrogen then QD Mlh1 Pms1 QD 705 Invitrogen was injected into the sample chamber and the signal from the different colored QDs was collected concurrently signal gaps in the real time data correspond to QD blinking see lower panels .

The chromatin diffusion experiments with Msh2 Msh6 were conducted essentially the same as with Mlh1 Pms1 in buffer containing 40 mM Tris HCl pH 7.7 50 mM NaCl 1 mM ADP 1 mM DTT and 0.4 mg mlBSA. ATP was omitted to prevent ATP triggered protein dissociation and the salt concentration was kept at 50 mM NaCl to increase the half life of the bound state in order to evaluate whether it was able to pass nucleosomes Gorman et al. 2007 . Under identical reaction conditions in the absence of nucleosomes many molecules 48 N 380 of Msh2 Msh6 reversibly enter a nondiffusive state immobile thought to mimic a pseudo damage recognition complex and the remaining Msh2 Msh6 complexes remain mobile 52 N 412 Gorman et al. 2007 . In the presence of high density nucleosomes 80 100 nucleosomes per DNA molecule the fraction of immobile molecules increases from 48 to 96.4 and we attribute this increase to Msh2 Msh6 molecules that are now trapped between nucleosomes and incapable of diffusing on DNA.

All diffusion coefficients represent the mean standard deviation of 25 particle tracking measurements and were calculated from MSD plots as described in Gorman et al. 2007. Traces were used to calculate diffusion coefficients only if the QDs could be tracked over 250 consecutive frames 50 seconds only QDs that blinked were used for tracking verifying they were single QDs and traces were excluded if collisions between two or more proteins prevented accurate tracking. We can calculate diffusion coefficients using fewer than 250 consecutive frames but the variance and error in the resulting data begins to increase significantly. Trajectories where the proteins approached to within 500 nm of one another were also excluded from the diffusion coefficient calculations as a quality control measure because error in the tracking algorithm increases below this distance. Two closely approaching QDs can be optically resolved from one another at distances 10 nm Lacoste et al. PNAS 2000 Lagerholm et al. Biophysics J. 2006 but as indicated above as a quality control measure we excluded any traces where proteins approached to within 500 nm of one another to ensure uniform accuracy in the particle tracking data.

Diffusion coefficients were calculated from the tracking data as previously described Gorman et al. Mol. Cell 2006 . In brief the movement of each protein complex in the y direction parallel to the long axis of the DNA was then analyzed to calculate the mean squared displacement MSD using 

The particle tracking algorithm used to monitor the movement of Msh2 Msh6 and Mlh1 Pms1 simultaneously records its position in the y direction parallel to the long axis of the DNA and in the x direction perpendicular to the long axis of the DNA . The values obtained for the x direction primarily reflect the thermal motions of the DNA molecules. All of the tracked proteins displayed x direction fluctuations ranging between 50 250 nanometers with a mean of 80 nm which is an order of magnitude below the motions observed along the helical axis i.e. the y direction of the DNA. DNAs more flexible than this were occasionally observed but they were omitted from the analysis because their flexibility caused the bound fluorescent Msh2 Msh6 to fluctuate too much within the evanescent field making them impossible to track accurately although qualitatively they displayed exactly the same diffusive behavior . The transverse fluctuations of the DNA and the temporal resolution of our detection system impose a lower limit of 1 10 msecfor the diffusion coefficients that can be measured anything slower than this will look like a stationary particle but this lower limit is well below any values reported in our study.

Those skilled in the art will recognize or be able to ascertain using no more than routine experimentation numerous equivalents to the specific substances and procedures described herein. Such equivalents are considered to be within the scope of this invention and are covered by the following claims.

