---

title: Genotypic tumor progression classifier and predictor
abstract: Actively dividing tumors appear to progress to a life threatening condition more rapidly than slowly dividing tumors. Assessing actively dividing tumors currently involves a manual assessment of the number of mitotic cells in a histological slide prepared from the tumor and assessed by a trained pathologist. Disclosed is a method for using cumulative information from a series of expressed genes to determine tumor prognosis. This cumulative information can be used to categorize tumor samples into high mitotic states or low mitotic states using a mathematical algorithm and gene expression data derived from microarrays or quantitative-Polymerase Chain Reaction (Q-PCR) data. The specific mathematical description outlines how the algorithm assesses the most informative subset of genes from the full list of genes during the assessment of each sample.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09037416&OS=09037416&RS=09037416
owner: H. Lee Moffitt Cancer Center and Research Institute, Inc.
number: 09037416
owner_city: Tampa
owner_country: US
publication_date: 20100322
---
This application is a continuation of prior filed International Application Serial Number PCT US2008 077190 filed Sep. 22 2008 which claims priority to U.S. provisional patent application No. 60 974 264 filed Sep. 21 2007 which is hereby incorporated by reference into this disclosure.

This invention was made with Government support under Grant No. DAMD 17 02 2 0051 awarded by the Department of Defense and Grant No. CA085052 awarded by the National Institutes of Health. The Government has certain rights in the invention.

This invention relates to oncogenic assessment using cumulative information obtained from expression of genes involved in regulation of cell cycle and mitosis.

In the United States and in many Western countries lung cancer represents the leading cause of cancer related death Jemal A. et al. Cancer Statistics 2006. CA Cancer J. Clin. 56 106 130 2006 . The National Cancer Institute of the U.S. National Institutes of Health estimated there were 215 020 new cases of lung cancer and 161 840 lung cancer related deaths within the U.S. in 2008. Lung cells cancers are classified into two main types small cell lung cancer SCLC and non small cell lung cancer NSCLC .

Cell classification and typing is typically performed using light microscopy immunohistochemistry physical examination chest x ray and chest computed tomography CT scan with infusion of contrast material. Diagnosis generally requires review of pathologic material by an experienced lung cancer pathologist to manually assess the number of mitotic cells in a histological slide prepared from the tumor. Tumor classification is vitally important because small cell lung cancer which responds well to chemotherapy and is generally not treated surgically can be confused on microscopic examination with non small cell carcinoma for which treatment normally consisting of surgical resection. Additionally staging procedures are important to distinguish localization and tumor aggressiveness. Determining cancer stage non surgically provides better assessment of prognosis and aides in treatment determination which is usually influenced by stage particularly when chest radiation therapy or surgical excision is added to chemotherapy for patients with limited stage disease.

SCLCs are the most aggressive pulmonary tumor with median survival from diagnosis of 2 to 4 months. Localized limited SCLC tumors are confined to the hemithorax of origin the mediastinum or the supraclavicular lymph nodes and treatment includes surgical rescission with or without chemotherapy. Extensive stage disease where tumor has spread beyond the supraclavicular area possess worse prognosis than limited stage tumors.

NSCLCs are a heterogeneous aggregate of tumors with the most common histologies epidermoid or squamous carcinoma adenocarcinoma and large cell carcinoma. These histologies are often classified together because approaches to diagnosis staging prognosis and treatment are similar. The first classification of tumors are surgically resectable generally stage I stage II and selected stage III tumors and possess the best prognosis which depends on a variety of tumor and host factors. In some cases tumors are alternatively treated with curative radiation therapy or chemotherapy. The second group of tumors is locally T3 T4 and or regionally N2 N3 advanced lung cancer which are treated with combination therapies such as radiation therapy in combination with chemotherapy or surgical resection and either preoperative or postoperative chemotherapy or chemoradiation therapy. The final tumor group is metastatic tumor with distant metastasis M1 at the time of diagnosis. Current treatment is radiation therapy or chemotherapy for palliation.

The 5 year overall survival rate of 15 has not improved over many decades mainly because approximately two thirds of lung cancers are discovered in advanced stages for which cure by surgical resection is no longer an option. Furthermore even among early stage patients who are treated to primarily by surgery with curative intent 30 55 will develop and die of metastatic recurrence. Recent multinational clinical trials IALT JBR10 ANITA UFT LACE conducted in several continents have demonstrated that adjuvant chemotherapy significantly improves the survival of patients with early stage IB II disease Booth C. M. Shapard F. A. Adjuvant chemotherapy for resected non small cell lung cancer. 2 180 187 2006 . Nevertheless it is clear that a proportion of patients with stage I disease have poorer prognosis and may benefit significantly from adjuvant chemotherapy whereas some with stage II disease with relatively good prognoses may not benefit significantly from adjuvant chemotherapies. It remains possible however that the latter patients could derive additional benefit from adjuvant targeted therapies Booth C. M. Shepherd F. A. Adjuvant chemotherapy for resected non small cell lung cancer. J. Thorac. Oncol. 2 180 187 2006 Gandara D. R. et al. Adjuvant chemotherapy of stage I non small cell lung cancer in North America. 7 suppl. 3 S125 S127 2007 Shepherd F. A. et al. Erlotinib in previously treated non small cell lung cancer. 353 123 132 2005 .

Therefore there is an urgent need to establish new diagnostic paradigms and validate in clinical trials methods for improving the selection of stage I II patients who are most likely to benefit from adjuvant chemotherapy.

Disclosed is a method of predicting clinical tumor outcome by providing gene expression from a tumor sample Shedden K. et al. ene expression based survival prediction in lung adenocarcinoma a multi site blinded validation study. Nat. Med. 14 8 822 7. 2008 . The gene expression may be obtained from any number of means known in the art including without limitation PCR Chip gene array microarrays or quantitative Polymerase Chain Reaction Q PCR and reverse transciptase Polymerase Chain Reaction rt PCR . At least one threshold value is defined for classifying the gene expression levels. In certain embodiments two threshold values are used. A vote of single gene classifiers is then determined by comparing the gene expressions to the threshold value or values followed by selecting genes with expression levels above at least one threshold value and selecting genes with expression levels below at least one threshold value. This data is then used to calculate the majority vote classifier.

In certain embodiments at least one threshold value consists of an upper threshold value and lower threshold value. These values are the upper 33 of the gene expression values the lower 33 of the gene expression values or the upper 15 of the gene expression values and the lower 15 of the gene expression values in specific embodiments. In alternative embodiments hazard ratios are determined for the risk score wherein a hazard ratio is calculated from the majority vote classifier and a score greater than 1 indicates poor clinical outcome.

Certain embodiments of the invention also assign numerical values to the gene expression 1 to gene expression values at and above the upper threshold value 1 to gene expression values at and below the lower threshold value and 0 to gene expression values above the lower threshold value and below the upper threshold value. In specific embodiments these values are added together to calculate the majority vote classifier. By further defining at least one majority vote threshold value the majority vote classifier is compared to at least one majority vote threshold value.

Alternatively only one threshold value is used with numerical values used for the gene expression 1 to gene expression values at and above the threshold value and 1 to gene expression values at and below the threshold value. In specific embodiments these values are added together to calculate the majority vote classifier. By further defining at least one majority vote threshold value the majority vote classifier is compared to at least one majority vote threshold value determining tumor outcome.

In some embodiments the method uses a probeset list generated by providing a first probeset list and testing the first probeset list against other gene expression data for a tumor cell. This gene expression data is compared to two threshold values and probes identified as significantly related to the first probeset list s gene expression either as through high expression or through low expression are used in a second probeset list test. A second probeset list including probes selected in the first test are compared to gene expression data for a tumor cell using the two threshold values as before. Probes identified as significantly related to gene expression in the second probeset list are included in a third probeset list test. This third probeset list is tested against gene expression data for a tumor cell as before. Probes selected from the third probeset are then included in the probelist.

Disclosed is a tumor prognosis predictor based on gene expression signatures of lung adenocarcinomas. Gene expression data is used to identify patients with slightly more aggressive forms of cancer like adenocarcinoma that might require closer monitoring or chemotherapeutic intervention. The invention uses cumulative expression information from a series of genes involved in the regulation of the cell cycle and the mitotic process to assess the overall mitotic state of a tumor sample. This cumulative information can be used to categorize tumor samples into high mitotic states or low mitotic states using a mathematical algorithm and gene expression data derived from microarrays or quantitative Polymerase Chain Reaction Q PCR data. The specific mathematical description outlines how the algorithm assesses the most informative subset of genes from the full list of genes during the assessment of each sample

A mechanistic classifier was produced using training data and tested for effectiveness using two different data sets. To ensure scientific validity of the results subject samples along with all relevant clinical pathological and outcome data were collected by investigators at four institutions using data from six lung cancer treatment sites with subject inclusion criteria defined a priori. Gene expression data on subsets of lung adenocarcinomas were generated by each of four different laboratories using a common platform and following a protocol previously demonstrated to be robust and reproducible Dobbin K. K. et al. Interlaboratory comparability study of cancer gene expression analysis using oligonucleotide microarrays. Clin. Cancer Res. 11 565 572 2005 .

Four different data sets were generated as described in designated as HLM UM MSK and CAN DF. Data generated at the HLM site was used as a training set and the results were validated using the independent data sets from the other three sites following a blinded protocol. Initial evaluation of the gene expression data suggested that the data from the UM HLM and MSK were broadly similar although distinguishable but the data from CAN DF showed some systematic differences from the other three sites due mainly to reduced signal intensity.

Approximately 505 adenocarcinoma samples varying from stage I to stage III were collected with about 60 of the samples represented by stage I tumors. The tumors were collected by surgical resection from patients prior to chemotherapy or radiation. Two years of follow up information was available for each tumor sample. Study pathologists reviewed the tumor permanent sections and frozen sections of the samples to identify stage diagnosis and regions of the frozen section containing the maximum tumor cellularity for macrodissection. Regions containing a minimum of 60 tumor cellularity were required and in most instances tumor cellularity of at least 70 90 was identified for inclusion in the sample for RNA isolation. A second pathological review of each case was performed to coordinate the pathological information across all tumor collection sites to verify the initial pathological findings. Where the pathological diagnosis and review disagreed the samples were excluded from the study. Clinical information was also collected for tumor staging history of prior cancers lymph node involvement by lymph node dissection sampling smoking history age gender operation type last follow up date and patient s status. Additional information useful to the study including CXR and CT scans pulmonary function tests time and site of recurrence and adjuvant treatment information was collected where available. Survival outcome was determined and most patients have reliable long term follow up. Patients were censored after 60 months of follow up or excluded for analysis where death occurred within 1 month of operation.

The clinical data collected from each individual site underwent two forms of review. A core set of critical covariates were identified for thorough quality assessment and reviewed by an NCl statistician to evaluate discrepancies in coding patterns between sites logical inconsistencies in the data and missing data. The complete dataset underwent a second round of quality review by the CALGB Statistical Center Data Operations prior to finalizing the clinical dataset.

Multiple comparison adjustment in the context of predictor development involves a tradeoff. Very strict Bonferroni type adjustment may reduce predictor performance by eliminating informative genes. Too lax adjustment may reduce predictor performance by inclusion of too many noise genes. It was thought that the significance level used in the sample size calculation represented a reasonable tradeoff between the two. A total of 486 tumor samples were arrayed and a total of 442 samples were retained after the QC evaluation of the clinical pathological and array data.

Frozen dissected tumor tissue was immersed in 1 ml of Trizol Reagent Invitrogen Corp. Carlsbad Calif. . Tissue sections were disrupted with a glass homogenizer or glass beads to facilitate dissolution in the Trizol reagent as necessary. Purification of the RNA from the Trizol dissolved samples followed the manufacturer s cleanup procedure Qiagen Inc. Valencia Calif. . The quality of total RNA was assessed by electrophoretic analysis on a bioanalyzer Agilent 2100 . RNA yields were determined by ODmeasurements.

RNA extracts were then labeled for hybridization. The poly A RNA pool of total RNA extracts were converted to cDNA and amplified and labeled with biotin following the procedure initially described previously Van Gelder R. N. et al. Amplified RNA synthesized from limited quantities of heterogeneous cDNA. 1990 87 5 1663 7 . First strand cDNA synthesis was carried out using the Superscript Choice System Invitrogen Corporation Carlsbad Calif. and the T7 promoter oligo dT primer 5 GGCCAGTGAATTGTAATACGACTCACTATAGGGAGGCGG dT 3 SEQ ID 1 from the T7 Oligo dT Promoter Primer Kit Affymetrix Corporation Santa Clara Calif. . Following annealing cDNA synthesis reaction solutions were added to establish final reaction conditions of 5 mg RNA 100 mol T7 T primer 500 mM each dNTP 10 mM DTT 50 mM Tris HCl pH 8.3 75 mM KCl 3 mM MgCl and 200 U of Superscript II reverse transcriptase Invitrogen . The reaction was incubated for 1 hr at 42 C. followed by a second strand cDNA synthesis at 16 C. for 2 hr in a total volume of 150 mL using 10 U of DNA ligase 40 U of DNA polymerase I and 2 U of RNase H in the presence of 200 mM of each dNTP 10 mM NHSO 1.3 mM DTT 26.7 mM Tris HCl pH 7.0 100 mM KCl 5 mM MgCl and 150 mM b NAD Invitrogen Corp. . Following the second strand DNA synthesis 10 U of T4 DNA Polymerase Invitrogen Corp. was added and the samples were incubated an additional 5 min at 16 C. The reaction was stopped by the addition of 0.5 M EDTA and the nucleic acids were purified using the Affymetrix GeneChip sample clean up modules 900371 Affymetrix . The nucleic acids were eluted with 25 ml of DEPC treated water. Twenty two microliters of the purified cDNA was used as the template for a transcription reaction performed with the BioArray HighYield RNA Transcript Labeling Kit according to manufacturer s instructions ENZO Life Sciences New York N.Y. which incorporates biotinylated UTP into the transcripts. The Biotin labeled RNA was purified using RNeasy columns Qiagen Inc. Valencia Calif. and fragmented to a size of 35 to 200 bases by incubating at 940 C for 35 minutes in fragmentation buffer 40 mM Tris acetate pH 8.1 100 mM potassium acetate 30 mM magnesium acetate . The integrity of the starting material and the products of each reaction were monitored on agarose gels to assess the size distribution of the products and compare them to the starting material.

Array hybridization solutions 20 g of fragmented RNA and 0.1 mg ml sonicated herring sperm DNA in 1 MES buffer containing 100 mM MES 1 M Na 20 mM EDTA and 0.01 Tween 20 were made. Hybridization standards were made by spiking aliquots with known concentrations of RNA from the bacterial genes BioB BioC and BioD and one phage gene Cre. The hybridization mixtures was heated to 99 C. for 5 min followed by incubation at 45 C. for 5 min before injection of the sample into an Affymetrix GeneChip array cartridge 133A. All hybridizations were carried out at 45 C. for 16 17 h with mixing on a rotisserie at 60 rpm. Following hybridization the solutions were removed and the arrays were rinsed with 1 MES. The arrays were washed and stained using the GeneChip Fluidics station protocol EukGE WS2 which consists of 10 cycles of 2 mixes per cycle with non stringent wash buffer 6 SSPE 0.01 Tween 20 at 25 C. followed by 4 cycles of 15 mixes per cycle with stringent wash buffer 100 mM MES 0.1 M Na and 0.01 Tween 20 at 50 C. The probe arrays were stained for 10 min in streptavidin phycoerythrin solution SAPE 1 MES solution 10 g ml SAPE Molecular Probes Eugene Oreg. and 2 g l acetylated BSA Invitrogen at 25 C. then washed for 10 cycles of 4 mixes per cycle at 25 C. The probe arrays were treated for 10 min with an antibody solution 1 MES solution 2 g l acetylated BSA 0.1 g l normal goat IgG Sigma Chemical St. Louis Mo. 3 g l biotinylated goat anti streptavidin antibody Vector Laboratories Burlingame Calif. at 25 C. followed by a second staining for 10 min in SAPE at 25 C. The final wash was 15 cycles of 4 mixes per cycle at 30 C. with non stringent wash buffer. The probe arrays were then scanned once at 1.56 m resolution using the Affymetrix GeneChip Scanner 3000 or at 3 m resolution using the Affymetrix GeneChip Scanner 2500.

Arrays were visually scanned for any defects or scanning artifacts that might compromise the final results. Data was then transferred to an honest broker system for further work. From a combined analysis by two independent testing facilities a small number of arrays were excluded under the suspicion of poor quality array data. The HLM dataset for all tumor stages and for stage 1 were used as the training set with the data from UM and MSK held out as an external validation datasets containing similar microarray data and the data from the DF site held out as a more challenging external test set. The datasets sites were quantile normalized as a group using the array NCl U133A61L as a reference file and imported into the software dChip Li et al 2001 Build version February 2006 and signal values for each probeset were calculated using the default settings. The calculated microarray data was then combined with the clinical data for each sample and assigned a sample number by the honest broker.

The data for the validation sets CAN DF and MSK were held by a third party honest broker during analysis of the training data. Risk scoring procedures were developed on the training data and a prognostic model developed on the training data. After the model was defined and documented the honest broker released the gene expression and clinical data but not the outcome data for the two validation data sets which was tested on the prognostic model to predict outcomes for each subject. These predicted risk scores were then passed back to the honest broker allowing evaluation of the performance of the prognostic models.

Overall 44 of the 486 arrayed samples including one Stratagene reference sample were excluded from the study based on violations of the clinical criteria that were discovered after the initial sample was defined or due to poor array quality. Eleven of the patient samples and the single excluded Stratagene sample were excluded due to quality control problems with the microarray data with the remaining 33 samples excluded due to violations of clinical and pathological criteria.

The probesets and thresholds for classification were selected using an iterative application of the classification process. An initial list of 614 probesets were developed using data collected from public databases and original data. Cell line data from fast growing cells versus slow growing cells was used to correlate expression of replication factor C and MCM7. This served as the initial gene set G which are believed to function in cell division or otherwise aid in progressing through the cell cycle. The Gdataset was used to classify the HLM data. The highest scoring subset of samples and lowest scoring subset were selected. A t test was performed to assign significance to each probeset and identify genes that best discriminated between these subsets of the HLM training data p

This classifier is a majority vote classifier using probesets identified in the training set and thresholds adjusted for the data set under investigation. The individual classifiers and if necessary the overall classifier utilize two thresholds so that a middle ground is established which constitutes a region of uncertainty and no vote is registered with respect to the two main classes. The classifier uses a mathematical algorithm for use on gene expression to provide a mechanism for identifying tumor samples with more ongoing mitosis. The algorithm description provides a general procedure for optimizing the genes and thresholds that will work within the framework of the data obtained from the samples. Microarray data from any platform or gene expression determined by RT PCR is usable provided an initial algorithm training step is first performed. Gene expression data from a group of selected genes is cumulatively assessed to determine tumor samples with a high rate of mitosis and thus a higher likelihood of having a poor response to treatment. The algorithm consists of three components the individual classifiers the majority vote classifier and the threshold selection. Thus the method uses 2 established thresholds and a null vote. Of note all classifiers started with the same set of expression summaries processed using the DChip algorithm Li C. Wong W. H. Model based analysis of oligonucleotide arrays expression index computation and outlier detection. 98 31 36 2001 so handling of the raw data was uniform.

S sign indicating trend relative to outcome high expression in Poor class high expression in Good class

For each gelement of G 3 quantiles are defined by 2 threshold values LO and HI. For this implementation of the classifier the LO threshold was set at the gene expression value closest to the 33percentile and the HI threshold was set at the gene expression value closest to the 66percentile in the dataset under investigation. The individual classifier is then defined as

This has the effect of voting 1 for samples with weak expression 1 for samples with high expression and 0 for samples with expression in the middle ground for probesets directly correlated to the defined classes. The vote is reversed if the gene is negatively correlated to the two classes.

The majority vote classifier is used to determine a final risk score. The sum of all individual classifiers is calculated provided by 002 .

This value can be used to develop a risk score. Alternatively one threshold or more can be set to define 2 or more classes.

For training purposes two thresholds were set to define the extreme classes used for gene selection as provide by

0.15 was heuristically determined. A Pred value of G or 1 G indicates complete agreement with one class whereas small values e.g. 0.15 G indicate uncertainty in classification.

The data generated by the classifer provides a binary indication of low or high risk for each gene expression. It was noted during analysis that low and medium gene expression do not widely differ and the classifier may be used to determine good prognosis from bad prognosis for each genetic marker i.e. gene expression with low and medium expression included as one group. The classifier alternatively is useful in assessing risk determining prognosis or discriminating between classes of tumors. In this instance gene expression scores are cumulatively assessed. This may be performed by adding the scores of individual gene expression in the binary 1 and 1 or tertiary 1 0 and 1 system. The cumulative scores are then applied to a continuous score range from G to G where G is the number of genes scored. This cumulative score is then used to assess risk determine prognosis or discriminate between classes of tumors. Specifically the score may be compared to predetermined threshold values or to cutoff thresholds. For example good prognosis may use the lower the 33 percentile of expression while the upper 66 designates poor prognosis. Other threshold values may be assigned.

The microarray data produced at DF was on a different scale than the data produced at the other institutions. The data was mathematically adjusted to account for the differences that exist in the DF data. The DF data as well as the remaining data was adjusted using two processes process. Quantile normalization was performed at the CEL file level which several publications claim produces superior results. It also allowed later normalization on future data without having to recreate all subsequent steps of the process. A software program was developed which utilizes histogram profile of the signal intensities of each of the individual samples in the training set sampled with a fairly typical profile to serve as the reference state for the distribution of signal intensities. Using quantile normalization places the DF data on nearly the same scale as the other data sets.

To estimate signal intensity for individual genes probesets in the samples array data models based methods such as RMA and dChip can also help to correct differences between independent data sets though the systems require all the data be processed together and significant problems arise if the data is processed separately. The data was processed together using dChip and to attempt to homogenize the data sets. Finally the classifiers were adjusted to compensate for the data. In some classifiers the fiftieth percentile was determined in the test data. In others an inflection point between high and low values and for the HLM classifier the 33and 66percentiles were identified in the test data and used to set the two threshold for the individual classifiers in the mitotic classifier. These final adjustments were needed due to problemeatic array files which could not adequately be recognized and corrected. Of note the adjusted thresholds may have reduced the classification algorithm s ability to stratify the data than might have been achieved with thresholds established from the training data. Using a larger clinical data set the thresholds can be set and no adjustment would be required for any future classification tasks.

The thresholds for individual classifiers were reset based on the distribution of the gene expression values in each test set. The majority vote classifier was calculated to generate a score. This score exists within a continuum of values between 313 and 313 which can serve as a risk score. Alternatively 1 or more thresholds can be set to serve as the discriminator between 2 or more classes. For a 2 class discrimination in this study the threshold was set at C or

The estimated hazard ratios for the risk scores produced by disclosed method with 95 confidence intervals is shown for the two validation sets in . Hazard ratios substantially greater than 1.0 indicate that subjects in the validation set with high predicted risk had poor outcomes. Confidence intervals in and the corresponding P values seen in indicate the method significantly determined the method performed significantly better than expected by chance.

For performance evaluation each predicted risk score was used as the covariate in a univariate Cox proportional hazards model with overall survival censored at 60 months as the outcome variable. For graphical representation risk scores were binned into tertiles and Kaplan Meier estimates of the survivor function were plotted for each subgroup seen in . This allowed for assessment of any dose response relationship. The disclosed method performed much better on sample sets containing all tumor stages compared to sample sets containing just stage I. This reflects an ability to stratify by stage even when stage is not explicitly included in the model. Further testing of the tumors classified with low mitosis and the tumors with intermediate mitosis seen in show the tumors with intermediate mitosis perform very similar to tumors with low mitosis. As such the tumors with intermediate mitosis were included in the low mitosis tumor group.

For prediction on subjects with stage 1 disease using gene expression data only the disclosed method gave hazard ratios exceeding 1 for both validation sets. The classifier used a binary indication of low or high risk for each gene expression. For example in the MSK all stages prediction patients 19L 37L REP and 67L are correctly predicted to die early whereas 41L is predicted to be low risk and does live for more than 5 years.

As another performance measure the concordance probability estimate CPE was calculated to measure agreement between subject outcomes and predicted risk scores Gonen M. Heller G. Concordance probability and discriminatory power in proportional hazards regression. 92 4 965 970 2005 . The CPE estimates the concordance probability which is the probability that for a given pair of subjects selected at random from the study population the subject with better prognosis has a better outcome. CPE values close to 0.5 indicate no concordance poor predictivity CPE values approaching 1.0 indicate strong concordance good predictivity . On the basis of these measures the method performed consistently well showing moderate concordance as seen in .

The CAN DF data set differed from the other datasets in that it lacked stage 3 samples. To give a realistic evaluation of how a prognostic method might be used in practice the combined data from HLM was used as the training set with MSK held out as a similar but external validation set and the CAN DF data held out as a second and more challenging external validation set.

For 3 year survival receiver operating characteristics ROC were constructed for the categorical predictor seen in . Sensitivity and specificity were calculated using Bayes theorem and Kaplan Meier estimates of the survivor function and hazard function to appropriately handle censoring. To calculate the sensitivity for a rule based on a given risk score cutoff k P R k Tk P Tk for sensitivity and Rt R

Using the majority vote classifier described above a training algorithm was used for calibration. A gene set G was selected from which the entire classification process is performed. The initial 614 mitosis related probesets the reference gene set M were considered by the algorithm. Gene set G is compared to reference gene set M using the majority vote classifier and setting G equal to set M. This generates two groups genes with high mitotic indices and those with low mitotic indices. Samples with intermediate expression levels are removed so that at test can be performed between the classified samples in the high mitotic group and low mitotic group.

The genes are individually separated into two groups for the expression of gene i Xit good Xit poor. At test is performed on the two gene expression groups Xit good Xit poor resulting in ti followed by significance testing to determine a p value pi for gene probeset i.

The data from the gene sets were analyzed using the two classes good prognosis low mitotic index versus bad prognosis high mitotic index seen in . The disclosed method successfully distinguished between the prognosis classes with tumors classified with low mitotic indices exhibiting significantly higher survival distributions. Applying the method to the test gene sets shows the method stratifies patient prognosis after normalizing expression values. The HLM gene set patients with good prognosis low mitotic indices showed initial higher mortality possibly due to post treatment complications seen in . After this initial period these patients exhibited increased survival with approximately 50 of patients surviving to the conclusion of the study compared to about 25 of the patients diagnosed with poor prognosis. DF and MSK patient data evidence better concordance between the prognosis and patient mortality as seen in . In both data sets patients with good prognosis had a much higher likelihood of survival out to over 100 months post treatment. Data for UMC showed less variation between good and poor prognosis mainly due to increased survival of patients with high mitotic indices seen in . However patients with low mitotic indices possessed a substantial increase in survival.

Each reference cited in the preceding disclosure is herein expressly incorporated by reference in its entirety to the same extent as if the reference was incorporated by reference individually.

In the preceding specification all documents acts or information disclosed does not constitute an admission that the document act or information or any combination thereof was publicly available known to the public part of the general knowledge in the art or was known to be relevant to solve any problem at the time of priority.

While there has been described and illustrated specific embodiments of a gene expression classifier it will be apparent to those skilled in the art that variations and modifications are possible without deviating from the broad spirit and principle of the present invention. It is also to be understood that the following claims are intended to cover all of the generic and specific features of the invention herein described and all statements of the scope of the invention which as a matter of language might be said to fall therebetween. Now that the invention has been described 

